{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!pip install cord-19-tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import cotools\n",
    "from pprint import pprint\n",
    "import os\n",
    "import sys"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Downloading the data\n",
    "\n",
    "First we have to download and extract the data. Currently the download function downloads the 4 sets of json data and puts them, extracted, into a directory:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['download.sh', 'comm_use_subset.tar.gz', 'noncomm_use_subset', 'biorxiv_medrxiv', 'pmc_custom_license', 'comm_use_subset']\n"
     ]
    }
   ],
   "source": [
    "downloaded = True # change me if you havent downloaded the data\n",
    "\n",
    "if not downloaded:\n",
    "    cotools.download(dir = 'data')\n",
    "    \n",
    "print(os.listdir('data'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Loading and using the data\n",
    "\n",
    "Next, we can load the data! We will use the `Paperset` class. We initialize it by specifying the directory of JSON data:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "64\n"
     ]
    }
   ],
   "source": [
    "comm_use = cotools.Paperset('data/comm_use_subset')\n",
    "print(sys.getsizeof(comm_use))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We access the data through indexing. Indices can either be ints or slices. Integer indices return dicts:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "376\n",
      "<class 'dict'>\n"
     ]
    }
   ],
   "source": [
    "# laziness\n",
    "print(sys.getsizeof(comm_use[0]))\n",
    "# returns dict\n",
    "print(type(comm_use[0]))\n",
    "# dict\n",
    "#pprint(comm_use[0])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Slice indices return lists of dicts:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "536\n",
      "<class 'list'>\n",
      "[<class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>, <class 'dict'>]\n"
     ]
    }
   ],
   "source": [
    "print(sys.getsizeof(comm_use[15:65]))\n",
    "print(type(comm_use[15:65]))\n",
    "print([type(x) for x in comm_use[15:65]])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Accessing the text\n",
    "\n",
    "To access the text, we have three methods: \n",
    "\n",
    "1. `cotools.text` gets the text of a single document\n",
    "2. `cotools.texts` gets the text of a list of documents (created through slicing)\n",
    "3. `cotools.Paperset.texts` gets a list of the text of all the documents!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('hypothesis-free approaches, like genome-wide association studies 15 , '\n",
      " 'managed to spark a progress of the field, indicated as a lack of replication '\n",
      " 'of results from such studies 10, [16] [17] [18] . Unfortunately, insights '\n",
      " 'derived from the fields of clinical infectology, microbiology, immunology, '\n",
      " 'epidemiology, as well as clinical, evolutionary and population genetics '\n",
      " 'remained largely isolated from one another [19] [20] [21] , preventing '\n",
      " 'systematic understanding of the entire field. The interest in host factors '\n",
      " 'associated with respiratory infectious diseases stems from their ability to '\n",
      " 'cause epidemics and pandemics. The infamous Spanish influenza of 1918 has '\n",
      " 'been implicated as the largest ever recorded pandemic, causing an estimated '\n",
      " '25-100 millions of deaths 22, 23 . On the other hand, tuberculosis is an '\n",
      " 'example of highly adaptive pathogen, which managed to coevolve with humans, '\n",
      " 'possibly as far as the initial waves of human migrations out of Africa [24] '\n",
      " '[25] [26] . The 2013 estimates suggested that 9 million of people suffered '\n",
      " 'from it globally, with 1.5 million of attributable deaths annually and '\n",
      " 'possibly as much as a third of all living humans being latent carriers 27 . '\n",
      " 'Pneumonia remains to be one of the main childhood mortality causes, with an '\n",
      " 'estimated 120 million episodes and 1.3 million of lethal outcomes globally '\n",
      " '28 . Similarly, respiratory syncytial virus (RSV) is attributable for 33.8 '\n",
      " 'million episodes of newly diagnosed acute lower respiratory infections '\n",
      " 'worldwide in children under 5 years, with at least 3.4 million episodes of '\n",
      " 'severe cases requiring hospital admission 29 . Finally, SARS-Coronavirus '\n",
      " 'outbreak in 2003 was an example of how quickly a novel respiratory pathogen '\n",
      " 'can spread on a global scale 30, 31 . Understanding the host genetic side '\n",
      " 'could be an invaluable tool in clinical medicine, especially because '\n",
      " 'susceptibility to an infectious agent lies at least partly hidden or masked '\n",
      " 'in inborn errors or immune response 32, 33 . This renders infectious '\n",
      " 'diseases a high-ranking research priority, having in mind the mobility of '\n",
      " 'modern human population and the ever changing pathogen nature. Therefore, '\n",
      " 'the aim of this study was to provide a field synopsis and systematic review '\n",
      " 'with meta-analyses of host genetic susceptibility to respiratory infectious '\n",
      " 'diseases. A special focus was set on the comparative analysis of different '\n",
      " 'clinical presentations and pathogens, aiming to increase the understanding '\n",
      " 'of both shared and specific disease pathways that could have been identified '\n",
      " 'on the basis of previously published candidate gene studies. We also provide '\n",
      " 'a critical overview of the entire field and generalized framework for '\n",
      " 'improvement of future studies in infect-omics -application of genomics, '\n",
      " 'proteomics metabolomics or other omics technologies in understanding of the '\n",
      " 'infectious disease development, progression and outcome. A total of 2,209 '\n",
      " 'data points were extracted from 386 studies. The majority of data addressed '\n",
      " 'tuberculosis (1,417 data points; 64.1%), followed by RSV (285; 12.9%), '\n",
      " 'SARS-Coronavirus (198; 9.0%) and influenza (84; 3.8%), pneumonia (172; '\n",
      " '7.8%), while the rest were on other respiratory infections such as otitis '\n",
      " 'media and bronchitis (53; 2.4%). There were 274 different genes in the data, '\n",
      " 'with most frequent results for TNFA (123; 5.6%), IL10 (117; 5.3%) and '\n",
      " 'SLC11A1 (92; 4.2%). A total of 949 distinctive markers were recorded in the '\n",
      " 'database, but only 89 met the inclusion criteria of at least four data '\n",
      " 'points available per marker, and were used in the subsequent meta-analysis. '\n",
      " 'Out of 1,963 data points with all three CSI score domains scored, only 107 '\n",
      " 'data points (5.5%) had credible CSI score (having only A or B grades), while '\n",
      " 'the remaining 94.5% of data points had at least one C grade and were thus '\n",
      " 'considered to have weak credibility (Fig. 1 ). Disease-specific CSI profiles '\n",
      " 'indicated better quality of studies on pneumonia, influenza and '\n",
      " 'tuberculosis, while CSI scores were lower in case of SARS-Coronavirus and '\n",
      " 'respiratory syncytial virus (RSV) (Supplementary Figure 1) . Among '\n",
      " 'investigated data points, a total of 1,952 were attributable to infectious '\n",
      " 'disease susceptibility (88.4%), 172 to disease severity (by comparing severe '\n",
      " 'vs. mild disease cases; 7.8%) and 38 to mortality (1.7%), while the '\n",
      " 'remaining studies were based on comparison of tuberculosis skin-test '\n",
      " 'positivity results with controls. We performed a total of 515 disease '\n",
      " 'susceptibility meta-analysis in all four genetic models, with a total of 86 '\n",
      " 'nominally significant results (Supplementary Tables 3-6 ). However, only two '\n",
      " 'genes retained noteworthiness for the mid/low BFDP level ( Table 1 ). There '\n",
      " 'were no significant results for severity models (Supplementary Table 7) , '\n",
      " 'while we did not perform mortality meta-analysis due to lack of at least '\n",
      " 'four published studies for any marker. We identified a total of 19 family '\n",
      " 'and linkage-based studies, which were not included in the meta-analysis due '\n",
      " 'to methodological restraints. These studies covered either broad genome '\n",
      " 'regions on chromosomes 3-7, 10-12, 15, 17, 19, 20, X, or specific genes such '\n",
      " 'as SLC11A1 (NRAMP1), IL8, CCR5, IL12RB1, SP-A, SP-D, SLC22A4, SCL22A5, '\n",
      " 'RAD50, FBOX11 and EVI1 gene (Supplementary Table 8 ). We also identified 11 '\n",
      " 'GWAS studies; seven were on tuberculosis, three on otitis media and one on '\n",
      " 'influenza. Two genome-wide significant results were recorded for '\n",
      " 'tuberculosis, belonging to WT1 gene and ASAP1 gene (Supplementary Table 9 ). '\n",
      " 'We also identified 54 published meta-analyses, of which the majority '\n",
      " 'addressed genetic susceptibility to tuberculosis. However, these '\n",
      " 'meta-analyses investigated Scientific RepoRts | 5:16119 | DOi: '\n",
      " '10.1038/srep16119 infection susceptibility in much wider set of clinical '\n",
      " 'appearances, often with manifestations not restricted to respiratory tract, '\n",
      " 'and were thus not directly comparable to this study. This is the first '\n",
      " 'extensive, systematic review and meta-analysis of all published studies that '\n",
      " 'were addressing host genetic factors implicated in five common respiratory '\n",
      " 'tract infectious diseases -tuberculosis, influenza, respiratory syncytial '\n",
      " 'virus (RSV), SARS-Coronavirus and pneumonia. Until now, majority of '\n",
      " 'published studies and meta-analyses were based on one pathogen -one '\n",
      " 'polymorphism approach, focusing on pathogen susceptibility. Only a handful '\n",
      " 'of published studies compared different respiratory tract diseases. In '\n",
      " 'addition, majority of previously published meta-analyses addressed '\n",
      " 'tuberculosis, and none was found for SARS-Coronavirus, RSV and influenza. '\n",
      " 'Therefore, by developing this entire resource we enabled not only '\n",
      " 'disease-specific analyses, but also analyses of the shared (pooled) '\n",
      " 'infectious disease mechanisms in the respiratory system -all in the '\n",
      " 'stringent evidence-based manner. In total, we found two results that '\n",
      " 'withstood multiple testing correction: IL4 in pooled model and TLR2 for '\n",
      " 'tuberculosis. Additionally, we established a role of CCL2 in tuberculosis, '\n",
      " 'by including only a subset of methodologically better studies, where '\n",
      " 'controls were of known exposure. IL4 has previously been described to have a '\n",
      " 'pivotal role in shaping the nature of immune response, by promoting and '\n",
      " 'stimulating both T-cell and B-cell differentiation 34 . It provides a '\n",
      " 'balance between Th1 and Th2 response, and therefore alteration of its '\n",
      " 'function may substantially affect immune response 34 . Most commonly '\n",
      " 'reported such alteration is associated with increased risk of atopy and '\n",
      " 'allergies 35 . Furthermore, IL4 seems to have a direct role in infectious '\n",
      " 'disease outcome -IL4-deficient mice were more susceptible to Legionella '\n",
      " 'pneumophila and had increased mortality rates compared to controls 36 . This '\n",
      " 'was the single result that was significant in pooled analyses, and marginal '\n",
      " 'in disease-specific meta-analyses. This reflects the need for studies with '\n",
      " 'more statistical power, but also suggests that this is one of the most '\n",
      " 'likely translatable results towards clinical application across the wider '\n",
      " 'spectrum of respiratory infectious diseases. TLR2 is a member of large '\n",
      " 'family of genes involved in pathogen recognition and signalling cascade of '\n",
      " 'innate immune response 37 . Previous studies have suggested its role in the '\n",
      " 'susceptibility to tuberculosis [38] [39] [40] , but also in other bacteria '\n",
      " 'such as Mycoplasma, Mycobacterium, Neisseria and other Gram-positive '\n",
      " 'bacteria 41 . Interestingly, despite being closely linked to CD14 mechanisms '\n",
      " '37 , the results of this study showed that CD14 had no association with '\n",
      " 'tuberculosis, while TLR2 had a very strong result, with odds ratio of 3.2. '\n",
      " 'This finding could point towards a specific role of TLR2 in tuberculosis, '\n",
      " 'with weaker association or even lack of such association in other elements '\n",
      " 'of the immune response. Interestingly, TLR2 effects seem to be very '\n",
      " 'context-specific, as the absence of TLR2 in knock-out mice did not affect '\n",
      " 'their clinical outcomes, nor did it seem to have a role in secondary '\n",
      " 'infections 42 . This suggests that the exact mechanisms of its function are '\n",
      " 'still elusive, and that more focused studies are needed, taking into account '\n",
      " 'the exact pathophysiological processes in patients with tuberculosis. CCL2 '\n",
      " 'gene has been associated with diseases that include monocytic infiltrations '\n",
      " '43 , migration and retention of monocytes in particular locations 44 , and '\n",
      " 'consequently in granuloma formation [45] [46] [47] . As such, its role in '\n",
      " 'tuberculosis has been described previously 48, 49 , with notable '\n",
      " 'ethnic-specific differences 50 . In this study, CCL2 gene was significant '\n",
      " 'only in a subset analysis, where studies with exclusively exposed controls '\n",
      " 'were sub-selected. This clearly demonstrates that methodological '\n",
      " 'improvements can have positive effect on gene discovery. Very complex '\n",
      " 'genetic architecture of infectious diseases development, coupled with errors '\n",
      " 'and omissions in primary study designs, might explain this accumulation of '\n",
      " 'large amount of poorly usable evidence. Therefore, one of the main '\n",
      " 'conclusions of this study is that the best way forward is through '\n",
      " 'improvement of primary studies and reduction of research waste. An overview '\n",
      " 'of family and linkage-based studies suggested frequent differences in study '\n",
      " 'designs, marked by dissimilarities in phenotype definitions, marker types '\n",
      " 'and analytic approaches. Nevertheless, the association of tuberculosis with '\n",
      " 'SLC11A1 was replicated in two studies 51, 52 , with a notable exception in '\n",
      " 'the third one (Supplementary Table 8) 53 . There was no overlap of '\n",
      " 'candidate-gene studies and GWAS 10,54 , thus prohibiting comparative '\n",
      " 'analysis. All GWAS studies provided a two significant and replicated result, '\n",
      " 'located in an intergenic region near WT1 gene (rs2057178) 16, 55 and ASAP1 '\n",
      " 'gene (rs10956514) 56 . Other reported signals were not replicated in '\n",
      " 'independent cohorts (Supplementary Table 9 ). This is likely a consequence '\n",
      " 'of largely underpowered study designs (with sample sizes rarely exceeding '\n",
      " '1,000 cases or controls), but more importantly methodological problems that '\n",
      " 'were carried over from the candidate-gene study designs. Up-scaling the '\n",
      " 'study in terms of genetic resolution (from single marker or just a few '\n",
      " 'markers to thousands and hundreds of thousands of markers in GWAS), increase '\n",
      " 'in the sample size and use of more advanced analytic methods will likely not '\n",
      " 'provide a substantial step forward before we manage to overcome '\n",
      " 'methodological and study design limitations. In addition, we are faced with '\n",
      " 'a high level of genetic complexity 57 , and duality between rare, Mendelian '\n",
      " 'variants, which may confer complete immunity, but have very low prevalence '\n",
      " 'in a population 58 , or common variants, which usually have much lower '\n",
      " 'explained variance, and therefore have a limited capacity for clinical '\n",
      " 'intervention. The principal step forward is to create larger-scale '\n",
      " 'collaborations and enable data sharing, built upon the common framework for '\n",
      " 'analysis, based on aligned protocols, larger-scale harmonizing and '\n",
      " 'collaborative efforts 59 . The results of this study fit into previously '\n",
      " 'implied mechanisms of respiratory infections, where CCL7-CCL2-CCR2 axis was '\n",
      " 'described to have a critical role in IL4 production and immune response and '\n",
      " 'regulation in both fungal 60 and viral infections 61 , as well as '\n",
      " 'atherosclerosis 62 and tumours 63 . Despite differences described from in '\n",
      " 'vitro and in vivo studies 64 , this could be the most interesting '\n",
      " 'translatable mechanism towards clinical medicine. In addition, WT1 gene from '\n",
      " 'GWAS studies 16, 55 , implied in susceptibility to tuberculosis, has been '\n",
      " 'described to be under control of CCL2, which seems to be having '\n",
      " 'target-responsive cytocidal activity on WT1-specific mechanisms 65 . This '\n",
      " 'mechanism could, at least partly, explain the relationship between some '\n",
      " 'infections and tumours 66 . However, to clarify their relationship and '\n",
      " 'provide a step forward, we must firstly be able to derive much better '\n",
      " 'evidence from primary studies. Primary study design was one of the main '\n",
      " 'barriers for the further field development, with almost 95% of all extracted '\n",
      " 'data points assessed as having strong risk of bias or confounding. The '\n",
      " 'improvements must be embodied into evidence-based principles, and have to be '\n",
      " 'undertaken in almost all domains of the field, including better clinical '\n",
      " 'definitions and phenotyping, improved controls selection and diagnostics, '\n",
      " 'appropriate statistical analysis, favourable study design, the use of novel '\n",
      " 'molecular technologies Scientific RepoRts | 5:16119 | DOi: 10.1038/srep16119 '\n",
      " 'and better reporting with larger sample sizes obtained through consortia '\n",
      " 'development. Firstly, we need to improve phenotype definitions and criteria '\n",
      " 'for diagnosis, especially in case of latent tuberculosis 13, 67 . The use of '\n",
      " 'proxy and extreme phenotypes, which was seen in HIV, with exposed uninfected '\n",
      " 'subjects, long-term non-progressors, fast progressors and elite controllers '\n",
      " '[68] [69] [70] , might also provide an interesting step forward 71 . It was '\n",
      " 'very common for authors to use the term \"healthy\" controls, but almost '\n",
      " 'universally used dissimilar diagnostic process for classification of cases '\n",
      " 'and controls. Thus, selection of controls and diagnostics process must be '\n",
      " 'fundamentally improved and harmonized (Supplementary Table 10) . A '\n",
      " 'substantial effort must go towards inclusion of controls that were exposed '\n",
      " 'to a pathogen, which is necessary for the disease development (but may not '\n",
      " 'be sufficient). Data analysis must also be improved, by adhering to the best '\n",
      " 'practices, including the use and report of odds ratios and confidence '\n",
      " 'intervals as measures of association, and multiple testing correction 72 . '\n",
      " 'Improvements can be made in the study design, where availability of finely '\n",
      " 'phenotyped life-long cohorts and biobanks provides an opportunity to use '\n",
      " 'this data in an understanding of life-long risk of developing respiratory '\n",
      " 'infectious disease, such as tuberculosis. The use of novel molecular '\n",
      " 'technologies, various omics, and their application to infectious disease '\n",
      " 'development, progression and outcome (infect-omics) presents an '\n",
      " 'unprecedented opportunity in this field 73 . However, these must be based on '\n",
      " 'evidence-based, always include estimates of repeatability and focus on '\n",
      " 'replication efforts in an unlinked population. Lastly, we must substantially '\n",
      " 'improve results reporting. This refers to the structured manuscript '\n",
      " 'preparation, reporting of all relevant information and manuscript '\n",
      " 'preparation against a given set of criteria like STREGA 74 , STROBE-ME 75 or '\n",
      " 'STREIS, where appropriate 76 . This also extends to the need to develop '\n",
      " 'individual-data public repositories and creation of large-scale consortia '\n",
      " 'that share common standard-operating procedures, similarly to these in '\n",
      " 'chronic disease genetic epidemiology. We do not neglect the proposed finer '\n",
      " 'scale analyses of various population subsets 67 , but we consider '\n",
      " 'improvements in study design to have much stronger potential impact. All '\n",
      " 'these suggestions are applicable not only for genomics, but also for any '\n",
      " 'type of infect-omics studies, defined as application of various -omics '\n",
      " 'technologies in understanding of the infectious disease development, '\n",
      " 'progression and outcome. In addition, we need to start developing framework '\n",
      " 'of systematic reviews, where we need to compare disease-specific genes and '\n",
      " 'pathways to disease-specific genes, as the final outcome, clinical disease, '\n",
      " 'might be attributable to different sets of genes that make a person more '\n",
      " 'susceptible to e.g. pneumonia regardless on the pathogen that is causing it. '\n",
      " 'Only by comparing such complementary results and insights will we be able to '\n",
      " 'better understand and harness this information in clinical care. These '\n",
      " 'results must be viewed in light of numerous limitations. We reviewed a very '\n",
      " 'diverse field with no unified reporting scheme, suggesting that the search '\n",
      " 'might have missed some relevant articles and data sources. In order to '\n",
      " 'accommodate for this problem we performed extensive search supplemented by '\n",
      " 'hand-search procedure described in Materials and methods section. Existence '\n",
      " 'of wider-scale unpublished data and occasionally our inability to obtain raw '\n",
      " 'data present another possible source of bias. This study also suffers from '\n",
      " 'the limited input, due to exclusion of HLA and KIR markers, haplotypes and '\n",
      " 'complete omission of any information on pathogen, thus providing rather '\n",
      " 'limited insight into the field and possibly missing out some of the key '\n",
      " 'element for more in-depth understanding 77 . Most of these limitations have '\n",
      " 'very negative impact on effect-to-bias ratio, where bias may be several '\n",
      " 'orders of magnitude greater than the size effects. We attempted to contrast '\n",
      " 'this by the use of the stringent analytic approach in an evidence-based '\n",
      " 'manner, including multiple testing correction, careful application of '\n",
      " 'inclusion criteria and quality assessment scoring, coupled with use of '\n",
      " 'random-effect meta-analysis. The study also suffers from inability to adjust '\n",
      " 'the analysis for basic covariates, such as age, sex and comorbidity '\n",
      " 'information. We also used a novel score for quality assessment that was '\n",
      " 'developed within this study and was not validated before. We thus provided '\n",
      " 'basic agreement indices, but wider use of this score would require an '\n",
      " 'in-depth reliability assessment and validation. If we turn back to '\n",
      " 'biological processes, earlier primary studies ignored microbiomes, while '\n",
      " 'some novel studies are suggesting that lungs are not sterile 78, 79 and that '\n",
      " 'respiratory tract microbiomes could have strong modifying effect on health '\n",
      " 'and disease mechanisms. Despite all these limitations, we attempted to '\n",
      " 'provide a field synopsis, supplemented by a generalized framework that could '\n",
      " 'be useful not only for respiratory infectious disease genetics, but can be '\n",
      " 'extended to all infectious diseases. However, these results are just the '\n",
      " 'first step it an attempt to provide a reflection of just one side of the '\n",
      " 'infectious disease story -the host. Only by combining the host genomics (and '\n",
      " 'other -omics) information with pathogen -omics will we be able to truly '\n",
      " 'advance this field and enable the development of personalized infectious '\n",
      " 'disease medicine. Protocol and registration. The review strategy and details '\n",
      " 'were registered prospectively in PROSPERO as the record 2014:CRD42014009072. '\n",
      " 'Eligibility criteria. We undertook an extensive and systematic search of all '\n",
      " 'published studies related to host genetics implicated in development or '\n",
      " 'outcome of respiratory tract infectious diseases. Instead of focusing on a '\n",
      " 'single-disease approach, we aimed at delivering systematic evidence on all '\n",
      " 'respiratory infections, including cases where a syndrome phenotype was '\n",
      " 'reported, such as pneumonia, without the specific pathogen information. '\n",
      " 'Scientific RepoRts | 5:16119 | DOi: 10.1038/srep16119 Information sources. '\n",
      " 'We used three bibliographic databases; PubMed (http://www.ncbi.nlm.nih.gov/ '\n",
      " 'pubmed), Web of Science (http://wok.mimas.ac.uk) and Scopus '\n",
      " '(http://www.scopus.com). We also performed an additional search of the HuGe '\n",
      " 'Literature Finder (http://www.hugenavigator.net), to identify articles that '\n",
      " 'might have been missed in PubMed. Search. Two of the joint first authors '\n",
      " 'performed the search and study selection, which was supervised by the last '\n",
      " 'author. The last author also resolved any discrepancies in scoring system or '\n",
      " 'data extraction process. The search was performed on May 06, 2014, with an '\n",
      " 'update performed on August 25, 2015. After duplicates removal, the search '\n",
      " 'yielded a total of 24,823 articles. All identified articles were firstly '\n",
      " 'checked for inclusion by reading the Title and Abstract. This led to '\n",
      " 'identification and exclusion of a total of 23,677 studies. The remaining 881 '\n",
      " 'articles were supplemented by hand search of their references, producing an '\n",
      " 'additional set of 265 articles. In total, we used a set of 1,146 articles '\n",
      " 'that were read in full and screened against inclusion criteria '\n",
      " '(Supplementary Figure 5) . The study selection process was based on '\n",
      " 'iterative steps. Firstly, the studies needed to report the number of cases '\n",
      " 'and controls for every analysed genotype. Therefore, we initially excluded '\n",
      " 'all of the family-based and linkage studies, as well as genome-wide '\n",
      " 'association (GWAS) studies from this meta-analysis, due to inability to '\n",
      " 'obtain complete set of GWAS results in the appropriate format. However, we '\n",
      " 'retained these studies and provided their qualitative summary, in order to '\n",
      " 'develop an inclusive field synopsis. Secondly, studies had to be based on '\n",
      " 'biallelic single-nucleotide polymorphisms (SNP) or biallelic '\n",
      " 'insertion-deletion marker type. Studies based on other marker types '\n",
      " '(including microsatellites or short tandem repeats), or those that reported '\n",
      " 'aggregated haplotypes were excluded. We also excluded all studies that '\n",
      " 'reported exclusively gene expression profiles without relevant genotyping '\n",
      " 'information. We then limited the review to studies that were published '\n",
      " 'during or after the year 2000 and were written in English language. Despite '\n",
      " 'similarity of disease patterns and mechanisms, we excluded all animal and '\n",
      " 'all in vitro studies and focused on susceptibility in previously healthy '\n",
      " 'subjects. We also excluded studies that reported: any outcome in previously '\n",
      " 'affected patients (either as a part of immunodeficiency syndromes or with '\n",
      " 'any form of disease such as asthma, HIV/AIDS or other comorbidities; where '\n",
      " 'reported), fungal infections (due to also very prevalent comorbidities), '\n",
      " 'nosocomial infections (such as ventilator-related or hospital-acquired '\n",
      " 'pneumonia), exclusively pathogen genotypes or host-pathogen interactions. '\n",
      " 'Studies that reported non-respiratory infection sites (such as '\n",
      " 'extra-pulmonary tuberculosis, meningitis or invasive pneumococcal disease) '\n",
      " 'were also excluded, as the focus of this study was on respiratory infectious '\n",
      " 'diseases. Lastly, we excluded all studies that were obviously reporting '\n",
      " 'previously published results or were re-using the cases that has already '\n",
      " 'been used elsewhere (when we managed to identify this), in order to remove '\n",
      " 'redundant data or publications from meta-analysis. To ensure a systematic '\n",
      " 'source of information, we additionally checked studies which provided '\n",
      " 'insufficient information. A total of 142 e-mails were sent to authors of '\n",
      " 'studies which we classified as having reported insufficient information, and '\n",
      " 'asked for clarifications or raw data sharing. We also contacted all authors '\n",
      " 'of identified GWAS studies to share their results. This entire process '\n",
      " 'results in a total of 10 responses (4 rejections, 2 responses that did not '\n",
      " 'meet the inclusion criteria and 4 cases of clarifications or shared raw '\n",
      " 'data, which are listed in the Acknowledgements). In total, we used 386 '\n",
      " 'articles in the data extraction process (Supplementary Figure 5 ). Data '\n",
      " 'collection process and data items. A total of 48 pieces of information were '\n",
      " 'extracted for every study (available in the on-line RISEdb -Respiratory '\n",
      " 'Infection SuscEptibility database, available at '\n",
      " 'http://www.prepare-europe.eu/risedb). All the data were extracted and '\n",
      " 'entered into a database by two authors, and checked by the third, in order '\n",
      " 'to ensure data integrity. Each identified study was indexed with a unique '\n",
      " 'identification code (RISEdb ID). Some studies provided more than one data '\n",
      " 'point (which corresponds to one marker -one study data entry), and were '\n",
      " 'consequently represented with more than one data entry in our database. In '\n",
      " 'all instances the data were extracted in the raw format as genotype counts, '\n",
      " 'or were calculated from the reported percentages and total sample sizes '\n",
      " 'where number of cases and controls was not available. When more cases or '\n",
      " 'control groups were available in one study, we either pooled them or '\n",
      " 'retained multiple data points from the same study, especially in case of '\n",
      " 'having discovery and replication datasets. In addition, all data points were '\n",
      " 'classified according to the underlying research question into three '\n",
      " 'categories: studies that investigated infectious disease susceptibility, '\n",
      " 'disease severity or disease mortality. In addition, we defined a subset of '\n",
      " 'methodologically better studies, where controls were of known exposure to '\n",
      " 'the pathogen. We developed a novel score for purposes of risk of bias '\n",
      " 'assessment. As the methodological design defines, a study may be prone to '\n",
      " 'three main domains of errors and bias -confounding, selection bias and '\n",
      " 'information bias. These three domains were the basis for the score '\n",
      " 'development, which was entitled Confounding-Selection-Information bias score '\n",
      " '-CSI (Supplementary Table 2 ). The elements of the score were developed on '\n",
      " 'the basis of several previous studies 80 and assessment scores, including '\n",
      " 'Venice criteria for assessing cumulative epidemiologic evidence in genetic '\n",
      " 'associations 81 , Newcastle-Ottawa case-control scale 82 , Cochrane risk of '\n",
      " 'bias tool 83 , and previously established quality scores in genetic '\n",
      " 'epidemiology 84 . Neither of these tools provided a good fit for the risk of '\n",
      " 'bias assessment in observational studies in genetic epidemiology, and we '\n",
      " 'therefore took elements of previous scales and derived a new one. In line '\n",
      " 'with Venice criteria, we scored all three domains in three grades of '\n",
      " 'credibility: high -graded as A; intermediate -B or weak -C. This scheme '\n",
      " 'provided estimates ranging from the best AAA to worst CCC score, which was '\n",
      " 'applied to every data point in RISE database (and not for the meta-analysis '\n",
      " 'results, as the Venice criteria is intended to). Since information bias can '\n",
      " 'affect several aspects of the study, we also divided information bias risk '\n",
      " 'domain in three separate sub-scores, relating to cases and controls '\n",
      " 'selection and genotyping procedures, with the worst of all three defining '\n",
      " 'the overall information bias risk. All CSI scoring was done independently by '\n",
      " 'two authors, and all discrepancies were settled by an agreement or input '\n",
      " 'from a third author (there were 313 or 14.2% data points with initially '\n",
      " 'dissimilar grades, providing good agreement with kappa coefficient of 0.717 '\n",
      " 'and standard error for kappa of 0.011). As an additional quality control '\n",
      " 'check, we re-calculated Hardy-Weinberg test for every control set included '\n",
      " 'in the analysis, using a two tailed chi-square test or an exact test, when '\n",
      " 'appropriate. Those data points that failed HWE test at the level of P < 0.05 '\n",
      " 'were downgraded to C in the CSI score (there were 225 out of 2,209, or 10.2% '\n",
      " 'of all analysed data points that failed HWE), in order to reflect possible '\n",
      " 'methodological limitations of failed HWE (which indicates both the risk for '\n",
      " 'selection and/or information bias). Where cases from a single study were '\n",
      " 'classified as moderate or severe, we did not determine Selection domain of '\n",
      " 'CSI score (a total of 246 data points). In such instances we assigned 0 to '\n",
      " 'selection bias risk, as there was no selection process involved, and '\n",
      " 'second-stage analysis was performed Summary measures and synthesis of '\n",
      " 'results. In the first step of meta-analysis we calculated all odds ratios '\n",
      " 'anew from raw data, in order to provide a common analytic framework and '\n",
      " 'reduce possible differences in data analysis. In order to better understand '\n",
      " 'the genetic architecture underlying gene-disease association, we performed '\n",
      " 'analyses using four separate genetic models: allelic, dominant, recessive '\n",
      " 'and heterozygote advantage. Model selection was based on commonly used '\n",
      " 'allelic model, two additional models that assume different inheritance '\n",
      " 'effects (dominant and recessive), followed by the heterozygote advantage, as '\n",
      " 'previous studies reported such results [85] [86] [87] . All calculations '\n",
      " 'were therefore based on allele counts, rather than exact cases and controls '\n",
      " 'counts. We compared the results of different models, in an attempt to '\n",
      " 'provide broader insight into genetic architecture. Minor allele frequencies '\n",
      " 'in Homo sapiens Short Variation (GRCh38) data set were obtained from Ensembl '\n",
      " 'Biomart on 08.12.2014 88 . We then performed a series of meta-analyses for '\n",
      " 'all SNPs where four or more data points were available for a single SNP. We '\n",
      " 'used random effects meta-analysis, assuming differences in designs and '\n",
      " 'ethnic composition of individual studies. We also calculated fixed-effects '\n",
      " 'model, in order to compare the results. We used I 2 statistics and '\n",
      " 'corresponding confidence intervals as a measure of heterogeneity. In order '\n",
      " 'to account for multiple testing as a result of numerous meta-analyses and '\n",
      " 'different genetic models being performed, we used Bayesian false-discovery '\n",
      " 'probability (BFDP) 72 . It was calculated for nominally significant results '\n",
      " 'only, with a threshold of 0.2, using an online free version of the BFDP '\n",
      " 'calculator (http://faculty.washington.edu/jonno/BFDP.xls). In addition, we '\n",
      " 'calculated BFDP using two prior probabilities, with medium/low prior level '\n",
      " '(0.05 to E-03), consistent with a candidate gene; and very low prior level '\n",
      " '(E-04 to E-06), consistent with a random SNP. The advantage of this approach '\n",
      " 'over the commonly used corrections is that it is not dependent on the number '\n",
      " 'of tests performed, as it relies on odds ratios and confidence intervals for '\n",
      " 'calculation. In addition, it provides conceptual improvement over the '\n",
      " 'similar methods, such as false-positive report probability (FPRP) 89 . Due '\n",
      " 'to substantial requirements for appropriate use of publication bias analysis '\n",
      " 'methods 90 , we only examined publication bias visually. All analyses were '\n",
      " 'performed in R, version 2.15.3 91 . Meta-analysis was performed with package '\n",
      " 'meta 3.2-0 and metabin, forest and funnel functions 92 . Hardy-Weinberg '\n",
      " 'equilibrium testing was performed in HardyWeinberg_1.5.2 package 93 , with '\n",
      " 'HWExactMat or HWChisqMat functions, based on the cell expected frequencies. '\n",
      " 'Risk of bias across studies. For purpose of meta-analysis credibility '\n",
      " 'assessment, we used Venice criteria 81 , in all instances of nominally '\n",
      " 'significant results. Study power was assessed on the basis of sample sizes '\n",
      " '(with sample sizes of up to a 1,000 graded as C, 1,001-10,000 graded as B, '\n",
      " 'and over 10,001 graded as A). Heterogeneity was based on I 2 statistics '\n",
      " '(graded as A in case of 0-25%, B in case of 26-50% and C in case of I2 being '\n",
      " 'over 50% ), while the third score domain was fixed as C (weak credibility), '\n",
      " 'due to very prevalent risk of bias in primary studies.')\n"
     ]
    }
   ],
   "source": [
    "pprint(cotools.text(comm_use[0]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['hypothesis-free approaches, like genome-wide association studies 15 , '\n",
      " 'managed to spark a progress of the field, indicated as a lack of replication '\n",
      " 'of results from such studies 10, [16] [17] [18] . Unfortunately, insights '\n",
      " 'derived from the fields of clinical infectology, microbiology, immunology, '\n",
      " 'epidemiology, as well as clinical, evolutionary and population genetics '\n",
      " 'remained largely isolated from one another [19] [20] [21] , preventing '\n",
      " 'systematic understanding of the entire field. The interest in host factors '\n",
      " 'associated with respiratory infectious diseases stems from their ability to '\n",
      " 'cause epidemics and pandemics. The infamous Spanish influenza of 1918 has '\n",
      " 'been implicated as the largest ever recorded pandemic, causing an estimated '\n",
      " '25-100 millions of deaths 22, 23 . On the other hand, tuberculosis is an '\n",
      " 'example of highly adaptive pathogen, which managed to coevolve with humans, '\n",
      " 'possibly as far as the initial waves of human migrations out of Africa [24] '\n",
      " '[25] [26] . The 2013 estimates suggested that 9 million of people suffered '\n",
      " 'from it globally, with 1.5 million of attributable deaths annually and '\n",
      " 'possibly as much as a third of all living humans being latent carriers 27 . '\n",
      " 'Pneumonia remains to be one of the main childhood mortality causes, with an '\n",
      " 'estimated 120 million episodes and 1.3 million of lethal outcomes globally '\n",
      " '28 . Similarly, respiratory syncytial virus (RSV) is attributable for 33.8 '\n",
      " 'million episodes of newly diagnosed acute lower respiratory infections '\n",
      " 'worldwide in children under 5 years, with at least 3.4 million episodes of '\n",
      " 'severe cases requiring hospital admission 29 . Finally, SARS-Coronavirus '\n",
      " 'outbreak in 2003 was an example of how quickly a novel respiratory pathogen '\n",
      " 'can spread on a global scale 30, 31 . Understanding the host genetic side '\n",
      " 'could be an invaluable tool in clinical medicine, especially because '\n",
      " 'susceptibility to an infectious agent lies at least partly hidden or masked '\n",
      " 'in inborn errors or immune response 32, 33 . This renders infectious '\n",
      " 'diseases a high-ranking research priority, having in mind the mobility of '\n",
      " 'modern human population and the ever changing pathogen nature. Therefore, '\n",
      " 'the aim of this study was to provide a field synopsis and systematic review '\n",
      " 'with meta-analyses of host genetic susceptibility to respiratory infectious '\n",
      " 'diseases. A special focus was set on the comparative analysis of different '\n",
      " 'clinical presentations and pathogens, aiming to increase the understanding '\n",
      " 'of both shared and specific disease pathways that could have been identified '\n",
      " 'on the basis of previously published candidate gene studies. We also provide '\n",
      " 'a critical overview of the entire field and generalized framework for '\n",
      " 'improvement of future studies in infect-omics -application of genomics, '\n",
      " 'proteomics metabolomics or other omics technologies in understanding of the '\n",
      " 'infectious disease development, progression and outcome. A total of 2,209 '\n",
      " 'data points were extracted from 386 studies. The majority of data addressed '\n",
      " 'tuberculosis (1,417 data points; 64.1%), followed by RSV (285; 12.9%), '\n",
      " 'SARS-Coronavirus (198; 9.0%) and influenza (84; 3.8%), pneumonia (172; '\n",
      " '7.8%), while the rest were on other respiratory infections such as otitis '\n",
      " 'media and bronchitis (53; 2.4%). There were 274 different genes in the data, '\n",
      " 'with most frequent results for TNFA (123; 5.6%), IL10 (117; 5.3%) and '\n",
      " 'SLC11A1 (92; 4.2%). A total of 949 distinctive markers were recorded in the '\n",
      " 'database, but only 89 met the inclusion criteria of at least four data '\n",
      " 'points available per marker, and were used in the subsequent meta-analysis. '\n",
      " 'Out of 1,963 data points with all three CSI score domains scored, only 107 '\n",
      " 'data points (5.5%) had credible CSI score (having only A or B grades), while '\n",
      " 'the remaining 94.5% of data points had at least one C grade and were thus '\n",
      " 'considered to have weak credibility (Fig. 1 ). Disease-specific CSI profiles '\n",
      " 'indicated better quality of studies on pneumonia, influenza and '\n",
      " 'tuberculosis, while CSI scores were lower in case of SARS-Coronavirus and '\n",
      " 'respiratory syncytial virus (RSV) (Supplementary Figure 1) . Among '\n",
      " 'investigated data points, a total of 1,952 were attributable to infectious '\n",
      " 'disease susceptibility (88.4%), 172 to disease severity (by comparing severe '\n",
      " 'vs. mild disease cases; 7.8%) and 38 to mortality (1.7%), while the '\n",
      " 'remaining studies were based on comparison of tuberculosis skin-test '\n",
      " 'positivity results with controls. We performed a total of 515 disease '\n",
      " 'susceptibility meta-analysis in all four genetic models, with a total of 86 '\n",
      " 'nominally significant results (Supplementary Tables 3-6 ). However, only two '\n",
      " 'genes retained noteworthiness for the mid/low BFDP level ( Table 1 ). There '\n",
      " 'were no significant results for severity models (Supplementary Table 7) , '\n",
      " 'while we did not perform mortality meta-analysis due to lack of at least '\n",
      " 'four published studies for any marker. We identified a total of 19 family '\n",
      " 'and linkage-based studies, which were not included in the meta-analysis due '\n",
      " 'to methodological restraints. These studies covered either broad genome '\n",
      " 'regions on chromosomes 3-7, 10-12, 15, 17, 19, 20, X, or specific genes such '\n",
      " 'as SLC11A1 (NRAMP1), IL8, CCR5, IL12RB1, SP-A, SP-D, SLC22A4, SCL22A5, '\n",
      " 'RAD50, FBOX11 and EVI1 gene (Supplementary Table 8 ). We also identified 11 '\n",
      " 'GWAS studies; seven were on tuberculosis, three on otitis media and one on '\n",
      " 'influenza. Two genome-wide significant results were recorded for '\n",
      " 'tuberculosis, belonging to WT1 gene and ASAP1 gene (Supplementary Table 9 ). '\n",
      " 'We also identified 54 published meta-analyses, of which the majority '\n",
      " 'addressed genetic susceptibility to tuberculosis. However, these '\n",
      " 'meta-analyses investigated Scientific RepoRts | 5:16119 | DOi: '\n",
      " '10.1038/srep16119 infection susceptibility in much wider set of clinical '\n",
      " 'appearances, often with manifestations not restricted to respiratory tract, '\n",
      " 'and were thus not directly comparable to this study. This is the first '\n",
      " 'extensive, systematic review and meta-analysis of all published studies that '\n",
      " 'were addressing host genetic factors implicated in five common respiratory '\n",
      " 'tract infectious diseases -tuberculosis, influenza, respiratory syncytial '\n",
      " 'virus (RSV), SARS-Coronavirus and pneumonia. Until now, majority of '\n",
      " 'published studies and meta-analyses were based on one pathogen -one '\n",
      " 'polymorphism approach, focusing on pathogen susceptibility. Only a handful '\n",
      " 'of published studies compared different respiratory tract diseases. In '\n",
      " 'addition, majority of previously published meta-analyses addressed '\n",
      " 'tuberculosis, and none was found for SARS-Coronavirus, RSV and influenza. '\n",
      " 'Therefore, by developing this entire resource we enabled not only '\n",
      " 'disease-specific analyses, but also analyses of the shared (pooled) '\n",
      " 'infectious disease mechanisms in the respiratory system -all in the '\n",
      " 'stringent evidence-based manner. In total, we found two results that '\n",
      " 'withstood multiple testing correction: IL4 in pooled model and TLR2 for '\n",
      " 'tuberculosis. Additionally, we established a role of CCL2 in tuberculosis, '\n",
      " 'by including only a subset of methodologically better studies, where '\n",
      " 'controls were of known exposure. IL4 has previously been described to have a '\n",
      " 'pivotal role in shaping the nature of immune response, by promoting and '\n",
      " 'stimulating both T-cell and B-cell differentiation 34 . It provides a '\n",
      " 'balance between Th1 and Th2 response, and therefore alteration of its '\n",
      " 'function may substantially affect immune response 34 . Most commonly '\n",
      " 'reported such alteration is associated with increased risk of atopy and '\n",
      " 'allergies 35 . Furthermore, IL4 seems to have a direct role in infectious '\n",
      " 'disease outcome -IL4-deficient mice were more susceptible to Legionella '\n",
      " 'pneumophila and had increased mortality rates compared to controls 36 . This '\n",
      " 'was the single result that was significant in pooled analyses, and marginal '\n",
      " 'in disease-specific meta-analyses. This reflects the need for studies with '\n",
      " 'more statistical power, but also suggests that this is one of the most '\n",
      " 'likely translatable results towards clinical application across the wider '\n",
      " 'spectrum of respiratory infectious diseases. TLR2 is a member of large '\n",
      " 'family of genes involved in pathogen recognition and signalling cascade of '\n",
      " 'innate immune response 37 . Previous studies have suggested its role in the '\n",
      " 'susceptibility to tuberculosis [38] [39] [40] , but also in other bacteria '\n",
      " 'such as Mycoplasma, Mycobacterium, Neisseria and other Gram-positive '\n",
      " 'bacteria 41 . Interestingly, despite being closely linked to CD14 mechanisms '\n",
      " '37 , the results of this study showed that CD14 had no association with '\n",
      " 'tuberculosis, while TLR2 had a very strong result, with odds ratio of 3.2. '\n",
      " 'This finding could point towards a specific role of TLR2 in tuberculosis, '\n",
      " 'with weaker association or even lack of such association in other elements '\n",
      " 'of the immune response. Interestingly, TLR2 effects seem to be very '\n",
      " 'context-specific, as the absence of TLR2 in knock-out mice did not affect '\n",
      " 'their clinical outcomes, nor did it seem to have a role in secondary '\n",
      " 'infections 42 . This suggests that the exact mechanisms of its function are '\n",
      " 'still elusive, and that more focused studies are needed, taking into account '\n",
      " 'the exact pathophysiological processes in patients with tuberculosis. CCL2 '\n",
      " 'gene has been associated with diseases that include monocytic infiltrations '\n",
      " '43 , migration and retention of monocytes in particular locations 44 , and '\n",
      " 'consequently in granuloma formation [45] [46] [47] . As such, its role in '\n",
      " 'tuberculosis has been described previously 48, 49 , with notable '\n",
      " 'ethnic-specific differences 50 . In this study, CCL2 gene was significant '\n",
      " 'only in a subset analysis, where studies with exclusively exposed controls '\n",
      " 'were sub-selected. This clearly demonstrates that methodological '\n",
      " 'improvements can have positive effect on gene discovery. Very complex '\n",
      " 'genetic architecture of infectious diseases development, coupled with errors '\n",
      " 'and omissions in primary study designs, might explain this accumulation of '\n",
      " 'large amount of poorly usable evidence. Therefore, one of the main '\n",
      " 'conclusions of this study is that the best way forward is through '\n",
      " 'improvement of primary studies and reduction of research waste. An overview '\n",
      " 'of family and linkage-based studies suggested frequent differences in study '\n",
      " 'designs, marked by dissimilarities in phenotype definitions, marker types '\n",
      " 'and analytic approaches. Nevertheless, the association of tuberculosis with '\n",
      " 'SLC11A1 was replicated in two studies 51, 52 , with a notable exception in '\n",
      " 'the third one (Supplementary Table 8) 53 . There was no overlap of '\n",
      " 'candidate-gene studies and GWAS 10,54 , thus prohibiting comparative '\n",
      " 'analysis. All GWAS studies provided a two significant and replicated result, '\n",
      " 'located in an intergenic region near WT1 gene (rs2057178) 16, 55 and ASAP1 '\n",
      " 'gene (rs10956514) 56 . Other reported signals were not replicated in '\n",
      " 'independent cohorts (Supplementary Table 9 ). This is likely a consequence '\n",
      " 'of largely underpowered study designs (with sample sizes rarely exceeding '\n",
      " '1,000 cases or controls), but more importantly methodological problems that '\n",
      " 'were carried over from the candidate-gene study designs. Up-scaling the '\n",
      " 'study in terms of genetic resolution (from single marker or just a few '\n",
      " 'markers to thousands and hundreds of thousands of markers in GWAS), increase '\n",
      " 'in the sample size and use of more advanced analytic methods will likely not '\n",
      " 'provide a substantial step forward before we manage to overcome '\n",
      " 'methodological and study design limitations. In addition, we are faced with '\n",
      " 'a high level of genetic complexity 57 , and duality between rare, Mendelian '\n",
      " 'variants, which may confer complete immunity, but have very low prevalence '\n",
      " 'in a population 58 , or common variants, which usually have much lower '\n",
      " 'explained variance, and therefore have a limited capacity for clinical '\n",
      " 'intervention. The principal step forward is to create larger-scale '\n",
      " 'collaborations and enable data sharing, built upon the common framework for '\n",
      " 'analysis, based on aligned protocols, larger-scale harmonizing and '\n",
      " 'collaborative efforts 59 . The results of this study fit into previously '\n",
      " 'implied mechanisms of respiratory infections, where CCL7-CCL2-CCR2 axis was '\n",
      " 'described to have a critical role in IL4 production and immune response and '\n",
      " 'regulation in both fungal 60 and viral infections 61 , as well as '\n",
      " 'atherosclerosis 62 and tumours 63 . Despite differences described from in '\n",
      " 'vitro and in vivo studies 64 , this could be the most interesting '\n",
      " 'translatable mechanism towards clinical medicine. In addition, WT1 gene from '\n",
      " 'GWAS studies 16, 55 , implied in susceptibility to tuberculosis, has been '\n",
      " 'described to be under control of CCL2, which seems to be having '\n",
      " 'target-responsive cytocidal activity on WT1-specific mechanisms 65 . This '\n",
      " 'mechanism could, at least partly, explain the relationship between some '\n",
      " 'infections and tumours 66 . However, to clarify their relationship and '\n",
      " 'provide a step forward, we must firstly be able to derive much better '\n",
      " 'evidence from primary studies. Primary study design was one of the main '\n",
      " 'barriers for the further field development, with almost 95% of all extracted '\n",
      " 'data points assessed as having strong risk of bias or confounding. The '\n",
      " 'improvements must be embodied into evidence-based principles, and have to be '\n",
      " 'undertaken in almost all domains of the field, including better clinical '\n",
      " 'definitions and phenotyping, improved controls selection and diagnostics, '\n",
      " 'appropriate statistical analysis, favourable study design, the use of novel '\n",
      " 'molecular technologies Scientific RepoRts | 5:16119 | DOi: 10.1038/srep16119 '\n",
      " 'and better reporting with larger sample sizes obtained through consortia '\n",
      " 'development. Firstly, we need to improve phenotype definitions and criteria '\n",
      " 'for diagnosis, especially in case of latent tuberculosis 13, 67 . The use of '\n",
      " 'proxy and extreme phenotypes, which was seen in HIV, with exposed uninfected '\n",
      " 'subjects, long-term non-progressors, fast progressors and elite controllers '\n",
      " '[68] [69] [70] , might also provide an interesting step forward 71 . It was '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'very common for authors to use the term \"healthy\" controls, but almost '\n",
      " 'universally used dissimilar diagnostic process for classification of cases '\n",
      " 'and controls. Thus, selection of controls and diagnostics process must be '\n",
      " 'fundamentally improved and harmonized (Supplementary Table 10) . A '\n",
      " 'substantial effort must go towards inclusion of controls that were exposed '\n",
      " 'to a pathogen, which is necessary for the disease development (but may not '\n",
      " 'be sufficient). Data analysis must also be improved, by adhering to the best '\n",
      " 'practices, including the use and report of odds ratios and confidence '\n",
      " 'intervals as measures of association, and multiple testing correction 72 . '\n",
      " 'Improvements can be made in the study design, where availability of finely '\n",
      " 'phenotyped life-long cohorts and biobanks provides an opportunity to use '\n",
      " 'this data in an understanding of life-long risk of developing respiratory '\n",
      " 'infectious disease, such as tuberculosis. The use of novel molecular '\n",
      " 'technologies, various omics, and their application to infectious disease '\n",
      " 'development, progression and outcome (infect-omics) presents an '\n",
      " 'unprecedented opportunity in this field 73 . However, these must be based on '\n",
      " 'evidence-based, always include estimates of repeatability and focus on '\n",
      " 'replication efforts in an unlinked population. Lastly, we must substantially '\n",
      " 'improve results reporting. This refers to the structured manuscript '\n",
      " 'preparation, reporting of all relevant information and manuscript '\n",
      " 'preparation against a given set of criteria like STREGA 74 , STROBE-ME 75 or '\n",
      " 'STREIS, where appropriate 76 . This also extends to the need to develop '\n",
      " 'individual-data public repositories and creation of large-scale consortia '\n",
      " 'that share common standard-operating procedures, similarly to these in '\n",
      " 'chronic disease genetic epidemiology. We do not neglect the proposed finer '\n",
      " 'scale analyses of various population subsets 67 , but we consider '\n",
      " 'improvements in study design to have much stronger potential impact. All '\n",
      " 'these suggestions are applicable not only for genomics, but also for any '\n",
      " 'type of infect-omics studies, defined as application of various -omics '\n",
      " 'technologies in understanding of the infectious disease development, '\n",
      " 'progression and outcome. In addition, we need to start developing framework '\n",
      " 'of systematic reviews, where we need to compare disease-specific genes and '\n",
      " 'pathways to disease-specific genes, as the final outcome, clinical disease, '\n",
      " 'might be attributable to different sets of genes that make a person more '\n",
      " 'susceptible to e.g. pneumonia regardless on the pathogen that is causing it. '\n",
      " 'Only by comparing such complementary results and insights will we be able to '\n",
      " 'better understand and harness this information in clinical care. These '\n",
      " 'results must be viewed in light of numerous limitations. We reviewed a very '\n",
      " 'diverse field with no unified reporting scheme, suggesting that the search '\n",
      " 'might have missed some relevant articles and data sources. In order to '\n",
      " 'accommodate for this problem we performed extensive search supplemented by '\n",
      " 'hand-search procedure described in Materials and methods section. Existence '\n",
      " 'of wider-scale unpublished data and occasionally our inability to obtain raw '\n",
      " 'data present another possible source of bias. This study also suffers from '\n",
      " 'the limited input, due to exclusion of HLA and KIR markers, haplotypes and '\n",
      " 'complete omission of any information on pathogen, thus providing rather '\n",
      " 'limited insight into the field and possibly missing out some of the key '\n",
      " 'element for more in-depth understanding 77 . Most of these limitations have '\n",
      " 'very negative impact on effect-to-bias ratio, where bias may be several '\n",
      " 'orders of magnitude greater than the size effects. We attempted to contrast '\n",
      " 'this by the use of the stringent analytic approach in an evidence-based '\n",
      " 'manner, including multiple testing correction, careful application of '\n",
      " 'inclusion criteria and quality assessment scoring, coupled with use of '\n",
      " 'random-effect meta-analysis. The study also suffers from inability to adjust '\n",
      " 'the analysis for basic covariates, such as age, sex and comorbidity '\n",
      " 'information. We also used a novel score for quality assessment that was '\n",
      " 'developed within this study and was not validated before. We thus provided '\n",
      " 'basic agreement indices, but wider use of this score would require an '\n",
      " 'in-depth reliability assessment and validation. If we turn back to '\n",
      " 'biological processes, earlier primary studies ignored microbiomes, while '\n",
      " 'some novel studies are suggesting that lungs are not sterile 78, 79 and that '\n",
      " 'respiratory tract microbiomes could have strong modifying effect on health '\n",
      " 'and disease mechanisms. Despite all these limitations, we attempted to '\n",
      " 'provide a field synopsis, supplemented by a generalized framework that could '\n",
      " 'be useful not only for respiratory infectious disease genetics, but can be '\n",
      " 'extended to all infectious diseases. However, these results are just the '\n",
      " 'first step it an attempt to provide a reflection of just one side of the '\n",
      " 'infectious disease story -the host. Only by combining the host genomics (and '\n",
      " 'other -omics) information with pathogen -omics will we be able to truly '\n",
      " 'advance this field and enable the development of personalized infectious '\n",
      " 'disease medicine. Protocol and registration. The review strategy and details '\n",
      " 'were registered prospectively in PROSPERO as the record 2014:CRD42014009072. '\n",
      " 'Eligibility criteria. We undertook an extensive and systematic search of all '\n",
      " 'published studies related to host genetics implicated in development or '\n",
      " 'outcome of respiratory tract infectious diseases. Instead of focusing on a '\n",
      " 'single-disease approach, we aimed at delivering systematic evidence on all '\n",
      " 'respiratory infections, including cases where a syndrome phenotype was '\n",
      " 'reported, such as pneumonia, without the specific pathogen information. '\n",
      " 'Scientific RepoRts | 5:16119 | DOi: 10.1038/srep16119 Information sources. '\n",
      " 'We used three bibliographic databases; PubMed (http://www.ncbi.nlm.nih.gov/ '\n",
      " 'pubmed), Web of Science (http://wok.mimas.ac.uk) and Scopus '\n",
      " '(http://www.scopus.com). We also performed an additional search of the HuGe '\n",
      " 'Literature Finder (http://www.hugenavigator.net), to identify articles that '\n",
      " 'might have been missed in PubMed. Search. Two of the joint first authors '\n",
      " 'performed the search and study selection, which was supervised by the last '\n",
      " 'author. The last author also resolved any discrepancies in scoring system or '\n",
      " 'data extraction process. The search was performed on May 06, 2014, with an '\n",
      " 'update performed on August 25, 2015. After duplicates removal, the search '\n",
      " 'yielded a total of 24,823 articles. All identified articles were firstly '\n",
      " 'checked for inclusion by reading the Title and Abstract. This led to '\n",
      " 'identification and exclusion of a total of 23,677 studies. The remaining 881 '\n",
      " 'articles were supplemented by hand search of their references, producing an '\n",
      " 'additional set of 265 articles. In total, we used a set of 1,146 articles '\n",
      " 'that were read in full and screened against inclusion criteria '\n",
      " '(Supplementary Figure 5) . The study selection process was based on '\n",
      " 'iterative steps. Firstly, the studies needed to report the number of cases '\n",
      " 'and controls for every analysed genotype. Therefore, we initially excluded '\n",
      " 'all of the family-based and linkage studies, as well as genome-wide '\n",
      " 'association (GWAS) studies from this meta-analysis, due to inability to '\n",
      " 'obtain complete set of GWAS results in the appropriate format. However, we '\n",
      " 'retained these studies and provided their qualitative summary, in order to '\n",
      " 'develop an inclusive field synopsis. Secondly, studies had to be based on '\n",
      " 'biallelic single-nucleotide polymorphisms (SNP) or biallelic '\n",
      " 'insertion-deletion marker type. Studies based on other marker types '\n",
      " '(including microsatellites or short tandem repeats), or those that reported '\n",
      " 'aggregated haplotypes were excluded. We also excluded all studies that '\n",
      " 'reported exclusively gene expression profiles without relevant genotyping '\n",
      " 'information. We then limited the review to studies that were published '\n",
      " 'during or after the year 2000 and were written in English language. Despite '\n",
      " 'similarity of disease patterns and mechanisms, we excluded all animal and '\n",
      " 'all in vitro studies and focused on susceptibility in previously healthy '\n",
      " 'subjects. We also excluded studies that reported: any outcome in previously '\n",
      " 'affected patients (either as a part of immunodeficiency syndromes or with '\n",
      " 'any form of disease such as asthma, HIV/AIDS or other comorbidities; where '\n",
      " 'reported), fungal infections (due to also very prevalent comorbidities), '\n",
      " 'nosocomial infections (such as ventilator-related or hospital-acquired '\n",
      " 'pneumonia), exclusively pathogen genotypes or host-pathogen interactions. '\n",
      " 'Studies that reported non-respiratory infection sites (such as '\n",
      " 'extra-pulmonary tuberculosis, meningitis or invasive pneumococcal disease) '\n",
      " 'were also excluded, as the focus of this study was on respiratory infectious '\n",
      " 'diseases. Lastly, we excluded all studies that were obviously reporting '\n",
      " 'previously published results or were re-using the cases that has already '\n",
      " 'been used elsewhere (when we managed to identify this), in order to remove '\n",
      " 'redundant data or publications from meta-analysis. To ensure a systematic '\n",
      " 'source of information, we additionally checked studies which provided '\n",
      " 'insufficient information. A total of 142 e-mails were sent to authors of '\n",
      " 'studies which we classified as having reported insufficient information, and '\n",
      " 'asked for clarifications or raw data sharing. We also contacted all authors '\n",
      " 'of identified GWAS studies to share their results. This entire process '\n",
      " 'results in a total of 10 responses (4 rejections, 2 responses that did not '\n",
      " 'meet the inclusion criteria and 4 cases of clarifications or shared raw '\n",
      " 'data, which are listed in the Acknowledgements). In total, we used 386 '\n",
      " 'articles in the data extraction process (Supplementary Figure 5 ). Data '\n",
      " 'collection process and data items. A total of 48 pieces of information were '\n",
      " 'extracted for every study (available in the on-line RISEdb -Respiratory '\n",
      " 'Infection SuscEptibility database, available at '\n",
      " 'http://www.prepare-europe.eu/risedb). All the data were extracted and '\n",
      " 'entered into a database by two authors, and checked by the third, in order '\n",
      " 'to ensure data integrity. Each identified study was indexed with a unique '\n",
      " 'identification code (RISEdb ID). Some studies provided more than one data '\n",
      " 'point (which corresponds to one marker -one study data entry), and were '\n",
      " 'consequently represented with more than one data entry in our database. In '\n",
      " 'all instances the data were extracted in the raw format as genotype counts, '\n",
      " 'or were calculated from the reported percentages and total sample sizes '\n",
      " 'where number of cases and controls was not available. When more cases or '\n",
      " 'control groups were available in one study, we either pooled them or '\n",
      " 'retained multiple data points from the same study, especially in case of '\n",
      " 'having discovery and replication datasets. In addition, all data points were '\n",
      " 'classified according to the underlying research question into three '\n",
      " 'categories: studies that investigated infectious disease susceptibility, '\n",
      " 'disease severity or disease mortality. In addition, we defined a subset of '\n",
      " 'methodologically better studies, where controls were of known exposure to '\n",
      " 'the pathogen. We developed a novel score for purposes of risk of bias '\n",
      " 'assessment. As the methodological design defines, a study may be prone to '\n",
      " 'three main domains of errors and bias -confounding, selection bias and '\n",
      " 'information bias. These three domains were the basis for the score '\n",
      " 'development, which was entitled Confounding-Selection-Information bias score '\n",
      " '-CSI (Supplementary Table 2 ). The elements of the score were developed on '\n",
      " 'the basis of several previous studies 80 and assessment scores, including '\n",
      " 'Venice criteria for assessing cumulative epidemiologic evidence in genetic '\n",
      " 'associations 81 , Newcastle-Ottawa case-control scale 82 , Cochrane risk of '\n",
      " 'bias tool 83 , and previously established quality scores in genetic '\n",
      " 'epidemiology 84 . Neither of these tools provided a good fit for the risk of '\n",
      " 'bias assessment in observational studies in genetic epidemiology, and we '\n",
      " 'therefore took elements of previous scales and derived a new one. In line '\n",
      " 'with Venice criteria, we scored all three domains in three grades of '\n",
      " 'credibility: high -graded as A; intermediate -B or weak -C. This scheme '\n",
      " 'provided estimates ranging from the best AAA to worst CCC score, which was '\n",
      " 'applied to every data point in RISE database (and not for the meta-analysis '\n",
      " 'results, as the Venice criteria is intended to). Since information bias can '\n",
      " 'affect several aspects of the study, we also divided information bias risk '\n",
      " 'domain in three separate sub-scores, relating to cases and controls '\n",
      " 'selection and genotyping procedures, with the worst of all three defining '\n",
      " 'the overall information bias risk. All CSI scoring was done independently by '\n",
      " 'two authors, and all discrepancies were settled by an agreement or input '\n",
      " 'from a third author (there were 313 or 14.2% data points with initially '\n",
      " 'dissimilar grades, providing good agreement with kappa coefficient of 0.717 '\n",
      " 'and standard error for kappa of 0.011). As an additional quality control '\n",
      " 'check, we re-calculated Hardy-Weinberg test for every control set included '\n",
      " 'in the analysis, using a two tailed chi-square test or an exact test, when '\n",
      " 'appropriate. Those data points that failed HWE test at the level of P < 0.05 '\n",
      " 'were downgraded to C in the CSI score (there were 225 out of 2,209, or 10.2% '\n",
      " 'of all analysed data points that failed HWE), in order to reflect possible '\n",
      " 'methodological limitations of failed HWE (which indicates both the risk for '\n",
      " 'selection and/or information bias). Where cases from a single study were '\n",
      " 'classified as moderate or severe, we did not determine Selection domain of '\n",
      " 'CSI score (a total of 246 data points). In such instances we assigned 0 to '\n",
      " 'selection bias risk, as there was no selection process involved, and '\n",
      " 'second-stage analysis was performed Summary measures and synthesis of '\n",
      " 'results. In the first step of meta-analysis we calculated all odds ratios '\n",
      " 'anew from raw data, in order to provide a common analytic framework and '\n",
      " 'reduce possible differences in data analysis. In order to better understand '\n",
      " 'the genetic architecture underlying gene-disease association, we performed '\n",
      " 'analyses using four separate genetic models: allelic, dominant, recessive '\n",
      " 'and heterozygote advantage. Model selection was based on commonly used '\n",
      " 'allelic model, two additional models that assume different inheritance '\n",
      " 'effects (dominant and recessive), followed by the heterozygote advantage, as '\n",
      " 'previous studies reported such results [85] [86] [87] . All calculations '\n",
      " 'were therefore based on allele counts, rather than exact cases and controls '\n",
      " 'counts. We compared the results of different models, in an attempt to '\n",
      " 'provide broader insight into genetic architecture. Minor allele frequencies '\n",
      " 'in Homo sapiens Short Variation (GRCh38) data set were obtained from Ensembl '\n",
      " 'Biomart on 08.12.2014 88 . We then performed a series of meta-analyses for '\n",
      " 'all SNPs where four or more data points were available for a single SNP. We '\n",
      " 'used random effects meta-analysis, assuming differences in designs and '\n",
      " 'ethnic composition of individual studies. We also calculated fixed-effects '\n",
      " 'model, in order to compare the results. We used I 2 statistics and '\n",
      " 'corresponding confidence intervals as a measure of heterogeneity. In order '\n",
      " 'to account for multiple testing as a result of numerous meta-analyses and '\n",
      " 'different genetic models being performed, we used Bayesian false-discovery '\n",
      " 'probability (BFDP) 72 . It was calculated for nominally significant results '\n",
      " 'only, with a threshold of 0.2, using an online free version of the BFDP '\n",
      " 'calculator (http://faculty.washington.edu/jonno/BFDP.xls). In addition, we '\n",
      " 'calculated BFDP using two prior probabilities, with medium/low prior level '\n",
      " '(0.05 to E-03), consistent with a candidate gene; and very low prior level '\n",
      " '(E-04 to E-06), consistent with a random SNP. The advantage of this approach '\n",
      " 'over the commonly used corrections is that it is not dependent on the number '\n",
      " 'of tests performed, as it relies on odds ratios and confidence intervals for '\n",
      " 'calculation. In addition, it provides conceptual improvement over the '\n",
      " 'similar methods, such as false-positive report probability (FPRP) 89 . Due '\n",
      " 'to substantial requirements for appropriate use of publication bias analysis '\n",
      " 'methods 90 , we only examined publication bias visually. All analyses were '\n",
      " 'performed in R, version 2.15.3 91 . Meta-analysis was performed with package '\n",
      " 'meta 3.2-0 and metabin, forest and funnel functions 92 . Hardy-Weinberg '\n",
      " 'equilibrium testing was performed in HardyWeinberg_1.5.2 package 93 , with '\n",
      " 'HWExactMat or HWChisqMat functions, based on the cell expected frequencies. '\n",
      " 'Risk of bias across studies. For purpose of meta-analysis credibility '\n",
      " 'assessment, we used Venice criteria 81 , in all instances of nominally '\n",
      " 'significant results. Study power was assessed on the basis of sample sizes '\n",
      " '(with sample sizes of up to a 1,000 graded as C, 1,001-10,000 graded as B, '\n",
      " 'and over 10,001 graded as A). Heterogeneity was based on I 2 statistics '\n",
      " '(graded as A in case of 0-25%, B in case of 26-50% and C in case of I2 being '\n",
      " 'over 50% ), while the third score domain was fixed as C (weak credibility), '\n",
      " 'due to very prevalent risk of bias in primary studies.',\n",
      " 'Molecular chaperones represent a large group of proteins that are essential '\n",
      " 'for maintaining cellular homeostasis and survival. As such, the roles of '\n",
      " 'these proteins are numerous; facilitating correct protein folding or '\n",
      " 'unfolding, assembly or disassembly of multimeric protein complexes, '\n",
      " 'participating in translocation of proteins and vesicles into organelles, '\n",
      " 'stabilising a wide range of signalling molecules and preventing aggregation '\n",
      " 'of non-native proteins (reviewed in [1, 2] ). Heat shock proteins (HSP) are '\n",
      " 'classified according to their molecular weight into several families: HSP40, '\n",
      " 'HSP60, HSP70, HSP90, HSP100, HSP110 and the small HSP (less than 34 kDa) [3] '\n",
      " '. The functional importance of the HSP70 and HSP90 families of molecular '\n",
      " 'chaperones is exemplified by their emerging implications in a variety of '\n",
      " 'diseases, including cancer [4, 5] , neurodegeneration [6] or viral infection '\n",
      " '[7, 8] . As such they have gained significant interest recently as potential '\n",
      " 'drug targets. Eukaryotes have multiple genes encoding for chaperones of the '\n",
      " 'HSP70 family, which are amongst the most conserved proteins in evolution [9] '\n",
      " '[10] [11] . The major Hsp70 isoforms are the constitutively expressed Hsc70, '\n",
      " 'the stress-inducible Hsp70 (iHsp70), the endoplasmic reticulum resident '\n",
      " '(Grp78) and the mitochondrial form (Grp75). All Hsp70 isoforms have an '\n",
      " 'N-terminal domain which harbours a highly conserved ATPase and a C-terminal '\n",
      " 'substrate binding domain [1] . Hsp90 isoforms which comprise the inducible '\n",
      " 'and constitutively-expressed isoforms (Hsp90α and Hsp90β respectively), the '\n",
      " 'ER resident (Grp74) and the mitochondrial form (TRAP1), also possess a '\n",
      " 'N-terminal ATP binding domain, although this has no similarity to the '\n",
      " 'ATP-binding domain found in the chaperones of the HSP70 family [5] . The '\n",
      " 'presence of ATPase pockets in both families of chaperones makes these '\n",
      " 'proteins desirable targets for small molecule inhibitors [12, 13] . The '\n",
      " 'therapeutic potential of these compounds is especially evident for several '\n",
      " 'HSP90 inhibitors, having already reached phase II and III clinical trials '\n",
      " '[14, 15] . Targeting of Hsp70 isoforms has been more challenging [12] , but '\n",
      " 'recently specific inhibitors have also undergone clinical trials [16, 17] . '\n",
      " 'Importantly, the development of highly specific inhibitors for Hsp70 '\n",
      " 'isoforms may have potential for the treatment of a diverse group of viruses '\n",
      " 'as the functional importance of Hsp70 isoforms in the life cycle of numerous '\n",
      " 'viruses has been highlighted over the past few years [8] . Distinct Hsp70 '\n",
      " 'isoforms are usurped to aid in many stages of viral replication as varied as '\n",
      " 'viral entry, uncoating, transcription, envelope protein maturation, '\n",
      " 'morphogenesis or DNA replication [8] . Therefore, the importance of these '\n",
      " 'chaperones in the life cycle of such a wide range of viruses suggests the '\n",
      " 'potential of these proteins as targets for broad-spectrum antivirals. '\n",
      " \"Kaposi's sarcoma-associated herpesvirus (KSHV) is the causative agent of \"\n",
      " \"several AIDSassociated malignancies, including Kaposi's sarcoma (KS), a \"\n",
      " 'highly vascular tumour of NE cannot be purified completely due to its '\n",
      " 'multiple subcellular connections. The outer nuclear membrane is continuous '\n",
      " 'with the endoplasmic reticulum and interacts with the cytoskeleton [36, 37] '\n",
      " 'while the inner nuclear membrane binds to host chromatin [38] [39] [40] . '\n",
      " 'Thus, we took advantage of its incomplete purity so that we could isolate '\n",
      " 'not only the NE and embedded nuclear pore complexes, but also components '\n",
      " 'found in the NE neighbourhood, such as RTCs. Utilising this novel approach '\n",
      " 'we demonstrate that cellular chaperones from the HSP70 family (Hsc70 and '\n",
      " 'iHsp70) are significantly increased in the NE-associated RTCs of reactivated '\n",
      " 'cells. Functional dissection further demonstrates that these chaperones were '\n",
      " 'specifically recruited to the periphery of incipient RTCs coinciding in time '\n",
      " 'with their formation. When actively replicated viral DNA was synthesised the '\n",
      " 'chaperones were recruited within RTCs. Strikingly, inhibition of Hsp70 '\n",
      " 'isoforms precluded RTC formation, curtailed chaperone redistribution within '\n",
      " 'RTCs and RNAPII recruitment to viral promoters. Importantly, abrogation of '\n",
      " 'lytic replication occurred without affecting cell viability, suggesting that '\n",
      " 'the cellular housekeeping functions carried out by these chaperones were not '\n",
      " 'compromised. As such, HSP70 inhibitors may provide a novel therapeutic '\n",
      " 'approach for the treatment of KSHV-associated malignancies, in particular it '\n",
      " 'would be interesting to determine the efficacy of combining the potential of '\n",
      " 'inhibiting lytic replication using HSP70 inhibitors with the previous '\n",
      " 'reported effect of HSP90 inhibitors to eradicate latent KSHV reservoirs [41] '\n",
      " '. The cellular chaperones Hsc70, iHsp70 and Grp78 were enhanced in the '\n",
      " 'NE-associated RTCs of reactivated HEK-293T rKSHV.219 cells To investigate '\n",
      " 'differential proteome changes which occur during KSHV lytic replication in '\n",
      " 'NE-associated RTCs, we utilised the HEK-293T cell line containing a latent '\n",
      " 'recombinant KSHV virus (rKSHV.219) [42] . This cell line can be reactivated '\n",
      " 'into a full lytic replication cycle via chemical induction. Unreactivated '\n",
      " 'cells were grown in isotopically labelled media (R6K4), while cells to be '\n",
      " 'reactivated were grown in label-free media (R0K0). After isotopic labelling '\n",
      " 'was complete, cells were reactivated for 48 h and nuclear envelopes (NEs) '\n",
      " 'were then successfully purified using a recently described method [35] , '\n",
      " 'with minor modifications. Western blot analysis of the NE preparations '\n",
      " 'demonstrated an enrichment of nucleoporins (Nups), lamins and histones and a '\n",
      " 'loss of cytoplasmic (GAPDH) and nucleolar (B-23, C-23) proteins compared '\n",
      " 'with whole cell (WC) lysates (Fig 1) . The essential KSHV mRNA processing '\n",
      " 'protein, ORF57 [43] , and viral RTA served as markers for lytic viral '\n",
      " 'replication and RTC enrichment, as both of these proteins are known to be '\n",
      " 'recruited to KSHV RTCs [31, 44] . The monoclonal antibody (Mab414), which '\n",
      " 'recognises the phenylalanine-glycine (FG)-repeat motif present in numerous '\n",
      " 'Nups, and the polyclonal antibodies against Nup160 and lamin B1 were used to '\n",
      " 'assess enrichment of the NE region. NE preparations showed a higher number '\n",
      " 'of Nups (closed arrows) than their respective WC preparations; although some '\n",
      " 'Nups were lost following 0.3 M salt wash (open arrow). Following LC-MS/MS '\n",
      " 'analysis and using a minimum of three unique peptides assigned to a single '\n",
      " 'protein, most proteins (1072) remained unchanged in their abundance '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'irrespectively of KSHV lytic infection and only five proteins had a '\n",
      " 'significant reduction (ratio cut-off <0.5) in NEs of reactivated cells. In '\n",
      " 'contrast, 216 proteins showed a significant increase (ratio cut-off >1.9) '\n",
      " 'during lytic replication. Importantly, multiple cellular proteins that are '\n",
      " 'known or expected to localize within herpesvirus RTCs; such as those '\n",
      " 'associated with KSHV ori-Lyt, the HCMV transactivator IE2-p86 protein or the '\n",
      " 'herpes simplex virus-1 (HSV-1) ICP8 protein were found significantly '\n",
      " 'increased in the NE regions of reactivated cells using the less stringent '\n",
      " 'ratio cut-off of 1.5 (S1 Table) . Some of these cellular proteins included '\n",
      " 'CSNK2A1 [45] , BLM [32] , topoisomerases I and II [31, 46, 47] and DEAD box '\n",
      " 'helicases DDX5 [32] and DDX17 [45] . Thus, LC-MS/MS results confirmed the '\n",
      " 'correct isolation of the NE region and accompanying RTCs. Importantly, many '\n",
      " 'of the 216 identified proteins most likely represent novel cellular proteins '\n",
      " 'hijacked by KSHV, not only due to the subcellular fractionation carried out '\n",
      " 'but also to the uncommon use of urea for protein extraction before LC-MS/ '\n",
      " 'MS. All proteins identified by LC-MS/MS can be seen on S1 Dataset. '\n",
      " 'Bioinformatical analysis revealed several upregulated pathways (ratio '\n",
      " 'cut-off >1.9) in reactivated cells (S2 Table) . Of particular interest was '\n",
      " 'an upregulated pathway which related to protein folding. This included '\n",
      " 'several Hsp70 isoforms and their associated co-chaperones from the HSP40 '\n",
      " '(DNAJ) family ( Table 1) . Notably, the constitutively expressed chaperone '\n",
      " 'Hsc70 presented a 4.1-fold increase with 41 unique peptides assigned. This '\n",
      " 'protein had the highest fold increase associated with the most unique '\n",
      " 'peptide number of all the proteins identified by LC-MS/MS. This could be due '\n",
      " 'to increased Hsc70 expression or to the redistribution of Hsc70 from the '\n",
      " 'cytoplasm into the NE region during KSHV lytic replication. Therefore, due '\n",
      " 'to the vital importance of Hsp70 isoforms in the replication cycle of a wide '\n",
      " 'range of viral families, we focussed our studies herein on the roles of the '\n",
      " 'three main Hsp70 isoforms (Hsc70, iHsp70 and Grp78) during KSHV lytic '\n",
      " 'replication. Hsc70 and iHsp70 are redistributed from the cytoplasm to both '\n",
      " 'the periphery and within KSHV-induced RTCs To verify the enrichment of Hsc70 '\n",
      " 'in the NE-associated RTCs of reactivated cells detected by our quantitative '\n",
      " 'proteomic approach, indirect immunofluorescence was used to label endogenous '\n",
      " 'Hsc70 protein in TREx BCBL1-RTA cells, a KSHV latently infected B-lymphocyte '\n",
      " 'cell line containing a Myc-tagged version of viral RTA under the control of '\n",
      " 'a doxycycline-inducible promoter [48] . Hsc70 protein was equally '\n",
      " 'distributed between the cytoplasm and nucleus of unreactivated cells in a '\n",
      " 'fine punctuate pattern (Fig 2Ai) . Similar Hsc70 localization was seen '\n",
      " 'during early lytic replication (12 h reactivation), when RTA protein was '\n",
      " 'diffuse in the nucleus, Successful enrichment of the nuclear envelope region '\n",
      " 'and associated KSHV RTCs in HEK-293T rKSHV.219 cells. Successful enrichment '\n",
      " 'of NE regions in unreactivated (-) and reactivated for 48 h (+) HEK-293T '\n",
      " 'rKSHV.219 cells was demonstrated by Western blotting. Equal amounts of total '\n",
      " 'protein extracted from whole cell (WC) and nuclear envelope (NE) fractions '\n",
      " 'were used. The monoclonal antibody Mab414 specifically recognises the '\n",
      " 'phenylalanine-glycine (FG)-repeat motif present in numerous nucleoporins '\n",
      " '(Nups). Closed arrows point to Nups that were only detected by MAb414 after '\n",
      " 'NE enrichment. Open arrow points to a nucleoporin lost during NE enrichment. '\n",
      " 'Several FG Nups, together with Nup160, histone H3 and lamin B1 were all '\n",
      " 'markedly enriched in the NE fractions compared with WC fractions. The '\n",
      " 'nucleolar proteins B-23 and C-23 and the cytoplasmic GAPDH protein were all '\n",
      " 'decreased in NEs fractions indicating correct subcellular fractionation. The '\n",
      " 'viral RTC-associated ORF57 and RTA proteins were used to monitor lytic '\n",
      " 'reactivation and assess RTC enrichment. ORF57 antibody detected both full '\n",
      " 'length ORF57 protein (FL-ORF57) and the caspase-7-cleaved ORF57 (CL-ORF57). '\n",
      " 'prior to RTC formation (Fig 2Aii) . In contrast, at later reactivation time '\n",
      " 'points (20 h), in which RTA was organised into small viral RTCs peripherally '\n",
      " 'located in the nucleus, numerous nuclear Hsc70 foci that were predominantly '\n",
      " 'adjacent to RTCs were observed (Fig 2Aiii) . During late reactivation time '\n",
      " 'points (26 h), cellular chromatin was marginalised to the nuclear periphery '\n",
      " '(Fig 2Aiv, nucleus highlighted in yellow) and larger Hsc70 foci avidly '\n",
      " 'accumulated within these fully-developed RTCs (Fig 2Aiv) . Reduced levels of '\n",
      " 'cytoplasmic Hsc70 were also observed at this time point (Fig 2Aiv arrows) , '\n",
      " 'suggesting that Hsc70 is redistributed from the cytoplasm to the nucleus '\n",
      " 'during KSHV lytic replication. This is further supported by the fact that '\n",
      " 'fractionation of TREx BCBL1-RTA cells into nuclear (N) and cytoplasmic (C) '\n",
      " 'fractions displayed an enrichment of Hsc70 in the nuclei of reactivated '\n",
      " 'cells which occurred without a noticeable increase in Hsc70 protein levels '\n",
      " 'in whole cell (WC) lysates ( Fig 2B) . Due to the observed redistribution of '\n",
      " 'Hsc70 to KSHV RTCs, further co-localization studies between Hsc70 and the '\n",
      " 'sites of viral DNA replication were performed in TREx BCBL1-RTA cells. Here, '\n",
      " 'cells were triple-labelled with Click-iT EdU Alexa Fluor 647 and antibodies '\n",
      " 'specific for RTA and Hsc70. In unreactivated cells, newly synthesized '\n",
      " 'cellular DNA during mid-S-phase (EdU incorporated) occurred mainly at the '\n",
      " 'nuclear periphery as previously observed in other cell types [49, 50] (Fig '\n",
      " '2Ci) . During early reactivation, Hsc70 was adjacent to RTA which was '\n",
      " 'present in small viral RTCs containing actively replicating viral DNA (Fig '\n",
      " '2Cii arrow) . At this stage, a proportion of Hsc70 also co-localised with '\n",
      " \"RTA (Fig 2Cii') . During late reactivation, when cellular chromatin was \"\n",
      " 'marginalised, much larger RTCs were visible and newly synthesized cellular '\n",
      " 'DNA was not apparent, here Hsc70 completely co-localised with newly '\n",
      " \"synthesized viral DNA and RTA (Fig 2Ciii and 2Ciii') . The location of the \"\n",
      " 'other two main Hsp70 isoforms (iHsp70 and Grp78) during KSHV lytic '\n",
      " 'replication was also investigated by indirect immunofluorescence microscopy. '\n",
      " 'iHsp70 was cytoplasmic in unreactivated TREx BCBL1-RTA cells (S1Ai Fig), in '\n",
      " 'contrast, an increase in nuclear iHsp70 labelling was observed in '\n",
      " 'reactivated cells, which displayed similar chaperone foci as those seen '\n",
      " 'during early reactivation for Hsc70 (S1Aii Fig). Occasionally, cells '\n",
      " 'displayed RTCs completely filled by iHsp70 (S1Aiii and S1Aiv Fig, asterisks) '\n",
      " '. Large iHsp70 foci positioned adjacent to RTCs were also seen in '\n",
      " 'reactivated cells at later time points (S1Aiv Fig  arrows) . Co-localisation '\n",
      " 'of iHsp70 with actively replicating viral DNA was also observed during late '\n",
      " 'reactivation (S1B Fig). Hsc70 and iHsp70 nuclear foci appeared at the same '\n",
      " 'time as KSHV RTCs were assembled, suggesting that these chaperones could be '\n",
      " 'involved in RTC assembly. Additionally, complete co-localization of Hsc70 '\n",
      " 'and iHsp70 with viral DNA indicated that these chaperones could also '\n",
      " 'participate in viral DNA replication and/or capsid assembly. In contrast, '\n",
      " 'the endoplasmic reticulum (ER) Hsp70 isoform, Grp78, was not redistributed '\n",
      " 'in reactivated TREx BCBL1-RTA cells (S2 Fig) , consistent with its ER '\n",
      " 'retention signal [51] . Nevertheless, reactivated cells seemed to accumulate '\n",
      " 'larger amounts of Grp78 in the ER. To confirm these results, '\n",
      " 'immunoflourescence studies were also performed using HEK-293T prior to RTC '\n",
      " 'formation, when RTA was diffuse in the nucleus (ii). At 20 h reactivation '\n",
      " 'viral RTA was assembled into incipient RTCs. Hsc70 was not detected in the '\n",
      " 'cytoplasm and instead numerous nuclear foci that were positioned '\n",
      " 'predominantly adjacent to RTCs were seen (iii). At later reactivation times '\n",
      " 'very large Hsc70 foci were completely recruited within RTCs. Hsc70 '\n",
      " 'cytoplasmic depletion is indicated (iv arrows). Nucleus highlighted in '\n",
      " 'yellow shows a typical KSHV fully developed-RTC with cellular chromatin '\n",
      " 'marginalised to the nuclear periphery. (B) Unreactivated (-) or reactivated '\n",
      " 'for 26 h (+) TREx BCBL1-RTA cells were fractionated into whole cell (WC), '\n",
      " 'cytoplasmic (C) and nuclear (N) fractions. Equal amounts of total protein '\n",
      " 'from each fraction were analysed by Western blotting. Nuclear fractions were '\n",
      " 'characterised by enrichment of lamin B1 and absence of cytoplasmic GAPDH, '\n",
      " 'while cytoplasmic fractions showed the inverse profile. A small decrease in '\n",
      " 'cytoplasmic Hsc70 which correlated with a small increase in nuclear Hsc70 '\n",
      " 'was detected in reactivated cells, further supporting that Hsc70 was '\n",
      " 'redistributed from the cytoplasm to the nucleus. (C) TREx BCBL1-RTA cells '\n",
      " \"remained unreactivated (i) or reactivated for 24 h (iiiii') followed by \"\n",
      " 'triple-labelling with antibodies specific for RTA and Hsc70 and Click-iT EdU '\n",
      " 'Alexa Fluor 647, the latter allowed visualization of newly synthesized DNA. '\n",
      " 'During early reactivation, a proportion of Hsc70 protein was adjacent to '\n",
      " 'small viral RTCs. Arrow points to a small KSHV RTC filled with viral DNA '\n",
      " \"(ii). Hsc70 also partially co-localised with viral DNA (ii) and RTA (ii') in \"\n",
      " 'small RTCs. During later reactivation times, Hsc70 completely moved within '\n",
      " 'fully-developed RTCs strongly co-localising with viral DNA (iii) and RTA '\n",
      " \"(iii'). rKSHV.219 cells, in which the presence of the recombinant virus is \"\n",
      " 'tracked by expression of the green fluorescent protein (GFP) from the '\n",
      " 'EF-1alpha promoter and lytic reactivation levels are monitored by expression '\n",
      " 'of the red fluorescent protein (RFP) from the KSHV lytic noncoding '\n",
      " 'polyadenylated nuclear (PAN) RNA promoter. Unreactivated cells displayed '\n",
      " 'cytoplasmic iHsp70 and Hsc70 labelling, whereas~40% of reactivated cells (24 '\n",
      " 'h reactivation) revealed nuclear iHsp70 and Hsc70 accumulations that '\n",
      " 'appeared to assemble within small RTCs (S3A Fig arrows and S3B Fig arrows '\n",
      " 'respectively) . The incomplete redistribution of iHsp70 and Hsc70 foci into '\n",
      " 'RTCs was likely due to a more asynchronous progression through the lytic '\n",
      " 'cycle in induced cells by TPA and sodium n-butyrate than in '\n",
      " 'doxycycline-induced TREx BCBL1-RTA cells. Similarly to TREx BCBL1-RTA cells, '\n",
      " 'Grp78 was not redistributed in HEK-293T rKSHV.219 cells, although larger '\n",
      " 'amounts appeared to accumulate in the ER of reactivated cells (S3C Fig), in '\n",
      " 'agreement with the significantly increased amounts of Grp78 detected in the '\n",
      " 'NE region of these cells ( Table 1) . These results clearly demonstrate that '\n",
      " 'KSHV specifically redistributes the molecular chaperones, Hsc70 and iHsp70, '\n",
      " 'from the cytoplasm to the nucleus, in contrast to Grp78, which coincides '\n",
      " 'with the initial formation of KSHV RTCs. Treatment with the small molecule '\n",
      " 'inhibitor VER-155008 abrogated viral protein synthesis at non-cytotoxic '\n",
      " 'concentrations Members of the HSP70 chaperone family possess an N-terminal '\n",
      " 'nucleotide binding domain with ATPase activity which is essential for their '\n",
      " 'function. To examine the implications of Hsc70 and iHsp70 redistribution '\n",
      " 'into KSHV RTCs, a small molecule inhibitor, VER-155008, (a '\n",
      " 'dibenzyl-8-aminoadenosine analog) was utilised. This is the only inhibitor '\n",
      " 'that has been demonstrated to specifically target the highly homologous '\n",
      " 'ATPase pocket present in the three main human Hsp70 isoforms [12, 13, 52, '\n",
      " '53] , which is highly divergent structurally from the ATPase pocket found in '\n",
      " 'chaperones of the HSP90 family [12, 54] . As such, VER-155008 functions as '\n",
      " 'an ATP mimetic that specifically inhibits the ATPase activity of members of '\n",
      " 'the HSP70 family. Initially, cytotoxicity of this compound was assessed in '\n",
      " 'unreactivated TREx BCBL1-RTA cells. Following 24 h inhibitor exposure, using '\n",
      " 'a non-radioactive MTS assay, which quantitatively assesses cell '\n",
      " 'proliferation, a drastic reduction in cell metabolic activity was seen for '\n",
      " 'inhibitor concentrations higher than 6 μM (S4 Fig) , thus concentrations '\n",
      " 'ranging from 1 to 4 μM were used for further cytotoxicity characterization '\n",
      " '(Fig 3Ai) . The inhibitor triggered apoptosis in a dose-dependent manner as '\n",
      " 'demonstrated by the caspase 3-mediated cleavage of full length poly '\n",
      " '[ADP-ribose] polymerase (FL-PARP1) protein into cleaved PARP1 (CL-PARP1) '\n",
      " '(Fig 3Aii) . Small amounts of CL-PARP1 were seen at 1 μM and 2 μM with a '\n",
      " 'significant increase of this form after 3 μM. These results were confirmed '\n",
      " 'with ApoTox-Glo Triplex Assay by quantitatively measuring viability, '\n",
      " 'cytotoxicity and activation of effector caspases-3/7 in the same sample well '\n",
      " 'after 24 h inhibitor treatment. A dose-dependent decrease in viability was '\n",
      " 'evident from 2 μM to 50 μM while cytotoxicity and activation of caspases-3/7 '\n",
      " 'were only considerably increased at concentrations higher than 3 μM ( Fig '\n",
      " '3B) . Next, TREx BCBL1-RTA cells were reactivated for 24 h in the presence '\n",
      " 'of drug vehicle DMSO (0.1%) or a range of increasing inhibitor '\n",
      " 'concentrations. Cells treated with the inhibitor at non-cytotoxic '\n",
      " 'concentrations (1 to 2.5 μM) revealed a drastic reduction in the levels of '\n",
      " 'early ORF57 and late minor capsid (mCapsid) proteins. A moderate reduction '\n",
      " 'in the immediateearly RTA protein was also seen ( Fig 3C) . Of note, when '\n",
      " 'detecting the fusion protein RTA-Myc, which expression is not from the KSHV '\n",
      " 'genome, with anti-Myc antibody, the decrease in RTA-Myc was not as dramatic '\n",
      " 'as that seen for viral RTA, suggesting that the decrease in viral proteins '\n",
      " 'was not due to a general cytotoxic effect of the inhibitor on the cells, and '\n",
      " 'that viral, but not cellular proteins were specifically affected. As an '\n",
      " 'additional cellular control, protein levels of the large subunit of RNAPII, '\n",
      " 'which has a half-life of 12-16 h [55] , was assessed with antibodies '\n",
      " 'specific for the different phosphorylated forms of RNAPII. Protein levels of '\n",
      " 'these forms were not significantly changed in the presence of the inhibitor, '\n",
      " 'nor were those of Hsc70 or Hsp90 proteins. Importantly, in the presence of '\n",
      " 'VER-155008, iHsp70 levels were not upregulated. iHsp70 upregulation is a '\n",
      " 'universal hallmark of Hsp90 inhibition not only in vitro [56] but also in '\n",
      " 'clinical trials [57] , pointing to selectivity for Hsp70 isoforms by '\n",
      " 'VER-155008. Hsp90 and Hsp70 chaperone machineries have been reported to be '\n",
      " 'crucial for the stability and/or maturation of multiple viral proteins [7, '\n",
      " '41, [58] [59] [60] [61] [62] . Therefore to ascertain whether Hsp70 isoforms '\n",
      " 'could stabilise the essential KSHV lytic proteins RTA and ORF57, TREx '\n",
      " 'BCBL1-RTA cells were reactivated for 24 h to allow sufficient viral protein '\n",
      " 'expression followed by addition of DMSO control or VER-155008 in conjunction '\n",
      " 'with cycloheximide (CHX) at 50 μg/ml to block de novo protein synthesis. '\n",
      " 'Protein lysates were then collected at different times after addition of '\n",
      " 'CHX. The half-life of RTA and ORF57 proteins from inhibitor-treated cells '\n",
      " 'were not altered compared with DMSO-treated cells (Fig 3D) . These results '\n",
      " 'indicate that the observed decrease in viral protein synthesis was prior to '\n",
      " 'translation and that neither viral RTA nor ORF57 protein were client '\n",
      " 'proteins of the Hsp70 isoforms. As such, this highlights a potentially novel '\n",
      " 'role of Hsp70 isoforms in the KSHV replication cycle independent of viral '\n",
      " 'protein stability and maturation. To further corroborate these results, '\n",
      " 'experiments were also repeated in HEK-293T rKSHV.219 cells. Again, cell '\n",
      " 'metabolic activity, PARP1 cleavage, viability, cytotoxicity and activation '\n",
      " 'of caspases-3/7 in unreactivated cells were all assessed at a range of '\n",
      " 'increasing inhibitor concentrations (Fig 4A and 4B ). On this occasion, the '\n",
      " 'inhibitor did not trigger apoptosis (Fig  4Aii and 4B ) but it caused a '\n",
      " 'pronounced cell cycle arrest at 24 h exposure at concentrations of 20 μM '\n",
      " 'demonstrated by a reduced number of metabolically active cells that '\n",
      " 'exhibited no increased cytotoxicity [63] (Fig 4B) . It is known that the '\n",
      " 'apoptotic potential of VER-155008 is cell line-dependent and that VER-155008 '\n",
      " 'can cause cell cycle arrest in human colon, breast and lung tumour cell '\n",
      " 'lines [54, 64] . HEK-293T rKSHV.219 cells were also reactivated for 24 h in '\n",
      " 'the presence of drug vehicle DMSO (0.1%) or increasing inhibitor '\n",
      " 'concentrations. Endogenous RTA, ORF57 and mCapsid protein levels were '\n",
      " 'moderately reduced in cells treated at an inhibitor concentration of 10 μM '\n",
      " 'and severely reduced at 40 μM while cellular proteins remain unaffected ( '\n",
      " 'Fig 4C) . These were relatively high inhibitor concentrations compared with '\n",
      " 'TREx BCBL1-RTA cells; nonetheless a concentration of 40 μM has been shown '\n",
      " 'before to be necessary for inhibition of Hsp70 isoforms in human carcinoma '\n",
      " 'cell lines [54] . As previously seen in TREx BCBL1-RTA cells, when blocking '\n",
      " 'de novo protein synthesis with CHX at 100 μg/ml in HEK-293T rKSHV.219 cells, '\n",
      " 'the half-life of RTA and ORF57 proteins were not reduced even in the '\n",
      " 'presence of VER-155008 at 50 μM ( Fig 4D) . This supports the findings seen '\n",
      " 'in TREx BCBL1-RTA cells, suggesting that the decrease in viral protein '\n",
      " 'production was due to a pretranslation event. using an MTS assay. (ii) A '\n",
      " 'dose-dependent cleavage of full length (FL) PARP1 protein into cleaved (CL) '\n",
      " 'PARP1 was observed in the presence of VER-155008. (B) Unreactivated cells '\n",
      " 'were exposed for 24 h to increasing inhibitor concentrations. Concentrations '\n",
      " 'higher than 3 μM resulted in increased cytotoxicity and activation of '\n",
      " 'effector caspases-3/7 as demonstrated by quantification with ApoTox-Glo '\n",
      " 'Triplex Assay. (C) Immunoblot analysis showing that reactivated cells '\n",
      " 'treated with non-cytotoxic inhibitor concentrations (1 to 2.5 μM) for 24 h '\n",
      " 'revealed a decrease in viral proteins compared with DMSO-treated samples '\n",
      " 'while cellular proteins remained unaffected. RNAPIIo denotes '\n",
      " 'hypophosphorylated RNAPII. Ser5 and Ser2 RNAPII denote serine 5-and serine '\n",
      " '2hyperphosphorylated RNAPII forms respectively. (D) Cells were reactivated '\n",
      " 'for 24 h to allow robust viral protein production. Then, DMSO control (0.1%) '\n",
      " 'or VER-155008 was added in conjunction with cycloheximide (CHX) at 50 μg/ml '\n",
      " 'to block de novo protein synthesis. Protein lysates were collected at '\n",
      " 'several times post-CHX addition (0, 4, 8, 16 and 24 h) and analysed by '\n",
      " 'Western blotting. VER-155008 did not alter the half-life of RTA or ORF57 '\n",
      " 'protein, thus these proteins were not clients of Hsp70 isoforms. The small '\n",
      " 'molecule inhibitor VER-155008 caused a significant reduction in viral '\n",
      " 'transcripts, viral DNA and progeny at non-cytotoxic concentrations As the '\n",
      " 'block in KSHV protein synthesis occurred pre-translationally, viral gene '\n",
      " 'expression was quantified in the absence or presence of VER-155008. TREx '\n",
      " 'BCBL1-RTA cells were reactivated for 24 h and two-step quantitative reverse '\n",
      " 'transcription PCR (qRT-PCR) was carried out to quantify a range of viral '\n",
      " 'transcripts. A significant decrease in early (PAN, ORF57, K12 and vGPCR), '\n",
      " 'late (gL and gB) viral transcripts and ori-Lyt transcripts was observed in a '\n",
      " 'dose-dependent manner, with all transcripts with the exception of vGPCR '\n",
      " 'being significantly reduced at an inhibitor concentration of 1 μM (Fig 5A) . '\n",
      " 'To determine whether cellular transcription was negatively affected in the '\n",
      " 'presence of VER-155008, firstly the stability of GAPDH transcript was '\n",
      " 'determined in mRNA decay assays using the transcriptional inhibitor '\n",
      " 'actinomycin D (AcD) (2.5 μg/ml) in TREx BCBL1-RTA cells. After 6 h of AcD '\n",
      " 'treatment, the amount of GAPDH mRNA was reduced by half (Fig 5B) , '\n",
      " 'indicating a short stability of GAPDH mRNA in this cell line. We then '\n",
      " 'plotted the raw cycle threshold (C T ) for GAPDH transcript from the same '\n",
      " 'samples in which viral transcripts had been quantified after 24 h of '\n",
      " 'VER-155008 treatment. As the same amount of total RNA was converted into '\n",
      " 'cDNA for all samples, if cellular transcription was not compromised a very '\n",
      " 'similar C T is expected for all samples. Indeed, samples treated with up to '\n",
      " '3 μM VER-155008 were all within 0.4 C T from the 12.7 C T of DMSO-treated '\n",
      " 'samples. Only after concentrations higher than 3 μM GAPDH mRNA levels were '\n",
      " 'significantly reduced compared to DMSO-treated samples as shown by a '\n",
      " 'significantly higher C T value ( Fig 5C) . This is consistent with the '\n",
      " 'cytotoxicity profile of VER-155008 in TREx BCBL1-RTA cells (Fig 3A and 3B ) '\n",
      " 'and the clear decrease in RTA-myc protein (which expression is not from the '\n",
      " 'KSHV genome) at inhibitor concentrations higher than 3 μM ( Fig  3C) . Taken '\n",
      " 'together, these results suggest that cellular transcription was compromised '\n",
      " 'at concentrations of VER-155008 higher than 3 μM while at concentrations '\n",
      " 'lower than 3 μM transcription was occurring normally. Interestingly, '\n",
      " 'transcription of viral genes which also require host RNAPII for their '\n",
      " 'expression was negatively affected even at VER-155008 concentrations lower '\n",
      " 'than 3 μM. Next, we assessed whether the inhibitor also caused a reduction '\n",
      " 'in viral DNA replication. TREx BCBL1-RTA cells were reactivated for 72 h, '\n",
      " 'total DNA was isolated and realtime qPCR was performed using primers '\n",
      " 'specific for ORF57. While DMSO-treated cells reached~9-fold increase in '\n",
      " 'viral DNA load, inhibitor concentrations of 2 μM or higher resulted in a '\n",
      " 'significant reduction (> 30%) in viral DNA ( Fig 5D) . Moreover, the '\n",
      " 'production of infectious KSHV virions in TREx BCBL1-RTA cells was evaluated '\n",
      " 'in the presence of VER-155008 at 2.5 μM or vehicle drug DMSO. For this, '\n",
      " 'cells were reactivated and treated for 72 h, culture medium was centrifuged '\n",
      " 'and incubated for 24 h with HEK-293T cells. Total RNA was then isolated and '\n",
      " 'qRT-PCR carried out. A significant reduction (~80%) in the release of '\n",
      " 'infectious viral progeny was observed in inhibitor-treated cells ( Fig 5E) . '\n",
      " 'Viral transcripts were also quantified at 24 h reactivation in HEK-293T '\n",
      " 'rKSHV.219 cells in the absence or presence of VER-155008. At non-cytotoxic '\n",
      " 'concentrations of 10 μM there was a drastic decrease for all PARP1 was '\n",
      " 'observed faintly. (B) Unreactivated cells were exposed for 24 h to '\n",
      " 'increasing inhibitor concentrations. A reduced number of metabolically '\n",
      " 'active cells accompanied by no increased cytotoxicity was observed in the '\n",
      " 'presence of the inhibitor. There was no activation of effector caspases even '\n",
      " 'in the presence of 60 μM VER-155008. This phenotype is consistent with cell '\n",
      " 'cycle arrest. Results were quantified with ApoTox-Glo Triplex Assay. (C) '\n",
      " 'Immunoblot analysis showing that reactivated cells treated with 40 μM '\n",
      " 'VER-155008 for 24 h resulted in abrogation of viral proteins compared with '\n",
      " 'DMSO-treated samples while cellular proteins remained constant. (D) Cells '\n",
      " 'were reactivated for 24 h to allow robust viral protein expression. Then, '\n",
      " 'DMSO control (0.1%) or VER-155008 was added in conjunction with '\n",
      " 'cycloheximide (CHX) at 100 μg/ml to block de novo protein synthesis. Protein '\n",
      " 'lysates were collected at several times post-CHX addition (0, 4, 8, 16 and '\n",
      " '24 h) and Western blotting was performed. As previously seen in TREx '\n",
      " 'BCBL1-RTA cells, VER-155008 did not alter the half-life of RTA or ORF57 '\n",
      " 'protein. Hsp70 Isoforms Are Essential for KSHV Lytic Replication Fig 6A) . '\n",
      " 'It is intriguing that ORF57 mRNA levels did not show a clear dose-response '\n",
      " 'with the inhibitor as seen for ORF57 mRNA in TREx BCBL1-RTA cells; however '\n",
      " 'the levels were decreased compared with DMSO-control cells. This is the only '\n",
      " 'transcript of all the viral transcripts tested in both cell lines that did '\n",
      " 'not show a dose-response. However, Western blotting did reveal a complete '\n",
      " 'reduction of ORF57 protein in cells treated with > 40 μM VER-155008 ( Fig '\n",
      " '4C) . This suggests that Hsp70 isoforms may also play a role in the folding '\n",
      " 'of ORF57 protein. In fact, Hsc70 has been previously reported to associate '\n",
      " 'with at least 15-20% of newly synthesized proteins during their biogenesis '\n",
      " '[65] , thus, a role for Hsp70 isoforms in folding viral proteins cannot be '\n",
      " 'ruled out. In order to evaluate cellular transcription activity in the '\n",
      " 'presence of VER-155008, we first determined GAPDH transcript stability in '\n",
      " 'mRNA decay assays using AcD (10 μg/ml) in HEK-293T rKSHV.219 cells. In '\n",
      " 'contrast to TREx BCBL1-RTA cells, GAPDH transcripts were very stable, with '\n",
      " 'no significant reduction in their levels after 10 h of AcD treatment ( Fig '\n",
      " '6B) . The half-lives of two cellular transcripts, SRAG (a cellular mRNA '\n",
      " 'export adapter) and histone H2A (H2A.1) were also determined in HEK-293T '\n",
      " 'rKSHV.219 cells. SRAG transcripts were reduced to 50% following 4 h AcD '\n",
      " 'treatment (Fig 6C) , while the H2A.1 mRNA was very unstable with nearly 100% '\n",
      " 'reduction after 2 h AcD treatment ( Fig 6D) . We then plotted the raw C T '\n",
      " 'values for GAPDH transcript from samples in which viral transcripts had been '\n",
      " 'quantified after 24 h of VER-155008 treatment; these did not significantly '\n",
      " 'change in the presence of VER-155008 ( Fig 6E) . Next, the unstable SRAG and '\n",
      " 'H2A.1 mRNAs were measured in the same samples in which viral transcripts had '\n",
      " 'been quantified after 24 h of VER-155008 treatment. In contrast to viral '\n",
      " 'transcripts, both cellular transcripts were not significantly reduced, '\n",
      " 'indicating that transcription of cellular genes was occurring normally even '\n",
      " 'in the presence of high VER-155008 concentrations ( Fig 6F) . Taken '\n",
      " 'together, these results demonstrate that inhibition of Hsp70 isoform '\n",
      " 'function abrogated the expression of viral genes from various temporal '\n",
      " 'classes; however cellular RNA-PII-mediated transcription was not compromised '\n",
      " 'when using VER-155008 at non-cytotoxic concentrations. Following '\n",
      " 'quantification of viral transcripts in both cell lines, it appeared that the '\n",
      " 'reduction seen in viral gene expression, protein production and infectious '\n",
      " 'virion production could be a consequence of a significant global reduction '\n",
      " 'of viral transcripts in inhibitor-treated cells. This led to the possibility '\n",
      " 'that Hsp70 isoforms could be implicated in activation of viral promoters and '\n",
      " 'subsequent transcription or alternatively Hsp70 isoforms were required for '\n",
      " 'KSHV RTC formation. Because viral RTA and Hsc70 co-localized in TREx '\n",
      " 'BCBL1-RTA cells and RTA is (orange colour), late (red colour) and ori-Lyt '\n",
      " 'viral transcripts were all significantly reduced at 1 μM VER-155008 compared '\n",
      " 'to the levels found in DMSOtreated samples. All samples were normalised to '\n",
      " 'GAPDH. Results show the mean of three biological replicates with error bar '\n",
      " 'as standard deviation. (B) The stability of GAPDH transcript was assessed in '\n",
      " 'unreactivated TREx BCBL1-RTA cells in the presence of the transcriptional '\n",
      " 'inhibitor actinomycin D (AcD) (2.5 μg/ml) or control DMSO (0.25%). Cells '\n",
      " 'were collected over the time points indicated and total RNA was extracted '\n",
      " 'followed by qRT-PCR. After 6 h AcD treatment, the amount of GAPDH '\n",
      " 'transcripts was reduced by more than half (C T = 18.09) compared with '\n",
      " 'DMSO-treated cells (C T = 16.24). A further reduction was observed at 8 h '\n",
      " 'treatment (C T = 19.68). The average of two biological replicates with error '\n",
      " 'bar as standard deviation is shown. (C) The amount of GAPDH transcripts did '\n",
      " 'not significantly decrease when using VER-155008 for 24 h at concentrations '\n",
      " 'lower than 3 μM compared to DMSO-treated samples. In contrast, '\n",
      " 'concentrations higher than 3 μM significantly showed a C T lower than DMSO '\n",
      " 'samples, indicating compromised cellular transcription at these inhibitor '\n",
      " 'concentrations. The same samples in which viral transcripts had been '\n",
      " 'quantified were used to plot all C T values. Results show the mean of three '\n",
      " 'biological replicates with error bar as standard deviation. (D) Cells were '\n",
      " 'reactivated for 72 h, total DNA was isolated and real-time qPCR was '\n",
      " 'performed. Viral DNA load was significantly decreased at 2 μM VER-155008. '\n",
      " 'Results show the mean of three biological replicates with error bar as '\n",
      " 'standard deviation. (E) TREx BCBL1-RTA cells were reactivated for 72 h in '\n",
      " 'the presence of VER-155008 at 2.5 μM or vehicle drug DMSO (0.1%). The '\n",
      " 'culture medium was then incubated for 24 h with HEK-293T cells followed by '\n",
      " 'total RNA extraction and qRT-PCR. A significant decrease in viral progeny '\n",
      " 'was seen in inhibitor-treated cells. Results show the mean of three '\n",
      " 'biological replicates with error bar as standard deviation. the master '\n",
      " 'latent-lytic transactivator for multiple KSHV immediate-early, delayed-early '\n",
      " 'and late promoters [66] [67] [68] [69] [70] , we further investigated the '\n",
      " 'possibility that Hsc70 was required for RTA-mediated transactivation. '\n",
      " 'Initially we assessed whether an interaction occurred between Hsc70 and RTA '\n",
      " 'in the absence of other viral proteins or DNA. For this, HEK-293T cells were '\n",
      " 'transiently transfected for 24 h with control pEGFP or pRTA-EGFP and '\n",
      " 'immunoprecipitations were carried out using a GFP-specific antibody. '\n",
      " 'RTA-EGFP precipitated endogenous Hsc70 in contrast to the control EGFP '\n",
      " 'protein (Fig 7A) . In addition, HEK-293T cells were transfected with control '\n",
      " 'pEGFP or pRTA-EGFP for 24 h and examined by immunofluorescence. In cells '\n",
      " 'expressing EGFP protein, endogenous Hsc70 remained cytoplasmic (Fig 7Bi) , '\n",
      " 'while in EGFP-RTA-expressing cells Hsc70 strongly co-localised with RTA in '\n",
      " 'the nuclei, suggesting RTA expression alone is sufficient to redistribute '\n",
      " 'Hsc70 into the nucleus (Fig 7Bii) . Similar nuclear redistribution was also '\n",
      " 'seen for endogenous iHsp70 (S5 Fig). Next, we determined whether VER-155008 '\n",
      " 'was able to disrupt the interaction between EGFP-RTA and Hsc70 in HEK-293T '\n",
      " 'cells. HEK-293T cells exhibited a very similar cytotoxicity profile to that '\n",
      " 'seen in HEK-293T rKSHV.219 cells (S6 Fig). HEK-293T cells were transiently '\n",
      " 'transfected with pRTA-EGFP or control pEGFP. To allow maximal protein '\n",
      " 'expression and avoid interference of the inhibitor with the transfection, '\n",
      " 'the inhibitor was added at 24 h post-transfection and incubated for a '\n",
      " 'further 24 h, prior to immunoprecipitations being performed. Western blot '\n",
      " 'analysis revealed that the inhibitor did not disrupt the interaction between '\n",
      " 'Hsc70 and RTA even at high inhibitor concentrations (55 μM) (Fig 7C) , '\n",
      " 'suggesting that the ATPase function of Hsc70 is not required for the '\n",
      " 'interaction with RTA protein. Therefore, to investigate whether Hsc70 was '\n",
      " 'required for RTA-mediated transactivation of the RTA-responsive ORF57 '\n",
      " 'promoter, a dual-luciferase reporter assay system was utilised. HEK-293T '\n",
      " 'cells were co-transfected with pRTA-EGFP along with the Renilla luciferase '\n",
      " 'vector and either the ORF57 promoter firefly luciferase reporter vector, or '\n",
      " 'the empty reporter vector (pGL3-BASIC). The same co-transfections were '\n",
      " 'performed using pEGFP, as a negative control. 24 h post-transfection, cells '\n",
      " 'were exposed for 2 h to increasing concentrations of VER-155008 and '\n",
      " 'luciferase activities were measured. Longer exposure times to the inhibitor '\n",
      " 'affected the formation of the control Renilla luciferase protein which has a '\n",
      " 'half-life of 3 h, suggesting that Hsp70 isoforms may be required for the '\n",
      " 'folding/maturation of this enzyme. In the presence of EGFP-RTA and the ORF57 '\n",
      " 'promoter reporter construct, the ORF57 promoter activity was '\n",
      " 'increased~30-fold, while the empty vector had a~4-fold increase. However, '\n",
      " 'ORF57 promoter activity was not significantly decreased in the presence of '\n",
      " 'VER-155008 ( Fig 7D) . To confirm this result, HEK-293T cells were also '\n",
      " 'transiently co-transfected with pPAN-WT, a plasmid encoding the genomic '\n",
      " 'region of wild type PAN RNA including its promoter region [71] , and either '\n",
      " 'pRTA-EGFP or control pEGFP. 24 h post-transfection either vehicle drug DMSO '\n",
      " 'or VER-155008 at 45 μM was added and incubated for a further 24 h followed '\n",
      " 'by qRT-PCR. In Quantification by RT-PCR of immediate-early (blue colour), '\n",
      " 'early (orange colour), late (red colour) and ori-Lyt viral transcripts '\n",
      " 'showed that all transcripts were significantly decreased at 25 μM VER-155008 '\n",
      " 'compared to the levels found in DMSO-treated samples. All samples were '\n",
      " 'normalised to GAPDH. Results show the mean of three biological replicates '\n",
      " 'with error bar as standard deviation. (B-D) The stability of cellular GAPDH, '\n",
      " 'SRAG and H2A.1 transcripts was determined in mRNA decay assays using '\n",
      " 'actinomycin D (AcD) (10 μg/ml) or control DMSO (1%) in HEK-293T rKSHV.219 '\n",
      " 'cells. Cells were collected over the time points indicated and total RNA was '\n",
      " 'extracted followed by qRT-PCR. The average of two biological replicates with '\n",
      " 'error bar as standard deviation is shown. (B) GAPDH transcripts were very '\n",
      " 'stable, with no significant reduction in their levels after 10 h of AcD '\n",
      " 'treatment. (C) At 4 h AcD treatment SRAG mRNA levels were reduced by 50% '\n",
      " 'showing a relatively short stability. (D) H2A.1 mRNA levels were very '\n",
      " 'unstable, with nearly 100% reduction after 2 h AcD treatment. (E) GAPDH '\n",
      " 'transcript levels remained unchanged following 24 h VER-155008 treatment in '\n",
      " 'reactivated HEK-293T rKSHV.219 cells. The same samples in which viral '\n",
      " 'transcripts had been quantified were used to plot all C T values. (F) The '\n",
      " 'unstable cellular SRAG and H2A.1 mRNAs were not significantly decreased in '\n",
      " 'the presence of VER-155008 treatment compared with DMSO treatment for 24 h '\n",
      " 'indicating that cellular transcription was not compromised even at high '\n",
      " 'concentrations of VER-155008. The same samples in which viral transcripts '\n",
      " 'had been quantified were used to quantify SRAG and H2A.1. All samples were '\n",
      " 'normalised to GAPDH. However, no significant decrease in the amount of PAN '\n",
      " 'RNA was seen in inhibitor-treated cells (Fig 7E) , indicating that '\n",
      " 'RTA-mediated promoter transactivation and subsequent synthesis of PAN RNA '\n",
      " 'was occurring normally in the presence of VER-155008. These data demonstrate '\n",
      " 'that Hsp70 isoforms did not directly enhance RTA-mediated transactivation. '\n",
      " 'To assess the transcription activity of cellular RNAPII in the presence of '\n",
      " 'the inhibitor, the half-life of PAN RNA was determined in HEK-293T cells '\n",
      " 'co-transfected with pPAN-WT and pRTA-EGFP. Following 24 h post-transfection, '\n",
      " 'AcD (5 μg/ml) or DMSO control (0.5%) was added. After 7 h of transcription '\n",
      " 'inhibition, PAN RNA levels were reduced to 25% compared to DMSO-treated '\n",
      " 'cells (Fig 7F) . This quick reduction in the stability of PAN RNA in the '\n",
      " 'absence of ORF57 protein is in agreement with previous reports [72] . Thus, '\n",
      " 'if VER-155008 was blocking general RNAPII transcription, a significant '\n",
      " 'reduction in PAN RNA levels should be observed after 7 h of inhibitor '\n",
      " 'treatment; however, PAN RNA levels were not reduced in cells treated with '\n",
      " 'VER-155008 for 24 h (Fig 7E) . A dramatic reduction in early, late and '\n",
      " 'ori-Lyt transcripts after 24 h treatment with VER-155008 was evident during '\n",
      " 'KSHV infection in both cell lines used. However, Hsp70 isoforms were not '\n",
      " 'required for RTA-mediated transactivation of KSHV promoters in transiently '\n",
      " 'transfected cells. Thus, we next monitored KSHV RTC formation in the absence '\n",
      " 'or presence of the inhibitor during KSHV lytic infection. TREx BCBL1-RTA '\n",
      " 'cells were reactivated and treated with either control DMSO or 2 μM '\n",
      " 'inhibitor. At 24 h reactivation cells were fixed and immunofluorescence was '\n",
      " 'performed using RTA-and Hsc70-specific antibodies. DMSO-treated cells '\n",
      " 'displayed abundant RTCs and numerous nuclear Hsc70 foci that partially '\n",
      " 'co-localised with RTCs. Hsc70 cytoplasmic depletion was also observed (Fig '\n",
      " \"8A and 8A') . In contrast, inhibitortreated cells showed diffuse nuclear RTA \"\n",
      " \"that was not able to assemble into RTCs (Fig 8B and  8B' ). In these cells, \"\n",
      " 'Hsc70 nuclear foci were still visible, but these were much less numerous and '\n",
      " 'smaller compared with the foci seen in DMSO-treated cells. Significantly, '\n",
      " 'following VER-155008 treatment, Hsc70 was observed in the cytoplasm of '\n",
      " \"reactivated cells (Fig 8B and  8B' ). Hsc70 subcellular localization was \"\n",
      " 'also analysed in DMSO-and inhibitor-treated cells by confocal profiling. '\n",
      " 'Profiling was performed by drawing a line long enough (~20 μm) to cover the '\n",
      " 'nucleus and cytoplasm at either side of the nucleus. If an Hsc70 pixel '\n",
      " 'intensity data point was equal or greater than to the data point in the '\n",
      " 'previous and subsequent pixel and above the background noise, it was '\n",
      " 'considered as an Hsc70 peak, representing an Hsc70 foci. analysed by '\n",
      " 'immunofluorescence. Endogenous Hsc70 was cytoplasmic in cells expressing '\n",
      " 'EGFP (i). In contrast, Hsc70 strongly co-localised with EGFP-RTA in the '\n",
      " 'nucleus, but not the nucleoli (ii). (C) HEK-293T cells were transiently '\n",
      " 'transfected with control pEGFP or pRTA-EGFP. 24 h posttransfection, '\n",
      " 'VER-155008 was added and incubated for a further 24 h. Then, '\n",
      " 'immunoprecipitations were carried out using a GFP-specific antibody. Even in '\n",
      " 'the presence of 55 μM VER-155008 the interaction between Hsc70 and EGFP-RTA '\n",
      " 'was not disrupted. (D) HEK-293T cells were co-transfected with pRTA-EGFP '\n",
      " 'along with the Renilla luciferase vector and either the ORF57 promoter '\n",
      " 'firefly luciferase reporter vector, or the empty reporter vector '\n",
      " '(pGL3-BASIC). The same co-transfections were performed using pEGFP as a '\n",
      " 'negative control protein. 24 h post-transfection, cells were exposed for 2 h '\n",
      " 'to VER-155008 and luciferase activities were measured. Comparable activation '\n",
      " 'of the RTA-responsive ORF57 promoter to that seen in DMSO-treated cells '\n",
      " 'occurred in cells treated with 55 μM VER-155008. The results of three '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'independent transfections were averaged with error bars as standard '\n",
      " 'deviation. (E) HEK-293T cells were transiently co-transfected with pPAN-WT '\n",
      " 'and either pRTA-EGFP or control pEGFP. 24 h post-transfection either vehicle '\n",
      " 'drug DMSO (0.1%) or 45 μM VER-155008 was added and incubated for a further '\n",
      " '24 h. Total RNA was then extracted and qRT-PCR performed. RTA-mediated '\n",
      " 'promoter transactivation and subsequent synthesis of PAN RNA occurred at a '\n",
      " 'similar rate in the presence of VER-155008 or control DMSO. The results of '\n",
      " 'three independent transfections were averaged with error bars as standard '\n",
      " 'deviation. (F) The stability of PAN RNA was determined in HEK-293T cells '\n",
      " 'that had been co-transfected with pPAN-WT and pRTA-EGFP. Following 24 h '\n",
      " 'post-transfection, actinomycin D (AcD) (5 μg/ml) or DMSO control (0.5%) was '\n",
      " 'added. Cells were collected over the time points indicated and total RNA was '\n",
      " 'extracted followed by qRT-PCR. The average of two biological replicates with '\n",
      " 'error bar as standard deviation is shown. '\n",
      " 'doi:10.1371/journal.ppat.1005274.g007 Hsp70 Isoforms Are Essential for KSHV '\n",
      " 'Lytic Replication DMSO-treated cells predominantly showed Hsc70 peaks only '\n",
      " 'within the DAPI boundaries, that is, within the nucleus (Fig 8A\") . '\n",
      " 'Inhibitor-treated cells displayed Hsc70 peaks outside the DAPI boundaries, '\n",
      " 'that is, in the cytoplasm (Fig 8B\" asterisk) more often than control cells. '\n",
      " 'A significant increase in cytoplasmic Hsc70 peaks was seen in '\n",
      " 'inhibitor-treated cells compared with DMSO control cells (Fig 8C) . '\n",
      " 'Fractionation of reactivated TREx BCBL1-RTA cells in the presence or absence '\n",
      " 'of VER-155008, also pointed to slightly higher levels of Hsc70 in the '\n",
      " 'cytoplasm and a decrease in nuclear Hsc70 in inhibitor-treated cells '\n",
      " 'compared with DMSO control cells (Fig 8D) . Cells were also labelled with '\n",
      " 'Click-iT EdU Alexa Fluor 647 and an antibody specific for Hsc70 (S7 Fig). '\n",
      " 'The percentage of assembled RTCs in DMSO-and inhibitor-treated cells was '\n",
      " 'also calculated. In DMSO-treated cells~44% of cells presented assembled RTCs '\n",
      " 'while only 13% of inhibitor-treated cells showed assembled RTCs (Fig 8E) . '\n",
      " 'This demonstrates that chaperone recruitment to the nucleus is essential for '\n",
      " 'the assembly of KSHV RTCs and treatment with VER-155008 was sufficient to '\n",
      " 'impair nuclear chaperone recruitment and KSHV RTC formation. The subcellular '\n",
      " 'localisation of RNAPII was also assessed by indirect immunoflourescent '\n",
      " 'labelling in TREx BCBL1-RTA cells with the monoclonal antibody CTD4H8, which '\n",
      " 'specifically recognises unphosphorylated and serine-5 phosphorylated RNAPII. '\n",
      " 'In unreactivated cells, RNAPII exhibited a nuclear localization, excluding '\n",
      " 'the nucleolus (Fig 9Ai arrow) irrespective of the presence of DMSO (Fig 9Ai) '\n",
      " 'or the inhibitor (Fig 9Aii) . However, in reactivated and DMSO-treated '\n",
      " 'cells, RNAPII was clearly hijacked to RTCs (Fig 9Bi) . In contrast, in '\n",
      " 'reactivated cells treated with the inhibitor, RNAPII was diffuse throughout '\n",
      " 'the nucleus, but excluding the nucleoli, and formed very small foci (Fig '\n",
      " '9Bii arrow) Multiple KSHV RTCs identified by RTA-labelling were formed in '\n",
      " 'DMSO-treated cells to which large and numerous Hsc70 foci were recruited. In '\n",
      " \"these cells, Hsc70 was depleted from the cytoplasm. (A') Confocal images \"\n",
      " 'were subjected to profiling analysis using Zeiss Zen 2011 software. '\n",
      " 'Profiling was conducted for each cell in two representative confocal images '\n",
      " 'taken with a 40-times objective; this included profiling a total of 49 '\n",
      " 'DMSO-treated cells. A representative line profile (yellow line) and '\n",
      " 'accompanying relative intensities of each channel at every pixel along the '\n",
      " 'line is shown for a DMSO-treated reactivated cell. The DAPI intensity '\n",
      " 'profile shows values below 25 outside the nucleus boundary. RTA profile '\n",
      " 'shows a peak with intensity of 160 that lies within the DAPI boundaries '\n",
      " 'corresponding to a KSHV replication compartment. Hsc70 peaks lie within the '\n",
      " 'DAPI boundaries demonstrating their nuclear location. An Hsc70 intensity '\n",
      " 'peak closely resembles the RTA intensity peak (arrows), indicating '\n",
      " 'co-localization of Hsc70 with RTA since both peaks occur at the same '\n",
      " 'position along the line. (B) In cells reactivated and treated with 2 μM '\n",
      " 'VER-155008 for 24 h, RTA was diffuse in the nucleus and not assembled into '\n",
      " 'KSHV RTCs. In addition, Hsc70 nuclear foci were smaller and less abundant '\n",
      " 'than in DMSO-treated cells and Hsc70 was not depleted from the cytoplasm. '\n",
      " \"(B'). Confocal profiling was performed for each cell in two representative \"\n",
      " 'confocal images taken with a 40-times objective; a total of 68 '\n",
      " 'inhibitor-treated cells were analysed. A representative line profile (yellow '\n",
      " 'line) is shown for an inhibitor-treated cell. An Hsc70 peak is seen outside '\n",
      " 'the DAPI boundaries corresponding to an Hsc70 cytoplasmic peak (asterisk). '\n",
      " '(C) Total Hsc70 peaks and total nuclear Hsc70 peaks per cell were combined '\n",
      " 'for each confocal image and experimental condition. Data was converted to '\n",
      " 'percentage of nuclear and cytoplasmic peaks. cells exhibited very small '\n",
      " 'RNAPII foci and diffuse nuclear Edu labelling ( S9Bii Fig). To confirm that '\n",
      " 'inhibition of Hsp70 isoform function was able to abolish RNAPII recruitment '\n",
      " 'to viral genomes, we also utilised chromatin immunoprecipitation (ChIP) '\n",
      " 'assays in TREx BCBL1-RTA A polyclonal antibody against RTA and a monoclonal '\n",
      " 'antibody (CTD4H8) against RNAPII were used for immunoflourescence analysis. '\n",
      " 'RNAPII protein was nuclear excluding the nucleoli (arrows) regardless of '\n",
      " 'inhibitor treatment. (B) TREx BCBL1-RTA cells were reactivated and treated '\n",
      " 'with either control DMSO (0.1%) or 2 μM VER-155008 for 24 h. In DMSO-treated '\n",
      " 'cells RNAPII and RTA were recruited to viral RTCs (i arrow) while in '\n",
      " 'inhibitor-treated cells RTA was diffuse in the nucleus and RNAPII formed '\n",
      " 'numerous small foci that excluded the nucleoli and resembled prereplicative '\n",
      " 'sites (ii arrow). (C) TREx BCBL1-RTA cells were either reactivated (R) in '\n",
      " 'the presence of DMSO (0.1%) or 2 μM VER-155008 for 24 h. Unreactivated (U) '\n",
      " 'cells treated with DMSO (0.1%) were used to assess levels of lytic '\n",
      " 'reactivation. ChIP assays were carried out with either monoclonal CTD4H8 '\n",
      " 'RNAPII antibody or mouse control antibody (IgG). In the presence of '\n",
      " 'VER-155008, significantly reduced amounts of RNAPII bound to viral promoters '\n",
      " 'were detected. The average of three independent experiments is shown with '\n",
      " 'error bars as standard deviation. cells that were either reactivated for 24 '\n",
      " 'h in the presence of DMSO or 2 μM VER-155008 ( Fig  9C) . In unreactivated '\n",
      " 'control cells, there was a clear enrichment of RNAPII at the promoter of '\n",
      " 'GAPDH gene, while RNAPII occupancy at viral promoters was minimal. However, '\n",
      " 'KSHV reactivation in DMSO-treated cells led to a drastic reduction of RNAPII '\n",
      " 'at the promoter of GAPDH and a significant increase of RNAPII at the viral '\n",
      " 'promoters in agreement with the previous immunofluorescence results, showing '\n",
      " 'RNAPII recruitment to RTCs (Fig 9Bi) . Conversely, upon treatment of the '\n",
      " 'HSP70 inhibitor, the amount of RNAPII bound at the promoters of ori-Lyt, K12 '\n",
      " 'and ORF59 was decreased by~65%,~70% and~50% respectively compared with '\n",
      " 'DMSO-treated reactivated cells. Importantly, these results indicate for the '\n",
      " 'first time that inhibition of Hsp70 isoforms leads to a severe impairment in '\n",
      " 'RNAPII recruitment at multiple viral promoters including that of ori-Lyt. To '\n",
      " 'further confirm the essential role of Hsc70 and iHsp70 in the formation of '\n",
      " 'KSHV RTCs, specific individual siRNA-mediated depletion of both isoforms was '\n",
      " 'performed in HEK-293T rKSHV.219 cells. Following four days post-siRNA '\n",
      " 'transfection, cells were reactivated for 24 h and RNA and protein were '\n",
      " 'extracted from the same sample. Hsc70 depletion was evaluated by Western '\n",
      " 'blotting and by qRT-PCR, the latter showing~85% Hsc70 mRNA knockdown ( Fig  '\n",
      " '10A) . In contrast, iHsp70 mRNA levels were not affected confirming '\n",
      " 'specificity of the Hsc70 siRNA. Despite a successful knockdown at the mRNA '\n",
      " 'level, significant amounts of Hsc70 protein remained in Hsc70-depleted cells '\n",
      " '(Fig 10B) . However, even with this modest amount of depletion at the '\n",
      " 'protein level, all viral transcripts (with the exception of PAN) displayed a '\n",
      " 'significant reduction in Hsc70 siRNA-treated samples compared with the '\n",
      " 'scramble siRNAtreated cells as demonstrated by qRT-PCR analysis (Fig 10C) . '\n",
      " 'ORF57, ORF74 and gL mRNA levels were decreased by~40% following Hsc70 '\n",
      " 'knockdown. Ori-Lyt and RTA transcripts were reduced by~30% and gB levels '\n",
      " 'by~20%. This suggests that depletion of Hsc70 impaired the expression of '\n",
      " 'viral genes from various temporal classes and thus Hsc70 may be necessary '\n",
      " 'for KSHV RTC formation. Viral DNA replication was also assessed following '\n",
      " 'Hsc70 knockdown. For this, after four days post-siRNA transfection, cells '\n",
      " 'were reactivated for a further 72 h. There were no significant differences '\n",
      " 'between scramble and depleted cells (Fig 10D) . The production of infectious '\n",
      " 'KSHV virions was also evaluated after 72 h reactivation. Again, no '\n",
      " 'significant differences were seen between scramble and depleted cells ( Fig '\n",
      " '10E) ; however this result is not surprising due to incomplete Hsc70 '\n",
      " 'depletion even after seven days post-transfection ( Fig 10F) . This '\n",
      " 'highlights the remarkable stability of Hsc70 protein in this cell line and '\n",
      " 'it suggests that Hsc70 depletion was enough to cause a reduction in viral '\n",
      " 'transcripts but not enough to cause a reduction in the amount of viral '\n",
      " 'proteins, thus KSHV lytic replication remained unaffected. It is also '\n",
      " 'possible that iHsp70 was able to functionally compensate for Hsc70. Next, '\n",
      " 'specific depletion of iHsp70 was performed in HEK-293T rKSHV.219 cells. '\n",
      " 'iHsp70 depletion at the mRNA level reached~75% knockdown (Fig 11A) which '\n",
      " 'correlated with efficient depletion at the protein level ( Fig 11B) . '\n",
      " 'However, in iHsp70-depleted cells, the majority of viral gene expression was '\n",
      " 'unaffected, apart from gL and PAN transcripts which were decreased bỹ 40% '\n",
      " 'and~20% respectively (Fig 11C) . Importantly, taken together these results '\n",
      " 'indicate that partial depletion of Hsc70 at the protein level is sufficient '\n",
      " 'to cause a reduction in viral transcription, suggesting an essential role of '\n",
      " 'this chaperone in the formation of KSHV RTCs, whereas iHsp70 may have a more '\n",
      " 'subtle effect on viral gene expression. To further support the essential '\n",
      " 'role of Hsc70 during KSHV RTC formation, Hsc70 was specifically silenced in '\n",
      " 'TREx BCBL1-RTA cells. For this, nucleofection was carried out. Transfection '\n",
      " 'efficiency was monitored co-transfecting the Hsc70 siRNA together with '\n",
      " 'pmaxGFP, which encodes maxGFP, a green fluorescent protein from the copepod '\n",
      " 'Pontellina p. (Fig 12A) . Note that higher transfection efficiency is '\n",
      " 'expected for the siRNA due to the smaller size of this compared with the '\n",
      " 'plasmid DNA. After four days post-nucleofection, Hsc70 mRNA levels '\n",
      " 'showed~90% knockdown (Fig 12B) with a minor depletion at the protein level '\n",
      " '(Fig 12C) . Due to the stability of Hsc70 protein, cells were incubated for '\n",
      " 'six days post-nucleofection followed by a further 24 h reactivation and '\n",
      " 'immunofluorescence for Hsc70 and RTA was carried out. RTC formation '\n",
      " 'dramatically decreased after nucleofection. Similar impairment in KSHV lytic '\n",
      " 'replication has previously been reported in electroporated TREx BCBL1-RTA '\n",
      " 'cells [74] ; nevertheless, in scramble siRNA-treated cells, groups of cells '\n",
      " 'could still be seen displaying RTCs to which Hsc70 was relocated (Fig 12D) . '\n",
      " 'In contrast, in Hsc70 siRNA-treated cells, fewer RTCs were visible and these '\n",
      " 'exhibited nuclear Hsc70 (Fig 12E yellow arrow) while cells fully depleted of '\n",
      " 'Hsc70, as identified by lack of Hsc70-labelling, did not form RTCs (Fig 12D  '\n",
      " 'white arrows) . This result strongly suggests that Hsc70 is an essential '\n",
      " 'chaperone for the formation of KSHV RTCs in TREx BCBL1-RTA cells. Current '\n",
      " 'quantitative proteomics approaches have become an invaluable tool for '\n",
      " 'large-scale, high-throughput identification of proteins in complex '\n",
      " 'biological samples. Moreover, advances in subcellular fractionation offer a '\n",
      " 'way to further reduce the complexity of the samples to be analysed by '\n",
      " 'LC-MS/MS, allowing identification of low abundance proteins. In this present '\n",
      " 'study, we have developed a novel quantitative proteomic approach enhanced by '\n",
      " 'subcellular fractionation that has enabled us to elucidate the cellular '\n",
      " 'protein composition of KSHV RTCs. This novel approach led to the '\n",
      " 'identification of several upregulated pathways in reactivated cells '\n",
      " 'associated with the NE fraction (S2 Table) . The first scored pathway was '\n",
      " 'RNA post-transcriptional modification, the second highlighted pathway was '\n",
      " 'protein synthesis with 26 different ribosomal proteins identified and the '\n",
      " 'third scored pathway was DNA replication, recombination and repair. In '\n",
      " 'addition, the isolation of the NE regions from unreactivated and reactivated '\n",
      " 'cells followed by the uncommon use of urea for protein extraction led to the '\n",
      " 'mass spectrometric identification of several Hsp70 isoforms and their '\n",
      " 'respective co-chaperones at significant levels in NE-associated RTCs of '\n",
      " 'reactivated cells. Immunoflourescence analysis confirmed that endogenous '\n",
      " 'Hsc70 and iHsp70 were redistributed from the cytoplasm to the periphery of '\n",
      " 'KSHV RTCs where they formed multiple nuclear foci during early lytic '\n",
      " 'replication. The formation of RTCs coincided in time with the appearance of '\n",
      " 'nuclear chaperone foci. Similar virus-induced-chaperone-enriched (VICE) '\n",
      " 'domains that form adjacent to HSV-1 RTCs, have also been observed in '\n",
      " 'HSV-1-infected cells and contain sequestered Hsc70, iHsp70, Hsp40 and Hsp90 '\n",
      " '[58, 75, 76] . HSV-1 induced-VICE domains also accumulate ubiquitinated '\n",
      " 'proteins and components of the proteasome and function to sequester '\n",
      " 'misfolded proteins away from RTCs and serve as protein quality control '\n",
      " 'centers [75, 77] . iHsp70 redistribution very similar to that seen in HSV-1 '\n",
      " 'induced-VICE domains was observed in multiple unreactivated and total RNA '\n",
      " 'and protein were extracted from the same sample. Despite achieving~85% Hsc70 '\n",
      " 'knockdown at the mRNA level (A), significant amounts of Hsc70 protein '\n",
      " 'remained in Hsc70 siRNA-treated samples (B) . Despite this small knockdown '\n",
      " 'at the protein level, in Hsc70-depleted cells there was a significant '\n",
      " 'decrease in the amount of multiple viral transcripts from various temporal '\n",
      " 'classes as quantified by RT-PCR analysis (C). Viral DNA replication was '\n",
      " 'assessed by qPCR following Hsc70-knockdown. Cells were reactivated for 72 h '\n",
      " 'after four days post-siRNA transfection. No significant differences were '\n",
      " 'observed between scramble and Hsc70-treated cells (D). Similar virion '\n",
      " 'production was detected in scramble and Hsc70-treated cells. Cells were '\n",
      " 'reactivated for 72 h after four days post-siRNA transfection, culture medium '\n",
      " 'was centrifuged and immediately incubated with HEK-293T cells for 24 h. '\n",
      " 'Total RNA was then isolated and qRT-PCR carried out (E). Incomplete '\n",
      " 'Hsc70-knockdown at the protein level was observed by Western blotting even '\n",
      " 'after seven days post-siRNA transfection in HEK-293T rKSHV.219 cells. This '\n",
      " 'demonstrates the remarkable stability of Hsc70 in this cell line (F). The '\n",
      " 'average of three independent transfections is shown with error bars as '\n",
      " 'standard deviation (A, C-E). doi:10.1371/journal.ppat.1005274.g010 Hsp70 '\n",
      " 'Isoforms Are Essential for KSHV Lytic Replication  Fig 11. The majority of '\n",
      " 'KSHV lytic gene expression was unaffected following specific depletion of '\n",
      " 'iHsp70 in HEK-293T rKSHV.219 cells. Cells were transfected twice with 100 nM '\n",
      " 'iHsp70-specific siRNA or 100 nM scramble siRNA. Following four days '\n",
      " 'post-siRNA transfection, cells were either reactivated cells undergoing KSHV '\n",
      " 'lytic replication, moreover this labelling was also observed for Hsc70 in '\n",
      " 'some cells. Hsc70 and iHsp70 remain positioned exclusively adjacent to HSV-1 '\n",
      " 'RTCs. However, in contrast; these chaperones were very dynamic during KSHV '\n",
      " 'lytic replication and surprisingly very large chaperone foci were recruited '\n",
      " 'within KSHV RTCs when viral DNA was actively synthesised (Fig 2C and S1B '\n",
      " 'Fig) . Therefore, our results strongly suggest that these chaperones may '\n",
      " 'also aid replication of KSHV genomes during lytic replication. We therefore '\n",
      " 'further suggest that Hsp70 isoforms may have an important role in the '\n",
      " 'assembly and activation of pre-initiation complexes on the origin of DNA '\n",
      " 'replication. This is supported by observations in prokaryotes, such as in '\n",
      " 'plasmid P1 [78, 79] , and eukaryotes, such as Saccharomyces cerevisiae [80] '\n",
      " ', human papillomavirus-11 [81, 82] , HSV-1 [83] and bacteriophage λ [84, 85] '\n",
      " '. Interestingly, Hsp70 isoforms have also been identified in the KSHV virion '\n",
      " '[86, 87] and therefore an additional role of these chaperones in KSHV capsid '\n",
      " 'assembly may also be possible. Cancer cells greatly rely on members of the '\n",
      " 'HSP70 and HSP90 chaperone families for their growth and survival [5, 88] , '\n",
      " 'consequently, significant efforts have been invested to design small '\n",
      " 'molecule inhibitors specific for these ATPases as novel anticancer '\n",
      " 'therapeutics. This has been successfully achieved for the HSP90 family with '\n",
      " 'several inhibitors undergoing phase III clinical trials [14] , although the '\n",
      " 'clinical efficacy of these inhibitors has been somewhat limited because of '\n",
      " 'the inevitably upregulation of iHsp70, when inhibiting Hsp90 [56, 57] . The '\n",
      " 'development of HSP70 inhibitors has substantially lagged behind that of '\n",
      " 'HSP90 inhibitors due to lack of natural product inhibitors specific for '\n",
      " 'Hsp70 isoforms, due to the highly polar nature of the Hsp70/ Hsc70 ATP '\n",
      " 'binding site and the high affinity for ATP displayed by these ATPases [12] . '\n",
      " 'Nevertheless, in recent years several HSP70 inhibitors have been designed '\n",
      " 'and tested in pre-clinical or clinical trials [88] . However, a major '\n",
      " 'challenge remains in finding inhibitors which can specifically discriminate '\n",
      " 'between iHsp70 and Hsc70. To date, only peptide aptamers targeting iHsp70 '\n",
      " 'are selective for a specific Hsp70 isoform [89] and also recently, methyl '\n",
      " 'blue was reported to specifically inhibit iHsp70 by oxidizing a cysteine in '\n",
      " 'its ATPase domain, the same residue is absent in Hsc70 allowing differential '\n",
      " 'targeting of the isoforms [90] . Because both Hsc70 and iHsp70 were '\n",
      " 'specifically redistributed to KSHV RTCs during lytic replication, we made '\n",
      " 'use of a recently developed small molecule inhibitor, VER-155008, which '\n",
      " 'targets the ATPase pocket of the main three human Hsp70 isoforms. This '\n",
      " 'inhibitor used at non-cytotoxic concentrations was able to effectively '\n",
      " 'abrogate early and late KSHV transcription together with viral protein '\n",
      " 'production, viral DNA replication and viral progeny. Therefore, our study '\n",
      " 'highlights the potential of VER-155008 or other novel HSP70 inhibitors to '\n",
      " 'prevent KSHV lytic replication in KSHV-associated tumours. One could expect '\n",
      " 'that HSP70 inhibition would be directly detrimental not only to cancer cell '\n",
      " 'survival but also the virus-specific functions that are dependent on Hsp70 '\n",
      " 'isoforms. These results may have exciting implications in combination with '\n",
      " 'the recently demonstrated efficacy of ATP-competitive HSP90 inhibitors in '\n",
      " 'blocking KSHV latent cycle in vitro and in a xenograft KSHV tumour model '\n",
      " '[41] . It may be the case that combining HSP70 inhibitors with HSP90 '\n",
      " 'inhibitors may lead to enhanced efficacy in eradicating latent KSHV '\n",
      " 'reservoirs. Excitingly, in our cell culture models VER-155008 abrogated '\n",
      " 'lytic replication without severely affecting cell viability or triggering '\n",
      " 'apoptosis. It is intriguing that inhibition of constitutively expressed '\n",
      " 'Hsc70 chaperone did not result in cell death. However, it is important to '\n",
      " 'highlight previous studies carried out on the susceptibility of tumour cells '\n",
      " 'versus normal cells to HSP90 inhibitors. In tumour cells, for 24 h or '\n",
      " 'remained unreactivated and total RNA and protein were extracted from the '\n",
      " 'same sample. Efficient iHsp70 knockdown was achieved both at the mRNA level '\n",
      " '(~75%) (A) and the protein level (B). A small decrease in the amount of gL '\n",
      " 'and PAN transcripts was observed in iHsp70-depleted cells (C). The average '\n",
      " 'of three independent transfections is shown with error bars as standard '\n",
      " 'deviation. doi:10.1371/journal.ppat.1005274.g011 Hsp90 is present entirely '\n",
      " 'in multi-chaperone complexes with high ATPase activity; in contrast, in '\n",
      " 'normal cells Hsp90 is in an uncomplexed conformation. These two distinct '\n",
      " 'Hsp90 presentations result in tumour Hsp90 exhibiting a 100-fold higher '\n",
      " 'binding affinity to an HSP90 inhibitor than normal Hsp90 [91] . Thus, it is '\n",
      " 'tempting to speculate that the multi-chaperone HSP70 foci that are recruited '\n",
      " 'to KSHV RTCs during lytic replication (Fig 2 and S1 Fig) are more sensitive '\n",
      " 'to VER-155008 than the chaperones not assembled in VICE domains which carry '\n",
      " 'out the housekeeping functions for cell survival. This would explain why '\n",
      " 'HSP70 inhibition had a profound effect on KSHV lytic replication without '\n",
      " 'affecting cell viability and support the idea that isoform specificity may '\n",
      " 'not be a requirement for treatment of KSHV-associated tumours. To establish '\n",
      " 'the essential role of Hsp70 isoforms during KSHV lytic replication, we '\n",
      " 'examined several possible functions. It could be hypothesised that Hsp70 '\n",
      " 'isoforms could be implicated in 1) stabilization of essential viral '\n",
      " 'proteins, 2) clearing stalled RNAPII during times of robust viral '\n",
      " 'transcription, 3) activation of viral promoters, 4) formation of KSHV RTCs. '\n",
      " 'Initially we hypothesised that Hsp70 isoforms may be necessary for '\n",
      " 'maintaining the stability of the key KSHV viral proteins RTA and/or ORF57. '\n",
      " 'This was deemed a possibility as during KSHV latency, the essential viral '\n",
      " 'latency associated nuclear antigen (LANA) protein has been recently shown to '\n",
      " 'be a client protein of the Hsp90 chaperone and several HSP90 inhibitors '\n",
      " 'reduced the expression of LANA [41] . In addition, the lytic KSHV K1 '\n",
      " 'glycoprotein has also been reported to be a client protein of Hsp90 [92] . '\n",
      " 'However, inhibition of Hsp70 isoforms did not alter the half-life of RTA or '\n",
      " 'ORF57 (Figs 3D and 4D) . Alternatively, Hsc70 has been observed at the '\n",
      " 'periphery of HSV-1 RTCs where it is also believed to aid in clearing stalled '\n",
      " 'RNAPII from viral genomes during times of active transcription [76] . '\n",
      " 'Indeed, the serine-2 phosphorylated form of RNAPII undergoes ubiquitination '\n",
      " 'and robust proteasomal degradation during HSV-1 infection [93] . In '\n",
      " 'contrast, in KSHV-reactivated cells, there was only a slight reduction of '\n",
      " 'serine-2 phosphorylated RNAPII in comparison with unreactivated cells even '\n",
      " 'at late times (24 h) post-reactivation ( Fig 3C) and a significant decrease '\n",
      " 'in the other RNAPII forms was not evident (Fig 3C) . This suggests that '\n",
      " 'although robust RNAPII degradation is a feature observed in virus infection, '\n",
      " 'such as HSV-1 and influenza virus [55, 93] , it may not be universally '\n",
      " 'conserved. It is interesting to note that expression of a dominant-negative '\n",
      " 'Hsc70 (K71M) that cannot hydrolyze ATP during HSV-1 infection resulted in '\n",
      " 'prevention of serine-2 RNAPII degradation and RTCs formation [76] . However, '\n",
      " 'inhibition of Hsp70 isoforms by VER-155008 did not prevent the slight '\n",
      " 'degradation of phospho-serine-2 RNAPII protein ( Fig  3C) . Using transient '\n",
      " 'transfections, we also demonstrated that Hsp70 isoforms were not directly '\n",
      " 'involved in the activation of viral promoters (Fig 7D and 7E) . Strikingly '\n",
      " 'however, inhibition of Hsp70 isoforms precluded KSHV RTCs formation (Fig 8) '\n",
      " 'and RNAPII re-localization to viral promoters (Fig 9) , thus, blocking KSHV '\n",
      " 'RTCs formation led to abolishment of global viral transcription and '\n",
      " 'subsequent protein synthesis and viral DNA replication. These results, taken '\n",
      " 'together with the differential Hsc70 and iHsp70 labelling seen in HSV-1 and '\n",
      " 'KSHV-infected cells, suggest that these chaperones may be playing additional '\n",
      " 'roles, such as participating in viral DNA replication and/or capsid '\n",
      " 'assembly, in KSHV lytic infection compared with HSV-1 infection. '\n",
      " 'Importantly, in both viruses, HSV-1 and KSHV, inhibition of Hsc70 ATPase '\n",
      " 'function leads to a clear impediment in RTCs formation and presents a novel '\n",
      " 'antiviral target for multiple herpesviruses. Moreover, as HSV-1 and KSHV '\n",
      " 'belong to different subfamilies of the Herpesviridae family (α and γ '\n",
      " 'subfamily respectively), the key role of Hsp70 isoforms in RTC formation may '\n",
      " 'be conserved across all subfamilies. In support of this notion is the fact '\n",
      " 'that Hsc70 and iHsp70 have also been reported to be incorporated in the '\n",
      " 'virion of the β-herpesvirus HCMV [94] and the γ-herpesvirus EBV [95] . Our '\n",
      " 'results also support that the finding of Hsc70 and iHsp70 chaperones in the '\n",
      " 'KSHV virion [86, 87] a decade ago was not casual, and that these chaperones '\n",
      " 'are essential for KSHV RTCs formation during lytic replication. iHsp70 has '\n",
      " 'also been detected in the HSV-1 virion [96] . To support the conserved role '\n",
      " 'of Hsp70 isoforms in herpesvirus infection, a recent study showed that '\n",
      " 'cellular depletion of Hsc70 protein significantly reduced HSV-1 viral output '\n",
      " 'in cell culture without adversely affecting cell viability. Depleting Hsc70 '\n",
      " 'from the HSV-1 virion also significantly reduced viral production by more '\n",
      " 'than 50% [97] . Hsp70 isoforms may be recruited to RTCs for several reasons. '\n",
      " 'Firstly, the chaperone may sequester misfolded, modified or unwanted '\n",
      " 'proteins away from RTCs. Alternatively, Hsp70 could produce a site for '\n",
      " 'protein remodelling and/or degradation which may regulate or delay cellular '\n",
      " 'pathways, such as the apoptosis cascade. Finally, it may aid subtlety to '\n",
      " 'clear stalled RNAPII complexes during robust viral transcription and '\n",
      " 'replication. An additional question which is yet to be addressed is the '\n",
      " 'mechanism by which Hsp70 isoforms are recruited to RTCs. One intriguing '\n",
      " 'possibility is observations made during HSV-1 infection. Here, HSV-1 ICP27 '\n",
      " 'has been shown to interact with Hsc70 and is required for Hsc70 nuclear foci '\n",
      " 'formation [76] . It will now be interesting to determine if the functional '\n",
      " 'KSHV homologue ORF57, also interacts with Hsc70 and whether its '\n",
      " 'nucleocytoplasmic shuttling ability is essential for Hsc70 nuclear import. '\n",
      " 'In summary, we have identified a new essential role for Hsp70 isoforms '\n",
      " 'during the formation of RTCs in KSHV lytic replication. Importantly, our '\n",
      " 'results suggest that HSP70 inhibitors have the potential as novel KSHV '\n",
      " 'antiviral agents and it would now be interesting to test these in '\n",
      " 'conjunction with other molecular chaperone inhibitors, specifically HSP90 '\n",
      " 'inhibitors [41] , which have the potential to eradicate latent KSHV '\n",
      " 'reservoirs in both in vitro and in vivo tumour models. TREx-BCBL-1-RTA cells '\n",
      " '(kindly provided by Dr. Jae Jung, University of Southern California) are a '\n",
      " 'BCBL-1-based, primary effusion lymphoma (PEL) B cell line that has been '\n",
      " 'engineered to inducibly express exogenous Myc-tagged RTA by the addition of '\n",
      " 'doxycycline, leading to a robust reactivation of the full KSHV lytic cycle '\n",
      " '[48] . The rKSHV.219 cell line (kindly provided by Dr. Jeffery Vieira, '\n",
      " 'University of Washington, Seattle, USA) maintains KSHV as a latent infection '\n",
      " 'and was generated by infecting HEK-293T cells (ATCC) with a recombinant KSHV '\n",
      " 'that contains a constitutively active puromycin resistance and GFP gene, and '\n",
      " 'an RFP gene that is fused to an RTA-responsive lytic cycle (PAN) promoter; '\n",
      " 'hence, expression of RFP can be used as a reporter of RTA activity [42] . '\n",
      " 'HEK-293T rKSHV.219 cells were grown in DMEM (Life Technologies) supplemented '\n",
      " 'with 10% foetal calf serum (FCS) (Life Technologies) and 1% '\n",
      " 'penicillin/streptomycin (P/S). This cell line was kept under puromycin '\n",
      " '(Sigma) selection (0.2 μg/ml). Reactivation into the lytic cycle was '\n",
      " 'achieved by addition of 12-O-tetradecanoylphorbol 13-acetate (TPA) (20 '\n",
      " 'ng/ml) and sodium n-butyrate (NaB) (Sigma) (4 mM). The TREx BCBL1-RTA cell '\n",
      " 'line was grown in RPMI 1640 medium (Life Technologies) supplemented with 10% '\n",
      " 'FCS and 1% P/S. This cell line was kept under hygromycin B (Life '\n",
      " 'Technologies) selection (100 μg/ml) and inductions were performed using 2 '\n",
      " 'μg/ml doxycycline hyclate (Sigma) as previously described [98] . All cells '\n",
      " 'were maintained at 37°C in a humidified incubator with 5% CO 2 . Plasmid '\n",
      " 'transfections were carried out using Lipofectamine 2000 (Life Technologies), '\n",
      " 'as previously described [99] . Luciferase assay plasmids Renilla luciferase '\n",
      " 'vector pRL-TK and firefly luciferase vector pGL3-BASIC were purchased from '\n",
      " 'Promega, pEGFP-N1 was obtained from Clontech, pRTA-EGFP and pPAN-WT have '\n",
      " 'been previously described [71, 100] . The monoclonal mouse antibodies to '\n",
      " 'anti-nuclear pore complex proteins (mAb414) (ab24609), GAPDH (6C5), rabbit '\n",
      " 'polyclonal anti-lamin B1 and anti-Ser2 RNAPII were purchased from Abcam. The '\n",
      " 'rabbit polyclonal anti-histone H3 C-terminus (39164) was purchased from '\n",
      " 'Active Motif. The rabbit polyclonal anti-Nup160 was obtained from Bethyl '\n",
      " 'Laboratories. Monoclonal antibodies to KSHV ORF57 (207.6), to Hsc70 (B-6), '\n",
      " 'to Grp78 (A-10), to Hsp90 (4F10), to B-23 (0412) and to C-23 (H6) were '\n",
      " 'obtained from Santa Cruz. The mouse monoclonal (C92F3A-5) anti-iHsp70 was '\n",
      " 'from Enzo Life Sciences. The rabbit polyclonal anti-PARP1 was purchased from '\n",
      " 'Cell Signalling. The mouse monoclonal (CTD4H8) anti-RNAPII was purchased '\n",
      " 'from Millipore. The mouse monoclonal (9E10) anti-c-Myc was from Sigma. Sheep '\n",
      " 'anti-KSHV minor capsid protein was purchased from Exalpha Biologicals, Inc. '\n",
      " 'The rabbit polyclonal anti-RTA was a gift from Professor David Blackbourn '\n",
      " '(University of Surrey, UK). The mouse monoclonal (JL-8) anti-GFP was '\n",
      " 'supplied by Clontech. The inhibitor for Hsp70 isoforms (VER-155008) was '\n",
      " 'obtained from Tocris Bioscience. For SILAC, HEK-293T cells rKSHV.219 were '\n",
      " 'fed with either medium (R6K4) or light (R0K0) labelled medium (Dundee Cell '\n",
      " 'Products) containing 10% dialysed FCS (Dundee Cell Products) for six '\n",
      " 'passages to allow incorporation of the isotopes, as previously described '\n",
      " '[101] . Subsequently, to induce lytic replication three T175 flasks were '\n",
      " 'reactivated with TPA (20 ng/ml) and NaB (4mM) for 48 h, while another three '\n",
      " 'T175 flasks remained unreactivated as control. To isolate NEs a protocol '\n",
      " 'published by Korfali et al., [35] was used with minor modifications. 75 '\n",
      " 'million cells were used per experimental condition. Cells were washed with '\n",
      " 'PBS and incubated in hypotonic lysis buffer (10 mM HEPES pH 7.4, 1.5 mM MgCl '\n",
      " '2 , and 10 mM KCl) for 30 min followed by homogenization with a tight Dounce '\n",
      " 'homogenizer. To stabilise and avoid lysing the nuclei after the hypotonic '\n",
      " 'swelling step, cells were resuspended in 2.2M SHKM (2.2 M sucrose, 50 mM '\n",
      " 'HEPES pH 7.4, 25 mM KCl, and 5 mM MgCl 2 ) and 1M KCl. The resuspended cells '\n",
      " 'were then underlayed with 30% SHKM (0.9 M sucrose, 50 mM HEPES pH 7.4, 25 mM '\n",
      " 'KCl, and 5 mM MgCl 2 ) and nuclei were pelleted at 2,000xg for 20 min at 4°C '\n",
      " 'in a Eppendorf centrifuge 5804 R. Nuclei were resuspended in 1.9 M SHKM, '\n",
      " 'underlayed with 2.2 M SHKM and transferred to 38.5-ml ultracentrifuge tubes '\n",
      " '(Beckman Coulter). Nuclei were then centrifuged at 82,000xg for 2h at 4°C in '\n",
      " 'a Sorvall Discovery 90SE ultracentrifuge. Pellets were resuspended in 0.25 M '\n",
      " 'SHKM (50 mM HEPES pH 7.4, 25 mM KCl, and 5 mM MgCl 2 ), treated with 1% '\n",
      " 'Triton-X in 10% SHM (0.3 M sucrose, 10 mM HEPES, pH 7.4, 2 mM MgCl 2 and 0.5 '\n",
      " 'mM Ca Cl 2 ) for 10 min and centrifuged at 2,000xg for 10 min. Nuclei were '\n",
      " 'then treated with RNase A (Thermo Scientific) and DNase I (Life '\n",
      " 'Technologies) for 15 min, pelleted at 6,000xg for 10 min, resuspended in 10% '\n",
      " 'SHM and treated again with RNase A and DNase I for 15 min. Nuclei were '\n",
      " 'centrifuged at 2,000xg for 10 min, resuspended and incubated for 15 min in '\n",
      " '10% SHM containing 0.3 M NaCl to remove nucleoplasmic contents. Nuclear '\n",
      " 'envelopes were then pelleted at 1,500xg for 15 min. Insoluble nuclear '\n",
      " 'envelope proteins were solubilised for 10 min in PBS supplemented with 0.1% '\n",
      " 'Triton-X100 and 6 M urea. Samples were centrifuged at 6,000 g for 2 min to '\n",
      " 'remove insoluble material and the supernatant containing nuclear envelope '\n",
      " 'proteins was stored at -80°C for further Western blotting and mass '\n",
      " 'spectrometry analysis. All solutions had freshly added 1x Complete, '\n",
      " 'EDTA-free protease inhibitors (Roche). DTT (2 mM) was also freshly added to '\n",
      " \"the solutions specified on Korfali's protocol. LC-MS/MS was performed as \"\n",
      " 'previously described [22] . Bioinformatical analysis was performed with the '\n",
      " 'Ingenuity Systems software packet, IPA 9.0 (Ingenuity Systems, Inc). Protein '\n",
      " 'samples were extracted using lysis buffer containing 50 mM Tris (pH 7.4), '\n",
      " '150 mM NaCl, 1% NP-40 and 1x Complete, EDTA-free protease inhibitors (Roche) '\n",
      " 'for 30 min on ice, as previously described [102] . Protein samples were run '\n",
      " 'on SDS-PAGE gels and transferred to nitrocellulose membranes (Amersham) via '\n",
      " 'wet transfer. Membranes were blocked with TBS + 0.1% Tween 20 and 5% dried '\n",
      " 'skimmed milk powder. Membranes were probed with relevant primary and '\n",
      " 'secondary antibodies, treated with EZ-ECL (Geneflow) and exposed to Amersham '\n",
      " 'hyperfilm ECL (GE Healthcare). Secondary antibodies were horseradish '\n",
      " 'peroxidase (HRP)-conjugated polyclonal goat anti-mouse and polyclonal goat '\n",
      " 'anti-rabbit (Dako). HRPconjugated polyclonal rabbit anti-sheep was from '\n",
      " 'Santa Cruz. GFP-Trap (Chromotek) experiments were performed as previously '\n",
      " 'described [103] . Nuclear/cytoplasmic fractionations were performed as '\n",
      " 'previously described [44] , with the exception that nuclear pellets were '\n",
      " 'solubilised for 20 min in PBS supplemented with 0.1% Triton-X100 and 6 M '\n",
      " 'urea. Vigorous pipetting and vortexing was applied to the nuclear pellet. '\n",
      " 'After urea treatment, insoluble material was removed by centrifugation at '\n",
      " '6,000 g for 2 min and the supernatant kept for further analysis. Cells were '\n",
      " 'cultured overnight on poly-L-lysine (Life Technologies) coated glass '\n",
      " 'coverslips in 24-well plates. Cells were fixed with 4% formaldehyde '\n",
      " '(Calbiochem) for 10 min and permeabilised with 0.1% Triton X-100 for 20 min '\n",
      " 'as previously described [104] . For labelling with Grp78 antibody cells were '\n",
      " 'fixed with ice-cold 100% methanol for five min. After permeabilization, '\n",
      " 'cells were then incubated in blocking solution (PBS with 1% BSA) for 1 h at '\n",
      " '37°C. Primary antibodies anti-Hsc70 (diluted 1:200), anti-iHsp70 (1:50), '\n",
      " 'anti-Grp78 (1:50), anti-RNAPII (CTD4H8) (1:500) or rabbit RTA (1:1,000) were '\n",
      " 'incubated for 1 h at 37°C. Coverslips were washed five times with PBS, '\n",
      " 'incubated with appropriate secondary antibody for 1 h at 37°C, washed five '\n",
      " 'times with PBS again and mounted in VECTASHIELD with DAPI (Vector Labs). '\n",
      " 'Images were obtained using a LSM 510 META confocal microscope (Carl Zeiss) '\n",
      " 'and processed using ZEN 2009 imaging software (Carl Zeiss) as previously '\n",
      " 'described [105] . Fluorescentlyconjugated secondary antibodies were all '\n",
      " 'obtained from Life Technologies: Alexa Flour 633 goat anti-mouse IgG, Alexa '\n",
      " 'Flour 488 goat anti-mouse IgG, Alexa Flour 488 goat anti-rabbit IgG, Alexa '\n",
      " 'Flour 546 donkey anti-mouse IgG and Alexa Flour 546 goat anti-rabbit IgG. '\n",
      " 'TREx BCBL1-RTA cells were labelled using the Click-iT EdU Alexa Fluor 647 '\n",
      " \"Imaging Kit (Life Technologies) according to the manufacturer's instructions \"\n",
      " 'with minor modifications as follows. Cells were seeded onto poly-L-lysine '\n",
      " 'treated coverslips in 24-well plates followed by induction and incubation at '\n",
      " '37°C for 24 hours. Prior to cell fixation, 10 μM EdU '\n",
      " \"(5-ethynyl-2'deoxyuridine) was added to each well for 45 min. Cells were \"\n",
      " 'then fixed for 10 min in 4% formaldehyde and permeabilised in 1% Triton '\n",
      " 'X-100 for 20 min. Edu detection was carried out adding the Click-iT reaction '\n",
      " 'cocktail for 30 min and immunofluorescent labelling for RTA and Hsc70 was '\n",
      " 'performed as above. Cells were mounted in VECTASHIELD with DAPI (Vector '\n",
      " 'Labs). Total RNA from cells was extracted using TRIzol (Life Technologies) '\n",
      " \"according to the supplier's protocol. DNA-free DNA Removal Kit (Ambion) was \"\n",
      " 'used to remove any contaminating DNA from RNA samples. Reverse transcription '\n",
      " 'was performed with ProtoScript II (NEB) and oligo(dT) primers and 1.5 μg of '\n",
      " 'total RNA. Negative control reactions were performed in the same manner but '\n",
      " 'without reverse transcriptase. Quantitative PCR (qPCR) reactions (20 μl) '\n",
      " 'included 1X SensiMix SYBR green master mix (Bioline), 0.5 μM of each primer '\n",
      " 'and 5 μl template cDNA (used at 1:200 dilution in RNase-free water). Cycling '\n",
      " 'was performed in a Rotor-Gene Q machine (Eppendorf). The cycling programme '\n",
      " 'was a 10 min initial preincubation at 95°C, followed by 40 cycles of 95°C '\n",
      " 'for 15 sec, 60°C for 30 sec and 72°C for 20 sec. After qPCR, a melting curve '\n",
      " 'analysis was performed between 65 and 95°C (with 0.2°C increments) to '\n",
      " 'confirm amplification of a single product. Relative expression compared to '\n",
      " 'control cells was calculated using the ΔΔC T method as previously described '\n",
      " '[106] . For each gene of interest and housekeeping gene (GAPDH) a standard '\n",
      " 'curve was constructed using a pool of cDNA derived from unreactivated and '\n",
      " 'reactivated cells. Six different dilutions of the standards were quantified, '\n",
      " 'these included 1:100, 1:200, 1:400, 1:800, 1:1,600 and 1:3,200 dilution. The '\n",
      " 'slope of the standard curve was used to calculate the amplification '\n",
      " 'efficiency (AE) of the primers using the formula: AE = (10 −1/slope ). The '\n",
      " 'mean cycle threshold (C T ) was determined from three independent biological '\n",
      " 'replicates. All genes of interest were normalised against the housekeeping '\n",
      " 'gene GAPDH (ΔC T ). ΔΔC T was calculated subtracting ΔC T of unreactivated '\n",
      " 'cells from ΔC T of reactivated cells and the fold change was then determined '\n",
      " \"using AE (-ΔΔC T ) . Statistical significance was validated by Student's \"\n",
      " 't-test. Unreactivated TREx BCBL1-RTA cells treated with DMSO (0.1%) were '\n",
      " 'used as control to assess viral reactivation. Reactivated cells were exposed '\n",
      " 'to doxycycline for 72 h. Total DNA was then isolated with the use of a '\n",
      " \"QIAamp DNA mini kit (Qiagen) as per the manufacturer's instructions. qPCR \"\n",
      " 'was carried out as described above. 10 ng of template DNA and primers '\n",
      " 'specific for the ORF57 gene were used. Quantification of GAPDH gene was used '\n",
      " 'to normalize between samples and the mean cycle threshold (C T ) was '\n",
      " 'determined from three independent biological replicates. Relative levels of '\n",
      " 'viral DNA compared with unreactivated cells were calculated using the ΔΔC T '\n",
      " 'method as previously described [106] . TREx BCBL1-RTA cells that had been '\n",
      " 'seeded on 12-well plates were reactivated and treated with control DMSO '\n",
      " '(0.1%) or VER-155008. Unreactivated cells treated with DMSO (0.1%) were used '\n",
      " 'as control to evaluate viral reactivation. After 72 h reactivation, 700 μl '\n",
      " 'of the RPMI 1640 culture medium was centrifuged at 800 g for five min, '\n",
      " 'immediately mixed with 300 μl of DMEM supplemented with 10% FCS and 1% P/S '\n",
      " 'and incubated for a further 24 h with HEK-293T cells that had been seeded in '\n",
      " '12-well plates the previous day. Total RNA was then extracted with TRIzol '\n",
      " '(Life Technologies) and qRT-PCR carried out as described above. Relative '\n",
      " 'expression compared to control cells was calculated using the ΔΔC T method '\n",
      " 'as previously described [106] . Determination of the cellular metabolic '\n",
      " 'activity was performed using a non-radioactive CellTiter 96 AQ ueous One '\n",
      " 'Solution Cell Proliferation Assay (MTS) (Promega), according to the '\n",
      " \"manufacturer's manual. 20,000 cells TREx BCBL1-RTA or 10,000 HEK-293T \"\n",
      " 'rKSHV.219 cells were seeded in triplicate in a flat 96-well culture plate '\n",
      " '(Corning). After 24 h inhibitor exposure, CellTiter 96 AQueous One Solution '\n",
      " 'Reagent was added and cells were incubated for 1 h in a humidified incubator '\n",
      " 'in 5% CO 2 at 37°C. Absorbance was measured at 490 nm using an Infinite F50 '\n",
      " '(Tecan) plate reader. Background control had culture medium without cells '\n",
      " 'and the signal from this was subtracted to all other absorbance values. This '\n",
      " 'assay allows evaluation of viability, cytotoxicity and effector caspases '\n",
      " 'activation within a single assay well. The assay was carried out as '\n",
      " \"specified on the supplier's manual. 20,000 TREx BCBL1-RTA cells or 10,000 \"\n",
      " 'HEK-293T rKSHV.219 cells were seeded in triplicate in tissue culture treated '\n",
      " 'black microplates (Greiner Bio-One). No-cell control (background) contained '\n",
      " 'only culture medium and the signal from this was subtracted to all other '\n",
      " 'absorbance and luminescence values. Fluorescence and luminescence readings '\n",
      " 'were collected using a GloMax System (Promega) (kindly provided by Dr. John '\n",
      " 'Boyle, University of Leeds, UK). Luciferase activity was detected using the '\n",
      " 'Dual-Luciferase Reporter Assay System (Promega) as previously described '\n",
      " '[107] . HEK-293T cells were seeded in triplicate in flat 96-well culture '\n",
      " 'plate (Corning) at a density of 10,000 cells per well. Following the '\n",
      " 'respective plasmid transfections and inhibitor exposure, media was removed '\n",
      " 'from the culture wells and cells washed gently with 100 μl PBS. 30 μl 1x '\n",
      " 'passive lysis buffer was added to the cell monolayer which was rocked for 15 '\n",
      " 'min and then 20 μl of each lysate was transferred to tissue culture treated '\n",
      " 'white microplates (Greiner Bio-One). Luciferase measurements were carried '\n",
      " 'out in a FLUOstar Optima microplate reader (BMG Labtech Ltd), with injectors '\n",
      " '1 and 2 being used to dispense 50 μl of Luciferase Assay Reagent II and Stop '\n",
      " '& Glo Reagent respectively. Firefly luciferase activity was normalized to '\n",
      " 'Renilla luciferase activity. Formaldehyde-crosslinked chromatin was prepared '\n",
      " 'using the Pierce Chromatin Prep Module (Thermo Scientific) following the '\n",
      " \"manufacturer's protocol. 2 x 10 6 cells were used per experimental sample \"\n",
      " 'and digested with six units of micrococcal nuclease (MNase) per 100 μl of '\n",
      " 'MNase Digestion buffer in a 37°C water bath for 15 min. These conditions '\n",
      " 'resulted in optimal sheared chromatin with most chromatin fragments ranging '\n",
      " 'from 150-300 base pairs. Immunoprecipitations were carried out using EZ-ChIP '\n",
      " \"kit (Millipore) according to the supplier's instructions and as previously \"\n",
      " 'described [44] . Immunoprecipitations were done overnight at 4°C and '\n",
      " 'contained 50 μl of digested chromatin (2 x 10 6 cells), 450 μl of ChIP '\n",
      " 'dilution buffer and 1.5 μg of RNAPII antibody (clone CTD4H8) (Millipore) or '\n",
      " 'isotype antibody, normal mouse IgG (Millipore). Both antibodies were '\n",
      " 'provided with the EZ-ChIP kit. Prior to qPCR analysis, eluted DNA was '\n",
      " 'subjected to a DNA clean up step using UltraClean PCR Clean-Up Kit (Mo Bio '\n",
      " 'Laboratories) according to the supplied protocol with the exception of using '\n",
      " '500 μl of SpinClean buffer instead of 300 μl. qPCR reactions were performed '\n",
      " \"as described above and using either 2 μl of ChIP'ed DNA or 2 μl of input DNA \"\n",
      " 'as template. siRNA Knockdown HEK-293T rKSHV.219 cells seeded on 12-well '\n",
      " 'plates were reverse transfected with either 100 nM of the specific Silencer '\n",
      " 'Select siRNA (Life Technologies) or 100 nM AllStars negative control siRNA '\n",
      " '(Qiagen) using 7 μl of siPORT NeoFX transfection agent (Life Technologies) '\n",
      " 'per transfection. The siRNA ID for Hsc70 and iHsp70 were s6985 and s6968 '\n",
      " 'respectively. s6968 siRNA targets the two major iHsp70 proteins (HSP70-1 and '\n",
      " 'HSP70-2). Two days post-transfection, cells were transfected again in the '\n",
      " 'same manner. Four days after the first transfection, cells were reactivated '\n",
      " 'and incubated for the desired time. Proteins and total RNA were isolated '\n",
      " 'with TRIzol (Life Technologies) and subsequent Western blot and qRT-PCR were '\n",
      " 'performed. 8 × 10 6 TREx BCBL1-RTA cells were transfected once with 100 μl '\n",
      " 'of Nucleofector solution V (Lonza) to which 2 μM siRNA (scramble or Hsc70) '\n",
      " 'was added. In addition, to monitor transfection efficiency, 1 μg of the '\n",
      " 'control plasmid pmaxGFP was also co-transfected. Cells were transfected '\n",
      " 'using program T-01 of an Amaxa nucleofector I (Lonza). After nucleofection '\n",
      " 'cells were maintained in six-well plates. Medium was freshly replaced every '\n",
      " 'day. Confocal images were subjected to profiling analysis using Zeiss Zen '\n",
      " '2011 software. This involved drawing a line in a confocal image to measure '\n",
      " 'the relative intensity of each channel at every pixel along the line. '\n",
      " 'Profiling was conducted for each cell in two representative confocal images '\n",
      " 'taken with a 40-times objective. These data were then analysed using '\n",
      " 'Microsoft Excel 2010. Firstly, a function was used to define whether the '\n",
      " 'relative intensity of a pixel could be defined as a \"peak\" and thus an Hsc70 '\n",
      " 'foci. This function asked whether the data point for one specific pixel of '\n",
      " 'the rhodamine channel was 20, this was set as the arbitrary threshold to '\n",
      " 'eliminate background noise. If this condition was met, the function then '\n",
      " 'asked whether the data point was greater than or equal to the data point in '\n",
      " 'the previous and subsequent pixel. If these conditions were true, then this '\n",
      " 'data point was counted as a peak. This was performed for every data point '\n",
      " 'measured in the line profile providing the total number of Hsc70 peaks in '\n",
      " 'the profile of one cell. Next, another function was used to determine '\n",
      " 'whether the relative intensity in the DAPI channel was 25, a threshold '\n",
      " 'determined from visualising the line profiling data as a graph. If a pixel '\n",
      " 'was shown to exceed the threshold in the DAPI channel and also in the '\n",
      " 'rhodamine channel, then it was counted as a nuclear Hsc70 peak. These '\n",
      " 'measurements were conducted for each pixel in each profile allowing counting '\n",
      " 'Hsc70 nuclear peaks in each profile. Hsc70 peaks outside the nucleus '\n",
      " 'corresponded to cytoplasmic Hsc70 peaks. Oligonucleotide primer sequences '\n",
      " 'are available upon request. All primers were purchased from Sigma (UK). '\n",
      " 'Supporting Information S1 Table. Cellular proteins previously reported to '\n",
      " 'localise to herpesvirus RTCs and found significantly increased in the NE of '\n",
      " 'reactivated cells. (XLS) S1 Fig. iHsp70 was redistributed from the cytoplasm '\n",
      " 'to both the periphery and within KSHV-induced RTCs. (A) TREx BCBL1-RTA cells '\n",
      " 'remained unreactivated or reactivated for either 20 h or 24 h. In '\n",
      " 'unreactivated cells iHsp70 was cytoplasmic (i). In contrast, at 20 h '\n",
      " 'reactivation an increase in nuclear iHsp70 labelling was seen with numerous '\n",
      " 'small iHsp70 foci found mainly adjacent to viral RTCs (ii). Some cells '\n",
      " 'displayed iHsp70 completely recruited within RTCs (iii and iv asterisks), '\n",
      " 'while other cells accumulated large iHsp70 adjacent to RTCs (iv arrows). (B) '\n",
      " 'TREx BCBL1-RTA cells remained unreactivated (i) or reactivated for 24 h (ii '\n",
      " 'and iii) followed by triple-labelling with antibodies specific for RTA and '\n",
      " 'iHsp70 and Click-iT EdU Alexa Fluor 647. Complete co-localisation between '\n",
      " 'iHsp70, RTA and actively replicated viral DNA (Edu-labelled) was observed in '\n",
      " 'both incipient RTCs (ii) and in fully-developed RTCs (iii). Note that in '\n",
      " 'these cells iHsp70 was not depleted from the cytoplasm. TREx BCBL1-RTA cells '\n",
      " 'remained unreactivated or reactivated for 24 h in the presence of control '\n",
      " 'DMSO (0.1%) or 2 μM VER-155008 followed by labelling with Click-iT EdU Alexa '\n",
      " 'Fluor 647 and an antibody specific for RNAPII (clone CTD4H8). (A) A high '\n",
      " 'proportion of unreactivated TREx BCBL1-RTA cells replicated their cellular '\n",
      " 'DNA (Edu-labelled) in the presence of control DMSO (0.1%) or 2 μM '\n",
      " 'VER-155008. Normal RNAPII localization was observed in these cells, with '\n",
      " 'nuclear RNAPII excluding the nucleoli. (B) In contrast, reactivated cells '\n",
      " 'entered cell cycle arrest as demonstrated by fewer Edu-labelled cells. In '\n",
      " 'the presence of DMSO, multiple RTCs were formed with some replicating viral '\n",
      " 'DNA (white arrows). In cells treated with VER-155008, multiple '\n",
      " 'pre-replicative sites were seen labelled by RNAPII antibody and '\n",
      " 'Edu-labelling was more diffused in the nucleus compared with DMSO-treated '\n",
      " 'cells. (TIF)',\n",
      " 'Coronaviruses are enveloped viruses with 27-32 kb single-strand '\n",
      " 'positive-sense RNA genomes encoding four structural proteins: nucleocapsid '\n",
      " '(N), spike (S), membrane (M) and envelope (E) [1, 2] . Translated on free '\n",
      " 'polysomes, highly basic N interacts with newly synthesized viral genomic RNA '\n",
      " 'to form helical nucleocapsids [3, 4] . The M, S and E viral membrane '\n",
      " 'proteins are translated on membrane-bound polysomes, inserted into the '\n",
      " 'endoplasmic reticulum (ER), and transported to the ER-Golgi intermediate '\n",
      " 'compartment (ERGIC) , where E and M interact and trigger budding [5, 6] . N '\n",
      " 'and S are incorporated into virions via interaction with M, with virions '\n",
      " 'accumulating in large, smooth-walled vesicles that eventually fuse with the '\n",
      " 'plasma membrane and release virions from cells [2, [7] [8] [9] [10] [11] . '\n",
      " 'Coronavirus E is a small integral membrane protein consisting of '\n",
      " 'approximately 76 to 109 amino acids and containing a hydrophobic domain. '\n",
      " 'Several researchers have suggested that coronavirus E functions as an ion '\n",
      " 'channel [12, 13] . The role of the coronavirus E ion channel in the virus '\n",
      " 'life cycle is not completely clear. The addition of hexamethylene amiloride '\n",
      " '(HMA, an ion channel inhibitor of mouse hepatitis coronavirus [MHV] and '\n",
      " 'human coronavirus 229E [HCoV229E] ion channel activity in vitro) to culture '\n",
      " 'medium blocks MHV and HCoV229E replication [12] , suggesting that the '\n",
      " 'coronavirus E ion channel plays a role in virus replication. Two or three '\n",
      " 'cysteine residues are located on the carboxyl side of the hydrophobic domain '\n",
      " 'in coronavirus E proteins, with some serving as targets for palmitoylation '\n",
      " '[14] [15] [16] [17] , which may contribute to virus assembly in the MHV. E '\n",
      " 'plus M has been shown to be sufficient for VLP formation in the MHV [18] , '\n",
      " 'transmissible gastroenteritis virus (TGEV) [19] , bovine coronavirus (BCoV) '\n",
      " '[19] , infectious bronchitis virus (IBV) [5] , and SARS-CoV [20] . Although '\n",
      " 'M and N co-expression is also sufficient for SARS-CoV VLP production [21, '\n",
      " '22] , VLP yields are further enhanced by E co-expression [23] . According to '\n",
      " 'these data, SARS-CoV E plays a supporting role in promoting virus assembly '\n",
      " 'and/or budding. E and N are thought to be packaged into VLPs through '\n",
      " 'separate and individual associations with M; it remains unknown whether E/N '\n",
      " 'interaction exists, which might contribute to enhanced virion production. '\n",
      " 'MHV and IBV E proteins are capable of release from cells as vesicles [5, 24] '\n",
      " ', implying a relationship between E release and its contribution to virus '\n",
      " 'production. Our goal for this study was to determine whether SARS-CoV N is '\n",
      " 'capable of interacting with E and cell release via an association with '\n",
      " 'Econtaining vesicles. Our results indicate no correlation between SARS-CoV E '\n",
      " 'capacity to release or interact with N, or its ability to promote VLP '\n",
      " 'production. Mammalian expression vectors encoding SARS-CoV M, N, S and E '\n",
      " 'were provided by G. J. Nabel [21] . A pair of upstream and downstream '\n",
      " 'primers was used to amplify Ecoding fragments via PCR-based overlap '\n",
      " 'extension mutagenesis [25] . Primers used to introduce an HA or FLAG epitope '\n",
      " 'tag to the E amino or carboxyl terminus are '\n",
      " '5′-TTCTGCGATATCGCCACCATGTACCCATACGAC GTGCCTGACTACGCCTACAGCTTCGTGAGCG-3′ '\n",
      " '(containing a flanking EcoRV restriction site and HA tagcoding nucleotides) '\n",
      " 'and 5′-GCGGATCCTCACTTGTC GTCGTCGTCCTTGTAGTCGCCCACCAGCAGGTCG GGCAC-3′ '\n",
      " '(containing a flanking BamHI restriction site and FLAG tag-coding '\n",
      " 'nucleotides). The forward primer is 5′-GTCTGAGCAGTACTCGTTGCTG-3 (referred to '\n",
      " 'as the N primer) and reverse primer 5′-GGAAAGGA CAGTGGGAGTGGCAC-3′ (referred '\n",
      " 'to as the anti-N primer). Primers used to construct designated E mutants '\n",
      " 'were EC3A, 5′-CTGAGGCTGGCCGCATATGCCGCG AACATCGTGAACGTGAGC-3′ (reverse); '\n",
      " '74LL/AA,5′-GCAGATCTGGATCCTAGTTCACACGGCCGCGTCG GGCACGCC-3′ (reverse); EΔ76V, '\n",
      " '5′-CAGATCTGGA TCCTAGTTCACAGCAGGTCGGGCAC-3′ (reverse). The N primer serves as '\n",
      " 'a forward primer while the anti-N primer was used as a reverse primer for '\n",
      " 'the second round PCR amplification. Purified PCR product was digested with '\n",
      " 'BamHI and EcoRV and ligated into the SARS-CoV E expression vector. When '\n",
      " 'constructing N-DsRed, the N primer served as the forward primer, using the '\n",
      " 'SARS-CoV N expression vector as a template and '\n",
      " '5′-GCGGATCCTGGGTGCTGTCGGCGCTG-3′ as the reverse primer. Amplified PCR '\n",
      " 'product was digested with BamHI and SalI and ligated into pDsRed-Monomer-N1 '\n",
      " '(Clontech). The cloned N-DsRed was digested with NheI and BamHI and ligated '\n",
      " 'into pEGFP-N1 (Clonetech), yielding construct N-EGFP. GST-M was constructed '\n",
      " 'by digesting M expression vector with EcoRV and NotI, ligated into '\n",
      " 'pCDNA3.1myc-HisA (Invitrogen). The resultant construct was then digested '\n",
      " 'with BamHI and Not1, and then the M coding sequence fragment was fused to '\n",
      " 'carboxyl terminus of GST, which is directed by a mammalian elongation factor '\n",
      " 'Ia promoter [26] . GST-N fusions, as described previously [27] , were '\n",
      " 'constructed by fusing SARS-CoV N coding sequences to the carboxyl terminus '\n",
      " 'of GST. Cell culture and transfection 293 T and HeLa cells were maintained '\n",
      " \"in Dulbecco's modified Eagle's medium (DMEM) supplemented with 10% fetal \"\n",
      " 'calf serum (GIBCO). Confluent cells were trypsinized and split 1:10 onto 10 '\n",
      " 'cm dishes 24 h prior to transfection. For each construct, cells were '\n",
      " 'transfected with 20 μg of plasmid DNA using the calcium phosphate '\n",
      " 'precipitation method; 50 μm chloroquine was added to enhance transfection '\n",
      " 'efficiency. Unless otherwise indicated, 10 μg of each plasmid was used for '\n",
      " 'co-transfection. Culture supernatant and cells were harvested for protein '\n",
      " 'analysis 2-3 d post-transfection. For HeLa transfection, plasmid DNA was '\n",
      " 'mixed with GenCarrier (Epoch Biolabs) at a ratio of 1 μg to 1 μl; the '\n",
      " \"transfection procedure was performed according to the manufacturer's \"\n",
      " 'protocols. At 48-72 h post-transfection, supernatant from transfected cells '\n",
      " 'was collected, filtered, and centrifuged through 2 ml of 20% sucrose in TSE '\n",
      " '(10 mM Tris-HCl [pH 7.5], 100 mM NaCl, 1 mM EDTA plus 0.1 mM '\n",
      " 'phenylmethylsulfonyl fluoride [PMSF]) at 4°C for 40 min at 274,000 x g. '\n",
      " 'Pellets were suspended in IPB (20 mM Tris-HCl [pH 7.5], 150 mM NaCl, 1 mM '\n",
      " 'EDTA, 0.1% SDS, 0.5% sodium deoxycholate, 1% Triton X-100, 0.02% sodium '\n",
      " 'azide) plus 0.1 mM PMSF. Cells were rinsed with ice-cold phosphate-buffered '\n",
      " 'saline (PBS), collected in IPB plus 0.1 mM PMSF, and microcentrifuged at 4°C '\n",
      " 'for 15 min at 13,700 x g to remove unbroken cells and debris. Supernatant '\n",
      " 'and cell samples were mixed with equal volumes of 2X sample buffer (12.5 mM '\n",
      " 'Tris-HCl [pH 6.8], 2% SDS, 20% glycerol, 0.25% bromphenol blue) and 5% '\n",
      " 'β-mercaptoethanol and boiled for 5 min or (for the M-containing samples) '\n",
      " 'incubated at 45°C for 10 min. Samples were resolved by electrophoresis on '\n",
      " 'SDS-polyacrylamide gels and electroblotted onto nitrocellulose membranes. '\n",
      " 'Membrane-bound M, M-FLAG, HA-E, E-FLAG or GST proteins were immunodetected '\n",
      " 'using a SARS-CoV M rabbit anitserum, anti-HA (LTK BioLaboratories, Taiwan), '\n",
      " 'anti-FLAG or anti-GST(Sigma) monoclonal antibody at a dilution of 1:1,000. '\n",
      " 'For SARS-CoV N or S detection, a mouse monoclonal antibody [28, 29] was used '\n",
      " 'at a dilution of 1:5,000. The secondary antibody was a sheep anti-mouse or '\n",
      " 'donkey anti-rabbit horseradish peroxidase-(HRP) conjugated antibody '\n",
      " '(Invitrogen), both at 1:5,000 dilutions. HeLa cells were split 1:80 onto '\n",
      " 'coverslips 24 h before transfection. Between 18 and 24 h post-transfection, '\n",
      " ', cells were washed with PBS and permeabilized at room temperature for 10 '\n",
      " 'min in PBS plus 0.1% Triton X-100 following fixation at 4°C for 20 min with '\n",
      " 'methanol/acetone (1:1). Samples were incubated with the primary antibody for '\n",
      " '1 h and with the secondary antibody for 30 min. Following each incubation, '\n",
      " 'samples were subjected to three washes (5 to 10 min each) with DMEM/calf '\n",
      " 'serum. Primary antibody concentrations were anti-HA at a dilution of 1:500. '\n",
      " 'A rabbit anti-mouse rhodamine-conjugated antibody at a 1:100 dilution served '\n",
      " 'as the secondary antibody (Cappel, ICN Pharmaceuticals, Aurora, OH). After a '\n",
      " 'final DMEM/calf serum wash, the coverslips were washed three times with PBS '\n",
      " 'and mounted in 50% glycerol in PBS for viewing. Images were analyzed and '\n",
      " 'photographs taken using the inverted laser Zeiss. Supernatants from '\n",
      " 'transfected 293 T cells were collected, filtered, and centrifuged through 2 '\n",
      " 'ml 20% sucrose cushions as described above. Viral pellets were suspended in '\n",
      " 'PBS buffer and laid on top of a pre-made 10-40% iodixanol (OptiPrep) '\n",
      " 'gradient consisting of 1.25 ml layers of 10, 20, 30 and 40% iodixanol '\n",
      " \"solution prepared according to the manufacturer's instructions (Axis-Shield, \"\n",
      " 'Norway). Gradients were centrifuged in a SW50.1 rotor at 40,000 rpm for 16 h '\n",
      " 'at 4°C; 500 μl fractions were collected from top to bottom and densities '\n",
      " 'were measured for each. Proteins in each fraction were precipitated with 10% '\n",
      " 'trichloroacetic acid (TCA) and subjected to Western immunoblotting. GST '\n",
      " 'pull-down protocols were as previously described [27] . Briefly, 500 μl of '\n",
      " 'PNS containing complete protease inhibitor cocktail was mixed with 30 μl of '\n",
      " 'glutathione agarose beads (Sigma). All reactions took place at 4°C overnight '\n",
      " 'on a rocking mixer. Immunoprecipitateassociated resin or bead-bound '\n",
      " 'complexes were pelleted, washed tree times with lysis buffer, two times with '\n",
      " 'PBS, eluted with 1X sample buffer, and subjected to SDS-10% PAGE as '\n",
      " 'described above. In a previous study we found that SARS-CoV M coexpression '\n",
      " 'with N leads to VLP formation, and that a FLAG tagged at the SARS-CoV M '\n",
      " 'carboxyl terminus (M-FLAG) significantly inhibits N packaging into VLPs [22] '\n",
      " '. Since N release into medium depends on an association with M, and since E '\n",
      " 'may promote VLP production, we set out to confirm whether E complements '\n",
      " 'M-FLAG in N packaging into VLPs. Toward this goal we coexpressed N with '\n",
      " 'M-FLAG in the absence or presence of SARS-CoV E tagged with either HA at the '\n",
      " 'amino terminus (HA-E) or with FLAG at the carboxyl terminus (E-FLAG). As '\n",
      " 'shown in Figure 1 , N that was barely detectable in medium became readily '\n",
      " 'detected following co-expression with HA-E (lanes 9 versus 10). Since N is '\n",
      " 'incapable of associating with M-FLAG, a simple explanation would be the '\n",
      " 'ability of E to promote N incorporation into VLPs via E/N interaction. '\n",
      " 'E-FLAG was barely HA-E denotes an HA tagged at the SARS-CoV E amino '\n",
      " 'terminus. DNA (5 μg) of each plasmid was used for each transfection, with '\n",
      " 'pBlueScript SK added to maintain a plasmid DNA level of 20 μg. At 48 h '\n",
      " 'post-transfection, supernatants and cells were collected and prepared for '\n",
      " 'protein analysis as described in Materials and Methods. Cell lysate samples '\n",
      " '(lanes 1 to 5) corresponding to 5% of total, and medium pellet samples '\n",
      " '(lanes 6 to 10) corresponding to 50% of total, were fractionated by 10% '\n",
      " 'SDS-PAGE and electroblotted onto nitrocellulose filters. SARS-CoV M was '\n",
      " 'probed with rabbit antiserum. SARS-CoV N and E were detected with a mouse '\n",
      " 'anti-N, anti-HA, or anti-FLAG monoclonal antibody. N proteins from medium or '\n",
      " 'cell samples were quantified by scanning N band densities from immunoblots. '\n",
      " 'Rations of N level in media to those in cells were determined for each '\n",
      " 'samples and normalized to that of samples without HA-E or E-FLAG '\n",
      " 'co-expression. detected in medium (Figure 1, lane 7) , suggesting that the '\n",
      " 'FLAG tagged at the E carboxyl terminus affected E release or E association '\n",
      " 'with M and/or N. To confirm that E-mediated N release is not due to HA tag, '\n",
      " 'we coexpressed N with E and found that E without HA tag can still promote N '\n",
      " 'release (data not shown), suggesting that HA tag exerts no major impacts on '\n",
      " 'E-mediated N release. Since SARS-CoV E and M are both capable of release '\n",
      " 'into medium, N detected in medium may be a result of an association with '\n",
      " 'released M-FLAG in the presence of E, but not a result of direct E/N '\n",
      " 'interaction. To rule out this possibility and to confirm that a specific N-E '\n",
      " 'interaction exists, we co-expressed N and either HA-E, E-FLAG, M or S. As '\n",
      " 'expected, N was readily detected when co-expressed with HA-E or with M, but '\n",
      " 'barely detectable in culture medium when expressed alone or coexpressed with '\n",
      " 'S ( Figure 2A , lanes 5 and 8) or E-FLAG (which was also barely detectable '\n",
      " 'in medium) ( Figure 2B , lane 3). These data support our hypothesis that '\n",
      " 'SARS-CoV N is capable of release into medium via association with E, and '\n",
      " 'suggests that FLAG tagged at the SARS-CoV E carboxyl terminus significantly '\n",
      " 'reduces E release capacity in addition to impairing E incorporation into '\n",
      " 'VLPs. To confirm that N release is associated with E in pelletable particle '\n",
      " 'form, we subjected culture medium from cells expressing E, E plus N, or E '\n",
      " 'plus N and M to iodixanol density gradient fractionation, and found that E '\n",
      " 'primarily sedimented with co-expressed N at a slightly lower density '\n",
      " 'fraction compared to E expression alone ( Figure 2C ). Particles formed by E '\n",
      " 'or E plus N exhibited an iodixanol density of 1.13 to 1.16 g/ml, whereas '\n",
      " 'VLPs formed by E, N and M displayed densities ranging from 1.11 to 1.13 '\n",
      " 'g/ml. Combined, these data support the idea that N released into medium is '\n",
      " 'associated with E vesicles when co-expressed with E. RNA is not required for '\n",
      " 'efficient E/N interaction SARS-CoV N contains an RNA binding domain [30] , '\n",
      " 'and results from our previous study indicate that RNA can enhance the N-N '\n",
      " 'interaction [22] . Here our goal was to determine whether the presence of '\n",
      " 'RNA is required for E/N interaction. HA-E or N was co-expressed with GST '\n",
      " 'tagged at the N amino terminus (GST-N). E or N association with GST-N was '\n",
      " 'assessed using a GST pull-down assay in the presence or absence of RNase. '\n",
      " 'GST by itself was not capable of pulling down HA-E or N ( Figure 3A, lanes '\n",
      " '5-6) . RNase treatment significantly reduced levels of co-pulled-down N ( '\n",
      " 'Figure 3A, lane 9) , which is consistent with past results [22] . However, '\n",
      " 'the same RNase condition did not reduce the amount of copulled-down HA-E, '\n",
      " 'suggesting that RNA is not required for E/N interaction. We frequently '\n",
      " 'observed slight increases in pulled-down HA-E following treatment with RNase '\n",
      " '( Figure 3A , lane 10 vs. lane 8). Since RNA binds to N readily, it is '\n",
      " 'likely that RNA-bound N molecules may associate less efficiently with E. To '\n",
      " 'map the N sequences involved in E association, we constructed a set of GST-N '\n",
      " 'fusions containing multiple N coding sequences fused to the GST carboxyl '\n",
      " 'terminus ( Figure 3B ). Each GST fusion construct was transiently '\n",
      " 'co-expressed with E, and culture supernatants and cell lysates were analyzed '\n",
      " 'by Western immunoblotting. As shown in Figure 3C , all GST fusions except '\n",
      " 'for GST-N7 (containing N residues 2 to 86, lane 9) were released into medium '\n",
      " 'when co-expressed with E. None of the GST fusions were readily detected in '\n",
      " 'medium without coexpressed E (data not shown), suggesting that the detected '\n",
      " 'GST-N fusions were released into medium via E association, and that the E '\n",
      " 'binding domains are largely located in the N carboxyl-terminal region '\n",
      " '(likely involving amino acid residues 87 to 421). Results from a GST '\n",
      " 'pull-down assay ( Figure 3D ) provide further support for this conclusion. '\n",
      " 'Cysteine residues are not required for E release, E/N interaction, or E '\n",
      " 'enhancement of VLP production SARS-CoV and MHV E proteins are '\n",
      " 'post-translationally modified with palmitic acid. Lack of palmitoylation '\n",
      " 'modification can markedly decrease SARS-CoV E membrane association [16] and '\n",
      " 'MHV E stability [17] . Substitution mutations for cysteines-the '\n",
      " 'palmitoylation targets in MHV E can reduce virus yields, implying their '\n",
      " 'importance for MHV virus production [14, 17] . Accordingly, we tried to '\n",
      " 'determine whether equivalent cysteines play a role in SARS-CoV E/N '\n",
      " 'interaction and VLP assembly by constructing an E mutant (designated EC3A) '\n",
      " 'with alanine replacements for SARS-CoV E cysteine residues C40, C43 and C44. '\n",
      " 'Supernatant and cell lysate samples containing HA-E were analyzed under '\n",
      " 'non-reducing conditions to confirm the presence of disulfide bonds between '\n",
      " 'SARS-CoV E molecules. Results indicate the presence of an 18 kDa band '\n",
      " '(equivalent to an HA-E dimer) ( Figure 4A , lanes 2 and 11, arrowheads), '\n",
      " 'which agrees with a previous report [31] . As expected, only the E monomer '\n",
      " 'was detected in EC3A-containing samples under non-reducing conditions ( '\n",
      " 'Figure 4A, lanes 3 and 12) . While most of intracellular E retains its '\n",
      " 'monomeric form, most released E was detected in dimeric form ( Figure 4A , '\n",
      " 'lane 2 vs. lane 11), suggesting that released SARS-CoV E proteins are '\n",
      " 'largely linked by disulfide bonds. Interestingly, EC3A was still efficiently '\n",
      " 'released into medium and apparently capable of promoting N release ( Figure '\n",
      " '4A, lanes 14  and 18) . The relatively low level of medium HA-E observed in '\n",
      " 'Figure 4A is due to low expression level (top panels, lanes 17 vs. 8). '\n",
      " 'Nevertheless, our data indicate that EC3A produced VLPs as efficiently as wt '\n",
      " 'when co-expressed with both M and N ( Figure 4B, lanes 10-12) , suggesting '\n",
      " 'that SARS-CoV E cysteines are not involved in E release, E/N interactions, '\n",
      " 'or VLP assembly. EC3A exhibited greater release efficiency compared to wt, '\n",
      " 'implying that SARS-CoV E cysteines are involved in the E trafficking '\n",
      " 'process. However, immunofluorescence results indicate that the EC3A '\n",
      " 'subcellular distribution pattern was similar to that of the wt. Removal of '\n",
      " 'last amino acid residue from the E carboxyl-terminal tail significantly '\n",
      " 'affects E release and E/N interaction E-FLAG was released at a lower '\n",
      " 'efficiency level compared to HA-E, suggesting that the carboxyl-terminal '\n",
      " 'tail domain may be involved in the E release process. We hypothesized that '\n",
      " 'the E-carboxyl tail is also involved in E/N interaction. To test this idea, '\n",
      " 'we constructed two E mutants: EΔ76V (with the final carboxyl-terminal '\n",
      " 'residue valine removed) and E74LL/AA (with the carboxylterminal dileucine '\n",
      " 'motif 74-LL-75 replaced with alanines). According to our results, (a) the '\n",
      " 'E74LL/AA mutant was capable of release, and (b) a positive correlation '\n",
      " 'exists between the release levels of co-expressed N and E74LL/AA ( Figure '\n",
      " '5A, lanes 10-15) . In addition, E74LL/AA was capable of producing VLPs as '\n",
      " 'efficiently as wt when co-expressed with M and N ( Figure 5A,  lanes 16 vs. '\n",
      " '17 and 18) . These results suggest that 74-LL-75 was not functionally '\n",
      " 'involved in E release, E/N interaction, or the promotion of VLP production. '\n",
      " 'Conversely, EΔ76A was not capable of efficiently promoting N release, and '\n",
      " 'exhibited a reduced release capacity ( Figure 5B, lanes 12 to 15) . GST '\n",
      " 'pull-down assay data suggest that the EΔ76V mutation markedly affected E/N '\n",
      " 'interaction, but exerted no major impacts on N/M association ( Figure 5C, '\n",
      " 'lanes 16 vs. 18) . The level of GST-Nassociated E was significantly lower '\n",
      " 'than that of GST-Massociated E, suggesting that E/N association is not as '\n",
      " 'strong as E/M association ( Figure 5C, lanes 15 vs. 17) . While pull-down '\n",
      " 'assay results suggest an association between EΔ76V and M, EΔ76V release was '\n",
      " 'not significantly enhanced following M co-expression ( Figure 5C , lane 18 '\n",
      " 'vs. Figure 5B, lanes 17 and 20) . A possible explanation is steric hindrance '\n",
      " 'during VLP assembly. Note that EΔ76V was still capable of promoting VLP '\n",
      " 'production as efficiently as the wt, despite defective packaging into VLPs ( '\n",
      " 'Figure 5B, lane 20) . This suggests that the SARS-CoV E contribution to VLP '\n",
      " 'production is independent of its ability to be packaged into VLPs. To '\n",
      " 'determine whether mutations affecting E subcellular localization also partly '\n",
      " 'accounted for altered release efficiency, we co-expressed each mutant with a '\n",
      " 'Golgi labeling marker (pECFP-Golgi). According to a confocal microscopy '\n",
      " 'analysis, wt and the EC3A, E74LL/AA, and EΔ76V mutants largely localized in '\n",
      " 'the Golgi area ( Figure 6A ), suggesting that none of the mutations '\n",
      " 'significantly affected E subcellular localization. When co-expressed with '\n",
      " 'EGFP-tagged N, a fraction of N co-localized with EΔ76V in the perinuclear '\n",
      " 'area; this staining pattern was barely distinguishable from that of cells '\n",
      " 'co-expressing HA-E and N-EGFP ( Figure 6B ). This implies that even though '\n",
      " 'EΔ76V may associate with N to a certain extent, the association is '\n",
      " 'insufficient for enabling EΔ76V co-precipitation with GST-N ( Figure 5C ). '\n",
      " 'To our knowledge, this is the first report of interaction between SARS-CoV E '\n",
      " 'and N proteins. Both SARS-CoV spike (S) and E proteins can be released into '\n",
      " 'medium; however, unlike E, S cannot promote the release of coexpressed N '\n",
      " '(Figure 2A ). This suggests that N is incapable of associating with S, and '\n",
      " 'supports the idea of specific E/N interaction. While RNA can enhance N/N '\n",
      " 'interaction, the presence of RNA is not necessary for E/ N interaction. '\n",
      " 'Genetic analyses suggest that E binding domains are largely located in the N '\n",
      " 'carboxyl-terminal region (Figure 3) . SARS-CoV M or E by itself can secrete '\n",
      " 'into medium as vesicles but not virus-like particles (VLPs) which can be '\n",
      " 'formed by M plus N, M plus E, E plus N or M plus N and E. It is likely that '\n",
      " 'there may be a combination of these different VLPs in culture medium when '\n",
      " 'cells are co-transfected with M plus E and N. The iodixanol density gradient '\n",
      " 'fractionation analyses suggest that released E vesicles and E-N VLPs exhibit '\n",
      " 'slightly higher densities compared to those of M-E-N VLPs ( Figure 2C ). '\n",
      " 'Since M is the major viral component, the strong presence of M molecules may '\n",
      " 'exert a spatial effect that explains, at least in part, the lower density '\n",
      " 'for M-E-N particles. We consistently observed marked E-N VLP yield '\n",
      " 'enhancement following M co-expression. Since M is the most abundant viral '\n",
      " 'structural protein capable of recruiting both E and N into VLPs, we do not '\n",
      " 'consider this a surprising result. In addition, M exhibits a noticeably '\n",
      " 'higher N binding capacity than E ( Figure 5C ). VLPs formed by M, E and N '\n",
      " 'look more morphologically homogeneous than E-N VLPs, possibly due to '\n",
      " 'additional intermolecular interactions between M/M, M/E and M/N. SARS-CoV E '\n",
      " 'is capable of undergoing oligomerization triggered by both hydrophobic '\n",
      " 'interaction and interchain disulfide bond formation between cysteine '\n",
      " 'residues [16, 31] . All three SARS-CoV E cysteine residues have been shown '\n",
      " 'to be post-translationally modified by palmitoylation [16] . Blocking MHV E '\n",
      " 'palmitoylation results in significantly impaired VLP assembly, suggesting '\n",
      " 'that palmitoylated E proteins are essential for murine coronavirus assembly '\n",
      " '[14] . In the present study we found that changing all three cysteines into '\n",
      " 'alanines (EC3A) did not exert negative effects on SARS-CoV E protein release '\n",
      " 'or VLP assembly (Figure 4 ). This suggests that E palmitoylation or '\n",
      " 'interchain disulfide bond formation is not required for SARS-CoV E protein '\n",
      " 'release or VLP assembly. Furthermore, we consistently detected higher levels '\n",
      " 'of EC3A compared to wt E in medium, suggesting that EC3A is released more '\n",
      " 'efficiently than wt E. One possible explanation is that post-translational '\n",
      " 'palmitoylation or interchain disulfide bond formation restricts E protein '\n",
      " 'secretion. Despite a higher release level compared to wt E, EC3A did not '\n",
      " 'produce higher VLP yields than wt E following co-expression with M and N, '\n",
      " 'suggesting no correlation between SARS-CoV E release capacity and its '\n",
      " 'contribution to efficient VLP production. Furthermore, EΔ76A was still '\n",
      " 'capable of enhancing VLP production even though it was defective in both '\n",
      " 'release and N association. Although EΔ76A is capable of M association (as '\n",
      " 'shown by GST assays), it is not efficiently incorporated into VLPs when '\n",
      " 'co-expressed with M. A possible explanation for this discrepancy is that the '\n",
      " 'disruption of cellular compartments allows E/M to associate at a higher '\n",
      " 'capacity, whereas less E/M association occurs in assembly and budding '\n",
      " 'compartments such as ER/Golgi, resulting in smaller amounts of E being '\n",
      " 'packaged into virions. This scenario is compatible with the proposal that E '\n",
      " 'may promote virus assembly and/or budding by facilitating membrane bending '\n",
      " 'and scissions without required packaging into virions. In support of this '\n",
      " 'hypothesis, one research team has reported that E deletion does not '\n",
      " 'significantly affect SARS-CoV replication, but that E-deleted mutants '\n",
      " 'exhibit 100-to 1,000-fold reductions in virus yields associated with '\n",
      " 'decreased efficiency during morphogenesis [32] . Although E/N interactions '\n",
      " 'may not be directly involved in virus assembly and budding, it remains to be '\n",
      " 'seen whether the SARS-CoV E contribution to virus production requires '\n",
      " 'efficient E/M interaction. Palmitoylation or interchain disulfide bond '\n",
      " 'formation appears to be dispensable for the SARS-CoV E enhancement of VLP '\n",
      " 'yields. The contribution of SARS-CoV E to efficient VLP production is also '\n",
      " 'independent of its release capacity, association with N, or VLP '\n",
      " 'incorporation. Additional experiments are required to clarify the biological '\n",
      " 'relevance of SARS-CoV E/N interaction, and to determine whether E/N '\n",
      " 'interaction also exists in other coronaviruses. (See figure on previous '\n",
      " 'page.) Figure 6 Subcellular localization of SARS-CoV E and N proteins. (A) '\n",
      " 'Subcellular localization of wt or mutant SARS-CoV E proteins. HeLa cells '\n",
      " 'were co-transfected with the indicated HA-tagged wt or mutant E expression '\n",
      " 'vectors plus pGolgi-ECFP, which encodes a Golgi apparatus labeling marker. '\n",
      " '(B) Co-localization of SARS-CoV E and N proteins. HeLa cells were '\n",
      " 'co-transfected with SARS-CoV N bearing carboxyl-terminal tagged EGFP '\n",
      " '(N-EGFP) and HA-tagged wt or Δ76V expression vectors. At 18 h '\n",
      " 'post-transfection, cells were fixed and labeled with a primary anti-HA '\n",
      " 'antibody and a secondary rhodamine-conjugated anti-mouse antibody. Images '\n",
      " 'represent the most prevalent phenotypes. Merged red/blue or red/green '\n",
      " 'fluorescent images are shown in right-hand column panels. Mock-transfected '\n",
      " 'cells failed to yield any signal (data not shown).',\n",
      " 'In both urban and rural Ghana, two species of frugivorous bats, the Gambian '\n",
      " 'epauletted fruit bat (Epomophorus gambianus) and the straw-coloured fruit '\n",
      " 'bat (Eidolon helvum), typically live close to humans [1, 2] . In the '\n",
      " 'township of Ve Golokuati, the Afadzato South District capital in the Volta '\n",
      " 'Region of Ghana, several \"camps\" of E. gambianus roost in mango and other '\n",
      " 'fruit trees that provide shade and other ecosystems services for residents. '\n",
      " 'Concerns have been raised about the closeness of bats to humans because of '\n",
      " 'the ability of the former to spread zoonotic diseases [3] [4] [5] [6] [7] . '\n",
      " 'Viruses isolated from bats include lyssaviruses, severe acute respiratory '\n",
      " 'syndrome (SARS) and related coronaviruses (SARS-related 2 Journal of '\n",
      " 'Environmental and Public Health have been most widespread in Africa [9] . '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'Ebola outbreaks have been recurring since the first major case in DR Congo '\n",
      " 'in 1995, which affected more than 260 humans and caused 186 deaths, was '\n",
      " 'reported [14] . The most recent outbreak of Ebola in West Africa in 2014 '\n",
      " 'infecting about 27,898 people and causing 11,296 reported deaths [17] raised '\n",
      " 'public apprehension about human-bat interactions. This is because the '\n",
      " 'outbreak has been traced back to a single incident of a young child playing '\n",
      " 'in the vicinity of a hollow tree frequented by bats [18] , now identified as '\n",
      " 'a potential source of spillover of the EBV [11, 14, 19] . The ability of '\n",
      " 'bats to spread zoonotic diseases and the fact they live close to humans '\n",
      " 'heighten the potential threat of disease spillover. Notwithstanding these '\n",
      " 'negative perceptions, bats play vital roles in providing ecosystems '\n",
      " 'services. They are well known for their roles in seed dispersal, '\n",
      " 'pollination, maintaining soil fertility, and aiding in nutrient distribution '\n",
      " '[6, 20, 21] . Muscarella and Fleming [20] noted that bats facilitate seed '\n",
      " 'dispersal in clear-cut strips as a result of how they defaecate and spit out '\n",
      " 'seeds during flight. They also observed that many fruit bats use one or more '\n",
      " 'feeding roosts each night and tend to deposit the vast majority of seeds '\n",
      " 'ingested far away from fruiting plants. Through this habit, bats help to '\n",
      " 'maintain species diversity by introducing seeds from outside disturbed '\n",
      " 'areas. Bats in general have further been identified as bioindicators through '\n",
      " 'their sensitivity to climate change, environmental degradation, and '\n",
      " 'contamination by agrochemicals and other toxins [22, 23] . Considering the '\n",
      " 'ecological importance of bats, vis-àvis their role as reservoirs of zoonotic '\n",
      " 'pathogens, a social management dilemma arises as to whether to encourage '\n",
      " 'cohabitation of humans with bats or to eradicate bats living close to '\n",
      " 'humans. According to Somphou et al. [24] , the lack of information regarding '\n",
      " 'the patterns of cohabitation with animals in urban areas hampers the design '\n",
      " 'of effective measures of disease prevention. The residents of Ve Golokuati '\n",
      " 'find themselves under this circumstance, which prompted this study. The '\n",
      " 'genesis of cohabitation of bats and humans in Ve Golokuati and the '\n",
      " 'complexities of this relationship in the township have not been documented, '\n",
      " 'neither has there been any disease surveillance in the area. This study '\n",
      " 'explored the various ways people come into contact with E. gambianus in Ve '\n",
      " \"Golokuati and the residents' perceptions of disease transmission risks from \"\n",
      " \"bats to humans. In general, the effects of settlement expansion on bats' \"\n",
      " 'habitats are not well understood particularly on a landscape scale. Gehrt '\n",
      " 'and Chelsvig [25] suggested that heterogeneous urban areas represent islands '\n",
      " 'of habitat for bats within larger landscapes. While some earlier studies '\n",
      " 'indicated that urbanization results in decreases in diversity and abundance '\n",
      " 'of bats [26] , others believe that the prevailing conditions and the '\n",
      " 'patterns of development in the urban periphery encourage bats to concentrate '\n",
      " 'in those areas [25, 27] . Urbanization can thus support bat-related '\n",
      " 'activities and enhance ecosystem services for humans. Ultimately, as most '\n",
      " 'emerging diseases exist within an intricately balanced host-agent continuum '\n",
      " 'among wildlife, domestic animals and human populations [28] changes to this '\n",
      " 'ecosystem can lead to changes in disease transmission potential. The '\n",
      " 'Epomophorus gambianus are generally found in woodland and savannah areas, '\n",
      " 'mostly in the tropical regions of Africa and Asia [29] . According to Fenton '\n",
      " '[30] , they roost in large hollow trees, thick foliage, and tree canopies '\n",
      " 'along stream banks and below the thatch of open sheds. They roost low in '\n",
      " 'trees during the day and are not disturbed by the presence of people [30] . '\n",
      " 'Apart from the nuisance created from noise [1] , not much has been '\n",
      " 'documented on how their proximity affects humans and, until recently, this '\n",
      " 'was not considered to be a problem. In addition, there have been no records '\n",
      " 'of any disease spillover from bats to humans in Ghana or in the study area. '\n",
      " 'The Ebola outbreak in parts of West Africa in 2014, as well as its link to '\n",
      " \"bats as a potential source of contagion [31] , has heightened people's \"\n",
      " 'concern [18] about bat-human interactions and their sense of risk. Thus '\n",
      " 'better understandings of the cohabitation of humans and wild animals and the '\n",
      " 'potential implications for human well-being are required urgently. The '\n",
      " '\"host-parasite continuum\" framework [28] , which suggests that wildlife and '\n",
      " 'domestic animals and human populations coexist and that disease spillover '\n",
      " 'occurs within a finely balanced host-agent continuum, provides the basis for '\n",
      " 'exploring bat-human interactions in this paper. The framework highlights the '\n",
      " 'ways in which underlying factors such as agricultural intensification, '\n",
      " 'translocation, and human encroachment are responsible for emerging '\n",
      " 'infectious diseases (EID). The study was carried out in Ve Golokuati, the '\n",
      " 'district capital of Afadzato South District ( Figure 1 ) between May 2013 '\n",
      " 'and April 2015. The district lies in the wet semiequatorial climatic zone of '\n",
      " 'Ghana, with annual rainfall values ranging between 1,016 mm and 1,210 mm. '\n",
      " 'The bimodal rainfall regime which used to start in April, with two peaks in '\n",
      " 'June and September, appears to be changing to a unimodal regime starting '\n",
      " 'from late April and ending in October. The area experiences an average of '\n",
      " 'four to five months of dry season. Temperatures are high throughout the '\n",
      " 'year, ranging from 26 ∘ C in the coolest months to about 32 ∘ C in the '\n",
      " 'hottest months. Mean monthly temperature is about 29 ∘ C [32] . The town is '\n",
      " 'located in the forest-savannah ecotone dominated by Guinea Savannah '\n",
      " 'Woodlands. However, anthropogenic drivers, notably land conversion through '\n",
      " 'rotational farming, lumbering, fuelwood extraction, and bush burning, have '\n",
      " 'degraded the forest cover considerably. The town has a population of about '\n",
      " '6,000 people, dominated by the Ewespeaking ethnic group of the Volta Region. '\n",
      " 'It is a \"nucleated\" settlement with houses built closely together without '\n",
      " 'any clear layout. Apart from the major roads that pass through the town from '\n",
      " 'east to west and north to south (see Figure 1 ), a few streets exist. Houses '\n",
      " 'and compounds are separated by winding footpaths. Residents share facilities '\n",
      " 'such as public places of convenience, water sources, and the open spaces '\n",
      " 'such as recreational fields. Epomophorus gambianus was the most dominant bat '\n",
      " 'species in the study area, but, in a few cases, small numbers of other '\n",
      " \"species such as the straw-coloured fruit bat (Eidolon helvum) and Peter's \"\n",
      " 'dwarf epauletted bat (Micropteropus pusillus) were observed. The M. pusillus '\n",
      " 'and E. gambianus both occur in the township and are distinguished mainly by '\n",
      " 'their facial looks and size, with M. pusillus being smaller. Also M. '\n",
      " 'pusillus roost mostly in the lower parts of the same trees as E. gambianus. '\n",
      " 'Residents believed that the two species were the same and that the '\n",
      " 'separation was by age, but ecological studies carried out as part of our '\n",
      " 'general studies on bats, which involved capturing of the bats, confirmed '\n",
      " 'that indeed they were two different species. According to the residents, the '\n",
      " 'bats migrated to Ve Golokuati in mid-to-late 1990s. It was, however, not '\n",
      " 'clear where the bats came from. Whereas some believed that the bats migrated '\n",
      " 'from Wli Water Fall area, a Wildlife Sanctuary located about 25 km from Ve '\n",
      " 'Golokuati, others were of the view that the bats came from nearby bushes '\n",
      " 'which were no longer habitable because of disturbances, notably incessant '\n",
      " 'bush fires and agricultural conversion. The congregation of bats in the '\n",
      " 'township, according to the residents, has been enhanced by the presence of '\n",
      " 'trees, particularly fruit trees, most of which have large crowns and provide '\n",
      " 'shade as well as other ecosystems services to residents. The most common '\n",
      " 'trees in this category, which also hosted colonies of bats, were mango '\n",
      " '(Mangifera indica) and neem (Azadirachta indica). The fruits of these trees '\n",
      " 'served as important sources of food for E. gambianus, although the bats are '\n",
      " 'known to feed on fruits such as guava (Psidium guajava), soursop fruit '\n",
      " '(Annona muricata), whistling pine fruit (Casuarina equisetifolia), cashew '\n",
      " '(Anacardium occidentale), and pawpaw (Carica papaya) [33] . Both '\n",
      " 'quantitative and qualitative data were used for the study. Quantitative data '\n",
      " 'were derived from questionnaire administration among 150 residents of the '\n",
      " 'study community through purposive sampling. Respondents targeted for '\n",
      " \"interview included those in households located close to bats' roosts and \"\n",
      " 'individuals who through their daily activities such as farming, trading, '\n",
      " 'recreation, and relaxation get exposed to bats either directly or '\n",
      " 'indirectly. Most of these individuals were identified through snowball '\n",
      " 'sampling where study subjects helped in the identification of other '\n",
      " 'subjects. Respondents ranged in age between 18 females and 41% males. The '\n",
      " 'questions posed to respondents centred mostly on activities that brought '\n",
      " 'residents close to bats and whether or not respondents experience fevers and '\n",
      " 'other strange diseases as a result of this exposure. Participatory appraisal '\n",
      " 'approach [34] was employed to solicit information on community perceptions '\n",
      " 'of human-bat interactions and whether or not there were any health related '\n",
      " 'issues as a result of such interactions. The approach involved participatory '\n",
      " 'mapping, interviews, a transect walk, and focus group discussions (FGDs). A '\n",
      " 'total of 13 elders and opinion leaders, ranging in age from 35 to 75 years, '\n",
      " 'participated in the mapping exercise. The selection was done with the help '\n",
      " 'of the Assemblyman for the area, who had a fair knowledge of individual '\n",
      " 'competences. The exercise was informative and served also as a guide in '\n",
      " \"determining the proximity of bats' roosts to residential areas and the \"\n",
      " 'selection of households for questionnaire administration. It also helped the '\n",
      " 'research team to design a transect line to facilitate the transect walk that '\n",
      " 'was undertaken following the mapping exercise. A six-member team including '\n",
      " 'one principal investigator undertook a transect walk from one end of town to '\n",
      " 'the other ( Figure 2 ). The walk was to identify the roosting and feeding '\n",
      " 'sites of bats, the proximity of residential areas to these sites, and '\n",
      " 'whether or not residents were exposed directly or indirectly to bats. During '\n",
      " 'the walk, major landmarks ca. 50-75 m astride the transect line were noted. '\n",
      " 'The exercise presented a further opportunity for the selection of households '\n",
      " 'and other target groups for interview. A total of 18 E. gambianus roosts and '\n",
      " '19 feeding sites were observed along the transect line. In addition, three '\n",
      " 'focus group discussions were conducted separately for men, women, and '\n",
      " 'opinion leaders of the town. Focus group discussions are a specific '\n",
      " 'technique that uses the services of a facilitator to moderate a small group '\n",
      " 'discussion. Each focus group discussion involved between eight (8) to 12 '\n",
      " 'participants. A total of 35 people comprising 15 men and 20 women '\n",
      " 'participated in the focus group discussions. The selection of participants '\n",
      " 'followed a similar process as what was used in participatory mapping. '\n",
      " 'Participants were between the ages of 25 and 76 and were made up of '\n",
      " 'residents who had lived in the town for more than 15 years. The discussions '\n",
      " 'were led by a facilitator and centred on the occurrence of bats in the town, '\n",
      " 'local perceptions and attitudes towards bats, and perceived risk of disease '\n",
      " 'spillover from bats to humans. On-site observation took the form of visits '\n",
      " \"to (i) bats' roosting sites to ascertain the means by which human and bats \"\n",
      " 'interact; (ii) the outskirts of town to determine the occurrence of bats '\n",
      " 'outside households where there was less protection; (iii) local market '\n",
      " 'centre to investigate whether or not bat meat was sold; (iv) public places '\n",
      " \"including the chief 's compound and premises of basic schools where trees \"\n",
      " 'colonized by bats provided shade. Journal of Environmental and Public Health '\n",
      " '5 In all the places visited, informal interviews were held randomly with '\n",
      " 'residents to understand human-bats interaction. On-site observation involved '\n",
      " 'also the identification of trees in which bats roost and fruit trees that '\n",
      " \"they feed on within the township. The researchers' own experience and local \"\n",
      " \"people's knowledge of native trees helped in the identification of the \"\n",
      " 'trees. The study also solicited information from health facilities on types '\n",
      " 'of common disease prevalent in the area. The research team had access to '\n",
      " 'information on the top 20 cases that accounted for out-patient morbidity for '\n",
      " 'the Afadjato North District and monthly out-patients morbidity returns for '\n",
      " 'selected months from 2012 to 2014 from the District Health Directorate and '\n",
      " 'the Ve Golokuati Health Centre, respectively. There was a follow-up visit to '\n",
      " 'the study area in April 2015, after the outbreak of Ebola virus in parts of '\n",
      " 'West Africa. During this visit, 27 community members including the chief and '\n",
      " 'elders of the town were engaged in a village square discussion [35] to find '\n",
      " 'out about their perceptions and relationship with bats after the Ebola '\n",
      " 'outbreak. Data analysis used descriptive statistical tools such as '\n",
      " 'statistical tables, graphs, and percentages to describe how the various '\n",
      " 'variables relate. Multivariate logistic regression analysis was carried out '\n",
      " 'to determine the association between experience of fevers and closeness of '\n",
      " \"respondent's activity to bats. The findings showed that about 48% of the \"\n",
      " 'residents were migrants from over 40 different towns and villages in the '\n",
      " 'Volta Region of Ghana. Among the respondents, 8% had tertiary education, 31% '\n",
      " 'secondary education, 42% junior secondary/middle school education, and 19% '\n",
      " 'little or no education. With regard to occupation, 75% of the respondents '\n",
      " 'were farmers, while the rest were engaged in petty-trading (8%), fashion '\n",
      " 'designing (3%), and other artisanal activities (2%). About 12%, mainly those '\n",
      " 'above 65 years and those below 20, claimed that they were unemployed. The '\n",
      " 'focus group discussions backed by on-site observation through transect walk '\n",
      " 'confirmed that human-bat interaction in the township was underpinned by the '\n",
      " 'solid greenery that characterised the landscape by virtue of assortment of '\n",
      " 'trees, which provided ecosystem services for humans and habitat and food for '\n",
      " 'bats. Ecosystems services provided by the trees to residents in the township '\n",
      " 'included food, medicine, shade, wind breaks, aesthetic beauty, and '\n",
      " 'enhancement of microclimate. Moreover, the fact that the bats were hardly '\n",
      " 'disturbed by residents made the township a protected niche for peaceful '\n",
      " 'coexistence of bats and humans. Though the exact number of the Epomophorus '\n",
      " 'gambianus in Ve Golokuati is not known, it was estimated that over 5,000 of '\n",
      " 'these bats lived with the people in the township. The occurrence of the '\n",
      " 'bats, according to some participants at the focus group discussion, was the '\n",
      " 'result of habitat disturbance, particularly wild bush fires and prolonged '\n",
      " 'droughts in the mid-1980s. Additionally, the annual bush burning which '\n",
      " 'precedes the farming season in the area, coupled with agricultural expansion '\n",
      " \"that had taken over bat's habitats, had also contributed to the migration of \"\n",
      " 'the bats into the township. Respondents believed that the population of bats '\n",
      " 'present in Ve Golokuati normally increased during the dry season (November '\n",
      " 'to January), which coincided with the fruiting of plants, and decreased in '\n",
      " 'the planting season (April to June). In spite of these seasonal '\n",
      " 'fluctuations, there has been net decrease in total bat population in the '\n",
      " 'town over the past decade from the perspective of respondents. The results '\n",
      " 'from the fieldwork showed that residents of the township came into contact '\n",
      " 'with bats directly or indirectly in various ways. These included living in '\n",
      " 'households with trees where bats roost, carrying out various vocations under '\n",
      " 'trees, hunting of bats, and eating bat bushmeat. With regard to households, '\n",
      " 'it was revealed that several of the residents carried out their daily chores '\n",
      " 'outdoors under trees colonized by E. gambianus (Figures 5 and 6 ). Household '\n",
      " 'chores that are noteworthy in relation to exposure to bats included food '\n",
      " 'preparation, dish washing and manual laundry activities, and serving and '\n",
      " 'eating food under trees. Participants in the focus group discussions '\n",
      " 'revealed that when fruits on mango and guava trees ripe, household members '\n",
      " 'may collect the partially eaten fruits discarded by bats as waste with their '\n",
      " 'bare hands. It was further revealed that several households had installed '\n",
      " 'rain harvesting facilities in the buildings to collect water for domestic '\n",
      " 'uses such as washing and cleaning. The discussants indicated that the '\n",
      " 'harvested rainwater was sometimes discoloured as a result of bat droppings, '\n",
      " 'yet they found nothing wrong with the water. Other daily activities that '\n",
      " 'brought people into contact with bats were economic activities including '\n",
      " 'farming and petty-trading at the market centre. On petty-trading, apart from '\n",
      " 'a few traders who had their wares displayed in stalls, most of the traders '\n",
      " 'plied their trade under mango trees, which incidentally also hosted large '\n",
      " 'colonies of bats. Foodstuffs were sold in their raw state on the bare '\n",
      " 'ground. All these activities invariably exposed both traders and buyers '\n",
      " \"constantly to bats' urine and faecal matter, as well as to irritating bat \"\n",
      " 'behaviours such as dropping of foliage as they move from one branch to the '\n",
      " 'other. During focus group discussion, one petty trader who operated under '\n",
      " 'one of the trees at the market reported the following: In some of the local '\n",
      " 'school premises, school pupils, according to participants at FGD, engaged in '\n",
      " 'outdoor recreational activities under trees that harbored several bat '\n",
      " \"colonies. Another vantage location of exposure to bats and to bats' \"\n",
      " \"activities was the chief 's palace, where mango trees providing shade for \"\n",
      " 'public gathering are colonized by E. gambianus.  The use of bats as food '\n",
      " 'brought people more directly into contact with bats than any other activity. '\n",
      " 'The sequence of events involved in preparing bat meat as food, according to '\n",
      " 'the findings, includes hunting, handling, butchering, dressing, smoking, and '\n",
      " 'cooking. In the process, people come into contact with the blood of bats in '\n",
      " 'addition to eating of the meat itself. With regard to the use of bats as '\n",
      " 'food, a multivariate logistic regression analysis showed that there was 98% '\n",
      " 'chance of more men eating bat bushmeat than women. This is statistically '\n",
      " 'significant and may be attributed to some cultural prohibitions on bat '\n",
      " 'consumption by pregnant women. For instance, the focus group participants '\n",
      " 'believed that bats have no rectum and, therefore, vomit their excreta. For '\n",
      " 'this reason, pregnant women were not allowed to eat bats; else they risked '\n",
      " 'transmitting the \"vomiting habit\" to their babies after birth. Among the '\n",
      " 'women respondents who claimed that they ate bat bushmeat, none of them had '\n",
      " 'done so when they were pregnant because of this belief, yet this did not '\n",
      " 'prevent women in general from eating bat bushmeat, as one middle aged woman '\n",
      " 'disclosed: Participants at FGD indicated that no bat meat was offered for '\n",
      " 'sale at the local market, though one could occasionally come across smoked '\n",
      " 'Eidolon helvum bushmeat in other markets such as Kpando, about 15 km away. '\n",
      " 'Investigation into the ages of people who hunted and ate bat bushmeat '\n",
      " 'revealed that it was mainly the adults that were involved ( Table 1 ). The '\n",
      " 'people of Ve Golokuati, however, would hardly eat E. gambianus but rather E. '\n",
      " 'helvum which used to be abundant in the area in the past but has become rare '\n",
      " 'locally. This would explain why most of the younger people did not eat bat '\n",
      " 'bushmeat, since the E. gambianus which were abundant at the time of the '\n",
      " 'study in the area are not appealing to most people because of their smaller '\n",
      " 'size. The results show further that only a small number of people across all '\n",
      " 'the ages hunted for bats in the area. However, with regard to eating of bat '\n",
      " 'bushmeat, people of age 45 and above, representing 16.7% of the total number '\n",
      " 'of respondents, were involved. This was followed by those between 36 and 45 '\n",
      " 'years representing 5.9%. However, none of the younger people (ages 15-25) '\n",
      " 'interviewed ate bat bushmeat. An analysis of the relationship between eating '\n",
      " 'of bat bushmeat and age was statistically insignificant (Chi square = '\n",
      " '11.835, df = 12, and = 0.459). Another way in which people came into contact '\n",
      " 'indirectly with bats was through the aerial movements of bats between '\n",
      " 'feeding and roosting sites. The bats reportedly moved on a daily basis '\n",
      " 'between 5:30 pm and 6:00 pm and again between 4:30 am and 5:00 am. During '\n",
      " 'these times when thousands of bats are in flight, the risk of contaminating '\n",
      " 'water harvesting systems by bat through aerosolization of bat droppings is '\n",
      " 'very high. The focus group discussion further revealed that domestic stock '\n",
      " '(e.g., sheep, goats, and poultry) also interacted both directly and '\n",
      " 'indirectly with the bats. Most of these animals were sheltered in pens and '\n",
      " 'coops within the households but were allowed to roam in search of food '\n",
      " 'during certain times of the day when they fed on foliage and fruits dropped '\n",
      " 'by bats (also see [36] ). Pets, notably cats and dogs, live within '\n",
      " 'households but are also left to roam about freely within the township to '\n",
      " 'hunt and scavenge for food. The threat of exposure to disease, through '\n",
      " 'human-bat interactions, is reinforced by the predatory behaviour of the pied '\n",
      " 'crow (Corvus albus) which targets the young E. gambianus just as they are '\n",
      " 'being weaned. According the respondents, the crows often left bat carcasses '\n",
      " 'which were often scavenged by dogs. This is noteworthy because, in the event '\n",
      " 'of any disease spillover, transmission via domestic animals cannot be '\n",
      " 'discounted. According to the findings, while most residents of Ve Golokuati '\n",
      " 'appreciated the ecological role of bats and were willing to live with bats, '\n",
      " 'others viewed them as awful animals whose activities, such as noise-making '\n",
      " 'and contamination of harvested rain water through defaecation and littering, '\n",
      " 'caused a lot of nuisance. Some of the residents only tolerated the presence '\n",
      " 'of the bats because of the hope that the bats would bring some benefits such '\n",
      " 'as ecotourism or research programmes to the community in the future and '\n",
      " 'create employment opportunities in the township. Figure 3 is a schematic '\n",
      " 'model that demonstrates the human-bat interaction in Ve Golokuati. The '\n",
      " 'figure illustrates the ecosystems services used by humans, activities of '\n",
      " 'bats that bring them into contact with humans, human actions and activities '\n",
      " 'that expose them to bats, and the potential dangers, as well as benefits of '\n",
      " 'these interactions. . Surprisingly, given their past experiences, the '\n",
      " 'constant exposure to bats, and the media coverage of bats and Ebola, the '\n",
      " 'residents of Ve Golokuati were strongly convinced that bats do not harbor '\n",
      " 'any diseases and that there has not been any disease spillover from bats to '\n",
      " 'humans. Their conviction came from the fact that there had not been any '\n",
      " 'strange disease outbreaks in the town since the arrival of the bats. Our '\n",
      " 'investigations into disease prevalence and common causes of human deaths in '\n",
      " 'the area during focus group discussions showed that malaria, locally known '\n",
      " 'as \"Nd}gbe,\" was the most common (note that there is a possibility of wrong '\n",
      " 'diagnoses as this assessment is based only on local perceptions). The '\n",
      " 'medical records obtained from the Out-Patient Department (OPD) of Ve '\n",
      " 'Golokuati Health Centre corroborate the finding from the focus group '\n",
      " 'discussions. common morbidity cases handled by the health centre were '\n",
      " 'malaria, anaemia, intestinal worms, acute respiratory tract infections, '\n",
      " 'diarrhoea, and skin diseases (Figure 4 ). The results indicated that, in all '\n",
      " 'cases, the frequency of disease was higher in the rainy season (June) than '\n",
      " 'in the dry season (December). A further investigation into disease records '\n",
      " 'from the District Health Directorate of the Afadzato South District showed '\n",
      " 'that there were no records of any health condition specifically associated '\n",
      " 'with bats in the district (Table 2) . Among the top 10 diseases recorded by '\n",
      " 'the District Directorate in 2013, malaria accounted for about 42% followed '\n",
      " 'by acute respiratory infection (18%) and anaemia (9.7%). Though the medical '\n",
      " 'records from health facilities in the area might not have recorded any '\n",
      " 'zoonotic diseases, one cannot discount misdiagnosis since state-of-the-art '\n",
      " 'equipment for diagnosing zoonoses may not be available in these facilities. '\n",
      " 'We assessed the experience of fevers among respondents in relation to the '\n",
      " 'closeness of their economic activities to bats. The analysis showed that the '\n",
      " 'various activities of the respondents were significantly associated with the '\n",
      " 'experience of fevers ( < 0.05). From a multivariate logistic regression '\n",
      " 'output (Table 3) , respondents whose activities brought them close to bats '\n",
      " 'were more likely to experience fevers than individuals whose activity were '\n",
      " 'not close to bats. Also farmers in the community who encountered activities '\n",
      " 'of bats such as feeding on their farms were more likely to contracting '\n",
      " 'fevers than those who do not encounter bats (odds ratio of 6.826 and '\n",
      " 'coefficient of 1.921). E. gambianus in the study area faced several threats, '\n",
      " 'such as habitat destruction, predation, and occasional hunting by humans for '\n",
      " 'bushmeat. The threats identified in this study confirm what the literature '\n",
      " 'summarizes as human disturbance and habitat destruction [37, 38] . These '\n",
      " 'threats were rooted in misinformation and incitement of people against the '\n",
      " 'existence of bats. The outbreak of Ebola in parts of West Africa and the '\n",
      " 'ensuing negative reportage about bats had led to the destruction of several '\n",
      " 'trees to get rid of bats in the township.  Interactions. The fundamental '\n",
      " 'question underpinning the presence of E. gambianus in the township of Ve '\n",
      " 'Golokuati is whether or not it is safe for people to live in close proximity '\n",
      " 'to bats. The question as to how long humans have been interacting with bats '\n",
      " 'in the township is also important as this may help in understanding the '\n",
      " 'disease dynamics as portrayed by the medical records. Indeed, the genesis of '\n",
      " 'occurrence of bats in the study area is not very certain, but research has '\n",
      " 'shown that, generally, bats relocate in response to a range of environmental '\n",
      " 'factors [39, 40] . According to Rebelo et al. [41] , climate change has the '\n",
      " 'potential to reduce the suitability of bat roosting sites, particularly tree '\n",
      " 'roosts. This corroborates the claims by respondents that habitat disturbance '\n",
      " 'caused by annual bush burning and agricultural conversion was the prime '\n",
      " 'cause of relocation of bats into the township. Cohabitation of humans and '\n",
      " 'bats, as was the case in Ve Golokuati, was possible because the township '\n",
      " 'contained trees that provided ecosystem services enjoyed by both humans and '\n",
      " 'bats and because the bats provided vital ecosystem services to humans. '\n",
      " 'Furthermore, the bats within the township were protected from threats and '\n",
      " 'other disturbances better than in the wild. In a number of locations in '\n",
      " 'Ghana where bats occur, they are under some form of protection for various '\n",
      " 'reasons such as traditional taboos, military presence, or deliberate '\n",
      " 'protection [42] . Human-bat interaction occurs for a number of reasons. '\n",
      " 'Hayman et al. [7] observed that such interaction occurs because humans are '\n",
      " \"encroaching on bats' habitats, just as bats are utilizing human structures \"\n",
      " 'as roosts. The interactions may be indirect, in the form of odour from '\n",
      " 'faecal droppings, urine, aerosolization of saliva, and glandular body '\n",
      " 'secretions [43] . It may also be direct through hunting and processing of '\n",
      " 'bat bushmeat for consumption [2, 10] . In Ve Golokuati, the findings showed '\n",
      " 'that human interaction with bats is both direct and indirect, which suggests '\n",
      " 'that the residents may be at risk to disease spillover. The \"host-parasite '\n",
      " 'continuum\" framework [28] , suggests that wildlife, domestic animals, and '\n",
      " 'human populations coexist and that disease spillover occurs within a finely '\n",
      " 'balanced host-agent continuum. The framework highlights the ways in which '\n",
      " 'underlying factors such as agricultural intensification, translocation, and '\n",
      " 'human encroachment are responsible for emerging infectious diseases. Though '\n",
      " 'only a few people ate bat bushmeat in the study area, this form of direct '\n",
      " 'interaction could easily spread disease pathogens among the people. '\n",
      " 'Shakespeare [44] discounted risk of disease transmission through eating of '\n",
      " 'bat bushmeat since the meat is normally well cooked but indicated that the '\n",
      " 'risk of transmission may occur during capture and slaughtering of bats, '\n",
      " 'during which time infection could occur through blood and body fluids or '\n",
      " 'bites and scratches. Currently there are no suspected cases of spillover of '\n",
      " 'disease from bats to humans in Ve Golokuati. Though this might be good news '\n",
      " 'to the residents, spillover of diseases cannot be completely ruled out since '\n",
      " 'no systematic surveillance has been done in the Ve Golokuati Health Centre, '\n",
      " 'which lacks the capacity in terms of personnel and equipment to do so. '\n",
      " 'Furthermore, most suspected cases of fevers are treated as \"malaria\" and '\n",
      " 'there is a possibility of misdiagnosis. In real terms, the official figures '\n",
      " \"and respondents' perceptions may overdiagnose malaria, while overlooking and \"\n",
      " 'underestimating the endemic zoonotic disease burden [45, 46] . While '\n",
      " 'attention is currently focused on bats and their potential to infect humans '\n",
      " 'with Ebola and other zoonotic diseases such as henipaviruses, the '\n",
      " 'limitations and difficulty to properly diagnose these diseases appear to be '\n",
      " 'off the public health radar. This therefore calls for great caution as '\n",
      " 'humans live with wildlife. The presence of bats in the Ve Golokuati Township '\n",
      " 'and their potential danger to humans in the face of emerging infectious '\n",
      " 'disease pose a serious conservation dilemma. The findings suggest that '\n",
      " 'residents of the township had been living with about 5000 E. gambianus over '\n",
      " 'the past two decades, and so far, there had been no suspected cases of '\n",
      " 'disease spillover from bats to human. However, the documented potential '\n",
      " 'epidemiological impacts of close human-bat interaction, particularly the '\n",
      " 'recent outbreak of Ebola virus in parts of West African, give cause for '\n",
      " 'concern. While getting rid of the bats by destroying the trees in Ve '\n",
      " 'Golokuati may not be environmentally and ecological prudent, the question of '\n",
      " 'how to reduce bat-human interaction to minimize the risk of any potential '\n",
      " 'future disease spillover still remains. A well-designed agenda for the '\n",
      " 'relocation of the bats to surrounding forested areas with the help of '\n",
      " 'experts from the Ghana Wildlife Division may offer a long term solution to '\n",
      " 'curtail the regular interactions and exposure of humans to bats. Yet, '\n",
      " 'relocating bats is not always easy, as demonstrated by the attempts at 37 '\n",
      " 'Military Hospital in the capital city of Accra, where thousands of bats '\n",
      " 'congregate (see [47] ). Nonetheless, establishment of a forest grove close '\n",
      " 'to the town, as a protected area, should be an option to facilitate a '\n",
      " 'possible relocation. This would provide a protected space for the bat '\n",
      " 'population and will, hopefully, decrease their frequent interaction with '\n",
      " 'humans. The initiative would also help to promote ecotourism and meet the '\n",
      " 'aspirations of the people. In the short term, the contamination of harvested '\n",
      " 'rainwater by bat urine and faeces can be minimized by delaying rainwater '\n",
      " 'harvesting after long spells of absence of rains. By doing so, bats '\n",
      " 'droppings on roof tops would be washed away before water harvesting starts. '\n",
      " 'Residents also need to be educated on the need to avoid drinking untreated '\n",
      " 'harvested rainwater. In order to reduce human exposures to bats, traditional '\n",
      " 'leaders including the chief and elders may have to enforce prohibitions on '\n",
      " 'hunting of bats to minimize disturbance and to create a safe haven for the '\n",
      " 'bats. In the long term, improved diagnostics and systematic surveillance of '\n",
      " 'zoonotic disease are required, coupled with regular health screening of the '\n",
      " 'residents of Ve Golokuati, as well as other communities in Ghana where bats '\n",
      " 'live in close proximity with humans. Finally, the Health Directorate '\n",
      " 'together with the District Assembly should sensitize the residents and, in '\n",
      " 'the future, possibly tourists coming to see the bats, on the potential '\n",
      " 'dangers of handling of bats, both dead and alive, and on appropriate ways to '\n",
      " 'promote hygienic and good health practices such as regular hand washing, '\n",
      " 'reporting all strange ailments promptly to medical centres and minimizing '\n",
      " 'direct contact with bats through hunting, handling, and eating. We recommend '\n",
      " 'future research on water quality of both harvested rain water in the '\n",
      " 'township and the nearby stream, which is the main source of potable water '\n",
      " 'for the township. The authors declare that there are no conflicts of '\n",
      " 'interest regarding the publication of this paper.',\n",
      " 'In the last decade there has been considerable interest and research in '\n",
      " 'addressing the needs and opportunities for on-site Nucleic Acid-based Tests '\n",
      " '(NATs) [1] [2] [3] [4] [5] [6] [7] [8] [9] [10] [11] [12] [13] [14] [15] '\n",
      " '[16] [17] [18] . These tests are also termed Nucleic Acid Amplification '\n",
      " 'Tests (NAATs) when incorporating sequence-specific enzymatic amplification '\n",
      " 'of nucleic acids such as PCR (polymerase chain reaction), or molecular '\n",
      " 'diagnostics when used in a medical context. Such in vitro diagnostic tests '\n",
      " 'implemented with low-cost, easy-to-operate, portable instruments and '\n",
      " 'miniaturized sample processing/analysis devices can be used outside of '\n",
      " 'medical laboratories for detecting pathogens in blood, urine, saliva and '\n",
      " 'other sample types. NAATs provide more timely results compared to '\n",
      " 'traditional culturing methods for diagnosis, and also work with the many '\n",
      " 'pathogens that cannot be cultured. More generally, pervasive, cheap, and '\n",
      " 'rapid NAATs will foster new paradigms for improved and sustainable medical '\n",
      " 'care (e.g., mobile healthcare using smartphones) and new tools for research '\n",
      " 'and discovery. For example, point-of-care (POC) diagnostics devices have '\n",
      " 'been developed for quantifying HIV viral load with finger-prick blood '\n",
      " 'samples. Closely related microfluidics technologies have been demonstrated '\n",
      " 'for food and water safety testing [19, 20] , rapid genetic tests [21] , '\n",
      " 'environmental monitoring [22] , surveillance for bioterrorism agents [23, '\n",
      " '24] , cancer screening [25] [26] [27] , assuring the health and hygiene of '\n",
      " 'laboratory animals [28] , analyzing biopsies [29, 30] , testing of livestock '\n",
      " 'and pets for parasites [31] , examining insects for genotype and infections '\n",
      " '[32, 33] , and first responder-administrated triage tests for stroke '\n",
      " 'biomarkers [34] . To date, most rapid tests for biomarkers are immunoassays '\n",
      " '[35] . The lateral flow strip test, or immunochromatographic assay, such as '\n",
      " 'found in home pregnancy tests, is an elegantly simple implementation of '\n",
      " 'immunoassays employed to detect antigens or antibodies in urine, blood, '\n",
      " 'food, and water. Molecular assays for DNA and RNA targets offer many '\n",
      " 'important advantages over immunoassays, including greater sensitivity '\n",
      " '(typically by a factor of 1000 or more) and higher specificity. The high '\n",
      " 'sensitivity is due to the ~million fold amplification of the target '\n",
      " 'sequence. The amplicons can be conveniently detected with intercalating dyes '\n",
      " 'or reporters that specifically label the amplicons, allowing measurements '\n",
      " 'with relatively simple fluorescence detectors such as CCD cameras or '\n",
      " 'photodiodes, or less commonly with electrochemical sensors. Further, unlike '\n",
      " 'immunoassays which rely on monoclonal antibodies and may require weeks or '\n",
      " 'months of development time, NAATs utilize nucleic acid oligos for '\n",
      " 'hybridization or as primers for enzymatic amplification, which can be '\n",
      " 'designed, synthesized, and validated in a short timeframe-in some cases as '\n",
      " 'short as one week-once the pathogen has been at least partially sequenced '\n",
      " '[36] . While immunoassays can be performed on crude, unprocessed samples '\n",
      " '(e.g., whole blood or raw food) at room temperature, more elaborate sample '\n",
      " 'processing and instrumentation is required for molecular diagnostics. In '\n",
      " 'particular, (1) virons and cells must be lysed to release nucleic acid '\n",
      " '(unless cell-free NAs are the target); (2) substances, such as heme in blood '\n",
      " 'specimens, that may inhibit downstream processes such as reverse '\n",
      " 'transcription and amplification, must be removed; (3) amplification reaction '\n",
      " 'temperatures must be closely regulated (typically within ±1 °C); (4) the '\n",
      " 'target NA must be concentrated so to reduce amplification reaction volume to '\n",
      " '10 to 50 µL in order to economize reagents and facilitate precise '\n",
      " 'temperature control and uniformity; and (5) amplicon detection schemes '\n",
      " 'should provide a statistically reliable means to discriminate between true '\n",
      " 'and false positives, and true and false negatives. Thus, in general, '\n",
      " 'although molecular diagnostics provide greater sensitivity and selectivity '\n",
      " 'than immunoassays, they require more sample preparation, more '\n",
      " 'instrumentation, more labor and expertise, and longer test times. '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'Fortunately, NAATs can be implemented in a microfluidics system for manual, '\n",
      " 'semi-automated, or fully-automated operation that can be readily adapted for '\n",
      " 'point-of-care use. A broad aim of microfluidics POC efforts is to make '\n",
      " 'molecular diagnostics nearly as easy to use as lateral flow strip '\n",
      " 'immunoassays. Microfluidics refers to various methods and technologies '\n",
      " 'wherein small quantities (1 nL to 1 mL) of liquids are manipulated, '\n",
      " 'processed, and analyzed using miniaturized, e.g., palm-sized, devices. In '\n",
      " 'biotechnology, microfluidic components and devices have been demonstrated '\n",
      " 'for, among other things, serial dilutions; cell fractionation, sorting, and '\n",
      " 'enrichment; cell and virus lysis; isolation, concentration, and purification '\n",
      " 'of nucleic acids; purification of proteins; immunoassays; reverse '\n",
      " 'transcription; labeling of biomolecules with reporters; enzymatic '\n",
      " 'amplification (e.g., PCR); electrophoresis; Micro total analytical systems '\n",
      " '(µTAS) combine these functions into an integrated sample-to-report device '\n",
      " 'for applications in medical diagnostics; environmental, food, and water '\n",
      " 'monitoring; detection of bioterrorism over conventional benchtop processing '\n",
      " 'agents, and as tools for research, discovery, and quality assurance. The '\n",
      " 'potential advantages of microfluidics include lower cost, faster results, '\n",
      " 'effective containment of hazardous or infection material, reduced '\n",
      " 'cross-contamination from other samples, better utilization of smaller '\n",
      " 'samples sizes, reduced consumption of reagents and reduced generation of '\n",
      " 'waste, portability, miniaturization, and ease of use for operation by '\n",
      " 'minimally trained people. Here, we review microfluidic \"lab on a chip\" (LOC) '\n",
      " 'technology integrating sample preparation and analysis for point-of-care '\n",
      " 'molecular diagnostics and other on-site NAATs. As such, microfluidic systems '\n",
      " 'can support and complement microarrays. While microarrays offer a high '\n",
      " 'degree of parallel processing for multiplex detection (hundreds or thousands '\n",
      " 'of targets), microfluidics can streamline and automate the serial sample '\n",
      " 'processing steps starting with crude, heterogeneous samples and yielding the '\n",
      " 'requisite concentrated and purified nucleic acids for enzymatic '\n",
      " 'amplification, microarray analysis, and sequencing. A 10-to 100-fold '\n",
      " 'concentration of target for amplification is often needed. For example, the '\n",
      " 'viral nucleic acid in 1 mL of blood plasma needs to be concentrated to a '\n",
      " 'volume of about 10 µL for amplification. The \"backend\" functions of LOC '\n",
      " 'molecular diagnostics systems which include amplification, labeling, and '\n",
      " 'detection are well established. However, the \"front end\" functions of '\n",
      " 'interfacing with the outside world through sample collection devices, as '\n",
      " 'well as sample preparation and processing (lysis and NA isolation) of '\n",
      " 'variegated heterogeneous samples are less developed. The \"backend\" functions '\n",
      " 'are generic and largely independent of sample type and application, but the '\n",
      " '\"front end\" functions need to be tailored to specific applications (e.g., '\n",
      " 'viral load measurements in blood vs. bacteria detection in food samples) '\n",
      " 'according to the required sample size determined by the concentration of '\n",
      " 'target and the sample and the desired Limits of Detection (LODs) [13] . '\n",
      " 'Other factors related to sample processing include the need for and '\n",
      " 'difficulty of lysis of tissue, cells and viruses containing the target NA, '\n",
      " 'the presence of inhibitors intrinsic to the sample or added to the sample to '\n",
      " 'effect lysis, and the stability of analytes, e.g., labile RNAs. Sample '\n",
      " 'processing steps may also include deactivation of nucleases, proteases, '\n",
      " 'infectious agents with reagents or heat. Some degree of multiplexing is '\n",
      " 'feasible in even simple microfluidic devices, and, as such, LOC devices may '\n",
      " 'be suitable as low-cost, convenient alternatives to microarrays when '\n",
      " 'multiplex detection of tens of targets suffices. This can be accomplished, '\n",
      " 'for example, by incorporating multiple amplification reactors on a single '\n",
      " 'chip, each containing primers for a specific target. The sample lysate is '\n",
      " 'split and loaded on the several membrane/chamber channels, which can be '\n",
      " 'imaged together using a CCD camera. Multiplexing in a single chamber would '\n",
      " 'require probes specific to a target and with a unique fluorescence emission '\n",
      " 'wavelength, combined with a color-sensitive imager. To support, supplement, '\n",
      " 'or complement microarrays, standardized or customized microfluidic devices '\n",
      " 'can be useful for rapid sample preparation and screening of samples where '\n",
      " 'labile targets may suffer degradation (especially mRNAs) or low yield (e.g., '\n",
      " 'microRNAs), quality control of reagents, detecting the presence of inhibitor '\n",
      " 'substances and other interfering components, before the samples are '\n",
      " 'subjected to more costly and time-consuming microarray analysis or '\n",
      " 'sequencing. Further, knowledge of sequences gained with microarrays can be '\n",
      " 'used to design amplification primers and probes for microfluidics-based '\n",
      " 'NAATs. Molecular diagnostics is comprised of the sequential steps of lysis '\n",
      " '(when the target nucleic acids are sequestered in cells or viruses); nucleic '\n",
      " 'acid isolation (extraction, concentration, and purification of DNA and RNA '\n",
      " 'from the lysate); reverse transcription for RNA targets, enzymatic '\n",
      " 'amplification of target DNA or RNA with sequence-specific primers; and '\n",
      " 'real-time or endpoint detection of amplification products. In the '\n",
      " 'laboratory, these unit operations are typically carried out in benchtop '\n",
      " 'instruments. First, the sample is incubated with lysing agents (salts, '\n",
      " 'detergents, enzymes, chaotropic agents), sometimes supplemented with '\n",
      " 'mechanical grinding, sonication, freeze-thaw, and heating, to release and '\n",
      " 'solubilize nucleic acids. Next, nucleic acids are isolated from the lysate, '\n",
      " 'commonly using a spin column kit for solid-phase extraction, where nucleic '\n",
      " 'acids are captured on a silica membrane. Lysing agents such as chaotropic '\n",
      " 'salts promote preferential binding of nucleic acids to silica glass or '\n",
      " 'cellulose fibers of the porous membrane. The membrane is then washed with '\n",
      " 'ethanol:water salt solutions to remove proteins and other debris. The '\n",
      " 'captured NA is then desorbed and eluted from the membrane with water. At '\n",
      " 'this stage, the captured NAs are concentrated and of sufficiently purity for '\n",
      " 'enzymatic amplification, typically by PCR in a thermal cycler instrument. In '\n",
      " 'the last five years, many new isothermal amplification methods, e.g., LAMP '\n",
      " '(Loop mediated AMPlification), helicase dependent amplification, NASBA '\n",
      " '(nucleic acid sequence-based amplification), and RPA (recombinase polymerase '\n",
      " 'amplification) have gained prominence for POC diagnostics [17] . '\n",
      " 'Traditionally, amplification products are detected and assessed by gel '\n",
      " 'electrophoresis, but nowadays, real-time monitoring of the amplification '\n",
      " 'process, using fluorescent intercalating dyes (SYBR ® green, SYTO ® green, '\n",
      " 'Evagreen ® ) or molecular probes (e.g., Taqman ® ) is preferred due '\n",
      " 'convenience and shorter protocols (obviating the need for post-amplification '\n",
      " 'analysis), the feasibility of relative quantification, and lower risk of '\n",
      " 'laboratory contamination by amplicons. Alternatives for detection of '\n",
      " 'amplification products include lateral flow strips analysis of amplicons '\n",
      " 'made with primers conjugated with antigens, or other real-time methods based '\n",
      " 'on monitoring various products or by-products of amplification such as '\n",
      " 'changes in pH, turbidity, electrical conductivity, phosphate or ATP '\n",
      " 'concentration, some of which can be monitored by luminescence reactions '\n",
      " 'using, e.g., luciferase/luciferin. Generally, this NAAT laboratory protocol '\n",
      " 'requires six or more hours, and is carried out by skilled technicians in '\n",
      " 'dedicated facilities equipped with hoods, centrifuges, temperature baths, '\n",
      " 'and benchtop thermal cycler instruments. The labile reagents (polymerases, '\n",
      " 'reverse transcriptases, proteases for lysis)-and often the sample too-need '\n",
      " 'to be maintained in a cold chain, and the laboratory work area and '\n",
      " 'procedures must minimize cross contamination of samples. The foregoing '\n",
      " 'benchtop procedure can be implemented with a credit-card sized microfluidic '\n",
      " 'cassette \"chip\", with total sample processing times of under one hour [18] . '\n",
      " 'A photo of a representative chip is shown in Figure 1 . The chip hosts three '\n",
      " 'parallel reaction chambers for the isothermal amplification of nucleic '\n",
      " 'acids. A porous cellulose (for DNA) or silica glass fiber (for RNA) '\n",
      " '\"membrane\" disc (~0.5-mm thick and 2 mm in diameter) is placed at the inlet '\n",
      " 'of each chamber. During the sample loading stage, NAs are captured on the '\n",
      " 'porous membrane solid phase, as sample lysate perfuses through the membrane. '\n",
      " 'The NAs bound on the membrane are then washed with ethanol-based solutions '\n",
      " 'to remove proteins and other contaminants. The chamber is then filled with '\n",
      " 'water and heated to the amplification temperature, typically 60-70 °C, '\n",
      " 'depending on the primer annealing temperatures and optimum temperature of '\n",
      " 'the polymerase enzyme. During the heating to amplification temperature, the '\n",
      " 'lyophilized reagents, enzymes, and dye that were pre-stored in the reaction '\n",
      " 'chamber are released and hydrated. By encapsulating the reagents in paraffin '\n",
      " 'that has a melting temperature below the amplification temperature, the '\n",
      " 'stored reagents are shielded from the sample loading and wash solutions that '\n",
      " 'flow through the chamber prior to amplification, and are reconstituted '\n",
      " 'during the heated amplification step after the chamber is filled with water. '\n",
      " 'The target NAs captured on the membrane are then amplified. RNA targets are '\n",
      " 'simultaneously reverse transcribed into cDNA templates by reverse '\n",
      " 'transcriptase, or an RNA/DNA dependent polymerase is used [37] . This '\n",
      " 'approach avoids the need for a separate elution step as include in the '\n",
      " 'benchtop spin column protocol. Specificity is dictated by appropriate design '\n",
      " 'of the amplification primers. As with benchtop real-time PCR, the increasing '\n",
      " 'amplicon concentration is monitored in real time during the amplification '\n",
      " 'process. The fluorescence emission can be detected with an inexpensive USB '\n",
      " 'microscope, photomultiplier, or smartphone (see Section 4). The '\n",
      " 'amplification time required for the emitted fluorescence intensity to cross '\n",
      " 'a specified threshold intensity is correlated with the concentration of '\n",
      " 'target NA in the sample. The chip performs multi tasks to simplify its '\n",
      " 'design, operation, and flow control and to reduce its cost. To summarize, '\n",
      " 'the chip combines the following unit operations in a single reactor: (1) A '\n",
      " 'porous silica or cellulose NA binding \"membrane\" is used to load sample '\n",
      " 'lysate in a flow-through mode, allowing NA to be isolated from large sample '\n",
      " 'volumes, such that the amplification volume is thus decoupled from the '\n",
      " 'sample size, which is crucial for detecting low abundance targets. This '\n",
      " 'feature is absent in many previously-reported POC NAAT devices; (2) in situ '\n",
      " 'amplification of NA captured on the membrane, which eliminates the need for '\n",
      " 'a separate elution step and simplifies flow control; (3) Isothermal '\n",
      " 'amplification such as loop mediated isothermal amplification (LAMP), which '\n",
      " 'simplifies the instrumentation compared to PCR and is less energy demanding '\n",
      " 'than thermal cycling. LAMP appears more tolerant of inhibitors compared to '\n",
      " 'PCR; (4) pre-storing in the amplification chamber paraffin-encapsulated, '\n",
      " 'lyophilized amplification reagents (polymerases, reverse transcriptases, '\n",
      " 'buffer salts, nucleotides, oligo primers, and DNA-binding dyes) further '\n",
      " 'reduces the number of flow control operations, thereby simplifying the chip. '\n",
      " 'Further, use of lyophilized reagents relieves both the need to add reagents '\n",
      " 'at the time of use, and the need for any cold storage of reagents. As an '\n",
      " 'added benefit, the method provides a \"hot start\" to amplification in that '\n",
      " 'reagents are released only once the reactor has reached its operating '\n",
      " 'temperature, reducing non-specific amplification. Moreover, it is feasible '\n",
      " 'to store different reagents with different primer sets in separate parallel '\n",
      " 'reactors, enabling the detection of multiple targets as well as providing '\n",
      " 'control and calibration reactors; (5) Our chips enable real-time detection '\n",
      " 'of the amplicon production, which provides more information than merely '\n",
      " 'end-point detection, and allows one to terminate the test based on the '\n",
      " 'target concentration, i.e., when targets are available in high abundance, '\n",
      " 'the test can be concluded in as little as 10-15 min; (6) The fluorescence '\n",
      " 'emission can be detected with a CCD camera which readily enables concurrent '\n",
      " 'monitoring of a large number of reactors and the utilization of smartphone '\n",
      " 'cameras for cost reduction. Figure 1 . Chip using single-chamber for NA '\n",
      " 'isolation, isothermal amplification, and real-time detection. Photo of chip '\n",
      " 'with three parallel chambers for multiplex detection (inset). Molecular '\n",
      " 'diagnostics process flow schematic (right): 1 Sample is mixed with '\n",
      " 'lysis/binding reagent (guanidium HCl), 2 sample is injected into chip, 3 '\n",
      " 'ethanol wash is injected into chip, washing NA (nucleic acid) capture '\n",
      " 'membrane (silica glass fiber or cellulose disc embedded in chip), 4 chamber '\n",
      " 'is filled with water and sealed. 5 chamber is heated to amplification '\n",
      " 'temperature (65 °C), melting paraffin encapsulation layer and reconstituting '\n",
      " 'lyophilized enzymes, reagents, and DNA intercalating fluorescent dye, 6 blue '\n",
      " 'light excitation from LED generates fluorescent signal (proportional to '\n",
      " 'amount of DNA amplicon) detected through filter (to block excitation light) '\n",
      " 'by CCD camera (Smarthphone). As a specific application example, we used the '\n",
      " 'chip described above to assess viral load in plasma samples (see below). '\n",
      " 'Several hundred microliters of plasma were mixed with an equal part of '\n",
      " 'lysing buffer (6 M guanidinium HCl), briefly incubated (~10 min), and '\n",
      " 'pipetted into the chip, through the embedded nucleic-acid isolation '\n",
      " 'membrane. Next, 100 µL of ethanol:water (50:50) was pipetted into the chip '\n",
      " 'to wash the membrane. Finally, the chip amplification chamber was filled '\n",
      " 'with water. The chip inlet and outlet ports were sealed with tape, and the '\n",
      " 'chip was then inserted into a portable palm-sized instrument with a heated '\n",
      " 'stage and mounting fixture for a cellphone. Figure 2 shows the results for '\n",
      " 'plasma samples spiked with three different concentrations of HIV virons plus '\n",
      " 'a negative control, indicating a LOD (limit of detection) of 350 copies per '\n",
      " 'mL of plasma sample.  Alternatives to real-time monitoring of the '\n",
      " 'amplification process for quantifying nucleic acids that are less demanding '\n",
      " 'of imaging and computational capabilities could prove more amenable to '\n",
      " 'low-cost POC diagnostics in resource-limited settings and may open new '\n",
      " 'opportunities for NAATs for identification of bacterial flora and gene '\n",
      " 'expression profiles. One such method that has been recently developed by our '\n",
      " 'group consists of an amplification-diffusion conduit (ADC), which provides a '\n",
      " 'spatial indication of the concentration of target nucleic acids in the '\n",
      " 'sample (Figure 3 ) [38] . This \"nuclemeter\" chip, shown in Figure 3a , hosts '\n",
      " 'four ADCs to illustrate a serial dilution effect of target concentration on '\n",
      " 'device operation. Briefly, the ADC extends from the embedded silica membrane '\n",
      " 'that captures NA, as described above, and which provides the source of '\n",
      " 'template NA for amplification and diffusion. The ADCs are typically 330 µm '\n",
      " 'wide × 330 µm deep × 12 mm long and are pre-filled with a stabilized gel '\n",
      " '(hydroxypropyl)methyl cellulose (HPMC) to retard diffusion and with all the '\n",
      " 'ingredients needed for the amplification reaction, sans the target itself. '\n",
      " 'The ADC is maintained at the amplification temperature (65 °C for the LAMP '\n",
      " 'process). As the target diffuses into the conduit, it amplifies. Eventually, '\n",
      " 'the amplification process runs its course (i.e., saturates). After a short '\n",
      " 'time from the beginning of the process, the reaction-diffusion conduit can '\n",
      " 'be viewed as consisting of two regions: A region in which the reaction has '\n",
      " 'been completed and a region into which the target nucleic acids have not yet '\n",
      " 'diffused and the amplification reaction has not yet started. When an '\n",
      " 'intercalating dye is used, the first region of the ADC, in which the '\n",
      " 'reaction has been completed, emits fluorescence and appears as a column of '\n",
      " 'light, while the second region, in which the amplification reaction has not '\n",
      " 'yet occurred, appears dark (Figure 3b ) which can be imaged with a '\n",
      " 'Smartphone camera (Figure 3c) . The two regions are separated by a '\n",
      " 'relatively sharp reaction front. Once the initial reaction region has '\n",
      " 'saturated, the reaction front moves with a constant velocity v (Figures '\n",
      " '3d-1,d-2 ). That is, the position of the front is given by the equation t0) '\n",
      " \". X0 is the front location when the amplification process near the ADC's \"\n",
      " 'entrance has saturated and t is time (Figure 3d (Figure 3d-4) , it serves a '\n",
      " 'similar role to that of the threshold time in real time machines. ADCs '\n",
      " 'operating with known quantities of target molecules can serve to calibrate '\n",
      " 'the length of the fluorescent column to enable quantification. The emission '\n",
      " 'from multiple columns can be conveniently recorded with a smart phone '\n",
      " 'camera. Microfluidic devices can leverage smartphone technology to image the '\n",
      " 'chip and detect the increasing level of fluorescence due to the '\n",
      " 'amplification process [39] [40] [41] [42] , as well as for computation and '\n",
      " 'control functions, communication of test results (including GPS), data '\n",
      " 'logging, and connect to sources of medical advisory information to the '\n",
      " 'tested person. In one implementation [42] , a blue LED (@ 20 mA current) '\n",
      " 'excites the DNA-binding fluorophore, and the longer-wavelength fluorescence '\n",
      " 'is detected by the CCD camera after optically filtering out the excitation '\n",
      " 'light. The CCD camera of a smartphone proves to be an adequate, and '\n",
      " 'widely-available, low-cost detector of green fluorescence (using SYTO green, '\n",
      " 'EvaGreen™ dye, or similar DNA intercalating dyes) for real-time monitoring '\n",
      " 'of amplification on the chip. The digitized image of the amplification '\n",
      " 'chamber, generated by custom APP (mobile application) software, can '\n",
      " 'quantitatively assess the increasing intensity of fluorescence during the '\n",
      " 'course of amplification to determine a threshold time which correlates with '\n",
      " 'the starting NA template concentration captured on the membrane. Further, '\n",
      " 'many parallel chambers can be simultaneously imaged for multiplex detection, '\n",
      " 'including several analytes, positive and negative controls, and calibration '\n",
      " 'standards. Figure 4 shows a fixture for mounting the cellphone over the chip '\n",
      " 'docked on a custom made, electricity-free heating stage. d Figure 3 . '\n",
      " 'Nucleometer chip. (a) The chip having a similar structure to that described '\n",
      " 'in Figure 1 . The amplification-diffusion column is pre-filled with all the '\n",
      " 'ingredients needed for the amplification process and a polymer to reduce '\n",
      " 'diffusion rate; (b) As the target diffuses into the column it amplifies. '\n",
      " 'After a short time that depends on initial target concentration, the column '\n",
      " 'consists of two regions. A region in which the reaction has been completed '\n",
      " '(column of light) and a region in which the reaction has not yet started '\n",
      " '(dark region); (c) The test results can be monitored, documented, and '\n",
      " 'transmitted with a smartphone. The front position of the fluorescence region '\n",
      " 'is extended as a function of time and can be used to estimate target (NA '\n",
      " 'template) concentration; (d-1) relative concentration of amplicon as '\n",
      " 'function of distance along the conduit, as determined by fluorescence; (d-2) '\n",
      " 'relative concentration of amplicon along the conduit, as determined by '\n",
      " 'fluorescence, as a function of time; (d-3) distance along conduit (XF) as a '\n",
      " 'function of time for different starting template concentrations (copies); '\n",
      " '(d-4) threshold time of fluorescence region extension vs. RNA template copy '\n",
      " 'number [39] . . Smartphone mounted on portable instrument that provides the '\n",
      " 'chip with temperature-regulated heating, optical excitation with a blue LED, '\n",
      " 'a filter to block the excitation light from the smartphone camera and '\n",
      " 'detection. The heating is provided with an exothermic reaction in which a '\n",
      " 'magnesium alloy powder interacts with water. The temperature is regulated '\n",
      " 'with a phase change material (PCM, e.g., paraffin) that acts as a ballast to '\n",
      " 'maintain the reaction chambers at the PCM melting temperature, since excess '\n",
      " 'heat production is absorbed as latent heat, maintaining the reaction chamber '\n",
      " 'at the PCM melting temperature [42] . Molecular tests for bloodborne '\n",
      " 'pathogens are usually based on detection of NA biomarkers in cell-free '\n",
      " 'plasma. Whole blood sample processing is problematic due to cells clogging '\n",
      " 'the NA binding membrane, heme from lysis of red blood cells inhibiting '\n",
      " 'enzyme action, and genomic DNA from white blood cells, for example, '\n",
      " 'containing integrated retroviruses, confounding and complicating test '\n",
      " 'results. Furthermore, prevailing standards for viral loads are based on '\n",
      " 'virus counts in plasma rather than in whole blood. In a laboratory setting, '\n",
      " 'the cell-free plasma component of blood is typically prepared as the '\n",
      " 'supernatant of centrifuged (e.g., 10,000 rpms for 10 min) whole blood. Many '\n",
      " 'POC venues do not have access to centrifuges, and therefore a simpler, '\n",
      " 'instrument-free method for extracting plasma from whole blood samples is '\n",
      " 'highly desired. Various microfluidic approaches for POC plasma extraction '\n",
      " 'often involve supporting equipment such as electric motors and actuators, or '\n",
      " 'vacuum pumps. Non-instrumented microfluidic devices rely on capillary flow '\n",
      " 'or filtration, but these techniques either have low yield (volume of plasma '\n",
      " 'extracted/volume of blood sample) or are limited to small blood sample sizes '\n",
      " '(~10 µL). In our work [43] , we have developed a plasma extraction device '\n",
      " 'based on combination of gravitational sedimentation and filtration through a '\n",
      " 'porous membrane to achieve practical performance levels for field use '\n",
      " '(figure 5). Our device is a rectangular cartridge (2 cm × 1 cm × 4 cm) with '\n",
      " 'a vertically oriented center chamber open at the top, and in which two '\n",
      " 'opposing sidewalls featuring arrays of micropillars (300 µm × 300 µm in '\n",
      " 'cross section, and 100 µm tall), machined or stamped, support polysulfone '\n",
      " 'membranes. The membrane has asymmetric pores (larger diameter facing the '\n",
      " 'center chamber) such that blood cells are gently trapped without lysis. The '\n",
      " 'advantage of our approach is in allowing cells to sediment to the bottom of '\n",
      " 'the chamber while plasma effuses through the vertical membranes, trickles '\n",
      " 'trough the pillar arrays into trench channels, from where it can be '\n",
      " 'collected by a pipette. The advantage of our approach is in reducing the '\n",
      " 'amount of cells that come into contact with the separation membrane, thereby '\n",
      " 'increasing separation membrane capacity. For example, 1.8 mL of whole blood '\n",
      " 'loaded into this device yielded nearly 300 µL of plasma in 7 min. In a newer '\n",
      " 'version of the device (not shown), we have nearly tripled the yield. An '\n",
      " 'assay for heme in the extract indicated negligible lysis of red blood cells. '\n",
      " 'Quantitative PCR of the plasma extracted from whole blood spiked with HIV '\n",
      " 'indicated that the plasma is of sufficient quality for enzymatic '\n",
      " 'amplification and that the virus recovery is over 80%. Figure 2 (shown '\n",
      " 'earlier) provides amplification results of a serial dilution sensitivity '\n",
      " 'data for HIV LAMP in the chip, using plasma samples extracted with our '\n",
      " 'plasma separator as described above.  The microfluidic devices described '\n",
      " 'above can be fabricated by several prototyping methods [44] . Indeed, this '\n",
      " 'is one of the advantages of microfluidics: Custom design and rapid '\n",
      " 'prototyping of components within a timeframe of one to two days is feasible. '\n",
      " 'Chips are made as bonded laminate structures from plastic sheet materials, '\n",
      " 'including PMMA (polymethylacrylate, \"acrylic\"), polycarbonate, polystryrene '\n",
      " 'or cyclic olefin copolymer. The fluidic circuits are drawn using computer '\n",
      " 'aided design (CAD) software (Solidworks™ or Autocad™). A middle layer is '\n",
      " 'patterned with the fluidic circuit by a CNC milling machinine, a desktop '\n",
      " 'engraver, or a CO2 laser cutter. Alternatively, the chip can be directly '\n",
      " 'formed using a 3D printer with clear acrylic-like polymer alloys. The '\n",
      " 'smallest feature size (i.e., channel width) is about 0.3 mm. Smaller depths '\n",
      " 'can be obtained with thin laminates. The component sheets are aligned and '\n",
      " 'bonded using adhesives, solvent bonding, thermal pressure bonding, or '\n",
      " 'ultrasonic welding. Both resistive heating or Peltier cooling modules can be '\n",
      " 'used for temperature regulation, along with a thermocouple temperature '\n",
      " 'sensors. The power consumption is on the order of 1 Watt. A commercial '\n",
      " 'temperature controller or microntroller can be used to regulate the '\n",
      " 'temperature, or even simpler analog electronics control circuits are '\n",
      " 'sufficient. Reagents are from commercial LAMP kits (Eiken Chemical, Tokyo, '\n",
      " 'Japan). Freeze-dried lamp reagents (including primers and fluorescent dyes) '\n",
      " 'as lyophilized spheres can be preloaded into the chip during assembly to '\n",
      " 'simplify operation. The feasibility and utility of microfluidic '\n",
      " 'lab-on-a-chip implementations of component subprocesses for NAATs, including '\n",
      " 'plasma extraction, lysis, NA isolation, amplification, and real-time '\n",
      " 'detection and end-point quantification, have been well demonstrated for '\n",
      " 'representative sample types and varied applications. POC devices and '\n",
      " 'operation are significantly simplified by (1) a multifunctional reaction '\n",
      " 'chamber that includes an embedded membrane for the capture of DNA and RNA, '\n",
      " 'storage of reagents, and the in situ amplification of NA on the membrane '\n",
      " 'which circumvents the need for a separate elution step and decouples sample '\n",
      " 'size from amplification reaction volume; and (2) isothermal (constant '\n",
      " 'temperature) amplification such as LAMP (Loop mediated amplification) with '\n",
      " 'real-time detection, using for example, a smartphone CCD camera as detector. '\n",
      " 'When blood samples are used, the hybrid sedimentation-filtration plasma '\n",
      " 'extraction device can replace centrifugation enabling plasma separation from '\n",
      " 'whole blood specimens at the point of care. The plasma separator can be '\n",
      " 'integrated with the nucleic acid amplification chip for seamless operation. '\n",
      " 'POC molecular diagnostics devices were first reported in the mid-1990s [45] '\n",
      " ', single-chamber PCR chips made in glass or silicon. In the last twenty '\n",
      " 'years, these chips have evolved to more viable polymer materials, have '\n",
      " 'integrated sample prep functions, included on-chip storage of reagents and '\n",
      " 'buffers, and accommodate low-cost detection schemes, such as smartphone '\n",
      " 'cameras, and as such now offer convenient, low-cost diagnostics suitable for '\n",
      " 'use outside of traditional clinical laboratories, including resource-limited '\n",
      " 'settings throughout the world. We also describe a new paradigm in target '\n",
      " 'nucleic acid quantification through the use of a chip-based '\n",
      " 'amplification-diffusion channel. This method facilitates end point detection '\n",
      " 'and enables the estimation of target (NA template) concentration from the '\n",
      " 'lengths of emitting columns, akin to reading temperature in a mercury '\n",
      " 'thermometer from the length of the mercury column. The various modules '\n",
      " 'described herein can be combined into total microanalytical systems for '\n",
      " 'sample-to-report POC diagnostics devices. In this review, we focused on NAAT '\n",
      " 'of HIV in blood, suggesting the application of similar designs for other '\n",
      " 'nucleic acid based tests where time, convenience, cost constraints would '\n",
      " 'preclude the use of a microarrays, while still achieving multiplexing '\n",
      " 'capability and quantification of both RNA and DNA targets with only modest '\n",
      " 'or no instrumentation. We use a solid-phase extraction for NA isolation '\n",
      " 'based on the common chaotrope-silica method used in commercial spin columns. '\n",
      " 'Related NA isolation methods based on ChargeSwitch ® Technology (CST) and '\n",
      " 'solid-phase reversible immobilization (SPRI) are also potential '\n",
      " 'alternatives. These point-of care molecular diagnostics devices complement '\n",
      " 'microarrays and provide an alternative to expensive laboratory-based '\n",
      " 'instruments. As a comparison, commercial PCR-based and related systems, such '\n",
      " 'as the COBAS AmpliPrep/Taqman HIV-1 (Roche), VERSANT HIV-1 RNA (Siemens), '\n",
      " 'RNA QT (bioMerieux), and RealTime m2000 HIV (Abbott) can process close to '\n",
      " '100 samples in a two-hour time span. Limits of detection typically range '\n",
      " 'from 50 to 100 copies/mL plasma [46] . These instruments cost over $100,000 '\n",
      " 'and their use is restricted to modern clinical laboratory facilities.',\n",
      " 'Coronaviruses (CoVs) cause human disease with symptoms ranging from mild '\n",
      " 'respiratory symptoms to severe pneumonia [1] . In September 2012, a novel '\n",
      " 'CoV termed the Middle East Respiratory Syndrome (MERS)-CoV emerged on the '\n",
      " 'Arabian Peninsula with 163 laboratory-confirmed cases including 71 deaths so '\n",
      " 'far (World Health Organization, December 2nd, '\n",
      " 'http://www.who.int/csr/don/2013_12_ 02/en/index.html). Due to its '\n",
      " 'distribution in several countries of the Arabian Peninsula there is a risk '\n",
      " 'of global spread through travel and pilgrimage, posing a potential threat to '\n",
      " 'global public health. The clinical picture of MERS-CoV infection is '\n",
      " 'characterized by acute atypical pneumonia and respiratory failure, '\n",
      " 'resembling symptoms caused by SARS-CoV [2] [3] [4] [5] . Acute renal failure '\n",
      " '(ARF) was described in a number of MERS cases, with potential influence on '\n",
      " 'disease severity [2, 3, [5] [6] [7] . ARF has neither been a typical feature '\n",
      " 'of SARS, nor has it been commonly observed in infections caused by any other '\n",
      " 'CoV, including human HCoV-NL63, -229E, -OC43, and -HKU1. Of note, the '\n",
      " 'majority (76%) of MERS-CoV patients were reportred to have underlying '\n",
      " 'medical conditions such as diabetes, chronic cardiac disease and chronic '\n",
      " 'renal disease [4, 8] . It has been shown recently that MERS-CoV enters '\n",
      " 'target cells not via the SARS-CoV receptor Angiotensinconverting-enzyme-2 '\n",
      " '(ACE-2), but via binding to Dipeptidyl-peptidase 4 (DPP-4) [9, 10] . Both, '\n",
      " 'ACE-2 and DPP-4 are expressed in several human tissues, including the kidney '\n",
      " '[11, 12] . However, kidney infection has not been compared between MERS-CoV '\n",
      " 'and SARS-CoV, and no post-mortem investigations on patients who succumbed to '\n",
      " 'MERS-CoV have been done. To assess the potential role of kidney affection in '\n",
      " 'MERS-CoV infection, we reviewed all peer-reviewed or communicated reports on '\n",
      " 'MERS cases for information on kidney involvement. The search was performed '\n",
      " 'in PubMed and Promed.Mail (http://www.promedmail.org) using the search term '\n",
      " '\"coronavirus\" on items published between 15th September 2012 and 16th '\n",
      " 'September 2013. The broad search term was chosen due to inconsistencies of '\n",
      " 'nomenclature before the consensus name MERS-CoV was announced [13] . In '\n",
      " 'total, the search revealed 655 publications, including 508 articles in '\n",
      " 'PubMed, and 147 reports in ProMed ( Figure 1 ). All identified publications '\n",
      " 'were reviewed in order to identify clinical descriptions of MERS-CoV '\n",
      " 'infection that included information on kidney function, irrespective of '\n",
      " 'whether they were pointing to normal or impaired function. In cases where '\n",
      " 'two reports on the same patient were available, only the report with the '\n",
      " 'more detailed clinical description was selected. From all identified '\n",
      " 'reports, we extracted information on patient sex and age, country of patient '\n",
      " 'origin and travel route, onset of illness, presence or absence of ARF, date '\n",
      " 'of onset of ARF, day of illness on which ARF occurred, as well as clinical '\n",
      " 'outcome (death or survival). Among 21 publications describing a total of 111 '\n",
      " 'MERS-CoV patients (doublereporting of the same patients could not be '\n",
      " 'excluded), a total of 7 publications were identified that specifically '\n",
      " 'addressed kidney function [2, 3, [5] [6] [7] 14, 15] . In these 7 '\n",
      " 'publications, a total of 12 MERS patients were described with disease onset '\n",
      " 'dates ranging from April 2012 to March 2013 ( Table 1 ). All patients were '\n",
      " 'male with a mean age of 47 years (range 25-73 years), originating from '\n",
      " 'Jordan, Saudi Arabia, and Qatar, including imported cases to the United '\n",
      " 'Kingdom (UK), Germany and France. Nine of those 12 patients (75%) developed '\n",
      " 'ARF during the acute course of MERS-CoV infection. For 6 of those 9 '\n",
      " 'patients, dates of onset of ARF were available, suggesting a median time of '\n",
      " '11 days (SD ±2,0 days) from first symptoms to onset of Figure 1 Flowchart of '\n",
      " 'study selection. Systematic search strategies were applied to identify all '\n",
      " \"reports on patients' kidney function during MERS-CoV infection. Data sources \"\n",
      " 'used were Pubmed (www.ncbi.nlm.nih.gov/pubmed/) and Promed.Mail '\n",
      " '(http://www.promedmail.org) by searching the term \"coronavirus\" with a date '\n",
      " 'range from 15th September 2012 (date of the first MERS-CoV report) to 16th '\n",
      " 'September 2013. All identified publications were reviewed for clinical data '\n",
      " 'on kidney function (irrespective of pointing to normal or impaired '\n",
      " 'function). Data on patient sex and age, country of patient origin and travel '\n",
      " 'route, onset of illness, presence or absence of ARF, date of onset of ARF, '\n",
      " 'day of illness on which ARF occurred, as well as clinical outcome (death or '\n",
      " 'survival) were extracted. In total, 7 publications were finally included '\n",
      " 'that described a total of 12 MERS-CoV patients and their kidney function '\n",
      " 'during the course of disease. ARF. Six of 9 (67%) patients with ARF had a '\n",
      " 'fatal outcome, while one of three patients without ARF died. However, the '\n",
      " 'differences found for the fatality rates in cases with and without ARF '\n",
      " 'during MERS-CoV infection were not statistically significant. To investigate '\n",
      " 'a laboratory surrogate for susceptibility of the kidney to MERS-CoV in '\n",
      " 'comparison to SARS-CoV, we analyzed cryoslides of a healthy human kidney for '\n",
      " 'ACE-2 and DPP-4 expression by immunofluorescence. Staining using polyclonal '\n",
      " 'goat-anti-human ACE-2 immunoglobulin (R&D systems) and polyclonal '\n",
      " 'goat-anti-human DPP-4 immunoglobulin (R&D systems) showed that both '\n",
      " 'receptors are abundantly expressed in the healthy human kidney, and both '\n",
      " 'receptors are predominantly found in the epithelial layer of the renal ducts '\n",
      " '(Figure 2A ), as well as in human primary epithelial cells derived from '\n",
      " 'these ducts and in epithelial cells from the human airway tract ( Figure 2B '\n",
      " '). To test whether kidney epithelial cells can actually be infected '\n",
      " 'in-vitro, we performed comparative infection experiments with MERS-and '\n",
      " 'SARS-CoV, using defined cultures of primary kidney epithelial cells (HREpC, '\n",
      " 'Promocell, Heidelberg). As the bronchoalveolar epithelium of the lung '\n",
      " 'constitutes the primary target compartment for both viruses, infection of '\n",
      " 'human primary bronchial epithelial cells (HBEpC, Promocell, Heidelberg) was '\n",
      " 'studied in parallel. As summarized in Figure 2 , primary cells up to passage '\n",
      " '4 were seeded at densities of 4x10 5 /mL and inoculated with either MERS-CoV '\n",
      " 'strain EMC/2012, or SARS-CoV strain Frankfurt-1 at multiplicities of '\n",
      " 'infection (MOI) of 0.5 for 1 hour. Cells were washed twice and supernatants '\n",
      " 'were harvested immediately after washing, as well as 20 and 40 hours later. '\n",
      " 'Cells were checked regularly for appearance of cytopathic effects (CPE) by '\n",
      " 'light microscopy. Replication of both viruses was quantified by titration of '\n",
      " 'supernatants in duplicate plaque assays on Vero cells as described '\n",
      " 'previously, as well as by real-time RT-PCR [9, 16] . Figure 1D ). The CPE in '\n",
      " 'HREpCs consisted of rounding and detachment of cells, leading to cell death '\n",
      " 'in the majority of cells already after 20 hours. Comparative quantification '\n",
      " 'of infectious virus production between SARS-and MERS-CoV showed that '\n",
      " 'MERS-CoV replicates to high titers in HREpC with almost 1000-fold higher '\n",
      " 'concentrations of infectious MERS-CoV progeny than in HBEpCs. In SARS-CoV '\n",
      " 'infection, only a small difference between primary kidney and bronchial '\n",
      " 'epithelial cells was seen ( Figure 1E ). Our results from in-vitro infection '\n",
      " 'experiments suggest differences in kidney cell involvement might exist '\n",
      " 'between SARS-and MERS-CoV infection. In a retrospective analysis of 536 SARS '\n",
      " 'cases, only in 36 (6.7%) patients acute renal impairment was diagnosed. ARF '\n",
      " 'occurred rather late in the course of disease after a median of 20 days from '\n",
      " 'the appearance of first symptoms [17] . In our literature review of MERS '\n",
      " 'cases, ARF occurred much earlier, namely in 6 of 9 sufficiently-described '\n",
      " 'patients after a median of 11 days. To date, it is difficult to compare '\n",
      " 'these clinical data with experiences from animal models. The only animal '\n",
      " 'that exhibits clinical signs similar to human MERS-CoV infection is the '\n",
      " 'rhesus macaque, but none of the available studies reported kidney failure in '\n",
      " 'macaques [18] [19] [20] . Only very small amounts of viral RNA were detected '\n",
      " 'in kidney tissue, and none in urine. Macaques might be of limited value for '\n",
      " 'the study of MERS-CoVspecific kidney pathology as seen in humans. While no '\n",
      " 'autopsy data have become available for patients infected with MERS-CoV, '\n",
      " 'post-mortem studies have been done systematically in SARS victims. '\n",
      " 'Histopathological findings revealed mainly acute tubular necrosis without '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'evidence of glomerular pathology, which is considered to be a consequence of '\n",
      " 'systemic inflammatory response in the context of multi-organ failure, rather '\n",
      " 'than a specific effect of viral infection of the kidney [17] . SARS-CoV has '\n",
      " 'never been successfully isolated from post mortem kidney tissue of infected '\n",
      " 'patients [21] . In-vitro studies of SARS-CoV with immortalized human '\n",
      " 'proximal tubular epithelial cells showed replication without cell impairment '\n",
      " 'as seen in our study, while no infection of podocyte cell lines and only '\n",
      " 'low-level replication in glomerular mesangial cells (MC) was seen, providing '\n",
      " 'further evidence against specific involvement of the renal tract in SARS-CoV '\n",
      " 'infection [22] . In contrast, MERS-CoV was shown to efficiently replicate in '\n",
      " 'a broad range of bat, primate and also human kidney epithelial cell lines '\n",
      " 'that are commonly used as laboratory models [9] . The abundant expression of '\n",
      " \"both viruses' entry receptors in kidney epithelium argues against \"\n",
      " 'receptor-dependent limitations to viral kidney tropism. Specific '\n",
      " 'interference of MERS-CoV with the induction of the interferon response '\n",
      " 'provides one of many possible explanations for an increased capacity of '\n",
      " 'MERS-CoV to replicate in kidney cells [23] . The clear differences in viral '\n",
      " 'permissiveness of primary kidney epithelial cells suggest these cells to be '\n",
      " 'appropriate models for the identification of host-specific restriction '\n",
      " 'factors in the future. We are aware that cell culture resembles a limited '\n",
      " 'model of viral infection which will not reflect the complexity of infection '\n",
      " 'in-vivo. Nevertheless, our results have some congruence with clinical '\n",
      " 'observations in a well-investigated MERS case [5] . Virus in the urine has '\n",
      " 'been detected as late as day 12 of illness, albeit at low concentrations. '\n",
      " 'Virus was detected in the urine but not in the blood, which could indicate '\n",
      " 'autonomous virus replication in the kidneys. Of note, by using a model to '\n",
      " 'assess the ability of CoVs to persist in the environment, it was shown that '\n",
      " 'MERS-CoV has similarities to viruses that are transmitted via the fecal-oral '\n",
      " 'route [24] . The authors suggest that this could hint for oral-urine '\n",
      " 'transmission of MERS-CoV. Further data on virus shedding in the urine over '\n",
      " 'the course of disease are urgently needed. If the kidney should indeed '\n",
      " 'constitute a site of primary virus replication, shedding of virus in the '\n",
      " 'urine might provide a possible source for human-to-human transmission, '\n",
      " 'especially in health-care settings or among close family contacts [3, 6, 7] '\n",
      " '. Clinical guidelines should consider the possibility that MERS patients may '\n",
      " 'benefit from early induction of renoprotective treatment.',\n",
      " 'and protein. Here, analyses using synonymous protein coding mutations '\n",
      " 'demonstrate that the region of the genome that harbors the frameshift signal '\n",
      " 'affects the regulation of genomic and subgenomic RNA production without '\n",
      " 'altering protein sequence. We discuss possible reasons for this. In an '\n",
      " 'effort to further our understanding of how the SARS frameshift signal '\n",
      " 'functions we performed deletion and mutagenesis studies an analyzed the '\n",
      " 'effects on frameshifting efficiency, RNA structure and viral RNA production. '\n",
      " 'There are at least three different types of structures used by the '\n",
      " 'coronaviruses to stimulate frameshifting. Some coronaviruses, such as '\n",
      " 'SARS-CoV, use a three stemmed pseudoknot, others such as human coronavirus '\n",
      " \"229E have an 'elaborated pseudoknot' or kissing stem-loops, while the avian \"\n",
      " 'infectious bronchitis virus utilizes a two stemmed pseudoknot [9, 12, 13] . '\n",
      " 'That these diverse coronaviruses retain some sequence between stems 1 and 2 '\n",
      " 'yet have quite different frameshift stimulating structures is intriguing. '\n",
      " 'One inference that can be made is that the RNA sequence, or a structure '\n",
      " 'formed by it, is involved in some other aspect of the virus lifecycle. Our '\n",
      " 'initial hypothesis was that a regulatory element was contained within this '\n",
      " 'sequence. Here we describe deletion and mutagenesis experiments with a dual '\n",
      " 'luciferase reporter to show that the effect the sequence between stems 1 and '\n",
      " '2 has on frameshifting efficiency is due to structural changes those '\n",
      " 'mutations cause in the pseudoknot. Further, we show by making synonomous '\n",
      " 'mutations in the virus that different levels of frameshifting efficiency '\n",
      " 'affect the production of genomic and subgenomic RNA differently. This could '\n",
      " 'be caused by disruption of a regulatory element or the altered ratio of '\n",
      " 'proteins derived from the coding region flanking the frameshift signal. The '\n",
      " 'individual residues and each stem in the SARS pseudoknot make unique '\n",
      " 'contributions to frameshifting efficiency [8] [9] [10] . For example, while '\n",
      " 'stem 2 is essential for frameshifting, stem 3 is not. Even small changes in '\n",
      " 'Stem 2 (e.g., replacing or deleting the bulged adenosine in stem 2 with a '\n",
      " 'cytosine) reduced frameshifting to levels similar to those observed with the '\n",
      " 'complete disruption of stem 2 [9] . In contrast, altering the bulged '\n",
      " 'adenosine in stem 3 or disruption of stem 3 promoted only modest changes in '\n",
      " 'frameshifting [9] . These observations engendered the hypothesis that stem 3 '\n",
      " 'formation may affect frameshifting by enhancing or facilitating the '\n",
      " 'formation of stem 2. Experiments were designed to test the effects of '\n",
      " 'changes to the length of stem 3 while still maintaining a stable structure. '\n",
      " 'Beginning with the wild-type stem 3 containing nine paired bases and a nine '\n",
      " 'residue loop, the following series of mutants were constructed. First, the '\n",
      " 'loop capping stem 3 was replaced with a shorter tetraloop; the L2Tetra '\n",
      " 'construct contains ten paired bases and a four residue loop. This was used '\n",
      " 'as the basis for construction of two additional tetraloop constructs with '\n",
      " 'progressively shorter stem 3 structures: L2Tetra/S3-4bp, containing only '\n",
      " 'four paired bases in the stem, and L2Tetra/S3-2bp, which has only two paired '\n",
      " \"bases. The 5'-uucg-3' tetraloop sequence used in the L2Tetra/S3-4bp mutant \"\n",
      " 'was selected because the last base-pair before the tetraloop affects its '\n",
      " 'formation. The closing pair for UNCG tetraloops is usually C-G compared to '\n",
      " 'the G-C closing base-pair found in CUUG tetraloops. All of the truncated '\n",
      " 'structures promoted frameshifting at levels equal to or greater than the '\n",
      " 'wild-type pseudoknot structure using a dual luciferase reporter assay ( '\n",
      " 'Figure 1) . Interestingly, the L2Tetra/S3-4bp mutant stimulated '\n",
      " 'frameshifting at 25.1% compared to the 15 to 17.5% observed for the other '\n",
      " 'stem 3 mutants. It is possible that the difference in the tetraloop '\n",
      " 'contributed to the slight increase in frameshifting efficiency. These '\n",
      " 'results, and experiments reported by Brierley et al., for another '\n",
      " 'coronavirus [12] indicate that the length of stem 3 does not play a critical '\n",
      " 'role in frameshifting, suggesting that it may serve some other function. '\n",
      " 'Removing stem 3 does not reduce programmed -1 ribosomal frameshifting (-1 '\n",
      " 'PRF) efficiency. The predicted secondary structure of the SARS coronavirus '\n",
      " 'pseudoknot is shown, along with a series of mutants with stem 3 truncated. '\n",
      " 'The stems are labeled S1 to S3 and the loops labeled L1 to L3. Site directed '\n",
      " 'mutagenesis was used to replace the wild-type loop 2 with a CUUG tetraloop '\n",
      " '(L2Tetra). Additional truncations to stem3 were made (L2Tetra/S3-4bp and '\n",
      " 'L2Tetra/S3-2bp) and frameshifting efficiency analyzed by dual luciferase '\n",
      " 'assay. Frameshifting efficiency is expressed as a percentage with standard '\n",
      " 'error as described in the experimental section.  the sequence of Stem 3 '\n",
      " 'itself, have similarly strong roles [9, 11] . However, given the ability of '\n",
      " 'tetraloop-capped stem 3 mutants to promote high levels of frameshifting, we '\n",
      " 'asked if these mutations could compensate for deletion of the bulged '\n",
      " 'adenosine in stem 2. To this end, the bulged adenosines in stems 2 and 3 '\n",
      " 'were deleted both individually and together in the context of the L2Tetra '\n",
      " 'construct ( Figure 2 ). L2Tetra was chosen as the baseline because, among '\n",
      " 'the tetraloop mutants, it most closely represents the wild-type sequence. '\n",
      " 'Deletion of the bulged adenosine in stem 2 (L2TetraS2Δ) reduced '\n",
      " 'frameshifting by 84.2% (17.5% versus 2.77%, p = 9.3 × 10 −19 , Figure 2 ). '\n",
      " 'When the bulged adenosine from stem 3 was removed (L2TetraS3Δ) a similar '\n",
      " 'reduction of 82.5% was observed (17.5% versus 3.1%, p = 1.1 × 10 −19 ). '\n",
      " 'Interestingly, removal of both adenosines (L2TetraS2ΔS3Δ) partially restored '\n",
      " 'frameshifting with only a 28.2% reduction in frameshifting (17.5% versus '\n",
      " '12.6%, p = 7.5 × 10 −5 ). Bulged adenosines, which are known to bend helices '\n",
      " '[14] , may be participating in formation of a functional pseudoknot '\n",
      " 'structure. Indeed, many frameshift-stimulating structures have bent '\n",
      " 'conformations that are postulated to help effect frameshifting [15] [16] '\n",
      " '[17] . Importantly, however, the contribution of the bulges in the SARS '\n",
      " 'pseudoknot is not the same in the context of the native stem 3/loop 2. '\n",
      " 'Specifically, removing both bulged adenosines (S2ΔS3Δ) from the wild-type '\n",
      " 'backbone decreased rates of frameshifting by 93% (15.3 versus 1.06%, p = 1.7 '\n",
      " '× 10 −18 ), similar to that observed with the removal of the stem 2 bulge '\n",
      " 'alone ( Figure 2 ). Together these results suggest that stabilization of '\n",
      " 'stem 3 assists in the formation or stability of stem 2, which in turn is '\n",
      " 'essential for frameshifting. However, the actual sequence of loop 2 or the '\n",
      " 'tetraloop-capping stem 3 also affected frameshifting efficiency, either '\n",
      " 'through the stabilization of stem 3 or the formation of stem 2 itself, thus '\n",
      " 'implicating the nucleotides in loop 2 or the tetraloop in tertiary structure '\n",
      " 'interactions. Further evidence comes from the chemical protection data '\n",
      " '(below), demonstrating that the altered sequence of loop 2 affects the '\n",
      " 'structure of stems 2 and 3. Although secondary structure predictions '\n",
      " 'indicate that the nucleotides in loop 2 are not part of a helix, those '\n",
      " 'nucleotides are susceptible to both single-and double-strand-specific '\n",
      " 'nucleases [9] suggesting that these nucleotides participate, at least part '\n",
      " 'of the time, in some ordered structural state. Stabilizing stem 3 with a '\n",
      " 'tetraloop preserves -1 PRF efficiency. Site directed mutagenesis was used to '\n",
      " 'change loop2 of the wild-type SARS pseudoknot into tetraloop (L2Tetra from '\n",
      " 'Figure 1 ). The bulged adenosines in stems 2 and 3 were removed by site '\n",
      " 'directed mutagenesis singly or together in conjunction with the tetraloop '\n",
      " '(L2TetraS2Δ, L2TetraS3Δ, and L2TetraS2ΔS3Δ). Both bulged adenosines were '\n",
      " 'removed from the wild-type pseudoknot (S2ΔS3Δ). Frameshifting efficiency is '\n",
      " 'expressed as a percentage with standard error as described in the '\n",
      " 'experimental section. While the high degree of phylogenetic conservation of '\n",
      " 'stem 3 in many coronaviruses [9] suggests that it serves an important '\n",
      " 'function, mutations in this element have less impact on frameshifting '\n",
      " 'efficiency than mutations in stem 2 [9, 11, 12) . Prior structural analyses '\n",
      " 'also suggested that the stability of the structure may contribute to '\n",
      " 'frameshifting efficiency. Here, specific changes were made to Stem 3 in '\n",
      " 'order to directly address this issue. A negative control plasmid was '\n",
      " 'constructed in which Stem 2 base-pairing was disrupted (S2D) by mutating '\n",
      " 'three residues in the third codon position so as to retain the amino acid '\n",
      " 'sequence of the frameshift protein pp1ab ( Figure 3A) . Consistent with '\n",
      " 'prior studies [9, 11, 18] , disruption of stem 2 resulted in a 99% reduction '\n",
      " 'in frameshifting (15% versus 0.19%, Figure 3A ) demonstrating that the '\n",
      " 'integrity of the second stem, and hence the pseudoknot, is required for '\n",
      " 'efficient frameshifting. A control plasmid, L2-UCC, with a synonymous '\n",
      " 'mutation in loop 2 was also made. The AGU codon  was changed to UCC because '\n",
      " 'the latter is present at the same position in the TGEV coronavirus and would '\n",
      " 'be expected to have a minimal impact. More detailed structural information '\n",
      " 'about the L2-UCC mutant is published elsewhere [19] . As expected, a '\n",
      " 'moderate change in frameshifting efficiency was observed (5.72% versus 15%, '\n",
      " 'Figure 3A ). S3b Next, a stem 3 mutant was designed to alter the structure '\n",
      " 'of stem 3 yet retain the same coding sequence so that the effects of these '\n",
      " 'mutations on virus propagation could also be tested in vivo ( Figure  3C ). '\n",
      " 'In the S3D construct, Stem-3 base-pairing was disrupted. The observed '\n",
      " 'effects in vivo should therefore reflect changes related to RNA structure '\n",
      " 'within the frameshifting signal, rather than changes in the function of the '\n",
      " 'encoded proteins. The silent codon changes in S3D promoted moderate '\n",
      " 'reductions in -1 PRF efficiency as compared to the silent S2 mutants (3.95% '\n",
      " 'versus 0.19%; compare Figures 3A and 3C. These moderate changes are similar '\n",
      " 'to the L2-UCC mutant and those observed for other stem 3 mutants [9, 11] . '\n",
      " 'Importantly these values are higher than 2.33%, the lowest level of '\n",
      " 'frameshifting previously observed for a viable SARS coronavirus [7] . To '\n",
      " 'determine if the synonymous mutations altered the structure of the '\n",
      " 'pseudoknot, their effects on this structure were evaluated using selective '\n",
      " \"2'-hydroxyl acylation analyzed by primer extension (SHAPE) analysis (see \"\n",
      " 'Experimental Section). The negative control S2D did not form a '\n",
      " 'pseudoknot-like structure (data not shown). The L2-UCC loop is more '\n",
      " 'structured than the wild-type loop and this reduced flexibility appeared to '\n",
      " 'affect the remaining structure of the pseudoknot as shown by diminished '\n",
      " 'reactivity to NMIA at and around the mutated bases in the loop (L2) compared '\n",
      " 'to the wild-type structure ( Figure 3B ). Additionally, there was decreased '\n",
      " 'reactivity with some bases in stem 3 and in the loops L1 and L3. These data '\n",
      " 'correlate well with prior chemical analysis showing that the nucleotides in '\n",
      " 'Loop 2 were susceptible to both single-and double-strand-specific nucleases '\n",
      " '[9] . Having established this baseline, S3D and S2ΔS3Δ mutants were '\n",
      " 'evaluated by SHAPE. In contrast to the L2-UUC pseudoknot mutant, the S3D and '\n",
      " 'S2ΔS3Δ pseudoknots were more reactive to NMIA indicating that they are less '\n",
      " 'structured ( Figure 3D ). In addition to the expected changes in reactivity '\n",
      " \"to the mutated bases in stem 3, bases in the 3' region of both loop 2 which \"\n",
      " 'caps stem 3 and the loop joining stems 2 and 3 (L3) also showed increased '\n",
      " 'reactivity for S3D. The S2ΔS3Δ mutant was also more reactive to NMIA. '\n",
      " 'However, the NMIA reactivity pattern of the S3D mutant, with more reactivity '\n",
      " \"in the 5' proximal end of stem 3 and the 3' portion of loop 2, differed from \"\n",
      " \"that seen for S2ΔS3Δ in which reactivity was more clustered in the 5' \"\n",
      " 'portion of loop 2 and adjacent stem 3 nucleotides. These data suggest that '\n",
      " 'the native structure of loop 2 and correct formation of stem 3 is dependent '\n",
      " 'on the presence of the bulged adenosines. Although most mutations in stem 3 '\n",
      " 'have a limited impact on frameshifting there is evidence of sequence '\n",
      " 'conservation between coronavirus sequences through this region [9] . To '\n",
      " 'determine the biological significance of stem 3, we tested a virus '\n",
      " 'containing synonymous stem 3 mutations and a virus containing synonymous '\n",
      " 'loop 2 mutations ( Figure 4A ). RNA was extracted from the infected cell '\n",
      " 'lysate, amplified by RT-PCR and sequenced to confirm that the desired '\n",
      " 'mutations were present. The S3D virus was viable but replicates to a lower '\n",
      " 'titer than the wild type ( Figure 4B ; Table 1 ). The effect on production '\n",
      " 'of genomic and subgenomic RNA was quantified by RT-PCR ( Figure 4C ). '\n",
      " 'Although the S3D virus could be detected by RT-PCR, the TCID 50 values were '\n",
      " 'below the limit of the assay (Table 1 ; Figure 4B ). The L2-UCC mutant was '\n",
      " 'detected by both methods. The RT-PCR analysis demonstrates that the genomic '\n",
      " 'RNA (gRNA) of L2-UCC accumulates to wild-type levels four days '\n",
      " 'post-infection ( Figure 4C ). In contrast, genomic RNA from the S3D mutant '\n",
      " 'only accumulated to levels approximately three orders of magnitude lower. '\n",
      " 'Interestingly, the genomic to subgenomic RNA ratios were inverted for the '\n",
      " 'S3D mutant: while the wild type and L2-UCC viruses had tenfold more genomic '\n",
      " 'than subgenomic RNA, the S3D mutant had tenfold more subgenomic RNA than '\n",
      " 'genomic RNA ( Figure 4C ). Similar changes in viral RNA ratios were seen '\n",
      " 'with viruses with mutated slippery sites, implicating a broader region of '\n",
      " 'this part of the genome as being involved in regulation of transcription [7] '\n",
      " '. While the changes in the previous report resulted in an altered protein '\n",
      " 'sequence, it is unknown if the pseudoknot structure in the slippery site '\n",
      " 'mutants was affected. The viruses used in this study all encode identical '\n",
      " 'proteins indicating that protein function itself is not causing the altered '\n",
      " 'transcription patterns. This suggests that the differences between the '\n",
      " 'viruses are due to RNA sequence and/or structural changes. Because the '\n",
      " 'RT-PCR quantification was performed on the supernatant from cells exhibiting '\n",
      " 'CPE and expected to contain representative amounts of cellular gRNA and '\n",
      " 'sgRNA, it is possible that the RT-PCR data could be indicative of differing '\n",
      " 'amounts of gRNA packaged in virions. The reduction in virion-associated gRNA '\n",
      " 'levels could occur if a packaging signal was disrupted. Disruption of a '\n",
      " 'packaging signal would result in fewer complete virions and lower TCID 50 '\n",
      " 'and gRNA values as observed for the S3D mutant. Although the efficiency of '\n",
      " 'frameshifting is similar for the L2-UCC and S3D mutants the gRNA levels '\n",
      " 'differ by several log. The stabilization of L2-UCC stem 3 structure and loss '\n",
      " 'of S3D stem 3 structure (see Figure 3 ) also support the idea that this '\n",
      " 'region may contain a packaging signal. Asterisks indicate values that were '\n",
      " 'at the lower limit of the assay. (C) Relative abundance of genomic and '\n",
      " 'subgenomic RNA in viral stocks. Plaque purified virus was used to infect '\n",
      " 'Vero cells. Four days post infection CPE was observed. Media and detached '\n",
      " 'cells were removed. RNA was extracted from a 100μL aliquot using Trizol. '\n",
      " 'Taqman analysis was used to determine the total number of genomic and '\n",
      " 'subgenomic RNA molecules compared to a reference RNA transcribed from a SARS '\n",
      " 'replicon. The number of copies per mL of viral stock is shown with standard '\n",
      " 'deviation. S3b  S3b Table 1 . Frameshifting efficiency affects infectivity. '\n",
      " 'SARS coronavirus mutants are indicated. -1 PRF efficiencies and standard '\n",
      " 'error as measured in VeroE6 cells as described in the Experimental Section. '\n",
      " 'The data for the WT and L2-UCC viruses has previously been published [7, 19] '\n",
      " '. TCID 50 values were calculated as described in the Experimental Section. '\n",
      " 'The error is the standard deviation of six measurements.  Construction of '\n",
      " 'dual luciferase plasmids. The parental plasmids pJD464 and pJD502, '\n",
      " 'containing the Renilla and firefly luciferase genes flanking the wild-type '\n",
      " 'SARS-CoV frameshift signal, have been described previously [9] . pJD502 is '\n",
      " 'the test construct for measuring frameshifting efficiency, and pJD464 is a '\n",
      " 'readthrough control plasmid to normalize against any defects in overall '\n",
      " 'translation that the introduced SARS sequence may cause. Site-directed '\n",
      " 'mutagenesis was used to introduce mutations into the test construct '\n",
      " \"(pJD502). Mutagenesis was performed using Stratagene's QuikChange II kit (La \"\n",
      " 'Jolla, CA). The mutations were confirmed by sequence analyses. Construction '\n",
      " 'of mutant viruses. Oligonucleotide site directed mutagenesis was used to '\n",
      " 'mutate SARS subclone D [20] . Each mutant subclone was assembled with the '\n",
      " 'other five subclones, and then T7 RNA polymerase and a GTP cap analog were '\n",
      " 'used to generate a full-length virus transcript as described previously [20] '\n",
      " '. Full-length genomic RNA was transfected into Vero cells resulting in a '\n",
      " 'productive, lytic viral infection. The virus was allowed to grow for 5 days '\n",
      " 'at 37 °C. Viral supernatants were plated onto Vero cells, and several clones '\n",
      " 'were plaque purified. The plaque-purified viruses were expanded by growth on '\n",
      " 'Vero cells. RT-PCR was used to detect subgenomic RNA, a marker of viral '\n",
      " 'replication. PCR amplicons from the frameshift region were sequenced to '\n",
      " 'verify that the correct mutations were introduced. Virus titer was measured '\n",
      " 'using a TCID 50 assay calculated according to the method of Reed and Muench '\n",
      " '[21] . All viral assays were conducted in BSL-3 safety facilities. The '\n",
      " 'abundance of gRNA and sgRNA was determined by quantitative real-time PCR '\n",
      " 'using SYBR green technology as previously described [7] . Primers '\n",
      " 'complementary to gRNA and sgRNA were used to detect RNA. Although the '\n",
      " 'primers complementary to the sgRNA can anneal to the gRNA, only the smallest '\n",
      " \"sgRNA has the 5' leader and 3' sequence close enough to allow PCR \"\n",
      " 'amplification under normal cycling conditions. RNA Structure Probing. The '\n",
      " \"structures of the pseudoknots were probed using the selective 2'-hydroxyl \"\n",
      " 'acylation analyzed by primer extension (SHAPE) procedure of Wilkinson et al. '\n",
      " '[22] . SHAPE analysis utilizes N-methylisatoic anhydride (NMIA), which tests '\n",
      " 'the local backbone flexibility of RNA. More flexible nucleotides assume '\n",
      " 'conformations that are reactive to NMIA which irreversibly acylates the '\n",
      " \"ribose 2'-hydroxyl groups. This then presents as a stop upstream of the \"\n",
      " 'modified base during primer extension. Thus, a more intense band in the '\n",
      " 'primer extension autoradiograph represents a more flexible region of the '\n",
      " 'RNA. Briefly, DNA was amplified from each mutant plasmid using PCR master '\n",
      " 'mix (Fermentas Inc., Glen Burnie, MD) and a forward-direction primer '\n",
      " 'containing the T7 RNA promoter; T7forSHAPE, '\n",
      " \"5'-TAATACGACTCACTATAGGGAAGATGCACCTGATGAAATGG-3' and the reverse-direction \"\n",
      " \"primer; revSHAPE, 5'-GCCCATATCGTTTCATAGCTTC-3'. RNA was synthesized from the \"\n",
      " 'PCR products using the Ambion T7 MEGAscript kit (Austin, TX) as per the '\n",
      " \"manufacturer's instructions. RNA (2 pmol) in a volume of 12 μL was denatured \"\n",
      " 'at 95 ºC and cooled on ice. Reaction buffer (6 μL) containing 333 mM HEPES '\n",
      " '(4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid) pH 8.0, 20 mM MgCl2, '\n",
      " 'and 222 mM NaCl was added to each RNA sample and the mixture was incubated '\n",
      " 'at 37 ºC for 20 minutes. The reactions were split into two equal aliquots, '\n",
      " 'one with 1 μL Dimethyl Sulfoxide, the other with 1 μL of 65 mM '\n",
      " 'N-methylisatoic anhydride (NMIA), and incubated at 37 ºC for 45 minutes. The '\n",
      " 'RNA was then precipitated with 90 μL H 2 O, 4 μL 5M NaCl, 1 μL 20 mg/mL '\n",
      " 'glycogen, 2 μL 100 mM EDTA pH 8.0, and 350 μL ethanol overnight at −80ºC. '\n",
      " 'After centrifugation the RNA was resuspended in 14 μL of 0.5 × TE. The '\n",
      " \"oligonucleotide 5'-GCCGGGCCTTTCTTTATG-3' (50 pmol; Integrated DNA \"\n",
      " 'Technology, Coralville, IA) was labeled with 30 μCi gamma 32 P-adenosine '\n",
      " 'triphosphate (ATP) using T4 kinase (Roche, Indianopolis, IN) and purified '\n",
      " 'through a G-25 column (GE Healthcare). RNA (7 μL) was incubated with '\n",
      " '32P-labeled oligonucleotide (3 μL) at 65 ºC for 5 minutes, 35 °C for 5 '\n",
      " 'minutes and cooled to 0 ºC. Reverse transcription reactions were performed '\n",
      " 'using SuperScript III enzyme (Invitrogen, Carlsbad, CA) at 52 ºC for 20 '\n",
      " 'minutes. Products were separated on an 8% polyacrylamide, 7.5M urea gel in '\n",
      " 'TBE buffer, dried and visualized by PhosphorImage analysis. Dual Luciferase '\n",
      " 'Assays. For each assay cells were transfected with either a test plasmid or '\n",
      " 'a control plasmid (described above) using FuGene6 from Promega (Madison, '\n",
      " \"WI). VeroE6 cells were grown overnight in Dulbecco's Modified Eagle Medium \"\n",
      " 'supplemented with 10% FBS at 37 ºC. Cells were disrupted using the passive '\n",
      " \"lysis buffer (Promega, Madison, WI) as per the manufacturer's instructions. \"\n",
      " 'Luminescence reactions were initiated by addition of 10 -20 μL of cell '\n",
      " 'lysates to 100 μL of the Promega LAR II buffer and completed by addition of '\n",
      " \"100 μL Stop'n'Glo reagent. Luminescence was measured using a Turner Design \"\n",
      " 'TD20/20 instrument. At least three replicates were performed within each '\n",
      " 'assay and all assays were repeated at least three times until the data were '\n",
      " 'normally distributed to enable statistical analyses both within and between '\n",
      " 'experiments. The frequency of frameshifting is expressed as a ratio of '\n",
      " 'firefly to Renilla luciferase from a test plasmid divided by the analogous '\n",
      " 'ratio from the read-through control plasmid multiplied by 100%. Fold change, '\n",
      " 'standard error and estimates of the P-values for ratiometric analyses were '\n",
      " 'performed as described previously [23] . In line with previous studies of '\n",
      " 'coronavirus frameshift signals, data presented here indicate that the '\n",
      " 'sequence and length of the third stem is not critical for frameshifting '\n",
      " 'efficiency [9, 12, 18] . However, as revealed by the structural analysis, '\n",
      " 'mutations in stem 3 can cause changes in frameshift efficiency by changing '\n",
      " 'stem 2 formation. Additionally, changes in stem 3 can be detrimental to '\n",
      " 'viral viability. Specifically, changes that destabilize stem 3 resulted in '\n",
      " 'reductions in -1 PRF and altered patterns of viral RNA production. It '\n",
      " 'remains to be determined if the latter change is due to a disruption in '\n",
      " 'frameshifting or disruption of a RNA regulatory element such as a packaging '\n",
      " 'signal. While many sequences involved in coronavirus subgenomic RNA '\n",
      " 'synthesis have been elucidated, sequences and structures involved in '\n",
      " 'regulating coronavirus genomic RNA synthesis are less well defined [24] . '\n",
      " 'Internal replication elements have been described in other plus-strand '\n",
      " 'non-segmented viruses including Poliovirus, Hepatitis C Virus and '\n",
      " 'Tombusviridae [25] [26] [27] . Each of these internal replication elements '\n",
      " 'is present in the region of the viral genome encoding non-structural '\n",
      " 'proteins. The Tombusviridae RNA-dependent RNA polymerase is translated as a '\n",
      " 'readthrough product and the internal replication element is downstream of '\n",
      " 'the readthrough signal [25] . Although the CRE element from poliovirus has '\n",
      " 'been studied for several years the precise roles of these elements are not '\n",
      " 'well defined [28] . In most instances it is not known, for example, if they '\n",
      " 'bind to specific viral or host proteins, or if they communicate with other '\n",
      " 'elements in the genome. Packaging signals for coronaviruses are also '\n",
      " \"ambiguous and have been identified in the 5' untranslated regions and the 3' \"\n",
      " 'end of ORF1a/b [29, 30] . Our work demonstrated that a less structured stem '\n",
      " '3 in the frameshift pseudoknot led to significantly reduced amounts of gRNA '\n",
      " 'when compared to viruses with more structured pseudoknots. However, the same '\n",
      " 'change in viral RNA patterns was observed with a slippery site mutant '\n",
      " 'suggesting that if a packaging signal is present it extends from, or is '\n",
      " 'affected by, the upstream slippery site. The correlation between stem 3 '\n",
      " 'structure and gRNA levels indicates that a structural requirement in stem 3 '\n",
      " 'is required for optimal RNA production but we cannot, at this stage rule out '\n",
      " 'a frameshifting threshold similar to that observed for the yeast '\n",
      " 'double-stranded RNA virus M1 [31] . That is, if frameshifting drops below a '\n",
      " 'certain frequency, the change in the ratio of pp1a to pp1ab results in a '\n",
      " 'dramatic reduction in gRNA production. Indeed, recent reports describing the '\n",
      " 'function of a nonstructural protein (nsp8) encoded upstream of the '\n",
      " 'frameshift signal provide an alternative hypothesis to regulatory element '\n",
      " 'hypothesis investigated in this work. Nsp8 was initially described as a '\n",
      " 'non-canonical RNA-dependent RNA polymerase and produced as part of the pp1a '\n",
      " 'protein whether or not frameshifting occurs [32] . In contrast, the '\n",
      " 'canonical RdRP Nsp12 is produced as part of the pp1ab protein only if '\n",
      " 'frameshifting occurs. This, in conjunction with our data suggests that the '\n",
      " 'nsp8 polymerse might direct sgRNA production and that the RdRP encoded in '\n",
      " 'nsp12 is predominantly used for gRNA production. It has been hypothesized '\n",
      " 'that coronavirus RNA synthesis involves structurally and functionally '\n",
      " 'separable RNA synthesising complexes [33] . The nsp7+nsp8 complex from three '\n",
      " 'different coronaviruses have recently been shown to have primer-independent '\n",
      " 'RNA polymerase activity [33, 34] in contrast to the nsp12 RdRP which is '\n",
      " 'primer dependent [35] . Currently the details regarding coronavirus RNA '\n",
      " 'synthesis are very limited and do not explain how the polymerases '\n",
      " 'distinguish between templates or prime synthesis. te Velthuis and colleagues '\n",
      " 'suggested that the separate RdRPs may influence plus and minus strand '\n",
      " 'synthesis [33] but our results indicate that the division might be between '\n",
      " 'genomic and subgenomic RNA synthesis. The RT-PCR for detection of gRNA and '\n",
      " 'sgRNA in our study generated a cDNA intermediate using random hexamers and '\n",
      " 'because of this the analyses performed here do not distinguish between the '\n",
      " 'production of positive-and negative-strands nor between mRNA and sgRNA. '\n",
      " 'Thus, we do not know if the regulatory sequences or proteins regulating '\n",
      " 'genomic RNA production act upon the positive-or negative-strand replication '\n",
      " 'intermediates or transcription intermediates. A slight difference in '\n",
      " 'frameshifting efficiency, such as that between the L2-UCC and S3D mutants, '\n",
      " 'would not be expected to have such significant effects on viral RNA levels. '\n",
      " 'Similar to previously described viruses containing mutations in the slippery '\n",
      " 'site of the frameshift signal [7] , here we show that mutations to the '\n",
      " 'SARS-CoV frameshift stimulating mRNA pseudoknot can also affect the '\n",
      " 'production of viral genomic RNA. In both instances, the abundance of viral '\n",
      " 'genomic RNA was reduced to 10-fold lower levels than the subgenomic RNA and '\n",
      " 'several orders of magnitude below that of the wild-type virus. It is not '\n",
      " 'clear if this result is due to a change in the abundance of gRNA available '\n",
      " 'to be transcribed to sgRNA or if it is attributable to other factors, such '\n",
      " 'as interactions between the gRNA and other proteins or RNA elements as '\n",
      " 'discussed above. It is also possible that the synonymous changes altered the '\n",
      " 'rate of translation and/or the folding of the RdRP [36] which, in turn, '\n",
      " 'could have affected production of genomic and/or subgenomic RNA. Regardless '\n",
      " 'of the mechanism, our observations provide a backdrop against which new '\n",
      " 'questions about coronvirus replication and transcription may be explored.',\n",
      " 'Wheat yellow mosaic is one of the most devastating soil-borne diseases of '\n",
      " 'winter wheat (Triticum aestivum L.). It was first reported in Japan in the '\n",
      " '1920s and China in the 1960s [1, 2] , and then spread continually in Japan '\n",
      " 'and China [3, 4] . According to the statistical data, the disease area was '\n",
      " 'more than 666,700 hectares in the 1990s, the yield loss was estimated to '\n",
      " 'range between 20-40% and could be up to 70-80% during a serious year, even '\n",
      " '100%. Wheat yellow mosaic virus (WYMV), the causal agent of wheat yellow '\n",
      " 'mosaic, belongs to the genus Bymovirus within the family Potyviridae. It is '\n",
      " 'a soil-borne pathogen and is transmitted by the fungus-like organism '\n",
      " 'Polymyxa graminis [5] . The genome of WYMV is comprised of two (+) '\n",
      " 'single-stranded RNAs, RNA1 encodes for coat protein (CP) and six others: P3, '\n",
      " '7 K, nuclear inclusion protein a (NIa), nuclear inclusion protein b (NIb), '\n",
      " 'cytoplasmic inclusion protein (CI), 14 K; RNA2 encodes for a polyprotein '\n",
      " 'that contains 28-kDa and 72-kDa proteins [6, 7] . Concerning virus '\n",
      " 'detection, several methods are used commonly to detect WYMV. ELISA is a '\n",
      " 'reliable method for detecting WYMV and suitable for high-throughput samples '\n",
      " '[8] [9] [10] ; RT-PCR is the most conventional method to detect RNA virus '\n",
      " '[11, 12] and western blotting detects the target protein for further '\n",
      " 'confirmation [13] [14] [15] [16] . However, the sensitivity of ELISA might '\n",
      " 'not be sufficiently high to detect low concentrations of WYMV, and '\n",
      " 'virus-specific antiserum is required. WYMV can serologically cross-react '\n",
      " 'with wheat spindle streak mosaic virus [17] , and RT-PCR is not perfect '\n",
      " 'either. Novel nucleic acid amplification methods, loopmediated isothermal '\n",
      " 'amplification (LAMP) for DNA and RT-LAMP for RNA, have been developed [18] . '\n",
      " 'The high specificity and sensitivity, rapid execution, performance under '\n",
      " 'isothermal condition, time-saving, easy observation of by-products [19] , '\n",
      " 'and low cost make RT-LAMP unrivaled among diagnostic techniques. It is easy '\n",
      " 'and simple to perform only with four appropriate primers, a reverse '\n",
      " 'transcriptase for RNA template, a DNA polymerase and a water bath or heat '\n",
      " 'block for reaction. Therefore, in recent years, many pathogenic viruses have '\n",
      " 'been detected by these methods, including human [20] [21] [22] [23] [24] , '\n",
      " 'animal [25] and plant [26] [27] [28] [29] [30] [31] [32] viruses. In the '\n",
      " 'present study, the RT-LAMP method was used successfully for detection of '\n",
      " 'WYMV for the first time. This method could result in more accurate diagnosis '\n",
      " 'for monitoring WYMV. Wheat samples were collected during field surveys from '\n",
      " 'different regions of China in March 2011 and stored in a freezer at-20°C. '\n",
      " 'The Chinese wheat mosaic virus (CWMV)-infected samples were collected from '\n",
      " 'Yantai, Shandong Province; the barley stripe mosaic virus (BSMV)-infected '\n",
      " 'samples were fresh wheat leaves inoculated with BSMV in our laboratory in '\n",
      " 'May 2011. The fresh or stored wheat samples were ground in a Retsch MM400 '\n",
      " 'mixer mill (Retsch, Haan, Germany) for 1 min at 30 Hz, the sample powder was '\n",
      " 'homogenized with 600 μl extraction buffer (0.1 M Tris-HCl, pH 7.4, 2.5 mM '\n",
      " 'NaCl, and EDTA) and 600 μl supercritical water-phenol, then centrifuged at '\n",
      " '12,000 rpm for 15 min. The aqueous phase was precipitated with 4 M LiCl. '\n",
      " 'After incubation at -20°C for 2 h or overnight, the precipitate was '\n",
      " 'collected by centrifugation (12,000 rpm) at 4°C for 15 min. The resultant '\n",
      " 'pellet was washed twice with 70% ethanol, dried at 37°C for about 5 min, and '\n",
      " 'dissolved in 60 μl deionized distilled water. The RNA extract was stored at '\n",
      " '-20°C. Based on published WYMV RNA1 and RNA2 sequences (accession numbers '\n",
      " 'AF067124 for RNA1and AF041041 for RNA2) [7] , four sets of primers were '\n",
      " 'designed by Primer Explorer version 4 (Fujitsu Ltd., Tokyo Japan, '\n",
      " 'http://primerexplorer.jp/elamp3.0.0/index.html). Four oligonucleotide '\n",
      " 'primers [F3, B3, FIP (F1c + F2), and BIP (B1c + B2)] that recognize a total '\n",
      " 'of six sequences of the CP gene and the 72 kDa gene were designed, '\n",
      " 'respectively. F3 and B3 were outer primers whereas FIP and BIP are inner '\n",
      " 'primers. Each inner primers has two distinct adjacent sequences in opposite '\n",
      " 'orientations. All primers were PAGE purified and synthesized by Invitrogen '\n",
      " 'or Sanggon (Shanghai, China). To choose the most appropriate primer set, '\n",
      " 'RT-LAMP reactions were conducted as described previously [18, 19, 33] . The '\n",
      " 'reaction was carried out in a 25-μl reaction system, containing 2.5 μl 10× '\n",
      " 'Thermopol buffer, 0.5 mM dNTP, 0.8 M betaine, 1.6 μM FIP and 1. Many '\n",
      " 'pyrophosphate ions were produced during RT-LAMP, and the production of a '\n",
      " 'white precipitate of magnesium pyrophosphate gave the tube a turbid '\n",
      " 'appearance that could be observed directly. The positive tube was cloudy and '\n",
      " 'the negative one was clear. With four sets of primers, the RT-LAMP reaction '\n",
      " 'mixture was observed by the naked eye, and the amplification reactions were '\n",
      " 'confirmed by complementary procedures, such as gel electrophoresis. Five '\n",
      " 'microliters of amplified DNA fragments were electrophoresed in 1.5% agarose '\n",
      " 'in TBE buffer. The RT mixture contained 2 μl RNA extract, 0.5 μl primer '\n",
      " '(Table 1) , 3 μl 5 × M-MLV buffer and 1 μl dNTP (5 mM each), 12 U RRI, 200 U '\n",
      " 'M-MLV reverse transcriptase (Promega, Madison, WI, USA) and 7.5 μl deionized '\n",
      " 'distilled water, added to a final volume of 15 μl. Reverse transcription was '\n",
      " 'performed at 37°C for 1.5 h, and 2.0 μl product of cDNA was used for PCR '\n",
      " 'amplification. Twenty-five microliters of PCR mixture that contained 2.5 μl '\n",
      " 'PCR buffer, 0.5 μl dNTP (5 mM each), 0.5 μM primers, 1.25 U Taq DNA '\n",
      " 'polymerase (Tiangen, Beijing, China), and 21.5 μl deionized distilled water '\n",
      " 'were added. The PCR was performed with denaturation at 94°C for 5 min and 30 '\n",
      " 'cycles of 94°C for 40 s, 63°C for 45 s, and 72°C for 40 s, followed by a '\n",
      " 'final 10-min extension step in a Bio-Rad Cycler(Bio-Rad, Hercules, CA, USA). '\n",
      " 'Five microliters of amplified DNA fragments were electrophoresed in 1.5% '\n",
      " 'agarose in TBE buffer. To determine the specificity of RT-LAMP, total RNA '\n",
      " 'from wheat leaves infected with WYMV, CWMV or BSMV were applied to the '\n",
      " 'RT-LAMP reaction solution separately, and RNA collected from the healthy '\n",
      " 'wheat served as a negative control. The primer sequences used for CWMV and '\n",
      " 'BSMV RT-PCR were shown in Table 1 . To compare the sensitivity of RT-LAMP '\n",
      " 'with RT-PCR, total RNA from WYMV-infected wheat was diluted serially in '\n",
      " '10-fold increments (10 0 -10 -8 ) with healthy wheat RNAs were then used as '\n",
      " 'templates for the two assays. The products were analyzed by agarose gel '\n",
      " 'electrophoresis (1.5% agarose, TBE). Four sets of RT-LAMP primers were '\n",
      " \"designed (I-IV). Primer sets I and II were located at 665-875 nt (3' \"\n",
      " \"terminal) and 106-332 nt (close to 5' terminal) of the CP gene; and primer \"\n",
      " \"sets III and IV were located at 715-916 nt (close to 5' terminal) and \"\n",
      " \"1335-1557 nt (close to 3' terminal) of the 72 kDa gene (Figure 1) . Each set \"\n",
      " 'contains four primers, F3 and B3 were outer primers whereas FIP and BIP were '\n",
      " 'inner primers. Each of the two inner primers had two distinct adjacent '\n",
      " 'sequences in opposite orientations. For detecting CP gene, the first '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'sequence (nt 752-731) of FIP-1 and sequence (nt 212-191) of FIP-2 were in '\n",
      " 'reverse orientation, whereas the second sequence (nt 689-706) and sequence '\n",
      " '(nt 142-161) of the primers were in the forward direction. In BIP-1 and '\n",
      " 'BIP-2, the forward sequence (nt 759-780) and sequence (nt 215-235) is '\n",
      " 'followed by the reverse sequence (nt 813-729) and sequence (nt 293-274). For '\n",
      " 'the 72 kDa gene: the first sequence (nt 813-729) of FIP-3 and sequence (nt '\n",
      " '1428-1407) of FIP-4 were in reverse orientation, whereas the second sequence '\n",
      " '(nt748-767) and sequence (nt 1355-1372) of the primers were in the forward '\n",
      " 'direction. In BIP-3 and BIP-4, the forward sequence (nt 814-833) and '\n",
      " 'sequence (nt 1429-1450) is followed by the reverse sequence (nt 889-871) and '\n",
      " 'sequence (nt 1498-1479). Their relative location in the virus genomes were '\n",
      " 'shown in Table 1 . To select the appropriate primer set, RT-LAMP using four '\n",
      " 'sets of primers was carried out with two healthy controls and two '\n",
      " 'virus-infected samples under isothermal condition at 65°C for different '\n",
      " 'times. Agarose gel electrophoresis showed that the primer set III had '\n",
      " 'positive bands after 30 min and the other three sets had positive bands '\n",
      " 'after 45 min. They were all available within 80 min, thus the primer set III '\n",
      " 'seemed more sensitive than the others and was selected for virus detection '\n",
      " 'under isothermal conditions at 65°C for 80 min in this study ( Figure 2 ). '\n",
      " 'For each primer set, AMV and M-MLV RT-LAMP was performed and similar results '\n",
      " 'were obtained, therefore, M-MLV reverse transcriptase was chosen for the '\n",
      " 'subsequent RT-LAMP assay because of its relatively cheap price (Figure 3 ). '\n",
      " 'The turbidity of pyrophosphate ions was observed directly after production '\n",
      " 'of large amounts of DNA fragments during the RT-LAMP reaction. RT-LAMP '\n",
      " 'reaction mixtures were used for turbidity observation with the primer sets '\n",
      " 'I-IV after reaction at 65°C for 80 min. The healthy controls were relatively '\n",
      " 'clear, whereas the positive samples were cloudy, and primer sets I and III '\n",
      " 'more readily gave negative or positive results (Figure 4 ). This '\n",
      " 'demonstrates that the RT-LAMP reactions are conveniently visualized by '\n",
      " 'directly judging the turbidity and it does not require specialized PCR and '\n",
      " 'electrophoresis equipment. The primer set III was chosen for the following '\n",
      " 'experiments. To evaluate the specificity of RT-LAMP, it was carried out '\n",
      " 'using two healthy controls, two WYMV-infected, two CWMV-infected and two '\n",
      " 'BSMV-infected samples with WYMV RT-LAMP primer set III. At the same time, '\n",
      " 'the samples were also detected by conventional RT-PCR, primers HC-157 F and '\n",
      " 'HC72-518R, which were located at nt 157-179 and 518-497 of the 72-kDa gene, '\n",
      " 'and used to amplify the 360-bp band of interest. The primers CWMV-F and '\n",
      " 'CWMV-R were used for CWMV RT-PCR and primers BSMV-F and BSMV-R were used for '\n",
      " 'BSMV RT-PCR (Table 1) . Agarose gel analysis of RT-LAMP revealed that WYMV '\n",
      " 'primers specifically amplified WYMV but not CWMV and BSMV, although CWMV and '\n",
      " 'BSMV could infect wheat with WYMV in some regions of China [12] (Figure 5) . '\n",
      " 'The sensitivity comparison showed that RT-LAMP could detect WYMV from total '\n",
      " 'RNA diluted up to 10 -5 , and RT-PCR detected total RNA diluted up to 10 -3 '\n",
      " '; therefore, RT-LAMP was 100-fold more sensitive than conventional RT-PCR ( '\n",
      " 'Figure 6 ). Figure 7 ). This result confirmed the reliability and '\n",
      " 'sensitivity of this RT-LAMP assay for routine detection. WYMV could be '\n",
      " 'detected by the RT-LAMP method using the designed four sets of primers '\n",
      " '(I-IV) and M- The primer set III seemed more sensitive than the others and '\n",
      " 'was selected for further virus detection in this study. The RT-LAMP assay '\n",
      " 'was performed at different times and virus was detected after 30-45 min by '\n",
      " 'agarose gel analysis, considering the field application where the turbidity '\n",
      " 'will be observed by naked eyes as the criteria to judge that the sample is '\n",
      " 'negative or positive, the reaction condition at 65°C for 80 min was used '\n",
      " 'according to previous studies and the results obtained here. The RT-LAMP '\n",
      " 'method had detection sensitivity about 100 times more than RT-PCR in the '\n",
      " 'present study, which was similar to that reported by Boubourakas [33] . '\n",
      " 'Therefore, RT-LAMP was demonstrated to be a simple and time-saving method '\n",
      " 'compared with RT-PCR for routine detection of WYMV, and it did not require '\n",
      " 'specialized PCR and electrophoresis equipment. Due to the high level of '\n",
      " 'detection sensitivity of the RT-LAMP method, careful and strict operation '\n",
      " 'was necessary during the whole process to avoid false-positive results [34] '\n",
      " '. Other factors to consider are DNA smear and turbidity. Similar to products '\n",
      " 'generated by PCR, DNA smear was present in the healthy control, which may '\n",
      " 'due to an excess amount of RNA used in the reaction and some non-specific '\n",
      " 'reactions [31] . If the DNA smear is strong enough, it can influence '\n",
      " 'observation of turbidity. Turbidity can be observed directly, but DNA smear '\n",
      " 'can also produce turbidity when the concentration of RNA template is '\n",
      " 'sufficiently high. Therefore, when turbidity is used to differentiate '\n",
      " 'positive and negative samples, the amount of RNA template should be '\n",
      " 'maintained at a low level. In conclusion, this study developed the RT-LAMP '\n",
      " 'assay for detecting WYMV. Compared with conventional RT-PCR, RT-LAMP yielded '\n",
      " 'more accurate results, and was more convenient and less time-consuming '\n",
      " 'especially for field detection. This method has potential application in '\n",
      " 'early diagnosis and screening of resistant wheat varieties, to reduce the '\n",
      " 'loss of yield. ',\n",
      " 'Dengue viruses (DENs) are the wildest transmitted arbovirus members of the '\n",
      " 'family Flaviviridae, genus Flavivirus, and compose four serotypes, DEN-1, 2, '\n",
      " '3, and 4. As the etiologic agents, DENs can cause severe flu-like illness '\n",
      " 'called dengue fever (DF), and sometimes lethal complication called dengue '\n",
      " 'haemorrhagic fever (DHF) and dengue shock syndrome (DSS) [1, 2] . They '\n",
      " 'transmit diseases to human beings primarily through mosquitoes, mainly Aedes '\n",
      " 'aegypti and Aedes albopictus. With dramatically growth in recent decades, DF '\n",
      " 'affects 100 million people and results in about 25,000 deaths annually, '\n",
      " 'mostly in tropical and sub-tropical regions. DHF has become a leading cause '\n",
      " 'of serious illness and death among children in some Asian countries [3] . '\n",
      " 'Unfortunately, effective vaccines or therapies against the infection are '\n",
      " 'still not available [4] . RNA interference (RNAi) is a sequence-specific RNA '\n",
      " 'degradation process in the cytoplasm of eukaryotic cells triggered by '\n",
      " 'double-stranded RNA (dsRNA), widely existing in many species from nematode '\n",
      " 'to human [5] [6] [7] [8] . Upon introduction into the cells, exogenous '\n",
      " 'dsRNAs are cut into 21-25 nt small interfering RNA (siRNA) by an RNase '\n",
      " 'III-like enzyme called Dicer. The siRNAs form RNA-induced silencing complex '\n",
      " '(RISC) with other cellular components, and lead to the cleavage of their '\n",
      " 'homologous transcript and eventually the silencing of specific gene [9] [10] '\n",
      " '[11] . RNAi is believed to be an effective endogenous mechanism for host '\n",
      " 'cells to defense against virus attack [12] , and has been applied as an '\n",
      " 'exogenous measure to inhibit viral replication, such as HIV [13, 14] , '\n",
      " 'influenza A virus [15] , HBV [16] and SARS-CoV [17] . DEN is one of the '\n",
      " 'first animal viruses that could be efficiently inhibited by RNAi [12, 18] . '\n",
      " 'Like other flaviviruses, DEN generates intracellular dsRNA as an '\n",
      " 'intermediate of their replication, which may induce RNAi in the host cells. '\n",
      " \"A new explanation for mosquitoes' non-pathogenic and persistent infections \"\n",
      " 'of DEN is that RNAi could be an important modulator [19] . Exogenous long '\n",
      " 'length dsRNA corresponding to DEN sequences, introduced by either plasmid or '\n",
      " 'Sindbis viruses, has been proven to mediate RNAi in mosquito C6/36 Cells and '\n",
      " 'lead to inhibition of DEN replication in cultured mosquito cells [20, 21] . '\n",
      " 'Genetically modified Aedes aegypti has been raised to develop dengue virus '\n",
      " 'resistance [22] [23] [24] . The mixtures of DEN specific small interfering '\n",
      " 'RNAs, the hallmark of RNAi, were detected in all aforementioned studies. But '\n",
      " 'little was known about the role of single siRNA with particular target '\n",
      " 'sequence in the inhibition of DEN replication. Our present study was '\n",
      " 'designed to investigate if a single siRNA has the inhibitory effect on DEN-1 '\n",
      " 'replication in mosquito cells. Effects of siRNA on C6/36 survival to DEN '\n",
      " 'challenge CPE in each group was observed at the 7 th dpi. The virus positive '\n",
      " 'control group and control siRNA group showed a large number of CPE up to '\n",
      " '++++, characterized by cell swelling and fusing, and reduced cell number; '\n",
      " 'while normal group and siRNA group showed no CPE and no observable decrease '\n",
      " 'in cell number. The cell survival rate of C6/36 cells, measured by MTT '\n",
      " 'assay, of siRNA group was 76.9% ± 4.5%, control-siRNA group was 43.9% ± '\n",
      " '3.6%, and virus positive control group was 23.6% ± 14.6%. Compared with '\n",
      " 'virus positive control group, the cell survival rate of siRNA group '\n",
      " 'increased by 2.26-fold (n = 5, P < 0.05), but siRNA control group showed no '\n",
      " 'significant difference (n = 5, P > 0.05) ( Figure 2 ). The amount of '\n",
      " 'intracellular DEN-1 viral loading was detected by Real time RT-PCR at the 7 '\n",
      " 'th dpi. The CT value of DEN-1 RNA of siRNA group was 19.91 ± 0.63, '\n",
      " 'control-siRNA group was 14.63 ± 0.91, virus-positive control group was 14.56 '\n",
      " '± 0.39, and no viral RNA detectable in normal control group. Compared with '\n",
      " 'the virus positive control group, siRNA group showed significant decrease of '\n",
      " 'viral loading (n = 5, P < 0.05), but control siRNA group showed no '\n",
      " 'significant difference (n = 5, P > 0.05) (table 2). The viral RNA amount of '\n",
      " 'siRNA group was reduced for 2 5 .34 fold (about 40 fold), the inhibition '\n",
      " 'rate was 97.54%. After transfection with FAM-labeled siRNA, fluorescent '\n",
      " 'signals can be detected in more than half of C6/36 cells cytoplasm under a '\n",
      " 'fluorescent microscope. During the incubation, the fluorescent intensity in '\n",
      " 'the cells gradually decreased ( Figure 1 ). And the percentage of FAM '\n",
      " 'positive cell at 6 h, days 1, 3, 5, and 7 were 66.0%, 52.1%, 32.0%, 13.5%, '\n",
      " 'and 8.9%, respectively, by flow cytometry (Figure 3 ). As a rapid developing '\n",
      " 'technology, RNAi has not only become a powerful tool for studying gene '\n",
      " 'function and development of gene-based therapies, but also been widely used '\n",
      " 'in anti-virus researches. Precious studies suggested that, at the molecular, '\n",
      " 'cellular and individual levels, RNAi can potentially be used to block viral '\n",
      " 'transmission and thus prevent the viral diseases [12, 18] . With the high '\n",
      " 'efficiency, specificity and low cytotoxicity, RNAi offered a new promise of '\n",
      " 'anti-viral therapy. For ssRNA viruses such as DEN, there genomes are exposed '\n",
      " 'within cytoplasm and become potential targets for RNAi. This likely happened '\n",
      " 'at the moment between the uncoating of viral RNA and the viral replication '\n",
      " '[25] . Virus resistance has been proven to be generated by RNAi triggered '\n",
      " 'from exogenous DEN-specific dsRNA. Adelman et al. [20] showed this '\n",
      " 'resistance by a 290 nt dsRNA homologous to the type II dengue virus PrM gene '\n",
      " 'which was expressed by plasmid, in cultured mosquito C6/36 cells. Travanty '\n",
      " 'et al. [23] developed transgenic mosquito lines that transcribed the same '\n",
      " '290 nt dsRNA with insect promoters, but failed to express in critical site '\n",
      " 'for DEN replication such as midguts. Franz et al. [24] increased the size of '\n",
      " 'dsRNA to 578 nt and succeeded in midgut expression and viral transmission '\n",
      " 'diminishing. In addition to DEN-2, alternation of the replication kinetics '\n",
      " 'of DEN-1 has also proven to be triggered by dsRNA in C6/36 cells [26] . '\n",
      " 'However, long dsRNA fragments are associated with higher cost in synthesis, '\n",
      " 'poorer stability, and raise the chance for mismatch rate. At the same time, '\n",
      " 'introduction of dsRNAs longer than 30 base pairs into mammalian cells will '\n",
      " 'activate the potent interferon and protein kinase R antiviral pathways, '\n",
      " 'resulting in non-sequencespecific effects that can include apoptosis [27] . '\n",
      " 'Therefore, these disadvantages largely restrict the use of long dsRNA in '\n",
      " 'pre-clinical and clinical applications. siRNAs are degraded products of '\n",
      " 'dsRNA with 21-25 nt in length, and have no such disadvantages because of '\n",
      " 'their short length. They are also described as \"hallmark\" of RNAi in all '\n",
      " 'previously published papers. We hypotheses that the siRNA derived from the '\n",
      " 'same fraction can have the same inhibitory effect as the dsRNA did. '\n",
      " 'Therefore, in this study, we designed and synthesized 21 nt siRNAs against '\n",
      " 'the DEN-I viral PrM gene, and investigated their inhibition effects of '\n",
      " 'dengue virus replication in transfected C6/36 cells. In four siRNAs we '\n",
      " 'designed, only one showed the function to reduce CPE after DEN infection. As '\n",
      " 'expected, the location of this siRNA in DEN-1 is inside the corresponding '\n",
      " 'location of 290 nt dsRNA reported before in DEN-2. With transfection of the '\n",
      " 'selected siRNA, C6/36 cells showed reduced CPE, and increased cell survival '\n",
      " 'by 2.26 folds and eliminated viral RNA by about 97.54% compared to virus '\n",
      " 'infection only group, at the 7 th dpi. These data indicate that siRNA '\n",
      " 'against DEN viral genome can effectively inhibit viral RNA replication in '\n",
      " 'the C6/36 cells, protect host cell from viral attack, suggesting its '\n",
      " 'potential role in prevention and treatment of dengue fever. Recent study '\n",
      " 'have shown [19] that Aedes aegypti can produce dsRNAs homologous to dengue '\n",
      " 'viral genes and trigger an intrinsic siRNA anti-viral action, but this '\n",
      " 'endogenous anti-viral mechanism can not effectively inhibit the replication '\n",
      " 'of dengue virus. Data presented in our study revealed that exogenous siRNA '\n",
      " 'in Aedes albopictus cells is effective in inhibiting viral DNA replication. '\n",
      " 'Therefore, it is possible that the anti-viral mechanisms mediated by '\n",
      " 'exogenous and endogenous siRNAs may act synergistically in the protection of '\n",
      " 'cells from viral attacks, although this hypothesis needs further researches '\n",
      " 'to prove. Because the short siRNAs are unstable inside the cells, and also '\n",
      " 'diluted by continuous cell division, the siRNA contents in the cells decline '\n",
      " 'over time resulting in the weakening of interference effect. From the data '\n",
      " 'in this study, siRNAs can be successfully transfected into 66% of the cells '\n",
      " 'but this percentage gradually reduced with time and siRNA molecules only '\n",
      " 'retained in 8.9% of the cells at the seventh day post transfection. '\n",
      " 'Therefore, maintaining the continuity of RNA interference is crucial for the '\n",
      " 'promotion and application of RNAi technology. Alternative form of vectors, '\n",
      " 'such as retrovirus [28] and nano device [29] , will be applied in the '\n",
      " 'future. Since currently there are no effective therapies or vaccines against '\n",
      " 'Dengue fever, the use of RNAi to suppress dengue virus replication and '\n",
      " 'protect cells from viral attacks will undutiful provide a new research '\n",
      " 'strategy for the prevention and treatment of this disease. Our data showed '\n",
      " 'that synthetic siRNA against the DEN membrane glycoprotein precursor gene '\n",
      " 'effectively inhibited DEN viral RNA replication and increased C6/36 cell '\n",
      " 'survival rate. siRNA may offer a potential new strategy for prevention and '\n",
      " 'treatment of DEN infection. The stability and inhibitory efficiency of siRNA '\n",
      " 'need further improvement in the future. Type 1 DEN strain GZ02-218 and Aedes '\n",
      " 'albopictus C6/ 36 cell line were both from the Department of '\n",
      " 'Virolo-gy&Immunology in Guangzhou Center for Disease Control and Prevention. '\n",
      " \"C6/36 cells were grown at 28°C, in Eagle's minimal essential medium (MEM, \"\n",
      " 'Gibco, USA) supplemented with 10% fetal bovine serum (FBS, Gibco, USA), 100 '\n",
      " 'μg/mL penicillin and streptomycin, pH 7.4 (maintain medium). Cells were '\n",
      " 'passaged every 5 to 7 days to maintain exponential growth. DEN-1 strain '\n",
      " 'GZ02-218 was passaged by infecting monolayers of C6/ 36 cells and viruses '\n",
      " 'were harvested at 12-14 days. TCID 50 of virus was measured for viral '\n",
      " 'challenge. Four pairs of siRNAs against different parts DEN-1 viral genome '\n",
      " 'were designed online (Qiagen, German) based on the common sequences of the '\n",
      " 'epidemic strain GZ02-218 in Guangzhou City of China in 2002 (GenBank access '\n",
      " 'number EF079826), DEN-1 reference virus strain (GenBank access number '\n",
      " 'EU848545), and other DEN strains. Four siRNA fragments (DenSi-1~4) were '\n",
      " 'synthesized and purified by PAGE electrophoresis (Ambion, USA). siRNAs was '\n",
      " 'also labeled with FAM and purified for visualization after transfection '\n",
      " '(Ambion, USA). A siRNA fragment inconsistent containing sequence with DEN '\n",
      " 'was used as the negative control (Qiagen, German).  C6/36 cells were '\n",
      " 'cultured in maintain medium at 28°C, and then inoculated in 24 well plates '\n",
      " 'at 1 × 10 6 /well in 0.5 ml medium the day before transfection. When 80% 90% '\n",
      " 'cells grew into monolayer, they were transfected with siRNA molecules '\n",
      " 'according to the manual for HiPerFect Transfection Reagent kit (Qiagen, '\n",
      " 'German). 1.0 μg siRNA and 3 μl HiPerFect Transfection Reagent were used for '\n",
      " 'each well. After cultured at 28°C for 4 hours, the medium with transfection '\n",
      " 'reagent was removed. Cells were washed by MEM, challenged by 100 TCID 50 '\n",
      " 'DEN-1 and cultured at 33°C in maintain medium for further investigation. The '\n",
      " 'cytopathic effect (CPE) of DEN-1 infection on C6/ 36 cells, including cell '\n",
      " 'rounding, syncytium formation and cell death, was evaluated under a light '\n",
      " 'microscope, and scored based on the severity from 0 (no CPE observed, no '\n",
      " 'cell death) to the most severe level ++++ (CPE observed in 100% cells). At '\n",
      " 'the 7 th dpi, MTT assay was applied to measure cell viability. 100 μl MTT '\n",
      " 'was added to every well and incubated for 4 hours then replaced with 1 ml '\n",
      " 'DMSO. The 24-well plate was shaken at 37°C for 10 minutes and OD 490 of '\n",
      " 'every well was obtained in a microplate reader. The OD value obtained from '\n",
      " 'the normal control group was set as 100% viability, and the cell survival '\n",
      " 'rate of other groups was calculated as their respective OD value divided by '\n",
      " 'that of the control group. 7. Real-time RT-PCR detection of dengue virus RNA '\n",
      " 'in C6/ 36 cells At the 7 th dpi, C6/36 cells were collected and viral RNA '\n",
      " 'was isolated with QIAamp Viral RNA Extraction Kit (Qiagen, German), and '\n",
      " 'quantified with the dengue virus real-time fluorescent RT-PCR detection kit '\n",
      " '(Shenzhen Taitai Genomics, China). PCR reaction conditions were: one cycle '\n",
      " 'of 50°C for 30 min and 95°C for 3 min, then 40 cycles of 95°C for 5 sec and '\n",
      " '60°C for 40 sec. The relative amount of viral load in each group was '\n",
      " 'represented by CT value, and the changes between groups were calculated by '\n",
      " 'comparative CT (ΔCT) values. C6/36 cells were transfected with FAM-labeled '\n",
      " 'siRNAs and cultured for 7 days. Fluorescence signal was observed under '\n",
      " 'fluorescence microscope at 6 hours, 1, 3, 5, and 7 days. Cells were '\n",
      " 'harvested at every time point, and measured by flow cytometry. siRNA '\n",
      " 'positive cell rate was calculated as the percentage of cells containing '\n",
      " 'fluorescent signals. All values were presented as mean ± S.D. Statistical '\n",
      " 'significance was evaluated using the two-tailed Mann-Whitney U-test; P < '\n",
      " '0.05 was considered significant.',\n",
      " 'The rhizomes and roots of aconitine species, a genus of the family '\n",
      " 'Ranunculaceae, are commonly used in treatment for various illnesses such as '\n",
      " 'collapse, syncope, rheumatic fever, joints pain, gastroenteritis, diarrhea, '\n",
      " 'edema, bronchial asthma, and tumors. They are also involved in the '\n",
      " 'management of endocrinal disorders such as irregular menstruation [1, 2] . '\n",
      " 'However, the usefulness of this aconitine species component intermingles '\n",
      " 'with toxicity after it is administered to a diseased patient. So far, few '\n",
      " 'articles have recorded the misuse of aconitine medicinals with strong '\n",
      " 'emphasis and thus have referenced that the misuse of this medicinal can '\n",
      " 'result in severe cardio-and neurotoxicity [3] [4] [5] [6] [7] . In our past '\n",
      " 'research, it was evidenced that the aconitine component is the main active '\n",
      " \"ingredient in this species' root and rhizome, and is responsible for both \"\n",
      " 'therapeutic and toxic effects [8] . This medicinal has been tested for '\n",
      " 'cancerological and dermatological activities. Its application to disease '\n",
      " 'conditions proved to exhibit an activity that slowed down cancer tumor '\n",
      " 'growth and to cure serious cases of dermatosis. It was also found to have an '\n",
      " 'effect on postoperative analgesia [9] [10] [11] [12] . However, a previous '\n",
      " 'safety study has revealed that aconitine toxicity is responsible for its '\n",
      " 'restriction in clinical settings. Further studies are needed to explain the '\n",
      " 'cause of aconitine toxicity as well as to show whether the toxicity '\n",
      " 'supersedes its usefulness. A combined network analysis and in silico study '\n",
      " 'was once performed to obtain insight on the relationship between aconitine '\n",
      " 'alkaloid toxicity and the aconitine structure, and it was found that the '\n",
      " 'cardiotoxicity of aconitine is the primary cause of patient death. The '\n",
      " 'aconitine poison is similar to the poison created by some pivotal proteins '\n",
      " 'such as the ryanodine receptor (RYR1 and RYR2), the Gap junction α-1 protein '\n",
      " '(GJA1), and the sodium-calcium exchanger (SLC8A1) [9] [10] [11] [12] . '\n",
      " 'However, among all existing studies about the aconitine medicinal, no one '\n",
      " 'has reported detail of its specific binding target protein linked to '\n",
      " 'toxicity. Protein-Protein Interactions (PPIs) participate in many metabolic '\n",
      " 'processes occurring in living organisms such as the cellular communication, '\n",
      " 'immunological response, and gene expression control [13, 14] . A systematic '\n",
      " 'description of these interactions aids in the elucidation of '\n",
      " 'interrelationships among targets. The targeting of PPIs with small-molecule '\n",
      " 'compounds is becoming an essential step in a mechanism study [14] . The '\n",
      " 'present study was designed and undertaken to identify the critical protein '\n",
      " 'that can affect the cardiotoxicity of aconitine alkaloids. A PPI network '\n",
      " 'built by the STRING database is a physiological contact for the high '\n",
      " 'specificity that has been established for several protein molecules and has '\n",
      " 'stemmed from computational prediction, knowledge transfer between organisms, '\n",
      " 'and interactions aggregated from other databases [15] . The analysis of the '\n",
      " 'PPI network is based on nodes and edges and is always performed via cluster '\n",
      " 'analysis and centrality measurements [16, 17] . In cluster analysis, highly '\n",
      " 'interconnected nodes and protein target nodes are divided and used to form '\n",
      " 'sub-graphs. The reliability of the PPI network is identified by the content '\n",
      " 'of each sub-graph [18] . The variability in centrality measurements is '\n",
      " 'connected to the quantitative relationship between the protein targets and '\n",
      " 'its weightiness in the network [18] . Hence, PPI networks with protein '\n",
      " 'targets related to aconitine alkaloid cardiotoxicity must enable us to find '\n",
      " 'the most relevant protein for aconitine toxicity and to understand the '\n",
      " 'mechanism at the network level. In our research, the evaluation and '\n",
      " 'visualization analysis of essential proteins related to cardiotoxicity in '\n",
      " 'PPIs were performed by the ClusterONE and CytoNCA plugins in Cytoscape 3.5, '\n",
      " 'designed to find the potential protein targets via combination with '\n",
      " 'conventional integrated pharmacophore matching technology built in the '\n",
      " 'PharmMapper platform. Structural modification of a familiar natural product, '\n",
      " 'active compound, or clinical drug is an efficient method for designing a '\n",
      " 'novel drug. The main purpose of the structural modification is to reduce the '\n",
      " 'toxicity of the target compound while enhancing the utility of the drug [19] '\n",
      " '. The identification of the structure-function relationship is an essential '\n",
      " 'step in the drug discovery and design, the determination of the 3D protein '\n",
      " 'structures was the key step in identifying the internal interactions in the '\n",
      " 'ligand-receptor complexes. X-ray crystallography and NMR were the only '\n",
      " 'accepted techniques of determining the 3D protein structure. Although the 3D '\n",
      " 'structure obtained by these two powerful techniques are accurate and '\n",
      " 'reliable, they are time-consuming and costly [20] [21] [22] [23] [24] . With '\n",
      " 'the rapid development of structural bioinformatics and Computer-Aided Drug '\n",
      " 'Design (CADD) techniques in the last decade, computational structures are '\n",
      " 'becoming increasingly reliable. The application of structural bioinformatics '\n",
      " 'and CADD techniques can improve the efficiency of this process [25] [26] '\n",
      " '[27] [28] [29] [30] [31] [32] [33] [34] . The ligand-based quantitative '\n",
      " 'structure-toxicity relationship (QSTR) and receptor-based docking technology '\n",
      " 'are regarded as effective and useful tools in analysis of structure-function '\n",
      " 'relationships [35] [36] [37] [38] . The contour maps around aconitine '\n",
      " 'alkaloids generated by comparative molecular field analysis (CoMFA) and '\n",
      " 'comparative molecular similarity index analysis (CoMSIA) were combined with '\n",
      " 'the interactions between ligand substituents and amino acids obtained from '\n",
      " 'docking results to gain insight on the relationship between the structure of '\n",
      " 'aconitine alkaloids and their toxicity. Scores from functions were used to '\n",
      " 'evaluate the docking result. The value-of-fit score in MOE software reflects '\n",
      " 'the binding stability and affinity of the ligand-receptor complexes. When '\n",
      " 'screening for the most potential target for cardiotoxicity, the experimental '\n",
      " 'data was combined with the value-of-fit score by the NDCG (Normalized '\n",
      " 'Discounted Cumulative Gain). The possibility of a protein being a target of '\n",
      " 'cardiotoxicity corresponds with the consistency of this experimental data. '\n",
      " 'Since the pioneering paper entitled \"The Biological Functions of '\n",
      " 'Low-Frequency Phonons\" [39] was published in 1977, many investigations of '\n",
      " 'biomacromolecules from a dynamic point of view have occurred. These studies '\n",
      " 'have suggested that low-frequency (or terahertz frequency) collective '\n",
      " 'motions do exist in proteins and DNA [40] [41] [42] [43] [44] . Furthermore, '\n",
      " 'many important biological functions in proteins and DNA and their dynamic '\n",
      " 'mechanisms, such as cooperative effects [45] , the intercalation of drugs '\n",
      " 'into DNA [42] , and the assembly of microtubules [46] , have been revealed '\n",
      " 'by studying the low-frequency internal motions, as summarized in a '\n",
      " 'comprehensive review [40] . Some scientists have even applied this kind of '\n",
      " 'low-frequency internal motion to medical treatments [47, 48] . Investigation '\n",
      " 'of the internal motion in biomacromolecules and its biological functions is '\n",
      " 'deemed as a \"genuinely new frontier in biological physics,\" as announced in '\n",
      " 'the mission of some biotech companies (see, e.g., Vermont Photonics). In '\n",
      " 'order to consider the static structural information of the ligand-receptor '\n",
      " 'complex, dynamical information should be also considered in the process of '\n",
      " 'drug discovery [49, 50] . Finally, molecular dynamics was carried out to '\n",
      " 'verify the binding affinity and stability of aconitine alkaloids and the '\n",
      " 'most potential target. This present study may be instrumental in our future '\n",
      " 'studies for the synergism and attenuation of aconitine alkaloids and for the '\n",
      " 'exploitation of its clinical application potential. A flowchart of '\n",
      " 'procedures in our study is shown in Figure 1 . of-fit score by the NDCG '\n",
      " '(Normalized Discounted Cumulative Gain). The possibility of a protein being '\n",
      " 'a target of cardiotoxicity corresponds with the consistency of this '\n",
      " 'experimental data. Since the pioneering paper entitled \"The Biological '\n",
      " 'Functions of Low-Frequency Phonons\" [39] was published in 1977, many '\n",
      " 'investigations of biomacromolecules from a dynamic point of view have '\n",
      " 'occurred. These studies have suggested that low-frequency (or terahertz '\n",
      " 'frequency) collective motions do exist in proteins and DNA [40] [41] [42] '\n",
      " '[43] [44] . Furthermore, many important biological functions in proteins and '\n",
      " 'DNA and their dynamic mechanisms, such as cooperative effects [45] , the '\n",
      " 'intercalation of drugs into DNA [42] , and the assembly of microtubules [46] '\n",
      " ', have been revealed by studying the low-frequency internal motions, as '\n",
      " 'summarized in a comprehensive review [40] . Some scientists have even '\n",
      " 'applied this kind of low-frequency internal motion to medical treatments '\n",
      " '[47, 48] . Investigation of the internal motion in biomacromolecules and its '\n",
      " 'biological functions is deemed as a \"genuinely new frontier in biological '\n",
      " 'physics,\" as announced in the mission of some biotech companies (see, e.g., '\n",
      " 'Vermont Photonics). In order to consider the static structural information '\n",
      " 'of the ligand-receptor complex, dynamical information should be also '\n",
      " 'considered in the process of drug discovery [49, 50] . Finally, molecular '\n",
      " 'dynamics was carried out to verify the binding affinity and stability of '\n",
      " 'aconitine alkaloids and the most potential target. This present study may be '\n",
      " 'instrumental in our future studies for the synergism and attenuation of '\n",
      " 'aconitine alkaloids and for the exploitation of its clinical application '\n",
      " 'potential. A flowchart of procedures in our study is shown in Figure 1 . The '\n",
      " 'whole framework of the comprehensive in silico method for screening '\n",
      " 'potential targets and studying the quantitative structure-toxicity '\n",
      " 'relationship (QSTR). The 33 compounds were aligned over, under the '\n",
      " 'superimposition of the common moiety and Template Compound 6. The '\n",
      " 'statistical parameters for database alignment-q 2 , r 2 , F, and SEE-were '\n",
      " 'The whole framework of the comprehensive in silico method for screening '\n",
      " 'potential targets and studying the quantitative structure-toxicity '\n",
      " 'relationship (QSTR). The 33 compounds were aligned over, under the '\n",
      " 'superimposition of the common moiety and Template Compound 6. The '\n",
      " 'statistical parameters for database alignment-q 2 , r 2 , F, and SEE-were '\n",
      " 'Table 1 . The CoMFA model with the optimal number of 6 components presented '\n",
      " 'a q 2 of 0.624, an r 2 of 0.966, an F of 124.127, and an SEE of 0.043, and '\n",
      " 'contributions of the steric and electrostatic fields were 0.621 and 0.379, '\n",
      " 'respectively. The CoMSIA model with the optimal number of 4 components '\n",
      " 'presented a q 2 of 0.719, an r 2 of 0.901, an F of 157.458, and an SEE of '\n",
      " '0.116, and the contributions of steric, electrostatic, hydrophobic, hydrogen '\n",
      " 'bond acceptor, and hydrogen bond donor fields were 0.120, 0.204, 0.327, '\n",
      " '0.216, and 0.133, respectively. The statistical results proved that the '\n",
      " 'aconitine alkaloids QSTR model of CoMFA and CoMSIA under the database '\n",
      " 'alignment have adequate predictability. Experimental and predicted pLD 50 '\n",
      " 'values of both the training set and test set are shown in Figure 2 , and the '\n",
      " 'CoMFA ( Figure 2A ) and CoMSIA ( Figure 2B ) model gave the correlation '\n",
      " 'coefficient (r 2 ) value of 0.9698 and 0.977, respectively, which '\n",
      " 'demonstrated the internal robustness and external high prediction of the '\n",
      " 'QSTR models. Experimental and predicted pLD50 values of both the training '\n",
      " 'set and test set are shown in Figure  2 Residuals vs. Leverage Williams '\n",
      " 'plots of the aconitine QSTR models are shown in Figure 3A ,B. All values of '\n",
      " 'standardized residuals fall between 3σ and −3σ, and the values of leverage '\n",
      " 'are less than h*, so the two models demonstrate potent extensibility and '\n",
      " 'predictability. Residuals vs. Leverage Williams plots of the aconitine QSTR '\n",
      " 'models are shown in Figure 3A ,B. All values of standardized residuals fall '\n",
      " 'between 3σ and −3σ, and the values of leverage are less than h*, so the two '\n",
      " 'models demonstrate potent extensibility and predictability. Under MeSH '\n",
      " '(Medical Subject Headings), a total of 491 articles (261 articles were '\n",
      " 'received from Web of Science, and others were received from PubMed) were '\n",
      " 'retrieved. After selecting cardiotoxicity-related and excluding repetitive '\n",
      " 'articles, 274 articles were used to extract the correlative proteins and '\n",
      " 'pathways for building a PPI network in the STRING server. The correlative '\n",
      " 'proteins or pathways are shown in Table 2 . All proteins were taken as input '\n",
      " 'protein in the STRING database to find its direct and functional partners '\n",
      " '[51] , and proteins and its partners were then imported into the Cytoscape '\n",
      " '3.5 to generate the PPI network with 148 nodes and 872 edges ( Figure  4 ). '\n",
      " 'Potassium voltage-gated channel H2 7 SCN3A Sodium voltage-gated channel type '\n",
      " '3, 3 SCN2A Sodium voltage-gated channel type 2 3 SCN8A Sodium voltage-gated '\n",
      " 'channel type 8 2 SCN1A Sodium voltage-gated channel type 1 2 SCN4A Sodium '\n",
      " 'voltage-gated channel type 4 1 KCNJ3 Potassium inwardly-rectifying channel '\n",
      " 'J3 1 During the case of screening of the essential proteins in PPI network, '\n",
      " 'three centrality measurements (Subgraph Centrality, Betweenness Centrality, '\n",
      " 'and Closeness Centrality) in CytoNCA were utilized to evaluate the weight of '\n",
      " 'nodes. After removing the central node \"AC,\" the centrality measurements of '\n",
      " '147 nodes were calculated by CytoNCA and documented in Table S1 . The top '\n",
      " '10% of three centrality measurement values of all node are painted with a '\n",
      " 'different color in Figure 4A . To screen the node with the high values of '\n",
      " 'each three centrality measures, nodes with three colors were overlapped and '\n",
      " 'merged into sub-networks in Figure 4B . Under MeSH (Medical Subject '\n",
      " 'Headings), a total of 491 articles (261 articles were received from Web of '\n",
      " 'Science, and others were received from PubMed) were retrieved. After '\n",
      " 'selecting cardiotoxicity-related and excluding repetitive articles, 274 '\n",
      " 'articles were used to extract the correlative proteins and pathways for '\n",
      " 'building a PPI network in the STRING server. The correlative proteins or '\n",
      " 'pathways are shown in Table 2 . All proteins were taken as input protein in '\n",
      " 'the STRING database to find its direct and functional partners [51] , and '\n",
      " 'proteins and its partners were then imported into the Cytoscape 3.5 to '\n",
      " 'generate the PPI network with 148 nodes and 872 edges ( Figure 4 ). Table 2 '\n",
      " '. Proteins related to aconitine alkaloids induced cardiotoxicity extracted '\n",
      " 'from 274 articles. Classification Frequency Ryanodine receptor 2 19 RYR1 '\n",
      " 'Ryanodine receptor 1 15 GJA1 Gap junction α-1 protein (connexin43) 13 SLC8A1 '\n",
      " 'Sodium/calcium exchanger 1 11 ATP2A1 Calcium transporting ATPase fast twitch '\n",
      " '1 9 KCNH2 Potassium voltage-gated channel H2 7 SCN3A Sodium voltage-gated '\n",
      " 'channel type 3, 3 SCN2A Sodium voltage-gated channel type 2 3 SCN8A Sodium '\n",
      " 'voltage-gated channel type 8 2 SCN1A Sodium voltage-gated channel type 1 2 '\n",
      " 'SCN4A Sodium voltage-gated channel type 4 1 KCNJ3 Potassium '\n",
      " 'inwardly-rectifying channel J3 1 During the case of screening of the '\n",
      " 'essential proteins in PPI network, three centrality measurements (Subgraph '\n",
      " 'Centrality, Betweenness Centrality, and Closeness Centrality) in CytoNCA '\n",
      " 'were utilized to evaluate the weight of nodes. After removing the central '\n",
      " 'node \"AC,\" the centrality measurements of 147 nodes were calculated by '\n",
      " 'CytoNCA and documented in Table S1 . The top 10% of three centrality '\n",
      " 'measurement values of all node are painted with a different color in Figure '\n",
      " '4A . To screen the node with the high values of each three centrality '\n",
      " 'measures, nodes with three colors were overlapped and merged into '\n",
      " 'sub-networks in Figure 4B . In the sub-networks, the voltage-gated calcium '\n",
      " 'and sodium channel accounted for a large proportion, which is consistent '\n",
      " 'with our research in clustering the network (Clusters 1, 2, and 9). All '\n",
      " 'proteins in the sub-networks will be utilized to predict the results of the '\n",
      " 'PharmMapper server to receive the potential target of cardiotoxicity induced '\n",
      " 'by aconitine alkaloids (in Figure 5A ,B). In the meantime, 2V7O (CAMK2G) and '\n",
      " '2VZ6 (CAMK2A) were identified as the potential targets with higher fit '\n",
      " 'scores. In the sub-networks, the voltage-gated calcium and sodium channel '\n",
      " 'accounted for a large proportion, which is consistent with our research in '\n",
      " 'clustering the network (Clusters 1, 2, and 9). All proteins in the '\n",
      " 'sub-networks will be utilized to predict the results of the PharmMapper '\n",
      " 'server to receive the potential target of cardiotoxicity induced by '\n",
      " 'aconitine alkaloids (in Figure 5A ,B). In the meantime, 2V7O (CAMK2G) and '\n",
      " '2VZ6 (CAMK2A) were identified as the potential targets with higher fit '\n",
      " 'scores. All compounds were docked into three potential targets. The values '\n",
      " 'of nDCG are shown in Table  3 . The Dock study of three proteins with an '\n",
      " 'NDCG of 0.8503 and 0.9122, respectively (the detailed In the sub-networks, '\n",
      " 'the voltage-gated calcium and sodium channel accounted for a large '\n",
      " 'proportion, which is consistent with our research in clustering the network '\n",
      " '(Clusters 1, 2, and 9). All proteins in the sub-networks will be utilized to '\n",
      " 'predict the results of the PharmMapper server to receive the potential '\n",
      " 'target of cardiotoxicity induced by aconitine alkaloids (in Figure 5A ,B). '\n",
      " 'In the meantime, 2V7O (CAMK2G) and 2VZ6 (CAMK2A) were identified as the '\n",
      " 'potential targets with higher fit scores. All compounds were docked into '\n",
      " 'three potential targets. The values of nDCG are shown in Table  3 . The Dock '\n",
      " 'study of three proteins with an NDCG of 0.8503 and 0.9122, respectively (the '\n",
      " 'detailed All compounds were docked into three potential targets. The values '\n",
      " 'of nDCG are shown in Table 3 . The Dock study of three proteins with an NDCG '\n",
      " 'of 0.8503 and 0.9122, respectively (the detailed docking result is shown in '\n",
      " 'Table S2 ) proves that the result of the Dock study of 2V7O is consistent '\n",
      " 'with the Experimental pLD 50 , so the protein 2V7O was utilized for the '\n",
      " 'ligand interaction analysis. Table 3 . Ranking results by experimental and '\n",
      " 'predicted pLD 50 and fit score. Experimental pLD 50 Fit Score (2V7O) Fit '\n",
      " 'Score (2VZ6)   6  1  3  3  20  2  1  12  12  3  4  9  1  4  2  4  11  5  7  '\n",
      " '2  14  6  8  13  16  7  5  6  7  8  17  15  8  9  10  11  27  10  23  17  '\n",
      " '13  11  12  19  15  12  11  5  32  13  18  18  5  14  22  8  33  15  13  29  '\n",
      " '21  16  15  1  25  17  9  20  22  18  25  25  17  19  20  16  28  20  24  '\n",
      " '30  9  21  16  32  29  22  32  14  2  23  30  24  30  24  31  26  18  25  '\n",
      " '21  27  10  26  26  21  23  27  29  31  31  28  33  7  26  29  14  23  4  '\n",
      " '30  28  33  3  31  6  10  19  32  27  28  24  33  19  22  NDCG  1 0.9122 '\n",
      " '0.8503 The 3D-QSTR contour maps were utilized to visualize the information '\n",
      " 'on the CoMFA and CoMSIA model properties in three-dimensional space. These '\n",
      " 'maps used characteristics of compounds that are crucial for activity and '\n",
      " 'display the regions around molecules where the variance of activities is '\n",
      " 'expected based on physicochemical property changes in molecules [52] . The '\n",
      " 'analysis of favorable and unfavorable regions of steric, electrostatic, '\n",
      " 'hydrophobic, HBD, and HBA atom fields contribute to the realization of the '\n",
      " \"relationship between the aconitine alkaloid's toxic activity and its \"\n",
      " 'structure. Steric and electrostatic contour maps of the CoMFA QSTR model are '\n",
      " 'shown in Figure 4A ,B, respectively. Hydrophobic, HBD, and HBA contour maps '\n",
      " 'of the CoMSIA QSTR model are shown in Figure 4C -E. Compound 6 has the most '\n",
      " 'toxic activity, so it was chosen as the reference structure for the '\n",
      " 'generation of the CoMFA and CoMSIA contour maps. In the case of the CoMFA '\n",
      " 'study, the steric contour map around Compound 6 is shown in Figure 6A . The '\n",
      " 'yellow regions near R2, R7, and R6 showed the substituents of the molecule, '\n",
      " 'which proved that these positions were not ideal for sterically favorable '\n",
      " 'functional groups. Therefore, Compounds 19, 24, and 26 (with pLD 50 values '\n",
      " 'of 1.17, 0.84, and 1.82, respectively), which consist of sterically '\n",
      " 'esterified moieties at Positions R2 and R7, were less toxic than Compounds 6 '\n",
      " 'and 20 (with pLD 50 values of 5.00 and 4.95), which were substituted by a '\n",
      " 'small hydroxyl group, and Compound 3 (with a pLD 50 value of 1.44) has less '\n",
      " 'toxic activity due to the esterified moiety in R6. The green regions, '\n",
      " 'sterically favorable The CoMFA electrostatic contour map is shown in Figure '\n",
      " '6B . The blue region near the R2 and R7 substitution revealed that the '\n",
      " 'replacement of electropositive groups is in favor of toxicity. This can be '\n",
      " 'proven by the fact that the compounds with hydroxy in these two positions '\n",
      " 'had higher pLD 50 values than the compound with acetoxy or no substituents. '\n",
      " 'The red region surrounding molecular scaffolds was not distinct, which '\n",
      " 'revealed that there was no connection between the electronegative and the '\n",
      " 'toxicity. The CoMSIA hydrophobic contour map is shown in Figure 6C . The R2, '\n",
      " 'R6, and R7 around the white region indicated that the hydrophobic groups '\n",
      " 'were unfavorable for the toxicity, so the esterification of hydrophilic '\n",
      " 'hydroxyl or dehydroxylation decreased the toxicity, which is consistent with '\n",
      " 'the steric and electrostatic contour map. The yellow contour map near the '\n",
      " 'R12 manifested that the hydrophilic hydroxy was unfavorable to the toxicity, '\n",
      " 'which can be validated by the fact that aconitine alkaloids with hydroxy '\n",
      " 'substituents in R12 (Compound 10, with a pLD 50 The PPI network of aconitine '\n",
      " 'alkaloids cardiotoxicity was divided into nine clusters using ClusterONE. '\n",
      " 'Statistical parameters are shown in Figure 5 . Six clusters, namely Clusters '\n",
      " '1, 3, 4, 5, 7, and 9, which possess quality scores higher than 0.5, a '\n",
      " 'density higher than 0.45, and a p-value less than 0.05, were selected for '\n",
      " 'further analysis (in Figure 7) . Clusters 1, 4, and 7 consisted of proteins '\n",
      " 'mainly involved in the effects of various calcium, potassium, and sodium '\n",
      " 'channels. Cluster 1 mainly The CoMSIA contour map of HBD is shown in Figure '\n",
      " '6D . The cyan regions at R2, R6, and R7 represented a favorable condition '\n",
      " 'for the HBD atom, which clearly validated the fact that the compounds with '\n",
      " 'hydroxy in this region show potent toxicity. A purple region was found near '\n",
      " 'R12, which proved that the HBD atom (hydroxyl) in this region has an adverse '\n",
      " 'effect on toxicity. The HBA contour map is shown in Figure 6 . The magenta '\n",
      " 'region around R1 substitution proved that this substitution was favorable to '\n",
      " 'the HBA atom, so Compounds 13, 15, 32, and 33 with the HBA atom in the R1 '\n",
      " 'substitution exhibit more potent toxicity (with pLD 50 values of 3.52, 3.30, '\n",
      " '3.16, and 2.84) than compounds with methoxymethyl substituents (Compounds '\n",
      " '19, 24, and 26 with pLD 50 values of 1.17, 0.84, and 1.82). The red contour '\n",
      " 'map where HBA atoms are unfavorable for the toxicity was positioned around '\n",
      " 'R2 and R6. These contours were well validated by the lower pLD 50 value of '\n",
      " 'compounds with carbonyl in these substitutions. The PPI network of aconitine '\n",
      " 'alkaloids cardiotoxicity was divided into nine clusters using ClusterONE. '\n",
      " 'Statistical parameters are shown in Figure 5 . Six clusters, namely Clusters '\n",
      " '1, 3, 4, 5, 7, and 9, which possess quality scores higher than 0.5, a '\n",
      " 'density higher than 0.45, and a p-value less than 0.05, were selected for '\n",
      " 'further analysis (in Figure 7) . Clusters 1, 4, and 7 consisted of proteins '\n",
      " 'mainly involved in the effects of various calcium, potassium, and sodium '\n",
      " 'channels. Cluster 1 mainly consisted of three channel types related to the '\n",
      " 'cardiotoxicity of aconitine alkaloids, Cluster 4 contained calcium and '\n",
      " 'sodium channels and some channel exchangers (such as RYR1 and RYR2), and '\n",
      " 'Cluster 7 mainly consisted of various potassium channels. All of these '\n",
      " 'findings are consistent with previous research about the arrhythmogenic '\n",
      " 'properties of the toxicity of aconitine alkaloids: the aconitine binds to '\n",
      " 'ion channels and affects their open state, and thus the corresponding ion '\n",
      " 'influx into the cytosol [53] [54] [55] . The channel exchangers play a '\n",
      " 'crucial role in keeping the ion transportation and homeostasis inside and '\n",
      " 'outside of the cell. Cluster 9 contained some regulatory proteins that can '\n",
      " 'activate or repress the ion channels through the protein expression level. '\n",
      " 'ATP2A1, RYR2, RYR1, CACNA1C, CACNA1D, and CACNA1S mediate the release of '\n",
      " 'calcium, thereby playing a key role in triggering cardiac muscle contraction '\n",
      " 'and maintaining the calcium homeostasis [56, 57] . Aconitine may cause '\n",
      " 'aberrant channel activation and lead to cardiac arrhythmia. Clusters 3 and 5 '\n",
      " 'consisted of cAMP-dependent protein kinase (cAPK), cGMP-dependent protein '\n",
      " 'kinase (cGPK), and guanine nucleotide binding protein (G protein). They have '\n",
      " 'not been fully studied to prove whether the cardiotoxicity induced by '\n",
      " 'aconitine alkaloids is linked to the cAPK, cGPK, and G proteins; however, '\n",
      " 'some studies have shown that cardiotoxicity-related protein KCNJ3 (potassium '\n",
      " 'inwardly-rectifying channel) is controlled by G proteins and the cardiac '\n",
      " 'sodium/calcium exchanger and is said to be regulated by cAPK and cGPK [58, '\n",
      " '59] . The result of ClusterONE indicated that the constructed network is '\n",
      " 'consistent with existing studies and that the network can be used to screen '\n",
      " 'essential proteins in the CytoNCA plugin. The protein 2V7O belonging to the '\n",
      " 'CaMKII (Calcium/Calmodulin (Ca 2+ /CaM)-dependent serine/threonine kinases '\n",
      " 'II) isozyme protein family plays a central role in cellular signaling by '\n",
      " 'transmitting Ca 2+ signals. The CaMKII enzymes transmit calcium ion (Ca 2+ ) '\n",
      " 'signals released inside the cell by regulating signal transduction pathways '\n",
      " 'through phosphorylation. Ca 2+ first binds to the small regulatory protein '\n",
      " 'CaM, and this Ca 2+ /CaM complex then binds to and activates the kinase, '\n",
      " 'which then phosphorylates other proteins such as ryanodine receptor and '\n",
      " 'sodium/calcium exchanger. Thus, these proteins are related to the '\n",
      " 'cardiotoxicity induced by aconitine alkaloids [60] [61] [62] . The excessive '\n",
      " 'activity of CaMKII has been observed in some structural heart disease and '\n",
      " 'arrhythmias [63] , and past findings demonstrate neuroprotection in neuronal '\n",
      " 'cultures treated with inhibitors of CaMKII immediately prior to excitotoxic '\n",
      " 'activation of the CaMKII [64] . The acute cardiotoxicity of the aconitine '\n",
      " 'alkaloids is possibly related to this target. Based on the analysis of the '\n",
      " 'PPI network above, CaMKII was selected as the potential target for further '\n",
      " 'molecular docking and dynamic simulation. The dock result of 2V7O is shown '\n",
      " 'in Figure 8A . Compound 20 has the highest fit scores, so it was selected as '\n",
      " 'the template for conformational analysis. The mechanisms of CaMKII '\n",
      " 'activation and inactivation are shown in Figure 8B . Compound 20 affects the '\n",
      " 'normal energy metabolism of the myocardial cell via binding in the '\n",
      " 'ATP-competitive site in Figure 8C . The inactive state of the CaMKII was '\n",
      " 'regulated by CASK-mediated T306/T307 phosphorylation, and this state can be '\n",
      " 'inhibited by the binding of Compound 20 in the ATP-competitive site. Such '\n",
      " 'binding moves CaMKII toward a Ca 2+ /CaM-dependent activation active state '\n",
      " 'and a Ca 2+ /CaM-dependent activation through structural rearrangement of '\n",
      " 'the inhibitory helix caused by Ca 2+ /CaM binding and the subsequent '\n",
      " 'autophosphorylation of T287 [65] , which will induce the excessive activity '\n",
      " 'of CaMKII and dynamic imbalance of the calcium ions in the myocardial cell, '\n",
      " 'eventually leading to heart disease and arrhythmias. Molecules 2018, 23, x '\n",
      " 'FOR PEER REVIEW 10 of 24 channel) is controlled by G proteins and the '\n",
      " 'cardiac sodium/calcium exchanger and is said to be regulated by cAPK and '\n",
      " 'cGPK [58, 59] . The result of ClusterONE indicated that the constructed '\n",
      " 'network is consistent with existing studies and that the network can be used '\n",
      " 'to screen essential proteins in the CytoNCA plugin. The protein 2V7O '\n",
      " 'belonging to the CaMKII (Calcium/Calmodulin (Ca 2+ /CaM)-dependent '\n",
      " 'serine/threonine kinases II) isozyme protein family plays a central role in '\n",
      " 'cellular signaling by transmitting Ca 2+ signals. The CaMKII enzymes '\n",
      " 'transmit calcium ion (Ca 2+ ) signals released inside the cell by regulating '\n",
      " 'signal transduction pathways through phosphorylation. Ca 2+ first binds to '\n",
      " 'the small regulatory protein CaM, and this Ca 2+ /CaM complex then binds to '\n",
      " 'and activates the kinase, which then phosphorylates other proteins such as '\n",
      " 'ryanodine receptor and sodium/calcium The information of a binding pocket of '\n",
      " 'a receptor for its ligand is very important for drug design, particularly '\n",
      " 'for conducting mutagenesis studies [28] . As has been reported in the past '\n",
      " '[66] , the binding pocket of a protein receptor to a ligand is usually '\n",
      " 'defined by those residues that have at least one heavy atom within a '\n",
      " 'distance of 5 Å from a heavy atom of the ligand. Such a criterion was '\n",
      " 'originally used to define the binding pocket of ATP in the Cdk5-Nck5a '\n",
      " 'complex [20] , which was later proved to be very useful in identifying '\n",
      " 'functional domains and stimulating the relevant truncation experiments. A '\n",
      " 'similar approach has also been used to define the binding pockets of many '\n",
      " 'other receptor-ligand interactions important for drug design [30, 31, 33, '\n",
      " '[67] [68] [69] [70] . The information of a binding pocket of CaMKII for the '\n",
      " 'aconitine alkaloids will serve as a guideline for designing drugs with '\n",
      " 'similar scaffolds, particularly for conducting mutagenesis studies. In '\n",
      " 'Figure 8A , four top fit scores-Compounds 1, 6, 12, and 20-generated similar '\n",
      " 'significant interactions with amino acid residues around the ATP-competitive '\n",
      " 'binding pocket. Four compounds formed with many Van Der Waals interactions '\n",
      " 'within the noncompetitive inhibitor pocket through amino acid residues such '\n",
      " 'as Asp157, Lys43, Glu140, Lys22, and Leu143. The ligand-receptor interaction '\n",
      " 'showed that the hydroxy in R2 formed a side chain donor interaction with '\n",
      " 'Asp157. In addition, the hydroxy in R6 and R7 also formed a side chain '\n",
      " 'acceptor interaction with Glu140 and Ser26, respectively (the docking result '\n",
      " 'of Compounds 6 and 12 in Figure 8A ). These results correspond to the CoMFA '\n",
      " 'and CoMSIA contour maps. However, the small electropositive and hydrophilic '\n",
      " 'group in R2, R6, and R7 possess a certain enhancement function to toxicity. '\n",
      " 'There were aromatic interactions between the phenyl group in R9 and amino '\n",
      " 'acid residues. The phenyl group in R9 formed aromatic interactions with '\n",
      " 'Leu20, Leu142, and Phe90, while the small group hydroxyl did not form any '\n",
      " 'interaction with Asp91, which demonstrate that bulky phenyl group is crucial '\n",
      " 'to this binding pattern and toxicity. This was mainly equal to the CoMFA '\n",
      " 'steric contour map, where R9 was ideal for sterically favorable groups. The '\n",
      " 'methoxymethyl R1 generated backbone acceptor with Lys43, which correspond to '\n",
      " 'the CoMSIA HBA contour map, where R1 was favorable for the HBA atom. '\n",
      " 'Compound 20 docked into 2V7O, and the ATP-competitive pocket was painted '\n",
      " 'green; the T287, T307, and T308 phosphorylation sites were painted green, '\n",
      " 'orange, and yellow, respectively; the inhibitory helix was painted red. The '\n",
      " 'result of MD simulation is shown in Figure 9 . The red plot represented the '\n",
      " 'RMSD values of the docked protein. The values of RMSD reached 2.41 Å in 1.4 '\n",
      " 'ns and then remained between 2 and 2.5 Å throughout the simulation for up to '\n",
      " '5 ns. The averaged value of the RMSD was 2.06 Å. The MD simulation '\n",
      " 'demonstrated that the ligand was stabilized in the active site. Finally, we '\n",
      " 'combined the ligand-based 3D-QSTR analysis with the structure-based '\n",
      " 'molecular docking study to identify the necessary moiety related to the '\n",
      " 'cardiotoxicity mechanism of the aconitine alkaloids (in Figure 10 ). '\n",
      " 'Finally, we combined the ligand-based 3D-QSTR analysis with the '\n",
      " 'structure-based molecular docking study to identify the necessary moiety '\n",
      " 'related to the cardiotoxicity mechanism of the aconitine alkaloids (in '\n",
      " 'Figure 10 ).  To build the PPI network of aconitine alkaloids, literature '\n",
      " 'from 1 January 2007 to 31 February 2017 was retrieved from PubMed '\n",
      " '(http://pubmed.cn/) and Web of Science (http://www.isiknowledge.com/) with '\n",
      " 'the MeSH word \"aconitine\" and \"toxicity\" and without language restriction. '\n",
      " 'All documents about cardiotoxicity caused by aconitine alkaloids were '\n",
      " 'collected. The proteins related to the aconitine alkaloids cardiotoxicity of '\n",
      " 'this decade were gathered and taken as the input protein in the STRING '\n",
      " '(https://string-db.org/) database [51, 71] , used to search for related '\n",
      " 'proteins or pathways that had been reported. Finally, all the proteins and '\n",
      " 'its partners were recorded in Excel in order to import information and build '\n",
      " 'a PPI network in Cytoscape software. Cytoscape is a free, open-source, Java '\n",
      " 'application for visualizing molecular networks and integrating them with '\n",
      " 'gene expression profiles [71, 72] . Plugins are available for network and '\n",
      " 'molecular profiling analyses, new layouts, additional file format support, '\n",
      " 'making connections with Figure 10 . Crucial requirement of cardiotoxicity '\n",
      " 'mechanism was obtained from the ligand-based 3D-QSTR and structure-based '\n",
      " 'molecular docking study. To build the PPI network of aconitine alkaloids, '\n",
      " 'literature from 1 January 2007 to 31 February 2017 was retrieved from PubMed '\n",
      " '(http://pubmed.cn/) and Web of Science (http://www.isiknowledge.com/) with '\n",
      " 'the MeSH word \"aconitine\" and \"toxicity\" and without language restriction. '\n",
      " 'All documents about cardiotoxicity caused by aconitine alkaloids were '\n",
      " 'collected. The proteins related to the aconitine alkaloids cardiotoxicity of '\n",
      " 'this decade were gathered and taken as the input protein in the STRING '\n",
      " '(https://string-db.org/) database [51, 71] , used to search for related '\n",
      " 'proteins or pathways that had been reported. Finally, all the proteins and '\n",
      " 'its partners were recorded in Excel in order to import information and build '\n",
      " 'a PPI network in Cytoscape software. Cytoscape is a free, open-source, Java '\n",
      " 'application for visualizing molecular networks and integrating them with '\n",
      " 'gene expression profiles [71, 72] . Plugins are available for network and '\n",
      " 'molecular profiling analyses, new layouts, additional file format support, '\n",
      " 'making connections with databases, and searching within large networks [71] '\n",
      " '. ClusterONE (Clustering with Overlapping Neighborhood Expansion) of '\n",
      " 'Cytoscape was utilized to cluster the PPI network into overlapping '\n",
      " 'sub-graphs of highly interconnected nodes. ClusterONE is a plugin for '\n",
      " 'detecting and clustering potentially overlapping protein complexes from PPI '\n",
      " 'data. The quality of a group was assessed by the number of sub-graphs, '\n",
      " 'p-values, and density. The cluster was discarded when the number of '\n",
      " 'sub-graphs was smaller than 3, the density was less than 0.45, the quality '\n",
      " 'was less than 0.5, and the p-value was under 0.05 [73] . The clustering '\n",
      " 'results of the ClusterONE are instrumental to understanding how the '\n",
      " \"reliability of the PPI network relates to aconitine alkaloids' \"\n",
      " 'cardiotoxicity. CytoNCA is a plugin in Cytoscape integrating calculation, '\n",
      " 'evaluation, and visualization analysis for multiple centrality measures. '\n",
      " 'There are eight centrality measurements provided by CytoNCA: betweenness, '\n",
      " 'closeness, degree, eigenvector, local average connectivity-based, network, '\n",
      " 'subgraph, and information centrality [74] . The primary purpose of the '\n",
      " 'centrality analysis was to confirm the essential proteins in the pre-built '\n",
      " 'PPI network. The three centrality measurements in the CytoNCA '\n",
      " 'plugin-subgraph centrality, betweenness centrality, and closeness '\n",
      " 'centrality-were used for evaluating and screening the essential protein in '\n",
      " 'the merged target network. The subgraph centrality characterizes the '\n",
      " 'participation of each node in all subgraphs in a network. Smaller subgraphs '\n",
      " 'are given more weight than larger ones, which makes this measurement an '\n",
      " 'appropriate one for characterizing network properties. The subgraph '\n",
      " 'centrality of node \"u\" can be calculated by [75]  µ l (u) is the uth '\n",
      " 'diagonal entry of the lth power of the weight adjacency matrix of the '\n",
      " 'network. v 1 , v 2 , . . . , v N is be an orthonormal basis composed of R N '\n",
      " 'composed by eigenvectors of A associated to the eigenvalues λ 1 , λ 2 , . . '\n",
      " '. , λ N v u v , which is the uth component of v v [75] . The betweenness '\n",
      " 'centrality finds a wide range of applications in network theory. It '\n",
      " 'represents the degree to which nodes stand between each other. Betweenness '\n",
      " 'centrality was devised as a general measure of centrality. It is applicable '\n",
      " 'to a wide range of problems in network theory, including problems related to '\n",
      " 'social networks, biology, transport, and scientific cooperation. The '\n",
      " 'betweenness centrality of a node u can be calculated by [76]  ρ (s, t) is '\n",
      " 'the total number of shortest paths from node s to node ρ (s, u, t), which is '\n",
      " 'the number of those paths that pass through u. Closeness centrality of a '\n",
      " 'node is a measure of centrality in a network, calculated as the sum of the '\n",
      " 'length of the shortest paths between the node and all other nodes in the '\n",
      " 'graph. Thus, the more central a node is, the closer it is to all other '\n",
      " 'nodes. The closeness centrality of a node u can be calculated by [77]  |Nu| '\n",
      " \"is the number of node u's neighbors, and dist (u, v) is the distance of the \"\n",
      " 'shortest path from node u to node v. PharmMapper serves as a valuable tool '\n",
      " 'for identifying potential targets for a novel synthetic compound, a newly '\n",
      " 'isolated natural product, a compound with known biological activity, or an '\n",
      " 'existing drug [78] . Of all the aconitine alkaloids in this research, '\n",
      " 'Compounds 6, 12, and 20 exhibited the most toxic activity and were used for '\n",
      " 'the potential target prediction. The Mol2 format of three compounds was '\n",
      " 'submitted to the PharmMapper server. The parameters of Generate Conformers '\n",
      " 'and Maximum Generated Conformations was set as ON and 300, respectively. '\n",
      " 'Other parameters used default values. Finally, the result of the ClusterONE '\n",
      " 'and PharmMapper will be combined together to select the potential targets '\n",
      " 'for the following docking study [78] . Comparative molecular field analysis '\n",
      " '(CoMFA) and comparative molecular similarity index analysis (CoMSIA) are '\n",
      " 'efficient tools in ligand-based drug design and are in use for contour map '\n",
      " 'generation and identification of favorable and unfavorable regions in a '\n",
      " 'moiety [52, 79] . The CoMFA consists of a steric and electrostatic contour '\n",
      " 'map of molecules that are correlated with toxic activity, while the CoMSIA '\n",
      " 'consists of hydrophobic field, hydrogen bond donor (HBD)/hydrogen bond '\n",
      " 'acceptor (HBA) [80] , and steric/electrostatic fields that are correlated '\n",
      " 'with toxic activity. The CoMFA and CoMSIA have been utilized to generate a '\n",
      " '3D-QSTR model [81] . All molecule models and the generation of 3D-QSTR were '\n",
      " 'performed with SYBYL X2.0. Alkaloids in mice with LD 50 values listed in '\n",
      " 'Table 4 were extracted from recent literature [70] . The LD 50 values of all '\n",
      " 'aconitine alkaloids were converted into pLD 50 with a standard TRIPOS force '\n",
      " 'field. These pLD50 values were used as a dependent variable, while CoMFA and '\n",
      " 'CoMSIA descriptors were used as an independent variable. The SKETCH function '\n",
      " 'of Sybyl X2.0 was utilized to illustrate structure and charges, and was '\n",
      " 'calculated by the Gasteiger-Huckel method. Additionally, the tripose force '\n",
      " 'field was utilized for energy minimization of these aconitine alkaloid '\n",
      " 'molecules [81] . The 31 molecules were divided into a ratio of 3:1. The '\n",
      " 'division was done in a way that showed that both datasets are balanced and '\n",
      " 'consist of both active and less active molecules [81] . The reliability of '\n",
      " 'the 3D-QSTR model depends on the database molecular alignment. The most '\n",
      " 'toxic aconitine alkaloids (Compound 6) was selected as the template '\n",
      " 'molecule, and the tetradecahydro-2H-3,6,12-(epiethane [1,1,2] '\n",
      " 'triyl)-7,9-methanonaphtho [2,3-b] azocine was selected as the common moiety. '\n",
      " 'PLS (partial least squares) techniques are associated with field descriptors '\n",
      " 'with activity values such as [80] Leave One Out (LOO) values, the optimal '\n",
      " 'number of components, the Standard Error of Estimation (SEE), '\n",
      " 'cross-validated coefficients (q 2 ), and the conventional coefficient (r 2 '\n",
      " '). These statistical data are pivotal in the evaluation of the 3D-QSTR model '\n",
      " 'and can be worked out in the PLS method [81] . The model is said to be good '\n",
      " 'when the q 2 value is more than 0.5 and the r 2 value is more than 0.6. The '\n",
      " \"q 2 and r 2 values reflect a model's soundness. The best model has the \"\n",
      " 'highest q 2 and r 2 values, the lowest SEE, and an optimal number of '\n",
      " 'components [80, 82, 83] . In the case of CoMFA and CoMSIA analysis, the '\n",
      " 'values of the optimal number of components, SEE, and q 2 can be worked out '\n",
      " 'by LOO validation, with USE SAMPLS turned on and components set to 5, while '\n",
      " 'in the process of calculating r 2 , the USE SAMPLS was turned off and the '\n",
      " 'column filtration was set to 2.0 kcal mol −1 in order to speed up the '\n",
      " 'calculation without the need to sacrifice information content [81] [82] [83] '\n",
      " '[84] . Therefore, components were set to 6 and 4, respectively, which were '\n",
      " 'optimal numbers of components calculated by performing a SAMPLS run. SEE and '\n",
      " 'r 2 were utilized to assess the non-cross validated model. The Applicability '\n",
      " 'Domain (AD) of the Topomer CoMFA and CoMSIA model was confirmed by the '\n",
      " 'Williams plot of Residuals vs. Leverage. Leverage of a query chemical is '\n",
      " 'proportional to its Mahalanobis distance measure from the centroid of the '\n",
      " 'training set [85, 86] . The leverages are calculated for a given dataset X '\n",
      " 'by obtaining the leverage matrix (H) with the equation below: X is the model '\n",
      " 'matrix, while XT is its transpose matrix. The plot of standardized residuals '\n",
      " 'vs. leverage values was drawn, and compounds with standardized residuals '\n",
      " 'greater than three standard deviation units (±3σ) were considered as '\n",
      " 'outliers [85] . The critical leverage value is considered 3 P/n, where p is '\n",
      " 'the number of model variables plus one, and n is the number of objects used '\n",
      " 'to calculate the model. h > 3 P/n mean predicted response is not acceptable '\n",
      " '[85] [86] [87] . (CADD) software program that incorporates the functions of '\n",
      " 'QSAR, molecular docking, molecular dynamics, ADME (absorption, distribution, '\n",
      " 'metabolism, and excretion), and homologous modeling. All of these functions '\n",
      " 'are regarded as conducive instruments in the field of drug discovery and '\n",
      " 'biochemistry. The molecular docking and dynamics technology were performed '\n",
      " 'in MOE2016 software to detect the stability and affinity between the ligands '\n",
      " 'and predictive targets [88, 89] . The docking process involves the '\n",
      " 'prediction of ligand conformation and orientation within a targeted binding '\n",
      " 'site. Docking analysis is an important step in the docking process. It has '\n",
      " 'been widely used to study the reasonable binding mode and obtain information '\n",
      " 'of interactions between amino acids in active protein sites and ligands. The '\n",
      " 'molecular docking analysis was carried out to determine the toxicity-related '\n",
      " 'moiety of aconitine alkaloids through the ligand-amino-acid interaction '\n",
      " 'function in MOE2015. The PDB format of 2V7O and 2VZ6 was downloaded from the '\n",
      " 'PDB (protein data bank) database (https://www.rcsb.org/), and the mol2 '\n",
      " 'format of compounds was from the SYBYL software of QSTR research. The '\n",
      " 'structure preparation function in MOE software will be carried out to '\n",
      " 'minimize the energy and optimize the structure of the protein skeleton. '\n",
      " 'Based on the London dG score and induced fit refinement, all compounds will '\n",
      " 'be docked into the active site of every potential target by taking score '\n",
      " 'values as the scoring function [90] . The DCG (Discounted Cumulative Gain) '\n",
      " 'algorithm was utilized to examine the consistency between the ranking result '\n",
      " 'of pLD 50 and our research (fit scores of dock study). They rely on the '\n",
      " 'formula that refers to pLD 50 . The IDCG (ideal DCG) refers to the ordered '\n",
      " 'pLD 50 values. The closer the Normalized Discounted Cumulative Gain (NDCG) '\n",
      " 'value is to 1, the better the consistency [91] . Preliminary MD simulations '\n",
      " 'for the model protein were performed using the program NAMD (NAnoscale '\n",
      " 'Molecular Dynamics program, v 2.9), and all files were generated using '\n",
      " 'visual molecular dynamics (VMD). NAMD is a freely available software '\n",
      " 'designed for high-performance simulation of large biomolecular systems [92] '\n",
      " '. During the MD simulation, minimization and equilibration of original and '\n",
      " 'docked proteins occurred in a 15 Å3 size water box. A CHARMM 22 force field '\n",
      " 'file was applied for energy minimization and equilibration with '\n",
      " 'Gasteiger-Huckel charges using Boltzmann initial velocity [93, 94] . '\n",
      " 'Integrator parameters also included 2 fs/step for all rigid bonds and '\n",
      " 'nonbonded frequencies were selected for 1 Å and full electrostatic '\n",
      " 'evaluations for 2 Å were used with 10 steps for each cycle [93] . The '\n",
      " 'particle mesh Ewald method was used for electrostatic interactions of the '\n",
      " 'simulation system periodic boundary conditions with grid dimensions of 1.0 Å '\n",
      " '[94] . The pressure was maintained at 101.325 kPa using the Langevin piston '\n",
      " 'and the temperature was controlled at 310 K using Langevin dynamics. '\n",
      " 'Covalent interactions between hydrogen and heavy atoms were constrained '\n",
      " 'using the SHAKE/RATTLE algorithm. Finally, 5 ns MD simulations for original '\n",
      " 'and docked protein were carried out for comparing and verifying the binding '\n",
      " 'affinity and stability of the ligand-receptor complex. The method combining '\n",
      " 'network analysis and the in silico method was carried out to illustrate the '\n",
      " 'QSTR and toxic mechanisms of aconitine alkaloids. The 3D-QSTR was built in '\n",
      " 'Sybyl with internal robustness and external high prediction, enabling '\n",
      " 'identification of pivotal molecule moieties related to toxicity in aconitine '\n",
      " 'alkaloids. The CoMFA model had q 2 , r 2 , optimum component, and '\n",
      " 'correlation coefficient (r 2 ) values of 0.624, 0.966, 6, and 0. 9698, '\n",
      " 'respectively, and the CoMSIA model had q 2 , r 2 , optimum component, and '\n",
      " 'correlation coefficient (r 2 ) values of 0.719, 0.901, 4, and 0.9770. The '\n",
      " 'network was built with Cytoscape software and the STRING database, which '\n",
      " 'demonstrated the reliability of cluster analysis. The 2V7O and 2VZ6 proteins '\n",
      " 'were identified as potential targets with the CytoNCA plugin with '\n",
      " 'PharmMapper server for interactions between the aconitine alkaloids and key '\n",
      " 'amino acids in the dock study. The result of the dock study demonstrates the '\n",
      " 'consistency of the experimental pLD 50 . The MD simulation indicated that '\n",
      " 'aconitine alkaloids exhibit potent binding affinity and stability to the '\n",
      " 'receptor CAMK2G. Finally, we incorporate pivotal molecule moieties and '\n",
      " 'ligand-receptor interactions to realize the QSTR of aconitine alkaloids. '\n",
      " 'This research serves as a guideline for studies of toxicity, including '\n",
      " 'neuro-, reproductive, and embryo-toxicity. With a deep understanding of the '\n",
      " 'relationship between toxicity and structure of aconitine alkaloids, '\n",
      " 'subsequent structural modification of aconitine alkaloids can be carried out '\n",
      " 'to enhance their efficacy and to reduce their toxic side effects. Based on '\n",
      " 'such research, aconitine alkaloids can bring us closer to medical and '\n",
      " 'clinical applications. In addition, as pointed out in past research [95] , '\n",
      " 'user-friendly and publicly accessible web servers represent the future '\n",
      " 'direction of reporting various important computational analyses and findings '\n",
      " '[96] [97] [98] [99] [100] [101] [102] [103] [104] [105] [106] [107] [108] '\n",
      " '[109] . They have significantly enhanced the impacts of computational '\n",
      " 'biology on medical science [110, 111] . The research in this paper will '\n",
      " 'serve as a foundation for constructing web servers for QSTR studies and '\n",
      " 'target identifications of compounds.',\n",
      " 'Severe Acute Respiratory Syndrome (SARS) originated in China towards the end '\n",
      " 'of 2002 and quickly spread to about 30 countries by 2003 causing over 800 '\n",
      " 'deaths worldwide [1] [2] [3] [4] . The etiological agent of the disease, the '\n",
      " 'SARS Coronavirus (SARS-CoV) was identified and classified as a unique member '\n",
      " 'of beta Coronavirus. The SARS-CoV carries a~30 kb positive-sense RNA genome '\n",
      " 'that contains 8 unique open reading frames (ORFs) in addition to the common '\n",
      " 'coronaviral genes [5, 6] . The Orf3A is the largest among these ORFs and '\n",
      " 'encodes a protein of 274 amino acids, the deletion of which reduces SARS-CoV '\n",
      " 'replication in cell cultures and murine models of infection [7] . Antibodies '\n",
      " 'against the 3a protein were also found in the sera of SARS patients [8] . '\n",
      " 'Following infection of Vero E6 or CaCo2 cells, the 3a protein was found to '\n",
      " 'be associated with virus particles [9, 10] and localized to the plasma '\n",
      " 'membrane and perinuclear regions of infected cells [11] . Several studies '\n",
      " 'including ours have shown the 3a protein to induce apoptosis in host cell '\n",
      " '[12] [13] [14] , which is also linked to its ability to form ion channels '\n",
      " '[15] . The 3a protein also promotes osteoclastogenesis by increasing NF-kB '\n",
      " 'activity [16] , and we have shown it to cause ER stress and induce '\n",
      " 'downregulation of the type 1 interferon receptor [14, 17] . The 3a protein '\n",
      " 'contains three transmembrane domains at the N-terminus and a C-terminal '\n",
      " 'cytoplasmic domain of~150 amino acids [18] . The cytoplasmic domain contains '\n",
      " 'a tyrosine based sorting motif, YXXΦ (where X can be any residue and Φ is a '\n",
      " 'residue with a bulky hydrophobic side chain) and a di-acidic EXD motif. It '\n",
      " 'has been hypothesized that these motifs assist the 3a protein in regulating '\n",
      " 'internalization of the viral Spike (S) protein from cell surface to '\n",
      " 'intracellular sites [19] . In a study by Tan et al., the 3a protein lacking '\n",
      " 'both of these motifs failed to express on the cell surface [11] . It has '\n",
      " 'also been reported that the 3a protein is released in membranous structure '\n",
      " 'from cell and mutation in the YXXΦ or EXD motif does not impact the release '\n",
      " '[20] . Tyrosine based sorting motifs are responsible for AP-2 mediated '\n",
      " 'internalization from the cell surface by interacting with μ2 subunit of the '\n",
      " 'clathrin complex [21] . However the YXXΦ motif can also mediate interaction '\n",
      " 'with other members of the clathrin complex including AP-1, AP-3 and AP-4 for '\n",
      " 'transport to different destinations inside the cell [22] . For example, the '\n",
      " 'YXXΦ motif has been shown to be required for lysosomal targeting of some '\n",
      " 'proteins like the CD3 chain of the T-cell receptor [23] . The di-acidic '\n",
      " 'motif functions as a canonical ER export signal by mediating interaction '\n",
      " 'with COPII vesicles [24] [25] [26] [27] and has been shown to mediate '\n",
      " 'efficient transport of the KAT1 ion channel protein to the plasma membrane '\n",
      " '[28] . However the functions of these individual motifs within the 3a '\n",
      " 'protein have not been understood properly. We previously reported the 3a '\n",
      " 'protein to localize to the plasma membrane and to interact with Caveolin-1 '\n",
      " '[29] , a protein that is part of lipid-rich regions of the membrane '\n",
      " '(caveolae) and has several functions in the cells, including the formation '\n",
      " 'of lipid droplets and modulation of lipolysis [30] [31] [32] . Lipid '\n",
      " 'droplets are intracellular storage organelles consisting of a core of '\n",
      " 'neutral lipids surrounded by a monolayer of phospholipids [33] . Many '\n",
      " 'proteins have been identified on lipid droplets that are also involved in '\n",
      " 'vesicular transport, membrane fusion and cytoskeletal mobility. Among these '\n",
      " 'are Perilipin A, Caveolins, Phospholipase D, and members of the Rab and ARF '\n",
      " 'families of small GTPases [34] [35] [36] [37] [38] . Among viral proteins, '\n",
      " 'the core proteins of hepatitis C virus (HCV) and GB virus B (GBV-B) are '\n",
      " 'known to be associated with lipid droplets [39] . Recent views of lipid '\n",
      " 'droplets emphasize that these might also act as storage vesicles for excess '\n",
      " 'and unfolded proteins [34, 40] . In this report we show that the tyrosine '\n",
      " 'based sorting motif (YXXΦ) of the 3a protein is responsible for its sorting '\n",
      " 'from the Golgi to plasma membrane. Whereas the wild type 3a protein traffics '\n",
      " 'to plasma membrane efficiently, most of the YXXΦ mutants are retained in '\n",
      " 'Golgi and lipid droplets. We also show that increased targeting of the 3a '\n",
      " 'protein to lipid droplets is associated with its lysosomal degradation. '\n",
      " 'These findings define the role of the YXXΦ motif in intracellular and '\n",
      " 'surface transport of the SARS-CoV 3a protein. The YXXΦ motif of the 3a '\n",
      " 'protein is required for its trafficking to the plasma membrane Several '\n",
      " 'motifs are present within the amino acid sequence of Orf3a that may be '\n",
      " 'crucial for its intracellular localization. The 274 amino acids (aa) long 3a '\n",
      " 'protein contains three potential transmembrane regions between residues '\n",
      " '34-56, 77-99 and 103-125, followed by a cytoplasmic domain of~150 aa ( '\n",
      " 'Figure 1 ). The cytoplasmic domain of 3a also contains a tyrosine-based '\n",
      " 'sorting motif, YXXΦ (aa 160-163) and a di-acidic motif, EXD (aa 171-173). We '\n",
      " 'compared the amino acid sequences of SARS-CoV 3a protein from several human '\n",
      " 'and bat isolates available in NCBI database (Table 1) . YXXΦ motif is '\n",
      " 'conserved in all the isolates studied where as the EXD motif is mutated in '\n",
      " 'all the seven bat isolates and one out of 21 human isolates. The EXD motif '\n",
      " 'was proposed to be an endoplasmic reticulum (ER) export signal whereas the '\n",
      " 'YXXΦ motif was shown to be crucial for endocytosis from the cell surface '\n",
      " '[19] . The topology of the 3a protein is such that the N-terminus is exposed '\n",
      " 'to the outside of plasma membrane [11] . To study its cell surface '\n",
      " 'distribution, we made expression constructs for the 3a protein and its '\n",
      " 'mutants with a N-terminal Myc-tag. The 3a protein was expressed in four '\n",
      " 'different cell lines -COS-7, HT29, MDCK and Huh7 and showed intracellular as '\n",
      " 'well as surface localization in all cell types (Figure 2A ). This result is '\n",
      " 'consistent with previous reports showing the membrane distribution of the 3a '\n",
      " 'protein [11] . To study the subcellular distribution of the 3a protein we '\n",
      " 'performed immunofluorescence staining of cells transfected with wild type '\n",
      " 'and mutant constructs. The cell surface distribution was observed in '\n",
      " 'non-permeabilized cells with the antibodies against the N-terminal Myc-tag. '\n",
      " 'Staining of permeabilized cells with antibodies against the cytoplasmic '\n",
      " 'region (aa 126-274) of the 3a protein was used to see its total expression '\n",
      " 'and intracellular distribution. The wild type 3a protein was found to '\n",
      " 'localize to the plasma membrane as well as to intracellular regions that '\n",
      " 'appeared to be perinuclear and showed a punctate distribution ( Figure 2B ). '\n",
      " 'The Cyto3a protein that lacks the transmembrane domains showed similar '\n",
      " 'intracellular distribution, but was expectedly not found on the plasma '\n",
      " 'membrane ( Figure 2B) . Surprisingly, the ΔYXXΦ protein with mutations in '\n",
      " 'the tyrosine-based sorting motif was mainly localized in the perinuclear '\n",
      " 'compartment ( Figure 2B ). On the other hand, the ΔEXD protein with the '\n",
      " 'mutations in the diacidic motif distributed similar to the wild type '\n",
      " 'protein. Since an N-terminal myc tag was present on all the proteins and was '\n",
      " 'likely to be exposed extracellularly, staining non-permeabilized cells with '\n",
      " 'the anti-myc antibodies confirmed the plasma membrane localization and '\n",
      " 'proper topology of the 3a proteins. Only the wild type and ΔEXD proteins '\n",
      " 'stained with anti-myc antibodies in non-permeabilized cells. These results '\n",
      " 'showed that the 3a proteins are expressed with the correct topology of '\n",
      " 'extracellular N-terminus and cytoplasmic C-terminus. The ΔYXXΦ mutant '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'protein saturated in the intracellular compartment, but was not found at the '\n",
      " 'plasma membrane, showing this tyrosine-based sorting motif to be important '\n",
      " 'for its trafficking to the cell surface. The di-acidic motif, which is a '\n",
      " 'canonical ER export signal, did not show any role in the intracellular '\n",
      " 'trafficking of the 3a protein. The YXXΦ motif in 3a protein is responsible '\n",
      " 'for its Golgi to plasma membrane sorting The subcellular distribution of the '\n",
      " 'ΔYXXΦ protein was very different from that of the wild type 3a protein. To '\n",
      " 'further characterize the compartments to which the mutant 3a proteins '\n",
      " 'localize, we expressed them in Huh7 cells together with markers for the ER '\n",
      " 'and Golgi compartments. The wild type and ΔEXD proteins were distributed to '\n",
      " 'the cell surface and punctate structures that are likely to represent '\n",
      " 'trafficking vesicles; these also showed minor colocalization with the Golgi '\n",
      " 'compartment ( Figure 3A ). However, the ΔYXXΦ mutant protein was localized '\n",
      " 'exclusively to the Golgi compartment ( Figure 3A ). Two subcellular '\n",
      " 'distribution patterns were observed for the proteins. While the first '\n",
      " 'consisted of a predominantly dispersed cytoplasmic and plasma membrane '\n",
      " 'localization with a minor fraction of the protein in the Golgi compartment, '\n",
      " 'the other showed a Golgi-saturated pattern. These distribution patterns for '\n",
      " 'the proteins were quantified in multiple cells using the Golgi marker. An '\n",
      " 'overwhelming majority of cells expressing wild type 3a or its ΔEXD mutant '\n",
      " 'showed the first pattern, while all of the cells expressing the YXXΦ mutant '\n",
      " 'showed the second pattern of localization ( Figure 3B ). The wild type 3a '\n",
      " 'and ΔEXD proteins appear to be rapidly sorted from the Golgi complex to the '\n",
      " 'plasma membrane. These results suggest that the di-acidic motif does not '\n",
      " 'function as an ER export signal in this protein; if this was the case, the '\n",
      " 'ΔEXD mutant protein would have been retained in the ER. None of the proteins '\n",
      " 'showed any significant localization with Amino acid sequences were obtained '\n",
      " 'from National Center for Biotechnology Information (NCBI). The YXXΦ (where X '\n",
      " 'is any amino acid and Φ is an amino acid with a bulky hydrophobic side '\n",
      " 'chain) and EXD motifs are shown in italics whereas mutated motifs are '\n",
      " 'highlighted in bold letters. the ER marker. The ΔYXXΦ mutant protein was '\n",
      " 'retained in the Golgi complex but not in the ER region or at the plasma '\n",
      " 'membrane, suggesting that the tyrosine-based sorting motif is crucial for '\n",
      " 'the sorting of the 3a protein from Golgi to the plasma membrane. Increased '\n",
      " 'Golgi retention of the 3a protein leads to increased targeting to lipid '\n",
      " 'droplets Cytoplasmic lipid droplets have been proposed to act as storage '\n",
      " 'vesicles for proteins [40] . Increased Golgi retention of Caveolin-2 is '\n",
      " 'reported to result in targeting of the protein to lipid droplets [41] . We '\n",
      " 'investigated whether this was also the case with Golgi-saturated 3a '\n",
      " 'proteins. To visualize lipid droplets we used the specific dye Nile Red to '\n",
      " 'stain transfected cells grown in culture media containing free fatty acids. '\n",
      " 'On costaining with Nile Red,~50% of the wild type 3a protein was found to be '\n",
      " 'associated with lipid droplets, but the Cyto3a protein did not show any '\n",
      " 'significant colocalization, indicating that the transmembrane region of the '\n",
      " '3a protein was required for its targeting to lipid droplets ( Figure 4) . '\n",
      " 'Presuming that these act as storage vesicles for excess or unfolded '\n",
      " 'proteins, Golgi overload would cause increased targeting to lipid droplets. '\n",
      " 'Expectedly, over 90% of the ΔYXXΦ mutant 3a protein localized to lipid '\n",
      " 'droplets ( Figure 4 ). Reduced levels of the ΔYXXΦ 3a protein are due to '\n",
      " 'increased lysosomal degradation In multiple transient transfections, we '\n",
      " 'observed that the ΔYXXΦ mutant protein was expressed at much lower levels as '\n",
      " 'compared to the wild type 3a protein ( Figure 5 A left panel). To '\n",
      " 'investigate this, semi-quantitative RT-PCR was performed, which showed the '\n",
      " 'mRNA levels of the wild type and mutant 3a proteins to be similar ( Figure '\n",
      " '5A right panel) . This indicated that the mutant protein was either unstable '\n",
      " 'or was actively degraded in the transfected cells. To explore this, we '\n",
      " 'performed an in vitro transcription-translation experiment and found similar '\n",
      " 'levels of the wild type and ΔYXXΦ mutant 3a proteins ( Figure 5B ), which '\n",
      " 'supported the intracellular degradation model. Since the mutant protein was '\n",
      " 'retained in lipid droplets, we hypothesized that this would lead to '\n",
      " 'increased autophagy or lysosomal degradation. In fact, the ΔYXXΦ mutant '\n",
      " 'protein localized with late endosome/ lysosomal marker Rab7 ( Figure 5C ). '\n",
      " 'To further confirm lysosomal degradation of the mutant protein, we used two '\n",
      " 'inhibitors -Brefeldin A, which blocks ER-to-Golgi transport and Aprotinin A, '\n",
      " 'which inhibits lysosomal proteases. Both of these led to 3 to 5 fold '\n",
      " 'increased levels of the ΔYXXΦ mutant protein ( Figure 5D ). These findings '\n",
      " 'clearly demonstrate that mutation of the tyrosine-based sorting motif causes '\n",
      " 'abnormal trafficking of the 3a protein leading to increased accumulation in '\n",
      " 'the Golgi compartment and lipid droplets and subsequent degradation in the '\n",
      " 'lysosomal compartment. The 3a protein is unique to SARS-CoV, not found in '\n",
      " 'other human coronaviruses, and is important for disease pathogenesis. It '\n",
      " 'functions as an ion channel [15] , which is crucial for its pro-apoptotic '\n",
      " 'role [42] , and for this the 3a protein must traffic to the cell surface. '\n",
      " 'This has been shown earlier [11] as well as in this report. However, the '\n",
      " 'motifs or domains of the 3a protein required for its cell surface '\n",
      " 'distribution have not been characterized. In many proteins, a tyrosine based '\n",
      " 'sorting motif is responsible for internalization of molecules from the '\n",
      " 'plasma membrane to intracellular sites by specific recognition of the cargo '\n",
      " 'containing this motif through the plasma membrane resident Clathrin AP-2 '\n",
      " 'complex [21] . Consequently, it was proposed that the YXXΦ motif within the '\n",
      " '3a protein would be important for its endocytosis [19] . If the YXXΦ motif '\n",
      " 'were important for mediating endocytosis from plasma membrane, the mutant 3a '\n",
      " 'protein lacking this motif would be retained in the plasma membrane. '\n",
      " 'However, this mutant protein was not found in the plasma membrane, but in '\n",
      " 'the Golgi compartment, suggesting a role for this motif in Golgi to plasma '\n",
      " 'membrane transport. The YXXΦ motif is also recognized by the AP-1, AP-3 and '\n",
      " 'AP-4 complexes, of which the AP-4 complex is responsible for Golgi to plasma '\n",
      " 'membrane transport [22] . However, in a preliminary experiment we did not '\n",
      " 'observe any interaction of the 3a protein with AP-4 in a yeast two-hybrid '\n",
      " 'assay (unpublished data). This, as well as alternate mechanisms need further '\n",
      " 'investigation. The di-acidic motif functions as a canonical ER export signal '\n",
      " 'in many proteins [24] [25] [26] [27] [28] . A mutant 3a protein lacking this '\n",
      " 'motif was efficiently transported from the ER to Golgi and the plasma '\n",
      " 'membrane. Thus, the EXD motif does not functions as an ER export signal in '\n",
      " 'the 3a protein. We also found the 3a protein of SARS-CoV to be localized to '\n",
      " 'lipid droplets. The transmembrane domains of the 3a protein are responsible '\n",
      " 'for targeting it to lipid droplets since the Cyto3a protein that lacks the '\n",
      " 'transmembrane domains did not localize to lipid droplets. It was shown '\n",
      " 'earlier that hydrophobic domains of Perilipin A and the core proteins of HCV '\n",
      " 'and GBV-B are crucial for lipid droplets targeting [35, 39] . Lipid droplets '\n",
      " 'are shown to act as storage vesicles for excess and unfolded proteins [34, '\n",
      " '40] . It has been reported that Golgi mislocalization of Caveolin-1 leads to '\n",
      " 'its targeting into lipid bodies [43] . Over-expressed Caveolin-2 is also '\n",
      " 'partially targeted to lipid droplets [38] . In addition Caveolin-1 '\n",
      " 'accumulates in lipid droplets when it is linked to an ERretrieval sequence '\n",
      " '(Cav-KKSL) [44] . A mutant Caveolin protein (Cav3 DGV ) that mislocalizes in '\n",
      " 'the Golgi, accumulates irreversibly in lipid droplets [41] . We observed '\n",
      " 'that the 3a ΔYXXΦ mutant that is retained in the Golgi is also associated '\n",
      " 'with lipid droplets. Targeting to lipid droplets could be a pathway for '\n",
      " 'protein degradation. In fact, inhibitors of autophagy and proteosomal '\n",
      " 'degradation led to increased protein levels and lipid droplet association of '\n",
      " 'Apolipoprotein B [37] . We observed that reduced protein levels of ΔYXXΦ '\n",
      " 'mutant were associated with increased lysosomal degradation. There was a 3 '\n",
      " 'to 5 fold increase in ΔYXXΦ protein levels on treatment with Brefeldin A or '\n",
      " 'Aprotinin. Based on these data, we put forward a new model of intracellular '\n",
      " 'distribution of the 3a protein ( Figure 6 ). We propose that the wild type '\n",
      " '3a protein is sorted efficiently from the ER and Golgi to the plasma '\n",
      " 'membrane. The ΔYXXΦ mutant fails to reach the plasma membrane as it gets '\n",
      " 'stuck in the Golgi compartment. This mutant protein is then targeted to '\n",
      " 'lipid droplets and ultimately to the protein degradation machinery. In '\n",
      " 'conclusion, surface expression of the 3a protein is mediated by the crucial '\n",
      " 'YXXΦ motif, in the absence of which the 3a protein is retained in the Golgi '\n",
      " 'compartment destined for lysosomal degradation via lipid droplets. This '\n",
      " 'finding demonstrates a novel role of the YXXΦ motif in intracellular protein '\n",
      " 'transport. All common reagents were purchased from Sigma Chemical Co. (St. '\n",
      " 'Louis, MO, USA) unless stated otherwise. Type Culture Collection (Manassas, '\n",
      " 'Va, USA), while MDCK and HT29 cells were obtained from the National Animal '\n",
      " 'Cell Repository, National Centre for Cell Sciences (Pune, India). All cell '\n",
      " 'lines were cultured at 37°C in 10% CO 2 in complete Dulbecco modified Eagle '\n",
      " 'medium (DMEM containing 1 g/lit glucose, 2 mM L-glutamine, 1.5 g/lit sodium '\n",
      " 'bicarbonate, 0.1 mM non-essential amino acids, 0.1 mg/ml streptomycin, 100 U '\n",
      " 'penicillin) and 5% fetal bovine serum (FBS). Anti-3a antibodies were '\n",
      " 'generated using the purified cytoplasmic domain of the 3a protein as '\n",
      " 'described earlier [29] . The orf3a (nucleotides 25,268 to 26,092) of the '\n",
      " 'SARS-CoV genome (GenBank accession number NC_004718, Tor2 isolate) and its '\n",
      " 'cytoplasmic domain were cloned into the eukaryotic plasmid vector pSGI-HA to '\n",
      " 'give expression vectors pSGI-3A-HA and pSGI-3ACyto-HA, as described earlier '\n",
      " '[29] . N-terminal Myc-tag was added to the 3A and its different mutants by '\n",
      " 'taking c-Mycepitope tag from pGBKT7 vectors. The ΔYXXΦ mutant of 3a was '\n",
      " 'created by mutating the tyrosine and valine residues in the YXXV motif (aa '\n",
      " '160-163) to alanine and glycine respectively, using primers YF1: '\n",
      " 'TGTATACCAGCTAACAGCCTCACAG and YR1: CTGTGGCACTGTTAGCTGGTATACA. The ΔEXD '\n",
      " 'mutant was created by mutating glutamic acid and aspartic acid residues in '\n",
      " 'the EXD motif (aa 171-173) into alanines, using primers EF1: '\n",
      " 'GTCGTTACTGCAGGTGCCGGCAT TTC and ER1: GAAATGCCG GCACCTGCAGTAACG AC. Site '\n",
      " 'directed mutagenesis was carried out using the QuickChange site directed '\n",
      " \"mutagenesis kit system (Stratagene) according to the manufacturer's \"\n",
      " 'protocol. All mutations were verified by DNA sequencing. Cells were '\n",
      " 'transfected using Lipofectin as described earlier [14] and were harvested in '\n",
      " 'PBS 48 hr post-transfection. For experiments with inhibitors, transfected '\n",
      " \"cells were treated with Brefeldin A or Aprotinin as per the manufacturer's \"\n",
      " 'instruction (Sigma) for 12 hr before harvest. Western blotting was done '\n",
      " 'after lysing the cells with lysis buffer (10 mM Tris pH7.6, 150 mM NaCl, 1% '\n",
      " 'Triton-X100) with protease inhibitors as described earlier [14] . RNA was '\n",
      " 'isolated from transfected cells 48 hr posttransfection using Trizol reagent '\n",
      " '(Invitrogen). Equal amounts of RNA were taken for reverse transcription '\n",
      " 'using Superscript reverse transcriptase (Life Technologies). '\n",
      " 'Semi-quantitative PCR was done using gene specific primers 3aF1, '\n",
      " 'GAATTCATGGATTTGT TTATGAGATTT, and 3aR1, AGATCTCAAAGGCACGCTAGTAGT. The PCR '\n",
      " 'reaction was carried out for only 20 cycles, which was found previously to '\n",
      " 'be in the linear phase before reaching saturation. For transient '\n",
      " 'transfection, cells were grown on coverslips to 40-50% confluence and '\n",
      " 'transfected with Lipofectin reagent (Invitrogen, USA) in antibiotic-free and '\n",
      " 'serum-free DMEM. Six hours post-transfection, the medium was removed and '\n",
      " 'replaced with complete DMEM containing 5% FBS. Around 48 hr '\n",
      " 'post-transfection, cells were washed with PBS and fixed with 2% '\n",
      " 'paraformaldehyde for 15 min at room temperature. For antibody staining, '\n",
      " 'cells were permeabilized with 100% methanol for 3 min at −20°C, blocked with '\n",
      " 'PBS containing 5% goat serum for 45 min and then incubated with a 1:200 '\n",
      " 'dilution of the primary antibodies in PBS for 1 hr at room temperature. The '\n",
      " 'cells were washed thrice for 5 min each with PBS and then incubated with a '\n",
      " '1:200 dilution of the relevant secondary antibodies conjugated with either '\n",
      " 'Alexa 488 or Alexa 594 (Molecular Probes, USA) in PBS for 1 hr. The '\n",
      " 'coverslips were mounted on slides with Antifade reagent (Bio-Rad, USA) and '\n",
      " 'sealed with a synthetic rubber-based adhesive (Fevicol; Pidilite Industries, '\n",
      " 'India). Confocal images were collected using a 60x objective in a Bio-Rad '\n",
      " 'Radiance 2001 laser-scanning confocal system attached to a Nikon Eclipse '\n",
      " 'TE-2000U inverted microscope. For subcellular localization, cells were '\n",
      " 'cotransfected to express the required protein and a relevant fluorescent '\n",
      " 'subcellular marker. For the studies reported here, the markers included '\n",
      " 'DsRed-ER and YFP-Golgi that are part of the Living Colors™ Subcellular '\n",
      " 'Localization Vector set (Clontech). To determine percent colocalization of '\n",
      " 'different proteins; images were analyzed by Nikon Elements Software and the '\n",
      " 'Pearson coefficient for colocalization was calculated. Transfected Huh7 '\n",
      " 'cells were transferred 48 hrs posttransfection to Glucose-free media (Sigma) '\n",
      " 'containing 1 mM fatty acids (0.5 mM Oleic acid and 0.5 mM Palmitoleic acid). '\n",
      " 'After 6 hr, the media was removed followed by a brief wash with PBS. Cells '\n",
      " 'were stained with 10 μM Nile Red (Sigma) in PBS for 20 min at room '\n",
      " 'temperature. The stain was removed and cells were washed twice with PBS. In '\n",
      " 'case of co-staining with anti-3a antibodies, cells were first stained for 3a '\n",
      " 'followed by Nile Red staining. The gene to be expressed was cloned '\n",
      " 'downstream of a T7 promoter in the pSGI expression vector [29] . The plasmid '\n",
      " 'was then added to the reaction mixture containing T7 polymerase, amino-acid '\n",
      " 'mixture (lacking methionine), appropriate buffer and rabbit reticulocyte '\n",
      " 'lysate using the TNT® quick coupled transcription/translation system kit '\n",
      " '(Promega Corporation, USA) according to the The wild type 3a protein or the '\n",
      " 'ΔEXD mutant are sorted from Golgi to plasma membrane whereas the ΔYXXΦ '\n",
      " 'mutant protein is retained in the Golgi and is targeted to lysosomal '\n",
      " 'compartment for degradation via lipid droplets.',\n",
      " '.2 shows the additional goodness-of-fit results arising from treating the '\n",
      " 'network as multilevel. In particular, results for female-bipartite, '\n",
      " 'male-bipartite, and female-bipartite-male network interactions are shown. '\n",
      " 'All statistics are fit to an acceptable standard. Because the same-sex '\n",
      " 'networks are fixed, graph statistics without bipartite edges show zero '\n",
      " 'variation. The ability of this model specification to closely capture all '\n",
      " 'these features of the empirical network, including ones not explicitly '\n",
      " 'modelled, is strong evidence that a useful model has been specified. ']\n"
     ]
    }
   ],
   "source": [
    "pprint(cotools.texts(comm_use[:12]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_text = comm_use.texts()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Dealing with abstracts\n",
    "\n",
    "We can get abstracts in a similar manner to getting texts:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'One single-nucleotide polymorphism from IL4 gene was significant for pooled respiratory infections (rs2070874; 1.66 [1.29-2.14]). We also detected an association of TLR2 gene with tuberculosis (rs5743708; 3.19 [2.03-5.02]). Subset analyses identified CCL2 as an additional risk factor for tuberculosis (rs1024611; OR = 0.79 [0.72-0.88]). The IL4-TLR2-CCL2 axis could be a highly interesting target for translation towards clinical use. However, this conclusion is based on low credibility of evidence -almost 95% of all identified studies had strong risk of bias or confounding. Future studies must build upon larger-scale collaborations, but also strictly adhere to the highest evidence-based principles in study design, in order to reduce research waste and provide clinically translatable evidence.'"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cotools.abstract(comm_use[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['␥ 9 ␦ 2 T cells provide a natural bridge between innate and adaptive immunity, rapidly and potently respond to pathogen infection in mucosal tissues, and are prominently induced by both tuberculosis (TB) infection and bacillus Calmette Guérin (BCG) vaccination. Mycobacterium-expanded ␥ 9 ␦ 2 T cells represent only a subset of the phosphoantigen {isopentenyl pyrophosphate [IPP] and (E)-4-hydroxy-3-methyl-but-2-enylpyrophosphate [HMBPP]}-responsive ␥ 9 ␦ 2 T cells, expressing an oligoclonal set of T cell receptor (TCR) sequences which more efficiently recognize and inhibit intracellular Mycobacterium tuberculosis infection. Based on this premise, we have been searching for M. tuberculosis antigens specifically capable of inducing a unique subset of mycobacterium-protective ␥ 9 ␦ 2 T cells. Our screening strategy includes the identification of M. tuberculosis fractions that expand ␥ 9 ␦ 2 T cells with biological functions capable of inhibiting intracellular mycobacterial replication. Chemical treatments of M. tuberculosis whole-cell lysates (MtbWL) ruled out protein, nucleic acid, and nonpolar lipids as the M. tuberculosis antigens inducing protective ␥ 9 ␦ 2 T cells. Mild acid hydrolysis, which transforms complex carbohydrate to monomeric residues, abrogated the specific activity of M. tuberculosis whole-cell lysates, suggesting that a polysaccharide was required for biological activity. Extraction of MtbWL with chloroform-methanol-water (10:10:3) resulted in a polar lipid fraction with highly enriched specific activity; this activity was further enriched by silica gel chromatography. A combination of mass spectrometry and nuclear magnetic resonance analysis of bioactive fractions indicated that 6-O-methylglucose-containing lipopolysaccharides (mGLP) are predominant components present in this active fraction. These results have important implications for the development of new immunotherapeutic approaches for prevention and treatment of TB.',\n",
       " '',\n",
       " 'Background. Influenza A(H7N9) viruses isolated from humans show features suggesting partial adaptation to mammals. To provide insights into the pathogenesis of H7N9 virus infection, we compared risk factors, clinical presentation, and progression of patients hospitalized with H7N9, H5N1, and 2009 pandemic H1N1 (pH1N1) virus infections.',\n",
       " 'Hantavirus infection, which causes zoonotic diseases with a high mortality rate in humans, has long been a global public health concern. Over the past decades, accumulating evidence suggests that long noncoding RNAs (lncRNAs) play key regulatory roles in innate immunity. However, the involvement of host lncRNAs in hantaviral control remains uncharacterized. In this study, we identified the lncRNA NEAT1 as a vital antiviral modulator. NEAT1 was dramatically upregulated after Hantaan virus (HTNV) infection, whereas its downregulation in vitro or in vivo delayed host innate immune responses and aggravated HTNV replication. Ectopic expression of NEAT1 enhanced beta interferon (IFN-␤) production and suppressed HTNV infection. Further investigation suggested that NEAT1 served as positive feedback for RIG-I signaling. HTNV infection activated NEAT1 transcription through the RIG-I-IRF7 pathway, whereas NEAT1 removed the transcriptional inhibitory effects of the splicing factor proline-and glutamine-rich protein (SFPQ) by relocating SFPQ to paraspeckles, thus promoting the expression of RIG-I and DDX60. RIG-I and DDX60 had synergic effects on IFN production. Taken together, our findings demonstrate that NEAT1 modulates the innate immune response against HTNV infection, providing another layer of information about the role of lncRNAs in controlling viral infections. IMPORTANCE Hantaviruses have attracted worldwide attention as archetypal emerging pathogens. Recently, increasing evidence has highlighted long noncoding RNAs (lncRNAs) as key regulators of innate immunity; however, their roles in hantavirus infection remain unknown. In the present work, a new unexplored function of lncRNA NEAT1 in controlling HTNV replication was found. NEAT1 promoted interferon (IFN) responses by acting as positive feedback for RIG-I signaling. This lncRNA was induced by HTNV through the RIG-I-IRF7 pathway in a time-and dose-dependent manner and promoted HTNV-induced IFN production by facilitating RIG-I and DDX60 expression. Intriguingly, NEAT1 relocated SFPQ and formed paraspeckles after HTNV infection, which might reverse inhibitive effects of SFPQ on the transcription of RIG-I and DDX60. To the best of our knowledge, this is the first study to address the regulatory role of the lncRNA NEAT1 in host innate immunity after HTNV infection. In summary, our findings provide additional insights regarding the role of lncRNAs in controlling viral infections. KEYWORDS long noncoding RNA, NEAT1, Hantaan virus, RIG-I, DDX60, beta interferon, SFPQ, interferons, innate immunity H antaviruses belong to the family Bunyaviridae and genus Hantavirus, and they have attracted worldwide attention as archetypal emerging pathogens (1). Hantaviruses contain a tripartite negative-sense single-stranded RNA (ssRNA) genome, including small (S), medium (M), and large (L) genes, which encode nucleocapsid protein (NP), F. 2017. The long noncoding RNA NEAT1 exerts antihantaviral effects by acting as positive feedback for RIG-I signaling. J Virol 91:e02250-16. Volume 91 Issue 9 e02250-16 jvi.asm.org 1 Journal of Virology']"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cotools.abstracts(comm_use[14:18])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_abstracts = comm_use.abstracts()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Manipulating the documents\n",
    "\n",
    "We can wrap up document manipulations with the `Paperset.apply` method. For example, if we wanted to get all the keys of each document, we could do:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'abstract',\n",
       " 'back_matter',\n",
       " 'bib_entries',\n",
       " 'body_text',\n",
       " 'metadata',\n",
       " 'paper_id',\n",
       " 'ref_entries'}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "keys = comm_use.apply(lambda x: list(x.keys()))\n",
    "# then lets combine them into a set\n",
    "\n",
    "set(sum(keys, []))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Please make PRs to make this more useful! To be added:\n",
    "\n",
    "* Metadata\n",
    "* Other datasets??"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Other Data\n",
    "\n",
    "## John Hopkins Data\n",
    "\n",
    "Getting the hopkins data with the `get_hopkins` function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "confirmed, deaths, recoveries = cotools.get_hopkins()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "dict_keys(['Province/State', 'Country/Region', 'Lat', 'Long', '1/22/20', '1/23/20', '1/24/20', '1/25/20', '1/26/20', '1/27/20', '1/28/20', '1/29/20', '1/30/20', '1/31/20', '2/1/20', '2/2/20', '2/3/20', '2/4/20', '2/5/20', '2/6/20', '2/7/20', '2/8/20', '2/9/20', '2/10/20', '2/11/20', '2/12/20', '2/13/20', '2/14/20', '2/15/20', '2/16/20', '2/17/20', '2/18/20', '2/19/20', '2/20/20', '2/21/20', '2/22/20', '2/23/20', '2/24/20', '2/25/20', '2/26/20', '2/27/20', '2/28/20', '2/29/20', '3/1/20', '3/2/20', '3/3/20', '3/4/20', '3/5/20', '3/6/20', '3/7/20', '3/8/20', '3/9/20', '3/10/20', '3/11/20', '3/12/20', '3/13/20', '3/14/20', '3/15/20', '3/16/20', '3/17/20'])\n",
      "<class 'dict'>\n",
      "[460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460, 460]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[str,\n",
       " str,\n",
       " float,\n",
       " float,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int,\n",
       " int]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(confirmed.keys())\n",
    "print(type(confirmed))\n",
    "print([len(x) for x in confirmed.values()])\n",
    "[type(x[0]) for x in confirmed.values()]"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
